## Polymerase Inhibitors

![EC50 Distribution of RdRP Compounds](covid-review-figs/rdrp-compounds-ec50.png#!width=35rem)

__RNA-dependent RNA polymerases (RdRp)__ catalyze phosphodiester bond formation between nucleoside triphosphates in an RNA-templated manner. RdRps are highly conserved in their structural and functional features, even among diverse RNA viruses belonging to different families[^10.3390/v11080745]. Nucleoside analog polymerase inhibitors are the most common antiviral compounds making up at least a plurality and possibly a majority of all licensed antivirals, including those for treating herpes viruses, HIV, hepatitis B, and hepatitis C. Most antiviral polymerase inhibitors lack a 3’-hydroxyl group and act as nucleoside analog chain terminators. Others contain a 3'-hydroxyl group yet still result in immediate or delayed chain termination. Finally, some nucleoside analogs become incorporated into viral genomes and inhibit replication by introducing mutations during subsequent rounds of virus replication.

Coronavirus RNA polymerization is more complex than that of other viruses. First, coronaviruses contain a 3’-to-5’ __exoribonuclease (ExoN; nsp14)__ which is required for replication fidelity[^10.1073/pnas.0508200103]. ExoN is responsible for the intrinsic resistance of coronavirus species to ribavirin and several other nucleoside analogs [^10.1371/journal.ppat.1003565][^10.1073/pnas.1718806115]. Second, coronavirus genomes are three times larger than most other RNA genomes and thus require increased processivity which may explains the RdRp (nsp12) requirement for two accessory proteins (nsp7 and nsp8).  Third, the coronavirus RdRp is required for transcribing multiple subgenomic mRNAs in addition to copying the full virus genome. 3D structures of the nsp12-nsp7-nsp8 complex of SARS-CoV [^10.1038/s41467-019-10280-3] and SARS-CoV-2 [^10.1016/j.cell.2020.05.034][^10.1126/science.abb7498][^10.1126/science.abc1560][^10.1038/s41586-020-2368-8] have been solved by cryo-EM.


### Remdesivir
Remdesivir (__GS-5734__) is the monophosphate prodrug of the parent 1’-cyano-substituted adenine C-nucleoside analogue __GS-441524__[^10.1021/acs.jmedchem.6b01594]. The presence of the phosphate group allows for more efficient metabolism of the prodrug to the active nucleoside triphosphate form bypassing the rate-limiting first phosphorylation step[^10.1038/nature17180]. Remdesivir is a nonobligate chain terminator in that it contains a 3’-OH group. It causes __delayed chain termination__ due to a steric clash after the addition of three trailing nucleosides[^10.1074/jbc.AC120.013056][^10.1074/jbc.RA120.013679][^10.1016/j.antiviral.2020.104793]. The fact that chain termination does not occur until additional nucleoside triphosphates have been added probably explains the mechanism by remdesivir eludes the coronavirus exonuclease.

Remdesivir has broad spectrum activity against multiple RNA viruses and inhibits SARS-CoV, MERS-CoV, and SARS-CoV-2 with most reported half-maximal effective concentrations (EC<sub>50</sub>s) below 1 μM[^10.1126/scitranslmed.aal3653][^10.1128/mBio.00221-18][^10.1038/s41467-019-13940-6][^10.1038/s41422-020-0282-0][^10.1101/2020.04.03.024257][^10.1016/j.antiviral.2020.104786][^10.1128/AAC.00819-20][^10.1101/2020.04.03.023846][^10.1101/2020.04.03.20052548][^10.1101/2020.04.27.064279]. It inhibits coronaviruses in multiple cell types including primary human airway epithelial cells[^10.1126/scitranslmed.aal3653][^10.1101/2020.04.27.064279]. Animal model experiments have demonstrated that remdesivir can reduce lung virus levels and lung damage in mice infected with MERS-CoV and SARS-CoV[^10.1126/scitranslmed.aal3653][^10.1016/j.antiviral.2020.104793], rhesus macaques infected with MERS-CoV[^10.1126/scitranslmed.aal3653][^10.1073/pnas.1922083117], and rhesus macaques infected with SARS-CoV-2[^10.1038/s41586-020-2423-5].

Remdesivir resistance in a related coronavirus has been selected in the murine hepatitis virus (MHV) during prolonged _in vitro_ passage in the presence of remdesivir[^10.1128/mBio.00221-18]. The remdesivir-resistant mutations are in the RdRp fingers domain. Introducing the homologous substitutions into SARS-CoV (F480L + V557L) resulted in a virus with reduced replication and about 6-fold reduced remdesivir susceptibility [^10.1128/mBio.00221-18].

Remdesivir is administered intravenously with a loading dose of 200 mg followed by 100 mg daily for 5 to 10 days in adult patients[^10.1002/phar.2398][^10.1016/j.onehlt.2020.100128]. Following intravenous administration, the achievable maximum remdesivir plasma concentration (Cmax) is predicted to be at least twice as high as its most commonly reported EC<sub>50</sub>s[^10.1002/cpt.1909]. There have been 3 published randomized clinical trials of remdesivir. The __NIH Adaptive Covid-19 Treatment Trial (ACTT)__[^10.1056/NEJMoa2007764] and a Chinese trial[^10.1016/S0140-6736(20)31022-9] were double-blinded placebo-controlled trials. The Gilead SIMPLE-Severe trial randomized patients to receive 5 vs. 10 days of treatment[^10.1056/NEJMoa2015301]. 

The NIH Adaptive Trial was the pivotal trial leading to the remdesivir __FDA Emergency Use Authorization__. It was a multicenter trial that randomized 1063 persons with severe disease to remdesivir x 10 days vs. placebo. The primary endpoint was the time to clinical improvement defined as being discharged from the hospital or no longer requiring supplemental oxygen. An interim DSMB review was performed after a total of 482 recoveries and 81 deaths. The review showed a statistically significant reduction in the time to recovery from 15 days to 11 days (p<0.001) and a non-statistically significant reduction in mortality by day 14 of 11.9% vs. 7.1% (p=0.06). A subgroup analysis showed that response to therapy was greater in those with less severe disease (not requiring supplemental oxygen or receiving oxygen via nasal cannula) than in those with severe disease (requiring high flow oxygen or mechanical ventilation)[^10.1056/NEJMoa2007764]. Serious adverse events occurred at similar rates in both groups (21.1% for remdesivir vs 27.0% for placebo). 

The phase Chinese placebo-controlled trial was discontinued prematurely after recruiting 237 (rather than the planned 453 patients) because of the reduction in new SARS-CoV-2 infections. Patients were randomized 2:1 to receive remdesivir or placebo. Although not statistically significant, patients receiving remdesivir had a faster time to clinical improvement than those receiving placebo (median 18 days \[IQR 12–28\] vs 23 days \[15–28\]; HR 1.52 \[0.95–2.43\]. The 28-day mortality was similar in both groups (14% for remdesivir and 13% for placebo). There was no difference in the rates of decline of nasopharyngeal viral RNA levels between the two groups. Adverse events leading to treatment discontinuation were 12% in the remdesivir group and 5% in the placebo group[^10.1016/S0140-6736(20)31022-9].

The Gilead SIMPLE-Severe trial enrolled 397 persons with radiographic evidence of pneumonia who were receiving supplemental oxygen or had an oxygen saturation <=94% on room air. In an unadjusted analysis, there was an improvement of 2 points along a 7-point ordinal scale by day 14 in 64% of those receiving treatment for 5 days and 54% in those receiving treatment for 10 days. However, after adjustment for baseline variables, which were somewhat worse in the 10 day group,  the study concluded that there was no difference between the two groups in the time to a 2 point clinical improvement by day 14.

It has been difficult to distinguish possible remdesivir adverse effects with those of COVID-19 disease. Gastrointestinal symptoms and transaminase elevations appear to the most common adverse effects. Thrombocytopenia and elevations in prothrombin time may also be related to RDV. Hypotension and hypersensitivity reactions have been rare. The RDV formulation contains a cyclodextrin-based solubility enhancer which would be expected to increase plasma exposure in patients with renal impairment. However, RDV does not appear to be associated with an increased risk of acute kidney injury and there are no recommendations for dose adjustments in those with an eGFR above 30. Although RDV is a CYP450 substrate, there have not yet been convincing reports of drug interactions - although this has not been systematically studied. Gilead sciences recommends against co-administration with rifampin (presumably because of the CYP450 metabolism) or hydroxychloroquine as there are some data suggesting that chloroquine analogs reduce remdesivir efficacy.

Based on the ACTT results, the FDA issued an Emergency Use Authorization for the use of remdesivir in hospitalized patients with COVID-19 who have severe disease defined as having an SpO2 ≤ 94% on room air or requiring supplemental oxygen, mechanical ventilation, or extracorporeal membrane oxygenation (ECMO). Further analysis of ACTT data containing results from a greater proportion of the participants and containing virological response data are likely to be published in the future.



Another recently published study reported that a five day course of therapy was as efficacious as a ten day course of therapy in hospitalized patients with an SpO2 ≤ 94% who did not require mechanical ventilation [^10.1056/NEJMoa2015301]. However, the patients in the ten day treatment arm had greater baseline disease severity than those in the five day treatment arm.

In 53 patients with severe SARS-CoV-2 pneumonia receiving remdesivir as part of two expanded access trials, 34 were receiving mechanical ventilation (including 3 on extracorporeal membrane oxygenation \[ECMO\]). Overall, 6 of the 34 ventilated patients and 1 of the 19 non-ventilated patients died[^10.1056/NEJMoa2007016]. This death rate appeared lower than that of historical controls with similar levels of disease severity.

Of note, the Chinese trial enrolled  did not reach the planned enrollment of 450 patients because the outbreak was controlled in China.

A phase 3 placebo-controlled trial begun in China in patients with moderate-to-severe disease was discontinued prematurely after recruiting 237 rather than the planned 453 patients. Patients were randomized 2:1 to receive remdesivir or placebo. Although not statistically significant, patients receiving remdesivir had a faster time to clinical improvement than those receiving placebo (median 18 days \[IQR 12–28\] vs 23 days \[15–28\]; HR 1.52 \[0.95–2.43\]. The 28-day mortality was similar in both groups (14% for remdesivir and 13% for placebo). There was no difference in the rates of decline of nasopharyngeal viral RNA levels between the two groups. Adverse events leading to treatment discontinuation were 12% in the remdesivir group and 5% in the placebo group[^10.1016/S0140-6736(20)31022-9].


___Conclusion: Remdesivir is the first anti-SARS-CoV-2 compound shown to be clinically beneficial. The optimal use of this drug will become clearer with further study. Remdesivir will likely be the main comparator drug in future trials of moderate-to-severe SARS-CoV-2 infections.___

### EIDD-2801
β-D-N4-hydroxycytidine (NHC) is a cytidine analogue that exerts its activity primarily through viral mutagenesis in that it incorporates into new RNA strands but leads to large numbers of replication errors during subsequent rounds of replication [^10.1016/j.coviro.2019.04.002]. β-D-N4-hydroxycytidine-5'-isopropyl ester (EIDD-2801) is an orally available NHC prodrug. The mechanism by which EIDD=2801 eludes the proof-reading function of the coronavirus exonucleases is not known.

EIDD-2801 has broad-spectrum antiviral activity against multiple coronaviruses including MERS-CoV, SARS-CoV, and SARS-CoV-2 with most EC<sub>50</sub>s below 1 μM[^10.1177/095632020401500102][^10.1177/095632020601700505][^10.1128/JVI.01348-19][^10.1126/scitranslmed.abb5883]. It also has activity in primary human airway epithelial cells and reduced virus levels, disease, and lung damage in mouse models of SARS-CoV and MERS-CoV[^10.1126/scitranslmed.abb5883]. Deep sequencing studies have confirmed that viral mutagenesis is the main mechanism by which EIDD-2801 inhibits coronaviruses [^10.1177/095632020401500102][^10.1177/095632020601700505][^10.1128/JVI.01348-19][^10.1126/scitranslmed.abb5883].

As of June 25, a phase I study in uninfected individuals and two phase II studies patients with mild-to-moderate disease have begun ([Clinical Trials of EIDD-2801](linic-triamplymerase)).

___Conclusion: The preclinical in vitro and animal model data associated with EIDD-2801 are comparable to those of remdesivir. However, EIDD-2801’s safety  and clinical efficacy have not yet been established. Should EIDD-2018 trials be successful, this drug may be a useful oral option in high-risk outpatients. ___

### Other Nucleoside Analogs
Ribavirin is a nucleoside analog with modest activity against several viruses. Although multiple mechanisms for its activity have been proposed, the most likely is an increase in viral mutagenesis. In cell culture studies, it has demonstrated little, if any, activity against coronaviruses including SARS-CoV, MERS-CoV, and SARS-CoV-2[^10.1016/s0140-6736(03)13615-x][^10.1177/095632020401500102][^10.1016/j.jinf.2013.09.029][^10.1099/vir.0.061911-0][^10.1038/srep01686][^10.1016/j.antiviral.2020.104786][^10.1016/S0140-6736(20)31022-9]. Structural and biochemical studies have shown that although ribavirin 5’-monophosphate can be incorporated during RNA synthesis, it is readily excised by nsp14, the coronavirus exonuclease. Multiple observational clinical studies of patients with SARS-CoV, and to a lesser extent in patients with MERS-CoV have shown no evidence for clinical benefit while demonstrating a high risk of toxicity[^10.1371/journal.pmed.0030343].

Favipiravir (T-705) is a purine analog prodrug that is ribosylated and phosphorylated intracellularly to form the active metabolite ribofuranosyl-5′-triphosphate (aka T-705-RTP). It has broad spectrum activity against multiple viral RNA polymerases and appears to act by causing viral mutagenesis[^10.1016/j.antiviral.2013.09.015][^10.1016/j.antiviral.2018.03.003][^10.1101/2020.05.15.098731]. Favipiravir is approved in Japan for treating pandemic influenza infections. It achieves high plasma concentrations following oral administration[^10.1002/cpt.1844][^10.1002/cpt.1909], but has demonstrated little, if any, inhibitory activity against SARS-CoV-2 in vitro, with EC<sub>50</sub>s ranging from 60 to >100 μM[^10.1038/s41422-020-0282-0][^10.1016/j.antiviral.2020.104786][^10.1128/AAC.00819-20][^10.1101/2020.05.15.098731]. It displays activity in a hamster model but only when used at very doses [^10.1101/2020.06.19.159053][^10.1101/2020.07.07.191775]. In an  open-label randomized trial of 240 patients, of whom just 40% had PCR-confirmed COVID-19 pneumonia, favipiravir was associated with an improved 7 day clinical recovery rate compared to umifenovir in a subgroup of patients without critical disease (71% vs 56%; p=0.02)[^10.1101/2020.03.17.20037432].  Another trial comparing favipiravir to lopinavir/r has been temporarily retracted[^10.1016/j.eng.2020.03.007]. As of June 25, there are 2 completed trials, 15 ongoing trials, and 14 planned trials that include favipiravir ([Clinical Trials of Favipiravir](linic-triampPolymerase)).

The FDA-approved nucleoside analogs sofosbuvir and tenofovir used for the treatment of hepatitis C and HIV-1, respectively, are being studied in several clinical trials. However, they have been shown to have either weak or no clinical activity in vitro[^10.1101/2020.04.29.067983][^10.1101/2020.06.15.153411], while tenofovir has been ineffective in ferrets[^10.1128/mBio.01114-20]. These nucleosides, as well as several others,  have been reported to inhibit SARS-CoV-2 RdRp in biochemical assays in the absence of exonuclease[^10.1101/2020.03.18.997585][^10.1016/j.antiviral.2020.104857] suggesting that their lack of activity in cell culture may result from SARS-CoV-2's proofreading ability. Galidesivir (BCX4430) is an adenosine analog that inhibits multiple viruses including filoviruses (Ebola virus and Marburg virus) _in vitro_ and in rodents and cynomolgus macaques[^10.1038/nature13027]. However, it has little, if any _in vitro_ activity against MERS-CoV and SARS-CoV with EC<sub>50</sub> levels of about 60 μM for MERS-CoV and SARS-CoV[^10.1038/nature13027] and of >100 μM for SARS-CoV-2[^10.1016/j.antiviral.2020.104786]. At least seven additional nucleoside analogs have been studied _in vitro_ against SARS-CoV, MERs-CoV, or SARS-CoV-2, including four that are either approved drugs (gemcitabine, mizoribine, or mycophenolate) or being studied in humans (merimepodib) as immunosuppressants or anticancer agents[^10.1099/vir.0.061911-0][^10.1016/j.antiviral.2005.01.003][^10.1128/JVI.00023-19][^10.1101/2020.04.07.028589][^10.1016/j.jinf.2013.09.029].

___Conclusions: There appears to be little benefit for the use or further study of the nucleoside analogs in this section either because they lack meaningful SARS-CoV-2 inhibitory activity or because they are immunosuppressive.___

## Protease Inhibitors
Coronaviruses contain two protease enzymes: 3 chymotrypsin-like cysteine protease (3CLpro or Main \[M\]-pro; non-structural protein 5 \[nsp5\]) and papain-like serine protease (PLpro; nsp3)[^10.1002/cmdc.202000223]. 3CLpro cleaves polyprotein 1a/b at 11 sites. It is conserved among several families of RNA viruses and has similar cleavage site specificity to the picornavirus family of 3C proteases. 3CLpro contains 306 amino acids and functions as a homodimer. PLpro is part of a large multidomain transmembrane protein. It cleaves polyprotein 1a/b at the nsp1/2, nsp2/3, and nsp3/4 boundaries and also cleaves host proteins important for innate immunity such as ubiquitin interferon-stimulated gene product 15 (ISG-15). The 3D structure of SARS-CoV-2 3CLpro[^10.1038/s41586-020-2223-y][^10.1126/science.abb3405] and [PLpro](https://www.wwpdb.org/pdb?id=pdb_00006w9c) have been solved and are available for structure-based drug design.

There are more candidate 3CLpro than PLpro inhibitors because of this enzyme’s similarities to those of other virus species. The recent SARS-CoV-2 3CLpro structures have also identified tightly binding compounds with nanomolar-range half-maximal inhibitory concentrations in biochemical experiments (IC<sub>50</sub>s) and half-maximal effective concentrations in cell culture (EC<sub>50</sub>s). Most of the protease inhibitors (PIs) that are currently being studied clinically are those approved for treating HIV-1.

### HIV-1 Protease Inhibitors
None of the HIV-1 protease inhibitors (PIs) are expected to inhibit coronaviruses as the HIV-1 protease is an aspartate protease with a different specificity than the coronavirus 3CLpro and PLpro enzymes. Nonetheless, one particular HIV-1 PI combination – lopinavir boosted by ritonavir, which is an inhibitor of lopinavir metabolism (lopinavir/r) – has been used frequently to treat SARS-CoV, MERS-CoV, and SARS-CoV-2 infection.  Each of the other HIV-1 associated PIs has also been studied for potential anti-coronavirus activity.

Lopinavir/r was first used to treat a coronavirus infection when it was administered to a cohort of 75 hospitalized patients with SARS-CoV in Hong Kong in 2003[^PMID14660806][^10.1136/thorax.2003.012658]. These 75 hospitalized patients included 44 newly diagnosed SARS-CoV patients and 31 who had more severe disease. Among the 44 newly diagnosed patients, 2% died and none required mechanical ventilation; the 31 patients with more severe disease, 13% died and 10% required intubation. The mortality rate and need for mechanical ventilation among the 44 newly diagnosed patients were lower than in the earliest SARS patients treated at their hospital.

Lopinavir displays weak inhibitory activity against SARS-CoV, MERS-CoV, and SARS-CoV-2 with EC<sub>50</sub>s of between 5 and >25 μM or higher[^10.1136/thorax.2003.012658][^10.1016/j.jcv.2004.03.003][^10.1016/j.bbrc.2004.04.083][^10.1177/095632020601700505][^10.1016/j.jinf.2013.09.029][^10.1128/AAC.03011-14][^10.1038/s41467-019-13940-6][^10.1128/AAC.00819-20][^10.1016/j.antiviral.2020.104786][^10.1101/2020.04.06.026476][^10.1101/2020.04.29.067983]. In addition, the selectivity index of lopinavir in cell culture is low suggesting that whatever inhibition is observed in cell culture may result from its toxic effects on the cells used to culture coronavirus rather than on the virus itself. Lopinavir has also been against SARS-CoV[^10.1073/pnas.0403596101] and SARS-CoV-2 3CLpro[^10.1038/s41422-020-0356-z] in biochemical assays.  Although ritonavir-boosted lopinavir (lopinavir/r) appeared to reduce lung pathology in two MERS-CoV-infected marmosets[^10.1093/infdis/jiv392], it was not beneficial in mice-infected with MERS-CoV[^10.1038/s41467-019-13940-6] or in SARS-CoV-2-infected ferrets[^10.1128/mBio.01114-20].

In an open-label randomized controlled trial of lopinavir/r vs. standard-of-care in 199 hospitalized adults with SARS-CoV-2 infection and SatO<sub>2</sub> levels 94% or PaO<sub>2</sub> /FiO<sub>2</sub>  <300, demonstrated no significant difference in days to virological clearance, clinical improvement, need for mechanical ventilation, or mortality[^10.1056/NEJMoa2001282].

The HIV-1 PIs nelfinavir and atazanavir have demonstrated variable in vitro activity against SARS-CoV-2 with EC<sub>50</sub>s ranging from below 1 to >50 μM[^10.1101/2020.06.16.153403][^10.1101/2020.04.06.026476][^10.1101/2020.04.29.067983][^10.1101/2020.04.14.039925][^10.1128/AAC.00819-20][^10.1101/2020.04.04.020925] while darunavir has consistently not displayed activity against SARS-CoV-2 in vitro[^10.1101/2020.04.03.20052548][^10.1101/2020.04.06.026476].

___Conclusion: Lopinavir/r does not inhibit coronaviruses in vitro and displays little or no clinical efficacy. However, as of June 25, there were 7 completed and 28 ongoing  clinical trials involving lopinavr/r. ___

### 3CLpro Investigational Inhibitors
Most coronavirus PI drug discovery efforts have targeted 3CLpro: it shares a conserved substrate recognition pocket with the 3C like protease (3Cpro) of human picornaviruses, which have already been successfully targeted with antiviral agents. 3CLpro can also be purified in large quantities and readily crystallized[^10.1016/j.virusres.2007.02.015].

There have been multiple drug screens and several structure-based designs targeting 3CLpro. Several compounds have inhibited SARS-CoV and SARS-CoV-2 in biochemical assays with IC<sub>50</sub>s ranging between 0.02 and 0.2 μM[^10.1021/jm050184y][^10.1038/s41422-020-0356-z][^10.1016/j.bmc.2016.05.013][^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb3405]. GC376 forms covalent bonds with three active site cysteines and inhibits multiple coronaviruses[^10.1128/JVI.01348-12]. Its IC<sub>50</sub> against SARS-CoV-2 3CLpro ranges from 0.03 to 1.5 μM while its EC<sub>50</sub> in cell culture ranges from 0.2 to 3.4 μM[^10.1038/s41422-020-0356-z][^10.1101/2020.05.03.073080][^10.1101/2020.06.07.138677]. GC376 also inhibits MERS-CoV (IC<sub>50</sub> = 1.7 μM) and displays sub-micromolar inhibitory against several other coronaviruses including the alphacoronavirus FIPV, TGEV, and HCoV 229E and the betacoronavirus BCV. It is highly effective in treating cats with the rapidly fatal coronavirus disease FIPV[^10.1371/journal.ppat.1005531]. Crystal structures of MERS-CoV 3CLpro bound to GC376 has led to the identification of piperidene peptidomimetics with greater SARS-CoV and MERS-CoV inhibitory activity[^10.1016/j.ejmech.2018.03.004].

The compounds with the most 3CLpro inhibitory activity identified to date are a series of peptidomimetic α-ketoamide and aldehyde derivatives[^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb4489] . Several of these compounds have been co-crystallized with SARS-CoV-2 3CLpro[^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb4489]. Preliminary pharmacokinetics evaluations have demonstrated metabolic stability in mouse and human microsomes and encouraging plasma and bronco-alveolar lavage fluid levels whether administered subcutaneously or by inhalation[^10.1126/science.abb3405][^10.1126/science.abb4489]. Another SARS-CoV-2 crystal structure identified of several inhibitory compounds; including an investigational compound ebselen, which has been studied in humans for a variety of other indications[^10.1038/s41586-020-2223-y].

Disulfiram, the licensed oral alcohol antagonist has been reported to inhibit SARS-CoV-2 3CL protease with an IC<sub>50</sub> of about 10 μM[^10.1038/s41586-020-2223-y] and to inhibit SARS-CoV and MERS-CoV PL protease with a slightly higher IC<sub>50</sub> [^10.1016/j.antiviral.2017.12.015]. However, disulfiram has not yet been studied in cell culture, and existing PK data suggests that standard doses may be insufficient to yield inhibitory drug levels.

___Conclusion: There are several promising lead compounds that could be developed into effective SARS-CoV-2 drugs. GC376 and ebselen display low μM activity and have been studied in cats and humans, respectively. However, they are probably not potent enough to be useful clinically without further chemical modifications. Other peptidomimetic compounds have greater inhibitory in vitro activity but have not yet been studied in animals or humans. ___

### PLpro Investigational Inhibitors
In contrast to 3CLpro, the PLpro enzyme is membrane associated, making it more challenging to study in vitro. Nonetheless, several crystal structures of SARS-CoV-2 PLpro have been deposited in the Protein Data Bank, although their publications are currently pending. PLpro has fewer homologues in other viruses, so there are fewer compounds that can serve as starting points for drug development. However, 6-thioguanine, a drug approved in the U.S. and used for the treatment of hematological malignancies and autoimmune conditions, has demonstrated activity in biochemical and cell culture assays[^10.1101/2020.07.01.183020]. Several lead compounds have been identified in other studies[^10.1016/j.antiviral.2014.12.015][^10.1002/cmdc.202000223].

## Entry Inhibitors
Coronavirus cell entry is mediated by its spike (S) glycoprotein. S is a membrane protein essential for the two main steps of the entry process: attachment to the host cell and fusion of the viral and cellular membranes. The functional S protein is a trimer of identical subunits. For most coronaviruses, S is processed by host proteases to generate two functional subunits, designated S1 and S2. The S1 subunit forms the apex of the S trimer including the receptor binding domains (RBDs)[^10.1016/bs.aivir.2019.08.002] and is responsible for attachment to cellular receptors. The S2 unit is responsible for fusion to the cell membrane.

The 3D structure of the SARS-CoV-2 S trimer has been determined by cryo-EM[^10.1126/science.abb2507]. The cellular receptor for SARS-CoV and SARS-CoV-2 is the human angiotensin-converting enzyme 2 (ACE2)[^10.1038/nature02145][^10.1038/s41586-020-2012-7]. An extended insertion within the structurally similar SARS-CoV and SARS-CoV-2 RBDs called the receptor-binding motif (RBM) contains most of the S amino acids responsible for ACE2 binding[^10.1074/jbc.C300520200][^10.1038/s41586-020-2180-5][^10.1007/s11427-020-1637-5]. S binding to ACE2 does not interfere with ACE2’s catalytic activity[^10.1126/science.1116480]. However, ACE2 internalization following infection leads to the loss of ACE2’s lung protective effects. S1-ACE2 affinity is higher for SARS-CoV-2 than SARS-CoV-1, possibly explaining its greater infectiousness[^10.1038/s41586-020-2180-5].

S2 contains two complementary heptad repeat (HR) regions, designated HR1 and HR2, which are alpha helices with repeated patterns of seven amino acids that have regions that can bind to one another. HR1 projects out to the host cell membrane whereas HR2 is about 170 amino acids upstream and closer to the virus membrane[^10.1016/j.virusres.2014.11.021]. HR1 and HR2 collapse onto one another to create a hairpin which brings the viral and cell membranes together promoting fusion. As S2 (like S1) functions as a trimer, the hairpin formed by the interaction of HR1 and HR2 is referred to as a six-helix bundle.

SARS-CoV and SARS-CoV-2 have two main cleavage sites. One is at the S1/S2 boundary and the other is at a site within S2 referred to as the S2’ site. These sites must be cleaved by host proteases in order for the S1 and S2 to undergo the necessary conformational changes to shift from receptor binding to cell fusion[^10.1016/bs.aivir.2019.08.002]. The specific cleavage sites, the responsible host enzymes, and the cellular locations at which cleavage occurs varies among different coronaviruses and likely also in different cell types; this variation is believed to play a role in cellular tropism and pathogenesis[^10.1016/j.virusres.2014.11.021][^10.1016/bs.aivir.2019.08.002]. Several candidate host proteases appear to be particularly important for processing S1 and S2, including TMPRSS2, a surface transmembrane serine protease, endosomal cathepsins, and furin. In contrast, to SARS-CoV, SARS-CoV-2 S harbors a furin cleavage site at the S1/S2 boundary, which is cleaved during biosynthesis[^10.1016/j.cell.2020.02.058][^10.1016/j.antiviral.2020.104742]. It has been speculated that the presence of this cleavage site may have expanded the transmissibility and tissue tropism of SARS-CoV-2 considering the more widespread expression of furin-like proteases in host cells.

This section describes interventions that interfere with virus entry including convalescent plasma, monoclonal antibodies that block S1 receptor binding, fusion inhibitors, and several other receptor binding inhibitors. Compounds that inhibit the host protease enzymes required for cleaving the S protein are described in the section on host-acting compounds.

### Monoclonal Antibodies
Neutralizing monoclonal antibodies (MAbs) have been used to treat Ebola[^10.1126/science.aad5224][^10.1056/NEJMoa1910993] and prevent respiratory syncytial virus[^10.2165/00003495-199958020-00009] infections. Neutralizing MAbs have also be shown to reduce virus levels in HIV-1-infected persons but resistance emerges unless a combination of neutralizing MAbs is administered[^10.1038/nm.4268][^10.1038/s41586-018-0531-2].

Recovery during SARS-CoV, MERS-CoV, and SARS-CoV-2 is usually associated with the development of neutralizing antibodies[^10.1007/s11684-017-0596-6][^10.3201/eid2607.200841][^10.1101/2020.05.03.20084442][^10.1101/2020.07.10.20150557]. These neutralizing antibodies nearly always target the S1 protein, particularly its receptor binding domain[^10.1002/rmv.706][^10.3390/v10120680][^10.1080/22221751.2019.1624482]. Within the past three months, there have been more than 20 published studies describing the isolation and characterization of SARS-CoV-2 neutralizing monoclonal antibodies (mAbs). Some studies describe the development of mAbs intended for COVID-19 treatment and prevention while other studies were performed primarily to characterize the genetic and structural basis of the SARS-CoV-2 antibody response. These studies include data on the (1) antibody source, (2) antigen bait used to isolate SARS-CoV-2 specific mAbs; (3) mAb cloning and expression; (4) functional screening; (5) structural studies; (6) escape mutations; (7) Fc-dependent effector mechanisms; and (8) activity in animal models. Four groups have begun clinical trials ([Clinical Trials of Monoclonal Antibody](linic-tria/#Monoclonalntibody)) including two that have published studies [^10.1038/s41586-020-2381-y][^10.1126/science.abd0827][^10.1126/science.abd0831].

**Antibody source:** The most common source of mAbs has been convalescent B cells from persons with a history of SARS-CoV-2. Convalescent B cells from persons with a history of SARS-CoV-1 have been used by several groups early in the pandemic to determine whether any SARS-CoV-1 mAbs cross-reacted with SARS-CoV-2 or to identify mAbs that might neutralize a broader spectrum of variants [^10.1126/science.abb7269][^10.1016/j.chom.2020.06.010][^10.1080/22221751.2020.1729069][^10.1038/s41586-020-2349-y][^10.1126/science.abc7424]. Two groups immunized mice including H2L2[^10.1038/s41467-020-16256-y] and VelocImmune[^10.1126/science.abd0827] mice. The mAbs isolated from immunized VelocImmune mice were reported to have similar sequences to those isolated by the same group from SARS-CoV-2 convalescent B cells. One group used a single-chain variable fragment (scFv) phage display libraries from naïve B cells [^10.1101/2020.05.13.093088] and another from a SARS-1 scFv library[^10.1101/2020.05.19.104281].

**Antigen bait:** Three main bait antigens have been used to identify reactive B cells, immunize transgenic mice, or pan phage display antibody fragments: the S1 receptor binding domain (RBD); the complete S1 protein; and the S trimer – often referred to as the trimeric extracellular or prefusion ectodomain. Less commonly used antigens have including the S1 N-terminal domain (NTD)[^10.1038/s41591-020-0998-x] and an S tetramer[^10.1126/science.abc7424]. Although most highly potent neutralizing mAbs bind to the S1 RBD, groups additional antigens would be expected to identify a broader spectrum of mAbs.

**Cloning, expression, and sequencing:** The most commonly used approach for cloning mAbs has involved sorting individual B cells followed by the physical linking of VH and VL regions through molecular techniques. Complete mAbs are then be expressed in IgG expression vectors or less commonly in immortalized B cells. The extent to which the above processes are automated through advances in microfluidics dictates the numbers of mAbs that can be generated and sequenced. High-throughput clonal sequencing has yielded insight into the variable heavy and light chain germ-line repertoire and extent of somatic hypermutation in antibodies recognizing SARS-CoV-2 antigens. Most antibodies have undergone minimal somatic hypermutation (0% to less than 10%) partly because convalescent plasma samples have been obtained within weeks following illness.  Most studies have reported that certain VH and VL genes are over-represented in neutralizing mAbs. Nonetheless, many neutralizing mAbs arise from diverse VH and VL repertoires.

**Functional screening:** Some groups have described cloning and sequencing of hundreds SARS-CoV-2 recognizing mAbs[^10.1126/science.abc5902][^10.1126/science.abc6952][^10.1038/s41586-020-2380-z][^10.1101/2020.06.17.153486][^10.1038/s41586-020-2456-9][^10.1126/science.abc7520]. To some extent, these sequences make it possible to filter out mAbs that are likely defective or duplicative. However, ultimately functional assays are required to identify which mAbs have therapeutic potential – including neutralizing assays using pseudoviruses and infectious virus assays. ELISA, biolayer-inferometry, and surface plasmon resonance assays are used to determine binding affinity to RBD, S1, and trimeric spike, to determine the extent to which mAbs compete with ACE2 or other mAbs with known epitopes, and to perform low-resolution epitope mapping.

**Structural studies:** X-ray crystallography has been used to determine the structure of the Fab portion of an mAb bound to the spike RBD [^10.1016/j.chom.2020.06.010][^10.1038/s41586-020-2380-z][^10.1038/s41586-020-2381-y][^10.1126/science.abb7269][^10.1126/science.abc2241] while CryoEM has been used to the structure of Fabs bound to trimeric S [^10.1016/j.cell.2020.05.025][^10.1126/science.abc6952][^10.1126/science.abd0827][^10.1038/s41586-020-2349-y]. Both types of structures make it possible to identify the spike residues interacting with an mAb and the surface area buried by the interaction. Low resolution structures determined using negative staining EM also contribute to epitope mapping. The most potent mAbs identified so far bind to the part of RBD that binds ACE2, also referred to as the receptor binding motif. Several groups have also identified non-ACE2 competing mAbs that bind to a part of the RBD called the core domain [^10.1038/s41586-020-2349-y][^10.1126/science.abc7424][^10.1126/science.abb7269][^10.1101/2020.05.22.111005]. While non-ACE2 competing mAbs have been on average less potent, they have a broader spectrum of activity, inhibiting SARS-CoV-1 and other members of the sarbecovirus clade.

**Resistance selection and mAb combinations:** The vulnerability of a mAb to mutational escape and the development of resistance can be inferred in part by the degree of conservation at each mAb-interacting spike residue. Molecular modeling and site-directed mutagenesis studies have also been performed to predict the effects of published spike variants[^10.1126/science.abd0831][^10.1038/s41586-020-2349-y][^10.1101/2020.05.22.111005]. One group selected for mAb-resistance mutations in virus culture[^10.1126/science.abd0831]. Several groups identified potentially synergistic mAbs that bind non-overlapping sites[^10.1126/science.abd0831][^10.1038/s41586-020-2349-y][^10.1126/science.abc2241].

**Fc-dependent effector mechanisms:** Two groups described assays to assess the effects of mAbs on antibody dependent cellular cytotoxicity (ADCC) and antibody dependent cellular phagocytosis (ADCP)[^10.1126/science.abd0827][^10.1038/s41586-020-2349-y]. In an attempt to reduce the likelihood of antibody-dependent enhancement of infection, another group introduced an Fc LALA mutation to reduce FcγR and C1q mAb binding[^10.1038/s41586-020-2381-y].

**Animal model studies:** Six groups evaluated mAbs in animal models including rhesus macaques[^10.1038/s41586-020-2381-y], Syrian hamsters[^10.1126/science.abc7520], and various transgenic mice species[^10.1126/science.abc7520][^10.1101/2020.05.13.093088][^10.1126/science.abc2241][^10.1101/2020.05.22.111005]. These studies aim to demonstrate _in vivo_ potency and to ascertain the risk of ADE[^10.1101/2020.05.22.111005].

___Conclusions: During the past month, more than 20 groups have described the development of mAbs able to inhibit SARS-CoV-2 in vitro. Four mAbs have begun clinical trials for the treatment and prevention of COVID-19. ___

### Fusion Inhibitors
Peptides that mimic HR2 can block virus-cell fusion by preventing the interaction between HR1 and HR2. The HIV-1 fusion inhibitor enfuvirtide is a 36 amino acid peptide that acts by this mechanism following subcutaneous administration[^10.1056/NEJMoa035026][^10.1056/NEJMoa035211][^10.1038/nrd1331]. Several peptide mimics of coronavirus HR2 have been shown to be potently inhibit MERS-CoV, SARS-CoV, and SARS-CoV-2 _in vitro_[^10.1016/j.bbrc.2004.05.046][^10.1038/ncomms4067][^10.1093/infdis/jiv325][^10.1021/acs.jmedchem.8b00890][^10.1126/sciadv.aav4580][^10.1128/JVI.00635-20] and reviewed by Tang et al[^10.1016/j.antiviral.2020.104792]. The interest in coronavirus fusion inhibitors derives in part because S2 is more conserved than S1 among different coronavirus species. For example, the S1 proteins of SARS-CoV and SARS-CoV-2 differ by more than 10% of their amino acids. However, their HR1 domains differ by 7% and their HR2 domains are essentially identical[^10.1038/s41423-020-0374-2]. Fusion inhibitors would be expected to have greater activity in preventing viral fusion occurring via the cell surface rather than the endosomal pathway of virus entry[^10.1128/JVI.01697-07][^10.1016/j.antiviral.2020.104792].

Prior to the SARS-CoV-2 pandemic, an HR2 mimicking peptide named EK1 was identified as a potential broad-spectrum coronavirus inhibitor as it inhibited both SARS-CoV and MERS-CoV at sub-micromolar levels and was found to safe and protective against MERS-CoV when administered intra-nasally to SARS-CoV-infected mice[^10.1126/sciadv.aav4580]. With the emergence of SARS-CoV-2, the crystal structure of the SARS-CoV-2 six helix bundle was solved, and several modifications were made to the EK1 peptide. In addition, a cholesterol group was added[^10.1038/s41422-020-0305-x] as lipidation had recently been shown to improve the pharmacokinetics and inhibitory activity of viral fusion inhibitors[^10.1128/JVI.00288-17]. This new lipopeptide, named EK1C4 was found to inhibit SARS-CoV-2 fusion 150 times more strongly than EK1, with SARS-CoV-2 EC<sub>50</sub>s of 0.04 μM and 0.004 μM in cell culture for SARS-CoV-2 and MERS-CoV, and with even lower EC<sub>50</sub>s in cell-cell fusion assays. Two other HR2-mimicking lipopeptide IPB02 and IPB04 have also been highly active in cell-cell fusion and pseudotype virus assays[^10.1128/JVI.00635-20].

___Conclusion: Further animal model studies or a phase I/II study are needed to determine the safety, pharmacokinetic, and efficacy of a fusion inhibitor such as EK1C4. ___

### Soluble ACE2 and Other Molecular Decoys
Soluble recombinant human ACE2 (rhACE2) has two potential beneficial effects. First, prior to the SARS-CoV-2 pandemic, rhACE2 was shown to protect lungs from injury during acute respiratory distress syndrome (ARDS)[^10.1038/nature03712][^10.1038/nm1267][^10.1186/s13054-017-1823-x][^10.1016/j.cell.2020.04.004]. Second, rhACE2 inhibits the binding of SARS-CoV and SARS-CoV-2 to ACE2-expressing cells[^10.1016/j.bbrc.2004.05.114][^10.1016/j.cell.2020.03.045] and has displayed dose-dependent activity _in vitro_ against SARS-CoV-2 in cultured cells and engineered human blood vessel and kidney organoids[^10.1016/j.cell.2020.04.004]. APN1 (Apeiron Biologics) is a recombinant form of human ACE2, that demonstrated safety in 89 non-SARS-CoV-2 patients and volunteers. It is now being evaluated in a 200-person placebo-controlled study of hospitalized persons with SARS-CoV-2 (NCT04335136). A similar approach involving the use of a fusion protein – in which an inactive ACE2 mutant is linked to a human Fc IgG – has also been shown to be highly active _in vitro_ [^10.1038/s41467-020-16048-4].

The complete ACE2 molecule may not be necessary to inhibit viral infection if a synthetic peptide can be constructed with a critical minimal subset of ACE2 amino acids capable of competing with cellular ACE2 for the SARS-CoV-2 receptor binding domain (RBD). One study suggested that multiple residues from different parts of the ACE2 molecule would be required including amino acids between positions 24–90 and 329–354[^10.1126/science.1116480] while another suggested that 23 amino acid N terminal fragments would bind the S1 receptor binding domain[^10.1101/2020.03.19.999318]. Two ACE2-derived peptides between positions 22–57 demonstrated some _in vitro_ activity against SARS-CoV[^10.1016/j.virol.2006.01.029]. Another peptide comprising two discontinuous ACE2 segments containing residues 22–44 and 351–357 artificially linked together by glycine, exhibited more potent _in vitro_ activity with an EC<sub>50</sub> of about 0.1 μM[^10.1016/j.virol.2006.01.029].

___Conclusion: rhACE2 may prove to be useful in patients with the most severe cases of SARS-CoV-2 pneumonia.___

### Additional Possible Entry Inhibitors
Two groups have reported that certain anticoagulants such as heparin, enoxaparin, and pixatimod bind to S1 receptor binding domains and  interfere with virus cell entry _in vitro_[^10.1101/2020.04.28.066761][^10.1101/2020.06.08.140236][^10.1101/2020.06.24.169334]. Administration of heparin intranasally has also been proposed as a method of preventing spread of SARS-CoV-2 after initial infection without increasing the risk of bleeding[^10.1101/2020.06.08.140236].

Spike-mimicking 20-mer peptides corresponding to different S1 and/or S2 regions were reported in one study to synergistically interfere with SARS-CoV entry into cells presumably by competing with complete virus for ACE2 binding[^PMID15918330]. However, few studies have described this approach and the inhibition of ACE2 enzymatic function or down-regulation of ACE2 levels is precarious as the absence of ACE2 is associated with the same pulmonary and cardiovascular disturbances observed during severe SARS-CoV and presumably SARS-CoV-2 infection.

Griffithsin, a 121 amino acid protein isolated from red algae, inhibits SARS-CoV and MERS-CoV _in vitro_ with EC<sub>50</sub>s in the range of 0.01 to 0.1 μM without inducing cell toxicity. It also inhibits SARS-CoV when delivered intranasally in a mouse model[^10.1128/JVI.02322-09][^10.1016/j.antiviral.2016.07.011]. Time course experiments suggest that griffithsin interferes with virus cell entry[^10.1016/j.antiviral.2016.07.011]. While ELISA assays showed that it binds directly to the S glycoprotein, it does not inhibit S binding to ACE2 suggesting it may prevent subsequent steps needed for viral entry[^10.1128/JVI.02322-09].

Umifenovir (aka arbidol) is an oral drug used in Russia and China to treat influenza and other viral infections[^10.1016/j.antiviral.2014.04.006][^10.1128/JVI.02077-15]. It is reported to have _in vitro_ activity against multiple viruses and to inhibit viral entry by binding viral surface glycoproteins[^10.1016/j.antiviral.2008.10.009]. Its SARS-CoV-2 EC<sub>50</sub>s have been reported to be between 5 and 10 μM with a selectivity index of about 8[^10.1101/2020.04.03.023846][^10.1038/s41421-020-0169-8]. A crystal structure of umifenovir in complex with influenza hemagglutinin showed it binding a hydrophobic cavity in the hemagglutinin trimer stem[^10.1073/pnas.1617020114]. There have been two open-label controlled trials in patients with SARS-CoV-2[^10.1101/2020.03.17.20037432][^10.1101/2020.03.19.20038984] and several small retrospective observational studies[^10.1016/j.jinf.2020.03.002][^10.1016/j.jinf.2020.03.060][^PMID32574262] which have demonstrated lack of clinical efficacy.

Defensins are small antimicrobial peptides produced by white blood cells and epithelial cells, which have a role in innate immunity. In a study of several peptides derived from mouse β-defensin-4, one 30 amino acid peptide called P9 was found to inhibit influenza A, MERS-Co-V, and SARS-CoV both _in vitro_ and in a mouse model[^10.1038/srep22008].

VE607 was reported in one study to inhibit SARS-CoV S protein-ACE2-mediated cell entry[^10.1016/j.chembiol.2004.07.013] but does not appear to have been studied further. HTCC polymers are cationically modified chitosan derivatives that appear to inhibit cell entry of multiple coronaviruses including MERS-CoV and SARS-CoV[^10.1101/2020.03.29.014183][^10.1371/journal.pone.0156552]. SARS-RBD-hexapeptide is weak inhibitor of SARS-CoV viral entry, but nonetheless suggests the possibility that small peptides may be able to inhibit the interaction of coronavirus S1 receptor binding domains and ACE2[^10.1016/j.antiviral.2011.12.012]. Emodin is a plant derivative that appears to weakly inhibit SARS-CoV S protein-ACE2-mediated cell entry[^10.1016/j.antiviral.2006.04.014], although it has also been proposed to inhibit SARS-CoV release from cells[^10.1016/j.antiviral.2011.02.008].

___Conclusion: Heparin and enoxaparin are increasingly being used to prevent clotting in persons with severe disease. They may also have a direct antiviral effect. Intranasal administration in early less severe infections should be studied. The remaining compounds in this section have not been  well studied.  ___


## Convalescent Plasma
There are few randomized controlled studies of convalescent plasma for the treatment of severe viral infections. A systematic review and meta-analysis of seven studies of convalescent plasma or immunoglobulin for SARS-CoV was considered inconclusive with respect to potential benefits and harm[^10.1371/journal.pmed.0030343]. A larger, more recent meta-analysis of 32 studies of severe SARS-CoV and influenza, reported that administration of convalescent plasma early after symptom onset showed a statistically significant 75% reduction in the pooled odds of mortality compared with placebo or no therapy[^10.1093/infdis/jiu396].

Two risk-benefit analyses and several experts have argued that the possible large benefit associated with convalescent plasma for SARS-CoV-2 outweighs the small risk of undetected bloodborne infection and allergic reaction[^10.1016/j.tmrv.2020.04.001][^10.1172/JCI138745][^10.1007/s11684-017-0596-6][^10.1172/JCI138003]. According to FDA guidance, the administration of SARS-CoV-2 convalescent plasma must be under an investigational new drug application. Donors must have had resolution of symptoms 28 days before their donation or 14 days before if they have negative nasopharyngeal or blood PCR tests. The recommend neutralizing antibody titers should be 1:160 (minimum >1:80). However, because neutralizing antibodies cannot readily be detected in most laboratories, this requirement is waived provided blood samples can be stored for possible eventual testing. Additionally, the magnitude of SARS-CoV-2 receptor-binding domain antibodies appears to correlate strongly with viral neutralization[^10.1101/2020.05.03.20084442].

Joyner et. al. reported the results of a safety review of the first 5,000 patients receiving convalescent plasma in the U.S. between April 3 and May 11 under the FDA Expanded Access Program[^10.1172/JCI140200]. 81% were classified as having severe or life-threatening diseases and 19% were judged to be at high risk of progressing to severe or life-threatening disease. 66% were in the ICU. 36 (0.72%) had a serious adverse event in the first four hours after transfusion including 25 events considered related to transfusion, including death in 4, transfusion-associated lung injury in 11, circulatory overload in 7, and allergic reaction in 3. Overall, the seven day mortality was 14.9%[^10.1172/JCI140200].

In an open-label randomized controlled trial of 103 persons with severe or life-threatening disease, there was no overall difference in the primary endpoint of a 2-point improvement on a 6-point scale by day 28[^10.1001/jama.2020.10044]. However, in the subgroup with severe (but not life-threatening) disease, 21/23 receiving convalescent plasma vs. 15/22 in the control group achieved the primary outcome (p=0.03)[^10.1001/jama.2020.10044]. A similar finding was observed in another observational study of 39 hospitalized patients receiving convalescent plasma in that the risk of dying compared to matched controls was lower among patients not receiving mechanical ventilation (HR=0.19, 95% CI: 0.05-0.72; p=0.02) compared to mechanically ventilated patients (HR=1.24, 95% CI:0.33-4.67)[^10.1101/2020.05.20.20102236]. However, a recent randomized controlled study was terminated prematurely because the enrolling patients had neutralizing antibodies titers that were similar to those of the plasma donors[^10.1101/2020.07.01.20139857].

There are currently about 70 ongoing clinical trials of convalescent plasma of which at least 10 are randomized controlled studies.

___Conclusions: If convalescent plasma is efficacious, it is likely to be more useful early in the course of disease. Efficacy may also depend on the titer of neutralizing antibodies in the donor sample. However, there is sufficient uncertainty about the clinical efficacy of convalescent plasma to justify the completion of the ongoing randomized trials.___


## Interferons
Interferons (IFNs) respond to the presence of cellular changes suggestive of a viral infection by inducing numerous genes (IFN-stimulated genes \[ISGs\]) involved in inflammation, signaling, and immunomodulation to protect against uncontrolled infection. They interfere with viral replication by slowing cell metabolism, interfering with the membrane formation required for virus replication, and inducing secretion of cytokines that promote adaptive immunity[^10.1016/j.antiviral.2020.104791]. There are three IFN families[^10.1038/nri3787]. Type I IFN includes 13 related IFN-α subtypes and a single IFN-β. Type II IFN consists of a single gene product IFN-γ. Type III IFN consists of IFN-λ1, IFN-λ2, and IFN-λ3. Several IFN-α products have been approved for the treatment of hepatitis B, hepatitis C, human papillomavirus, and several malignancies. IFN-β, IFN-γ, and IFN-λ have been approved for the treatment of multiple sclerosis, congenital immunodeficiency chronic granulomatous disease, and hepatitis delta infection, respectively, Commercial IFN-α products vary in the extent to which they include different IFN-α subtypes. The biological and clinical significance of the differences between the different IFN-α products are not known[^10.1016/j.smim.2019.101300]. Pegylated interferons have a longer half-life because the attached polyethylene glycol causes interferon to be cleared more slowly.

The importance of IFN is underscored by the fact that many viruses, particularly coronaviruses encode multiple proteins that antagonize cellular IFN signaling pathways[^10.1016/j.smim.2019.101300][^10.1038/nrmicro.2016.81]. SARS-CoV-2 encodes approximately 10 proteins that allow the virus to escape or counteract the induction or antiviral action of IFN [^10.1371/journal.ppat.1008737][^10.1016/j.coviro.2012.04.004] and SARS-CoV-2 infection is associated with a reduced IFN response[^10.1128/JVI.00985-20][^10.1126/science.abc6027][^10.1016/j.cell.2020.04.026][^10.1371/journal.ppat.1008737].

As of August 2020, CoV-RDB contains data from nearly 30 cell culture studies, 10 animal model studies, and 12 clinical studies of different IFN preparations. Most of the cell culture and animal model studies have involved the inhibition of SARS-CoV or MERS; only five have involved SARS-CoV-2. IFN-α, IFN-β, and IFN-γ products were used in about 50%, 30%, and 10% of experiments, respectively. IFN-λ and IFN-β in combination with IFN-γ were used in the remaining 10% of experiments. Both IFN-α and IFN-β appear to inhibit coronavirus _in vitro_ and in animal models, although several authors it is commonly stated that IFN-β is more effective than IFN-α at inhibiting coronaviruses[^10.1016/j.antiviral.2020.104791][^10.1371/journal.ppat.1008737]. Two _in vitro_ studies suggests that SARS-CoV-2 is more susceptible to Type I IFNs than SARS-CoV possibly due to a deletion in ORF3b, a 154 amino acid protein known to antagonize one of the proteins involved in the cellular IFN response[^10.1101/2020.03.07.982264][^10.1074/jbc.AC120.013788].

Seven clinical studies with at least 75 SARS-CoV-2 patients have been published. Four of these studies, enrolling a total of approximately 400 patients, were randomized controlled open-label studies. Two evaluated IFN-β[^10.1016/S0140-6736(20)31042-4][^10.1128/AAC.01061-20] and two evaluated novaferon (an inhaled IFN-α formulation)[^10.1101/2020.04.24.20077735][^10.1101/2020.04.24.20077735]. The largest of these randomized 127 patients with mild-to-moderate disease 2:1 to receive IFN-β-1b subcutaneously + ribavirin + lopinavir/r vs. lopinavir/r alone[^10.1016/S0140-6736(20)31042-4]. The patients receiving triple therapy had a significantly shorter time to a negative nasopharyngeal PCR (7 vs. 12 days) and a more rapid clinical improvement by several metrics. As ribavirin has not been efficacious in treating coronavirus infections, IFN-β may have been the compound most likely responsible for the improved outcome. The other three randomized trials suggested possible virological or clinical benefit from IFN. Two of the three observational studies also suggested either a virological or clinical benefit from the use of inhaled IFN-α-2b[^10.3389/fimmu.2020.01061][^10.1016/j.chom.2020.07.005][^ 10.1128/AAC.01061-20]. There has been a recent press release describing significantly reduced progression to severe disease among those receiving an inhaled formulation of IFN-β ([https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf](https://www.synairgen.com/wp-content/uploads/2020/07/200720-Synairgen-announces-positive-results-from-trial-of-SNG001-in-hospitalised-COVID-19-patients.pdf) in a blinded randomized controlled trial.

Theoretical and empirical data suggest that IFNs should be administered as early as possible in the course of infection[^10.1172/JCI126363] when innate immunity is likely to be most needed; this would also avoid exacerbating the unregulated exuberant deleterious immune response that often occurs in the later stages of severe coronavirus infections. However, a recent report that type I and type II IFNs may upregulate ACE2 production has provided a note of caution in that the potential immune benefits of IFNs may be counteracted by an increased number of virus receptors[^10.1016/j.cell.2020.04.035]. Although far fewer data are available for IFN-λs, they have garnered much interest because they act at epithelial barriers and thus are theoretically optimal for treating viral respiratory tract infections and because they are reported to suppress viral spread without causing inflammation[^10.15252/emmm.201606413][^10.1093/cid/ciaa453][^10.1016/j.celrep.2020.107863][^10.1016/j.chom.2020.05.008].

___Conclusion: Considering the multiple proteins produced by coronaviruses to down-regulate host innate immunity and the generally favorable response to IFN in vitro and in animal models, sufficiently powered randomized clinical trials of the early use of IFNs following SARS-CoV-2 infection should be one of the highest treatment priorities.___

## Additional Host-Targeting Compounds
With the exception of interferons, all approved antivirals target specific steps in a virus’s replication cycle. However, the concept of developing drugs that target host pathways has gained increasing interest during the past several years and during the SARS-CoV-2 pandemic. Host-acting drugs have a number of theoretical advantages for treating acute pandemic infections. First, host acting antiviral drugs may have already been approved or undergone human testing for other conditions, making it possible to repurpose them to treat SARS-CoV-2. Second, host-acting drugs might be able to inhibit multiple viruses, since many viruses depend on similar host factors and pathways. Third, although drugs targeting host factors may have a higher risk of cellular toxicity compared to drugs that specifically target a virus protein, such toxicity may be acceptable for the relatively few days required to treat an acute viral infection.

During the past 17 years, many compounds have been identified that appear to inhibit coronaviruses _in vitro_ by inhibiting a cellular protein, influencing a signaling pathway, or modifying a cellular organelle. For some of these compounds, it has been possible to determine their mechanism of action. However, it is difficult to pinpoint the mechanism of action for many other compounds. This section reviews compounds that appear to act primarily by influencing the host rather than the virus and that are described in multiple studies, demonstrate _in vitro_ potency, and have favorable safety profiles. These compounds have been categorized into those that (i) inhibit host proteases required for S protein cleavage; (ii) appear to inhibit virus endosomal trafficking;  (iii) have been reported to stimulate host immunity; or (iv) act by additional or uncertain mechanisms.

### Host Protease Inhibitors
The proteolytic activation of coronavirus spike proteins by host cell proteases is critical for the virus to transition from receptor attachment to cell fusion. Many host proteases have been shown to proteolytically process the spike (S) protein including cell surface proteases such as transmembrane protease/serine proteases (TMPRSSs), endosomal cathepsins, furin, and trypsin[^10.1016/j.virusres.2014.11.021][^10.1016/j.antiviral.2020.104792]. Coronaviruses usually require that S is cleaved at two sites – the S1/S2 boundary and at a second site within S2 referred to as S2’. The specific enzymes involved depend in part on the virus species, the amino acid sequence at the S1/S2 boundary and at the S2’ site, and the infected cells.

In contrast to SARS-CoV, SARS-CoV-2 contains a functional poly-basic furin cleavage site at the S1/S2 boundary[^10.1016/j.cell.2020.02.058]. This observation and accumulating data strongly suggest that furin and TMPRSS2 are likely to mediate cleavage at the S1/S2 boundary and at S2’, respectively, and thus are likely to be particularly important host proteases for SARS-CoV-2[^10.1016/j.cell.2020.02.052][^10.1101/2020.04.15.042085]. However, the endosomal cysteine proteases cathepsin B and L are considered essential host proteases for SARS-CoV[^10.1073/pnas.0505577102][^10.1128/JVI.02232-10][^10.1128/JVI.02062-10][^10.1128/JVI.00094-12][^10.1016/j.virol.2017.11.012][^10.1016/j.cell.2020.02.052][^10.1101/2020.04.15.042085] and, together with TMPRSS2, may also be involved in SARS-CoV-2 virus entry in certain cell types[^10.1101/2020.04.15.042085][^10.3390/v12060629][^10.1038/s41467-020-15562-9].

Two serine protease inhibitors used in Japan for the treatment of pancreatitis and disseminated intravascular coagulation, camostat and nafamostat, inhibit coronaviruses through their inhibition of TMPRSS2. Both drugs inhibit MERS-CoV, SARS-CoV, and SARS-CoV-2 in pseudovirus entry assays and in cell culture[^10.1016/j.cell.2020.02.052][^10.1128/AAC.00754-20][^10.3390/v12060629][^10.1101/2020.04.03.024257]. Camostat has also protected mice from fatal SARS-CoV infection[^10.1016/j.antiviral.2015.01.011]. Both camostat and nafamostat appear to inhibit SARS-CoV-2 at lower concentrations than SARS-CoV[^10.1016/j.cell.2020.02.052][^10.1128/AAC.00754-20][^10.3390/v12060629][^10.1101/2020.04.03.024257]. In Calu-3 lung epithelial cells, camostat inhibited SARS-CoV and SARS-CoV-2 with EC<sub>50</sub>s generally below 1 μM, while the EC<sub>50</sub>s for nafamostat have been about 10 to 100-fold lower. However, the EC<sub>50</sub>s for both drugs against both viruses are considerably higher in Vero cells [^10.1128/AAC.00819-20][^10.3390/v12060629][^10.1016/S0140-6736(20)31022-9] suggesting that TMPRSS2 inhibition alone may not be sufficient to suppress SARS-CoV-2 in some cells[^10.1101/2020.04.15.042085]. Two studies have reported that aprotonin, a third serine protease inhibitor available in Russia as an aerosol preparation displayed comparable inhibitory activity to camostat[^10.1101/2020.04.03.024257][^10.1101/2020.04.15.042085].

Several small molecule cathepsin inhibitors have also been shown to inhibit coronaviruses _in vitro_. K11777, an investigational inhibitor of Trypanosoma cruzi, the agent of Chagas disease, was found to inhibit a variety of cysteine proteases including human cathepsins[^10.1016/j.antiviral.2015.01.011]. It also potently inhibited SARS-CoV in cell culture and virus entry assays at sub-micromolar concentrations[^10.1016/j.antiviral.2015.01.011]. Several K11777 derivatives including SMDC256159 and SMDC256160 were also shown to inhibit SARS-CoV replication. K11777 and each of its derivatives synergistically inhibited SARS-CoV when administered in combination with camostat. MG132, MDL28170, and other calpain inhibitors have been found to inhibit SARS-CoV with EC<sub>50</sub>s in the low μM range[^10.1128/JVI.01001-12][^10.1073/pnas.0505577102][^10.1177/095632020601700505].

Several synthetic furin inhibitors have also shown to inhibit coronaviruses _in vitro_ including MI-1851, which inhibited SARS-CoV-2 alone and synergistically in combination with TMPRSS2 inhibitors[^10.1101/2020.04.15.042085], while dec-RVKR-CMK, which weakly inhibited MERS-CoV in a pseudovirus entry assay[^10.1073/pnas.1407087111].

___Conclusion: As of July, there are eight ongoing and eight planned clinical trials of camostat, nafamostat, or another host protease inhibitor. This class of inhibitors currently appear to be the most promising among existing repurposed drugs. ___

### Additional Host-Acting Compounds
__Apilimod__ inhibits the intracellular trafficking of several viruses including SARS, MERS, and SARS-CoV-2 during the early stages of virus replication. It inhibits a kinase enzyme, phosphatidylinositol-3-phosphate 5-kinase (PIKfyve) involved in endosomal membrane formation. Although it does not alter the pH of endosomes or inhibit cathepsin B or L, it blocks entry of multiple viruses in pseudovirus and cell culture assays[^10.1101/2020.04.21.053058][^10.1038/s41467-020-15562-9]. In two large scale screening assays, it was found to be one of the most inhibitory compounds with EC50s in the range of 10 to 90 nanomolar[^10.1038/s41586-020-2577-1][^10.1016/j.cell.2020.06.034]. It is an investigational compound that has been studied in human clinical trials as an anticancer agent and is now being studied in a phase II trial for the treatment of COVID-19.

__Chloroquine (CQ) and its analogs__ have been reported to have multiple intracellular actions that interfere with virus replication and/or the immune response to viral infections. CQ analogs are weak bases, that in their non-protonated form, penetrate and concentrate within acidic intracellular organelles such as endosomes and lysosomes. In these organelles, CQ analogs may interfere with the trafficking of viruses that require a pH-dependent step for entry[^10.1016/s1473-3099(03)00806-5][^10.1371/journal.ppat.1004502]. CQ analogs have also been reported to impair ACE2 glycosylation, which could theoretically diminish receptor binding by SARS-CoV and SARS-CoV-2[^10.1186/1743-422X-2-69]. CQ analogs also anti-inflammatory, and have been used for autoimmune conditions. CQ and hydroxychloroquine (HCQ) have modest inhibitory activity _in vitro_ against MERS-CoV[^10.1128/AAC.03036-14][^10.3390/v10060283][^10.1128/AAC.03011-14], SARS-CoV[^10.1186/1743-422X-2-69][^10.1177/095632020601700505][^10.1016/j.bbrc.2004.08.085][^10.1128/AAC.03011-14], and SARS-CoV-2[^10.1038/s41422-020-0282-0][^10.1038/s41421-020-0156-0][^10.1093/cid/ciaa237][^10.1128/AAC.00819-20][^10.1101/2020.04.04.020925][^10.1101/2020.03.29.014407][^10.1101/2020.04.03.023846] with most EC<sub>50</sub>s ranging between 1 μM and 10 μM.

However, two large blinded and three large open-label randomized control trials totaling more than 6,000 patients failed to show any clinical or virological benefit from the use of hydroxychloroquine[^10.1101/2020.07.15.20151852][^10.7326/M20-4207][^10.1056/NEJMoa2019014][^10.1101/2020.04.10.20060558][^10.1093/cid/ciaa1009]. Two additional randomized controlled studies totaling more than 3,000 patients failed to show a reduction in the risk of infection among exposed patients receiving HCQ[^10.1101/2020.07.20.20157651][^10.1056/NEJMoa2016638].  The lack of efficacy likely reflects the fact that  at standard dosages, CQ and HCQ do not achieve sufficiently high inhibitory drug levels[^10.1111/j.1365-2125.1989.tb03439.x][^10.2165/00003088-199631040-00003][^10.1128/AAC.00339-08][^10.1002/cpt.1856][^10.1002/cpt.1909] and because cytoplasmic entry of SARS-CoV-2 appears to be dependent primarily on the membrane based enzyme TMPRSS2 rather than on endosomal fusion and acidification[^10.1101/2020.07.22.216150].

__Niclosamide__ is an oral anti-helminthic drug that has been used in millions of persons worldwide. It appears to exert it anti-helminthic effects by inhibiting oxidative phosphorylation and stimulating mitochondrial adenosine triphosphatase activity[^10.1021/acsinfecdis.0c00052]. In high-throughput drug repurposing screens, niclosamide has been found to have several additional biological effects that appear to be associated either with its effect on endosomal acidification or with its influence on signaling pathways[^10.1016/j.cellsig.2017.04.001]. Five research groups reported that niclosamide inhibits SARS-CoV[^10.1021/jm070295s][^10.1073/pnas.0403596101], MERS-CoV[^10.1101/2020.02.25.965582], or SARS-CoV-2[^10.1101/2020.04.15.997254][^10.1128/AAC.00819-20] in cell culture with EC<sub>50</sub>s consistently below 1 μM. One study suggested that SARS-CoV-2 facilitates AKT1/SKPT-dependent degradation of autophagy-initiating Beclin-1 and that reversal of this effect by niclosamide and the AKT inhibitor MK-2206 markedly inhibits SARS-CoV-2[^10.1101/2020.04.15.997254]. Although niclosamide dosing has generally been designed to achieve high intestinal levels, a recent pharmacokinetic review suggested that even at current doses, its maximal plasma concentration may be sufficient to inhibit SARS-CoV-2 replication[^10.1002/cpt.1909].

__Chlorpromazine__ is an antipsychotic drug used in treatment of schizophrenia that has been reported to inhibit several viruses by interfering with viral entry into cells by inhibiting clathrin-mediated endocytosis[^10.1038/nrd.2015.37][^10.7150/ijbs.45498]. Four studies have reported EC<sub>50</sub>s in the range of 1 to 14 μM for SARS-CoV and MERS-CoV[^10.1128/AAC.03036-14][^10.1128/AAC.03011-14][^10.1371/journal.pone.0194868][^10.1101/2020.03.25.008482].

__K22__ is a small molecule found to inhibit the double membrane formation and reticular vesicular network required for the replication of coronaviruses and other positive strand RNA viruses[^10.1371/journal.ppat.1004166][^10.1016/j.virusres.2018.01.002]. Although it has weak _in vitro_ activity, it is of interest because it appears to act at a novel stage of infection; furthermore, K22-resistant viruses containing nsp6 mutations were selected after 10 passages of HCoV-229E suggesting that this compound may also have specific antiviral activity[^10.1371/journal.ppat.1004166].

__Signaling pathway inhibitors__: High-throughput experiments have been performed for some viral infections to identify host proteins essential for virus infection but not cell viability. These have not yet been reported for coronavirus infections. However, several transcriptomic and proteomic analyses of SARS-CoV and MERS-CoV infection have described perturbations in cellular signaling pathways raising the possibility that drugs that influence the kinase proteins involved in these pathways might influence coronavirus replication[^10.1128/mBio.00165-13][^10.1128/mBio.00611-12][^10.1128/AAC.03659-14]. One encyclopedic proteomic SARS-CoV-2 study described the many interactions between virus and host proteins but was not primarily focused on signaling pathways[^10.1038/s41586-020-2286-9].

The kinase inhibitors dasatinib and imatinib were found to inhibit SARS-CoV and MERS-CoV with EC<sub>50</sub>s of 2.1 and 5.4 μM for dasatinib and 9.8 and 17.7 μM for imatinib[^10.1128/AAC.03036-14]. Further mechanistic studies demonstrated that imatinib inhibits the early stages of infection with SARS-CoV, MERS-CoV and other coronaviruses possibly by affecting viral fusion with endosomal membranes; inhibition was demonstrated to occur specifically through ABL kinase inhibition[^10.1128/JVI.01429-16][^10.1099/jgv.0.001047]. At least one pharmacokinetic study suggested, however, that standard doses of imatinib would be associated with plasma levels considerably lower than the EC<sub>50</sub>s reported for SARS-CoV and MERS-CoV[^10.1002/cpt.1909]. As of July, imatinib is listed in eight ongoing or planned clinical trials including two placebo-controlled trials. It appears, however, that in these studies imatinib is being studied primarily for its role in treating pulmonary fibrosis. Another kinase inhibitor, saracatinib, has also recently been shown to inhibit several coronaviruses in the low μM range[^10.3390/v10060283]

__Nitazoxanide__: Nitazoxanide is licensed for the treatment of cryptosporidium infections. It has been reported to inhibit a wide range of viruses in vitro. It has been proposed to interfere with host regulated pathways involved in viral replication including those involving interferon or mTORC1[^10.1016/j.jiph.2016.04.001]. In one study its EC<sub>50</sub> for SARS-CoV-2 was 2.1 μM[^10.1038/s41422-020-0282-0]. One pharmacokinetic analysis suggested that nitazoxanide plasma levels could reach concentrations above its one reported EC<sub>50</sub> level[^10.1002/cpt.1909]. As of July, it is being studied in five ongoing and nine planned clinical trials.

__Ciclesonide__: Ciclesonide is an inhaled corticosteroid discovered in a high-throughput drug screen to inhibit coronavirus replication in the low μM range[^10.1101/2020.03.11.987016][^10.1128/AAC.00819-20][^10.1101/2020.02.25.965582]. Although its mechanism of action is not known, virus passage in the presence of ciclesonide led to the selection of a ciclesonide-resistant mutant, characterized by a point mutation in the NSP15 gene. A single mutation in this gene was responsible for ~100-fold reduced susceptibility. A small case series of three patients from Japan who received inhaled ciclesonide has been published[^10.1016/j.jiac.2020.04.007], As of April 21, there are two ongoing and six planned studies of inhaled ciclesonide.

__Emetine__: Emetine is an FDA-approved drug for treating amebiasis. It is produced from the ipecac root and has been used to induce vomiting. Emetine has been shown in four studies to inhibit multiple coronaviruses including SARS-CoV, MERS-CoV, and SARS-CoV-2 _in vitro_, with EC<sub>50</sub>s consistently below 1.0 μM[^10.1128/AAC.03036-14][^10.1128/JVI.00023-19][^10.1101/2020.02.25.965582][^10.1016/j.antiviral.2020.104786]. Its mechanism of action is not known for certain, although it did inhibit MERS-CoV in an entry inhibitor assay using a pseudotype virus expressing S protein[^10.1128/JVI.00023-19]. After oral administration, its levels in the lungs are much higher than in the plasma and well above its reported EC<sub>50</sub> levels[^10.3390/ph13030051]. Emetine is currently not being studied for the treatment of SARS-CoV-2

__Interferon inducers__: Several compounds appear to inhibit viruses by inducing interferon. Ampligen® is a mismatched double-stranded RNA polymer that stimulates the production of interferon. It has been reported to have _in vitro_ activity against several viruses[^10.4049/jimmunol.178.8.5200] including SARS-CoV [^10.1177/095632020601700505]. It has also been reported to protect mice against SARS-CoV challenge in three studies[^10.1177/095632020601700505][^10.1016/j.virol.2009.09.023][^10.1016/j.antiviral.2016.12.007]. Hiltonol® is synthetic double-stranded polyriboinosinic-polyribocytidylic acid reported to protect mice from lethal SARS-CoV challenge when administered intranasally prior to infection[^10.1016/j.antiviral.2016.12.007]. Tilorone is a small molecule reported to inhibit MERS-CoV and other viruses _in vitro_ by inducing interferon[^10.1128/JVI.00023-19][^10.1128/AAC.00440-20].

__Cyclophilin inhibitors__: Cyclosporin and FK-506 are two immunosuppressive drugs commonly used to prevent rejection in transplant recipients. The discovery of these two compounds led to the identification of the family of cyclophilin host proteins that bind to cyclosporin and FK-506. Cyclophilins are involved in a wide range of cellular processes including protein folding, protein trafficking, and cell signaling. They have also been shown to play a role in the replication of several viruses including HIV-1, hepatitis C, influenza, and several coronavirus species[^10.1016/j.virol.2018.06.011]. Alisporivir is a non-immunosuppressive cyclophilin inhibitor which has been studied primarily for the treatment of hepatitis C virus. However, these compounds have generally had just weak inhibitory effects of on coronavirus replication and alisporivir did not protect mice from SARS-CoV infection[^10.1099/vir.0.034983-0][^10.1099/vir.0.052910-0][^10.1016/j.virusres.2016.11.011][^10.1016/j.virusres.2014.02.010][^10.1016/j.antiviral.2018.05.007][^10.1128/AAC.00819-20].

__Ivermectin__: Ivermectin is one of an increasing number of additional compounds found to inhibit SARS-CoV-2 without a defined mechanism of action. It has a reported EC<sub>50</sub> of about 2 μM[^10.1016/j.antiviral.2020.104787]. However, its plasma levels following oral administration appear too low to inhibit coronavirus replication [^10.4269/ajtmh.20-0271][^10.1002/cpt.1909][^10.1101/2020.04.21.20073262]. In one retrospective study, ivermectin use was associated with significantly reduced mortality compared to persons at the same hospitals not receiving ivermectin, particularly in patients with severe pulmonary disease[^10.1101/2020.06.06.20124461]. As of July, ivermectin is being studied in 19 ongoing and 18 planned clinical trials.

<!-- start of references details -->

[^10.3390/v11080745]: Peersen OB. "A Comprehensive Superposition of Viral Polymerase Structures" _Viruses_, 2019. [10.3390/v11080745](https://doi.org/10.3390/v11080745)
[^10.1073/pnas.0508200103]: Minskaia E, Hertzig T, et int., Ziebuhr J. "Discovery of an RNA virus 3'->5' exoribonuclease that is critically involved in coronavirus RNA synthesis" _Proc. Natl. Acad. Sci. U.S.A._, 2006. [10.1073/pnas.0508200103](https://doi.org/10.1073/pnas.0508200103)
[^10.1371/journal.ppat.1003565]: Smith EC, Blanc H, et int., Denison MR. "Coronaviruses lacking exoribonuclease activity are susceptible to lethal mutagenesis: evidence for proofreading and potential therapeutics" _PLoS Pathog._, 2013. [10.1371/journal.ppat.1003565](https://doi.org/10.1371/journal.ppat.1003565)
[^10.1073/pnas.1718806115]: Ferron F, Subissi L, et int., Imbert I. "Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA" _Proc. Natl. Acad. Sci. U.S.A._, 2018. [10.1073/pnas.1718806115](https://doi.org/10.1073/pnas.1718806115)
[^10.1038/s41467-019-10280-3]: Kirchdoerfer RN, Ward AB. "Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors" _Nat Commun_, 2019. [10.1038/s41467-019-10280-3](https://doi.org/10.1038/s41467-019-10280-3)
[^10.1126/science.abb7498]: Gao Y, Yan L, et int., Rao Z. "Structure of the RNA-dependent RNA polymerase from COVID-19 virus" _Science_, 2020. [10.1126/science.abb7498](https://doi.org/10.1126/science.abb7498)
[^10.1126/science.abc1560]: Yin W, Mao C, et int., Xu HE. "Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir" _Science_, 2020. [10.1126/science.abc1560](https://doi.org/10.1126/science.abc1560)
[^10.1021/acs.jmedchem.6b01594]: Siegel D, Hui HC, et int., Mackman RL. "Discovery and Synthesis of a Phosphoramidate Prodrug of a Pyrrolo[2,1-f][triazin-4-amino] Adenine C-Nucleoside (GS-5734) for the Treatment of Ebola and Emerging Viruses" _J. Med. Chem._, 2017. [10.1021/acs.jmedchem.6b01594](https://doi.org/10.1021/acs.jmedchem.6b01594)
[^10.1038/nature17180]: Warren TK, Jordan R, et int., Bavari S. "Therapeutic efficacy of the small molecule GS-5734 against Ebola virus in rhesus monkeys" _Nature_, 2016. [10.1038/nature17180](https://doi.org/10.1038/nature17180)
[^10.1074/jbc.AC120.013056]: Gordon CJ, Tchesnokov EP, et int., Gotte M. "The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus" _J. Biol. Chem._, 2020. [10.1074/jbc.AC120.013056](https://doi.org/10.1074/jbc.AC120.013056)
[^10.1074/jbc.RA120.013679]: Gordon CJ, Tchesnokov EP, et int., Götte M. "Remdesivir is a direct-acting antiviral that inhibits RNA-dependent RNA polymerase from severe acute respiratory syndrome coronavirus 2 with high potency" _J. Biol. Chem._, 2020. [10.1074/jbc.RA120.013679](https://doi.org/10.1074/jbc.RA120.013679)
[^10.1016/j.antiviral.2020.104793]: Shannon A, Tuyet Le NT, et int., Canard B. "Remdesivir and SARS-CoV-2: structural requirements at both nsp12 RdRp and nsp14 Exonuclease active-sites" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104793](https://doi.org/10.1016/j.antiviral.2020.104793)
[^10.1126/scitranslmed.aal3653]: Sheahan TP, Sims AC, et int., Baric RS. "Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses" _Sci Transl Med_, 2017. [10.1126/scitranslmed.aal3653](https://doi.org/10.1126/scitranslmed.aal3653)
[^10.1128/mBio.00221-18]: Agostini ML, Andres EL, et int., Denison MR. "Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease" _mBio_, 2018. [10.1128/mBio.00221-18](https://doi.org/10.1128/mBio.00221-18)
[^10.1038/s41467-019-13940-6]: Sheahan TP, Sims AC, et int., Baric RS. "Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against MERS-CoV" _Nat Commun_, 2020. [10.1038/s41467-019-13940-6](https://doi.org/10.1038/s41467-019-13940-6)
[^10.1038/s41422-020-0282-0]: Wang M, Cao R, et int., Xiao G. "Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro" _Cell Res._, 2020. [10.1038/s41422-020-0282-0](https://doi.org/10.1038/s41422-020-0282-0)
[^10.1101/2020.04.03.024257]: Bojkova D, McGreig JE, et int., Cinatl JN. "SARS-CoV-2 and SARS-CoV differ in their cell tropism and drug sensitivity profiles" _bioRxiv_, 2020. [10.1101/2020.04.03.024257](https://doi.org/10.1101/2020.04.03.024257)
[^10.1016/j.antiviral.2020.104786]: Choy K, Yin-Lam Wong A, et int., Yen H. "Remdesivir, lopinavir, emetine, and homoharringtonine inhibit SARS-CoV-2 replication in vitro" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104786](https://doi.org/10.1016/j.antiviral.2020.104786)
[^10.1128/AAC.00819-20]: Jeon S, Ko M, et int., Kim S. "Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs" _Antimicrobial Agents and Chemotherapy_, 2020. [10.1128/AAC.00819-20](https://doi.org/10.1128/AAC.00819-20)
[^10.1101/2020.04.03.023846]: Touret F, Gilles M, et int., Coutard B. "In vitro screening of a FDA approved chemical library reveals potential inhibitors of SARS-CoV-2 replication" _bioRxiv_, 2020. [10.1101/2020.04.03.023846](https://doi.org/10.1101/2020.04.03.023846)
[^10.1101/2020.04.03.20052548]: Meyer SD, Bojkova D, et int., Ciesek S. "Lack of Antiviral Activity of Darunavir against SARS-CoV-2" _medRxiv_, 2020. [10.1101/2020.04.03.20052548](https://doi.org/10.1101/2020.04.03.20052548)
[^10.1101/2020.04.27.064279]: Pruijssers AJ, George AS, et int., Sheahan TP. "Remdesivir potently inhibits SARS-CoV-2 in human lung cells and chimeric SARS-CoV expressing the SARS-CoV-2 RNA polymerase in mice." _bioRxiv_, 2020. [10.1101/2020.04.27.064279](https://doi.org/10.1101/2020.04.27.064279)
[^10.1073/pnas.1922083117]: de Wit E, Feldmann F, et int., Feldmann H. "Prophylactic and therapeutic remdesivir (GS-5734) treatment in the rhesus macaque model of MERS-CoV infection" _Proc. Natl. Acad. Sci. U.S.A._, 2020. [10.1073/pnas.1922083117](https://doi.org/10.1073/pnas.1922083117)
[^10.1038/s41586-020-2423-5]: Williamson BN, Feldmann F, et int., de Wit E. "Clinical benefit of remdesivir in rhesus macaques infected with SARS-CoV-2" _Nature_, 2020. [10.1038/s41586-020-2423-5](https://doi.org/10.1038/s41586-020-2423-5)
[^10.1002/phar.2398]: Barlow A, Landolf KM, et int., Heavner MS. "Review of Emerging Pharmacotherapy for the Treatment of Coronavirus Disease 2019" _Pharmacotherapy_, 2020. [10.1002/phar.2398](https://doi.org/10.1002/phar.2398)
[^10.1016/j.onehlt.2020.100128]: Amirian ES, Levy JK. "Current knowledge about the antivirals remdesivir (GS-5734) and GS-441524 as therapeutic options for coronaviruses" _One Health_, 2020. [10.1016/j.onehlt.2020.100128](https://doi.org/10.1016/j.onehlt.2020.100128)
[^10.1002/cpt.1909]: Arshad U, Pertinez H, et int., Owen A. "Prioritisation of Anti-SARS-Cov-2 Drug Repurposing Opportunities Based on Plasma and Target Site Concentrations Derived from their Established Human Pharmacokinetics" _Clin. Pharmacol. Ther._, 2020. [10.1002/cpt.1909](https://doi.org/10.1002/cpt.1909)
[^10.1056/NEJMoa2007016]: Grein J, Ohmagari N, et int., Flanigan T. "Compassionate Use of Remdesivir for Patients with Severe Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2007016](https://doi.org/10.1056/NEJMoa2007016)
[^10.1016/S0140-6736(20)31022-9]: Wang Y, Zhang D, et int., Wang C. "Remdesivir in adults with severe COVID-19: a randomised, double-blind, placebo-controlled, multicentre trial" _The Lancet_, 2020. [10.1016/S0140-6736(20)31022-9](https://doi.org/10.1016/S0140-6736(20)31022-9)
[^10.1056/NEJMoa2007764]: Beigel JH, Tomashek KM, et int., ACTT-1 Study Group Members. "Remdesivir for the Treatment of Covid-19 - Preliminary Report" _N. Engl. J. Med._, 2020. [10.1056/NEJMoa2007764](https://doi.org/10.1056/NEJMoa2007764)
[^10.1056/NEJMoa2015301]: Goldman JD, Lye DC, et int., Subramanian A. "Remdesivir for 5 or 10 Days in Patients with Severe Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2015301](https://doi.org/10.1056/NEJMoa2015301)
[^10.1016/j.coviro.2019.04.002]: Pruijssers AJ, Denison MR. "Nucleoside analogues for the treatment of coronavirus infections" _Curr Opin Virol_, 2019. [10.1016/j.coviro.2019.04.002](https://doi.org/10.1016/j.coviro.2019.04.002)
[^10.1177/095632020401500102]: Barnard DL, Hubbard VD, et int., Sidwell RW. "Inhibition of severe acute respiratory syndrome-associated coronavirus (SARSCoV) by calpain inhibitors and beta-D-N4-hydroxycytidine" _Antivir. Chem. Chemother._, 2004. [10.1177/095632020401500102](https://doi.org/10.1177/095632020401500102)
[^10.1177/095632020601700505]: Barnard DL, Day CW, et int., Sidwell RW. "Evaluation of immunomodulators, interferons and known in vitro SARS-coV inhibitors for inhibition of SARS-coV replication in BALB/c mice" _Antivir. Chem. Chemother._, 2006. [10.1177/095632020601700505](https://doi.org/10.1177/095632020601700505)
[^10.1128/JVI.01348-19]: Agostini ML, Pruijssers AJ, et int., Denison MR. "Small-Molecule Antiviral β-d-N4-Hydroxycytidine Inhibits a Proofreading-Intact Coronavirus with a High Genetic Barrier to Resistance" _J. Virol._, 2019. [10.1128/JVI.01348-19](https://doi.org/10.1128/JVI.01348-19)
[^10.1126/scitranslmed.abb5883]: Sheahan TP, Sims AC, et int., Baric RS. "An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice" _Sci Transl Med_, 2020. [10.1126/scitranslmed.abb5883](https://doi.org/10.1126/scitranslmed.abb5883)
[^10.1016/s0140-6736(03)13615-x]: Cinatl J, Morgenstern B, et int., Doerr HW. "Glycyrrhizin, an active component of liquorice roots, and replication of SARS-associated coronavirus" _Lancet_, 2003. [10.1016/s0140-6736(03)13615-x](https://doi.org/10.1016/s0140-6736(03)13615-x)
[^10.1016/j.jinf.2013.09.029]: Chan JFW, Chan K, et int., Yuen K. "Broad-spectrum antivirals for the emerging Middle East respiratory syndrome coronavirus" _J. Infect._, 2013. [10.1016/j.jinf.2013.09.029](https://doi.org/10.1016/j.jinf.2013.09.029)
[^10.1099/vir.0.061911-0]: Hart BJ, Dyall J, et int., Hensley L. "Interferon-β and mycophenolic acid are potent inhibitors of Middle East respiratory syndrome coronavirus in cell-based assays" _J. Gen. Virol._, 2014. [10.1099/vir.0.061911-0](https://doi.org/10.1099/vir.0.061911-0)
[^10.1038/srep01686]: Falzarano D, de Wit E, et int., Feldmann H. "Inhibition of novel β coronavirus replication by a combination of interferon-α2b and ribavirin" _Sci Rep_, 2013. [10.1038/srep01686](https://doi.org/10.1038/srep01686)
[^10.1371/journal.pmed.0030343]: Stockman LJ, Bellamy R, Garner P. "SARS: systematic review of treatment effects" _PLoS Med._, 2006. [10.1371/journal.pmed.0030343](https://doi.org/10.1371/journal.pmed.0030343)
[^10.1016/j.antiviral.2013.09.015]: Furuta Y, Gowen BB, et int., Barnard DL. "Favipiravir (T-705), a novel viral RNA polymerase inhibitor" _Antiviral Res._, 2013. [10.1016/j.antiviral.2013.09.015](https://doi.org/10.1016/j.antiviral.2013.09.015)
[^10.1016/j.antiviral.2018.03.003]: Delang L, Abdelnabi R, Neyts J. "Favipiravir as a potential countermeasure against neglected and emerging RNA viruses" _Antiviral Res._, 2018. [10.1016/j.antiviral.2018.03.003](https://doi.org/10.1016/j.antiviral.2018.03.003)
[^10.1101/2020.05.15.098731]: Shannon A, Selisko B, et int., Canard B. "Favipiravir strikes the SARS-CoV-2 at its Achilles heel, the RNA polymerase" _bioRxiv_, 2020. [10.1101/2020.05.15.098731](https://doi.org/10.1101/2020.05.15.098731)
[^10.1002/cpt.1844]: Du Y, Chen X. "Favipiravir: Pharmacokinetics and Concerns About Clinical Trials for 2019-nCoV Infection" _Clinical Pharmacology & Therapeutics_, 2020. [10.1002/cpt.1844](https://doi.org/10.1002/cpt.1844)
[^10.1101/2020.06.19.159053]: Kaptein SJ, Jacobs S, et int., Delang L. "Antiviral treatment of SARS-CoV-2-infected hamsters reveals a weak effect of favipiravir and a complete lack of effect for hydroxychloroquine" _bioRxiv_, 2020. [10.1101/2020.06.19.159053](https://doi.org/10.1101/2020.06.19.159053)
[^10.1101/2020.07.07.191775]: Driouich J, Cochin M, et int., Nougairède A. "Favipiravir and severe acute respiratory syndrome coronavirus 2 in hamster model" _bioRxiv_, 2020. [10.1101/2020.07.07.191775](https://doi.org/10.1101/2020.07.07.191775)
[^10.1101/2020.03.17.20037432]: Chen C, Huang J, et int., Wang X. "Favipiravir versus Arbidol for COVID-19: A Randomized Clinical Trial" _MedRxiv_, 2020. [10.1101/2020.03.17.20037432](https://doi.org/10.1101/2020.03.17.20037432)
[^10.1016/j.eng.2020.03.007]: Cai Q, Yang M, et int., Liu L. "Experimental Treatment with Favipiravir for COVID-19: An Open-Label Control Study" _Engineering (Beijing)_, 2020. [10.1016/j.eng.2020.03.007](https://doi.org/10.1016/j.eng.2020.03.007)
[^10.1101/2020.04.29.067983]: Liu S, Lien C, et int., Wang T. "Evaluation of 19 antiviral drugs against SARS-CoV-2 Infection" _bioRxiv_, 2020. [10.1101/2020.04.29.067983](https://doi.org/10.1101/2020.04.29.067983)
[^10.1101/2020.06.15.153411]: Sacramento CQ, Fintelman-Rodrigues N, et int., Souza TML. "The in vitro antiviral activity of the anti-hepatitis C virus (HCV) drugs daclatasvir and sofosbuvir against SARS-CoV-2" _bioRxiv_, 2020. [10.1101/2020.06.15.153411](https://doi.org/10.1101/2020.06.15.153411)
[^10.1128/mBio.01114-20]: Park S, Yu K, et int., Choi YK. "Antiviral Efficacies of FDA-Approved Drugs against SARS-CoV-2 Infection in Ferrets" _mBio_, 2020. [10.1128/mBio.01114-20](https://doi.org/10.1128/mBio.01114-20)
[^10.1101/2020.03.18.997585]: Chien M, Anderson TK, et int., Ju J. "Nucleotide Analogues as Inhibitors of SARS-CoV-2 Polymerase" _bioRxiv_, 2020. [10.1101/2020.03.18.997585](https://doi.org/10.1101/2020.03.18.997585)
[^10.1016/j.antiviral.2020.104857]: Jockusch S, Tao C, et int., Ju J. "A Library of Nucleotide Analogues Terminate RNA Synthesis Catalyzed by Polymerases of Coronaviruses that Cause SARS and COVID-19" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104857](https://doi.org/10.1016/j.antiviral.2020.104857)
[^10.1038/nature13027]: Warren TK, Wells J, et int., Bavari S. "Protection against filovirus diseases by a novel broad-spectrum nucleoside analogue BCX4430" _Nature_, 2014. [10.1038/nature13027](https://doi.org/10.1038/nature13027)
[^10.1016/j.antiviral.2005.01.003]: Saijo M, Morikawa S, et int., Kurane I. "Inhibitory effect of mizoribine and ribavirin on the replication of severe acute respiratory syndrome (SARS)-associated coronavirus" _Antiviral Res._, 2005. [10.1016/j.antiviral.2005.01.003](https://doi.org/10.1016/j.antiviral.2005.01.003)
[^10.1128/JVI.00023-19]: Shen L, Niu J, et int., Tan W. "High-Throughput Screening and Identification of Potent Broad-Spectrum Inhibitors of Coronaviruses" _J. Virol._, 2019. [10.1128/JVI.00023-19](https://doi.org/10.1128/JVI.00023-19)
[^10.1101/2020.04.07.028589]: Bukreyeva N, Mantlo EK, et int., Zeldis J. "The IMPDH inhibitor merimepodib suppresses SARS-CoV-2 replication in vitro." _bioRxiv_, 2020. [10.1101/2020.04.07.028589](https://doi.org/10.1101/2020.04.07.028589)
[^10.1002/cmdc.202000223]: Ghosh AK, Brindisi M, et int., Mesecar AD. "Drug Development and Medicinal Chemistry Efforts Toward SARS-Coronavirus and Covid-19 Therapeutics" _ChemMedChem_, 2020. [10.1002/cmdc.202000223](https://doi.org/10.1002/cmdc.202000223)
[^10.1038/s41586-020-2223-y]: Jin Z, Du X, et int., Yang H. "Structure of Mpro from COVID-19 virus and discovery of its inhibitors" _Nature_, 2020. [10.1038/s41586-020-2223-y](https://doi.org/10.1038/s41586-020-2223-y)
[^10.1126/science.abb3405]: Zhang L, Lin D, et int., Hilgenfeld R. "Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors" _Science_, 2020. [10.1126/science.abb3405](https://doi.org/10.1126/science.abb3405)
[^PMID14660806]: Chan KS, Lai ST, et int., Yuen KY. "Treatment of severe acute respiratory syndrome with lopinavir/ritonavir: a multicentre retrospective matched cohort study" _Hong Kong Med J_, 2003. [PMID14660806](https://pubmed.ncbi.nlm.nih.gov/14660806)
[^10.1136/thorax.2003.012658]: Chu CM, Cheng VCC, et int., HKU/UCH SARS Study Group. "Role of lopinavir/ritonavir in the treatment of SARS: initial virological and clinical findings" _Thorax_, 2004. [10.1136/thorax.2003.012658](https://doi.org/10.1136/thorax.2003.012658)
[^10.1016/j.jcv.2004.03.003]: Chen F, Chan KH, et int., Yuen KY. "In vitro susceptibility of 10 clinical isolates of SARS coronavirus to selected antiviral compounds" _J. Clin. Virol._, 2004. [10.1016/j.jcv.2004.03.003](https://doi.org/10.1016/j.jcv.2004.03.003)
[^10.1016/j.bbrc.2004.04.083]: Yamamoto N, Yang R, et int., Yamamoto N. "HIV protease inhibitor nelfinavir inhibits replication of SARS-associated coronavirus" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.04.083](https://doi.org/10.1016/j.bbrc.2004.04.083)
[^10.1128/AAC.03011-14]: de Wilde AH, Jochmans D, et int., Snijder EJ. "Screening of an FDA-approved compound library identifies four small-molecule inhibitors of Middle East respiratory syndrome coronavirus replication in cell culture" _Antimicrob. Agents Chemother._, 2014. [10.1128/AAC.03011-14](https://doi.org/10.1128/AAC.03011-14)
[^10.1101/2020.04.06.026476]: Yamamoto N, Matsuyama S, et int., Yamamoto N. "Nelfinavir inhibits replication of severe acute respiratory syndrome coronavirus 2 in vitro." _bioRxiv_, 2020. [10.1101/2020.04.06.026476](https://doi.org/10.1101/2020.04.06.026476)
[^10.1073/pnas.0403596101]: Wu C, Jan J, et int., Wong C. "Small molecules targeting severe acute respiratory syndrome human coronavirus" _Proceedings of the National Academy of Sciences_, 2004. [10.1073/pnas.0403596101](https://doi.org/10.1073/pnas.0403596101)
[^10.1038/s41422-020-0356-z]: Ma C, Sacco MD, et int., Wang J. "Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease" _Cell Research_, 2020. [10.1038/s41422-020-0356-z](https://doi.org/10.1038/s41422-020-0356-z)
[^10.1093/infdis/jiv392]: Chan JF, Yao Y, et int., Yuen K. "Treatment With Lopinavir/Ritonavir or Interferon-β1b Improves Outcome of MERS-CoV Infection in a Nonhuman Primate Model of Common Marmoset" _J. Infect. Dis._, 2015. [10.1093/infdis/jiv392](https://doi.org/10.1093/infdis/jiv392)
[^10.1056/NEJMoa2001282]: Cao B, Wang Y, et int., Wang C. "A Trial of Lopinavir–Ritonavir in Adults Hospitalized with Severe Covid-19" _NEJM_, 2020. [10.1056/NEJMoa2001282](https://doi.org/10.1056/NEJMoa2001282)
[^10.1101/2020.06.16.153403]: Bakowski MA, Beutler N, et int., Rogers TF. "Oral drug repositioning candidates and synergistic remdesivir combinations for the prophylaxis and treatment of COVID-19" _bioRxiv_, 2020. [10.1101/2020.06.16.153403](https://doi.org/10.1101/2020.06.16.153403)
[^10.1101/2020.04.14.039925]: Ohashi H, Watashi K, et int., Wakita T. "Multidrug treatment with nelfinavir and cepharanthine against COVID-19" _bioRxiv_, 2020. [10.1101/2020.04.14.039925](https://doi.org/10.1101/2020.04.14.039925)
[^10.1101/2020.04.04.020925]: Fintelman-Rodrigues N, Sacramento CQ, et int., Souza TML. "Atazanavir inhibits SARS-CoV-2 replication and pro-inflammatory cytokine production" _bioRxiv_, 2020. [10.1101/2020.04.04.020925](https://doi.org/10.1101/2020.04.04.020925)
[^10.1016/j.virusres.2007.02.015]: Grum-Tokars V, Ratia K, et int., Mesecar AD. "Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery" _Virus Res._, 2008. [10.1016/j.virusres.2007.02.015](https://doi.org/10.1016/j.virusres.2007.02.015)
[^10.1021/jm050184y]: Shie J, Fang J, et int., Wong C. "Discovery of potent anilide inhibitors against the severe acute respiratory syndrome 3CL protease" _J. Med. Chem._, 2005. [10.1021/jm050184y](https://doi.org/10.1021/jm050184y)
[^10.1016/j.bmc.2016.05.013]: Kumar V, Tan K, et int., Liang P. "Identification, synthesis and evaluation of SARS-CoV and MERS-CoV 3C-like protease inhibitors" _Bioorg. Med. Chem._, 2016. [10.1016/j.bmc.2016.05.013](https://doi.org/10.1016/j.bmc.2016.05.013)
[^10.1021/acs.jmedchem.9b01828]: Zhang L, Lin D, et int., Hilgenfeld R. "α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment" _J. Med. Chem._, 2020. [10.1021/acs.jmedchem.9b01828](https://doi.org/10.1021/acs.jmedchem.9b01828)
[^10.1128/JVI.01348-12]: Kim Y, Lovell S, et int., Chang K. "Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses" _J. Virol._, 2012. [10.1128/JVI.01348-12](https://doi.org/10.1128/JVI.01348-12)
[^10.1101/2020.05.03.073080]: Vuong W, Khan MB, et int., Lemieux MJ. "Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication" _bioRxiv_, 2020. [10.1101/2020.05.03.073080](https://doi.org/10.1101/2020.05.03.073080)
[^10.1101/2020.06.07.138677]: Luan X, Shang W, et int., Zhang S. "Structure Basis for Inhibition of SARS-CoV-2 by the Feline Drug GC376" _bioRxiv_, 2020. [10.1101/2020.06.07.138677](https://doi.org/10.1101/2020.06.07.138677)
[^10.1371/journal.ppat.1005531]: Kim Y, Liu H, et int., Pedersen NC. "Reversal of the Progression of Fatal Coronavirus Infection in Cats by a Broad-Spectrum Coronavirus Protease Inhibitor" _PLoS Pathog._, 2016. [10.1371/journal.ppat.1005531](https://doi.org/10.1371/journal.ppat.1005531)
[^10.1016/j.ejmech.2018.03.004]: Galasiti Kankanamalage AC, Kim Y, et int., Groutas WC. "Structure-guided design of potent and permeable inhibitors of MERS coronavirus 3CL protease that utilize a piperidine moiety as a novel design element" _Eur J Med Chem_, 2018. [10.1016/j.ejmech.2018.03.004](https://doi.org/10.1016/j.ejmech.2018.03.004)
[^10.1126/science.abb4489]: Dai W, Zhang B, et int., Liu H. "Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease" _Science_, 2020. [10.1126/science.abb4489](https://doi.org/10.1126/science.abb4489)
[^10.1016/j.antiviral.2017.12.015]: Lin M, Moses DC, et int., Chou C. "Disulfiram can inhibit MERS and SARS coronavirus papain-like proteases via different modes" _Antiviral Res._, 2018. [10.1016/j.antiviral.2017.12.015](https://doi.org/10.1016/j.antiviral.2017.12.015)
[^10.1101/2020.07.01.183020]: Swaim CD, Perng Y, et int., Huibregtse JM. "6-Thioguanine blocks SARS-CoV-2 replication by inhibition of PLpro protease activities" _bioRxiv_, 2020. [10.1101/2020.07.01.183020](https://doi.org/10.1101/2020.07.01.183020)
[^10.1016/j.antiviral.2014.12.015]: Báez-Santos YM, St John SE, Mesecar AD. "The SARS-coronavirus papain-like protease: structure, function and inhibition by designed antiviral compounds" _Antiviral Res._, 2015. [10.1016/j.antiviral.2014.12.015](https://doi.org/10.1016/j.antiviral.2014.12.015)
[^10.1016/bs.aivir.2019.08.002]: Tortorici MA, Veesler D. "Structural insights into coronavirus entry" _Adv. Virus Res._, 2019. [10.1016/bs.aivir.2019.08.002](https://doi.org/10.1016/bs.aivir.2019.08.002)
[^10.1126/science.abb2507]: Wrapp D, Wang N, et int., McLellan JS. "Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation" _Science_, 2020. [10.1126/science.abb2507](https://doi.org/10.1126/science.abb2507)
[^10.1038/nature02145]: Li W, Moore MJ, et int., Farzan M. "Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus" _Nature_, 2003. [10.1038/nature02145](https://doi.org/10.1038/nature02145)
[^10.1038/s41586-020-2012-7]: Zhou P, Yang X, et int., Shi Z. "A pneumonia outbreak associated with a new coronavirus of probable bat origin" _Nature_, 2020. [10.1038/s41586-020-2012-7](https://doi.org/10.1038/s41586-020-2012-7)
[^10.1074/jbc.C300520200]: Wong SK, Li W, et int., Farzan M. "A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2" _J. Biol. Chem._, 2004. [10.1074/jbc.C300520200](https://doi.org/10.1074/jbc.C300520200)
[^10.1038/s41586-020-2180-5]: Lan J, Ge J, et int., Wang X. "Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor" _Nature_, 2020. [10.1038/s41586-020-2180-5](https://doi.org/10.1038/s41586-020-2180-5)
[^10.1007/s11427-020-1637-5]: Xu X, Chen P, et int., Hao P. "Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission" _Sci. China Life Sci._, 2020. [10.1007/s11427-020-1637-5](https://doi.org/10.1007/s11427-020-1637-5)
[^10.1126/science.1116480]: Li F, Li W, et int., Harrison SC. "Structure of SARS coronavirus spike receptor-binding domain complexed with receptor" _Science_, 2005. [10.1126/science.1116480](https://doi.org/10.1126/science.1116480)
[^10.1016/j.virusres.2014.11.021]: Millet JK, Whittaker GR. "Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis" _Virus Res._, 2015. [10.1016/j.virusres.2014.11.021](https://doi.org/10.1016/j.virusres.2014.11.021)
[^10.1016/j.cell.2020.02.058]: Walls AC, Park Y, et int., Veesler D. "Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein" _Cell_, 2020. [10.1016/j.cell.2020.02.058](https://doi.org/10.1016/j.cell.2020.02.058)
[^10.1016/j.antiviral.2020.104742]: Coutard B, Valle C, et int., Decroly E. "The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade" _Antiviral Research_, 2020. [10.1016/j.antiviral.2020.104742](https://doi.org/10.1016/j.antiviral.2020.104742)
[^10.1126/science.aad5224]: Corti D, Misasi J, et int., Sullivan NJ. "Protective monotherapy against lethal Ebola virus infection by a potently neutralizing antibody" _Science_, 2016. [10.1126/science.aad5224](https://doi.org/10.1126/science.aad5224)
[^10.1056/NEJMoa1910993]: Mulangu S, Dodd LE, et int., PALM Consortium Study Team. "A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics" _N. Engl. J. Med._, 2019. [10.1056/NEJMoa1910993](https://doi.org/10.1056/NEJMoa1910993)
[^10.2165/00003495-199958020-00009]: Scott LJ, Lamb HM. "Palivizumab" _Drugs_, 1999. [10.2165/00003495-199958020-00009](https://doi.org/10.2165/00003495-199958020-00009)
[^10.1038/nm.4268]: Caskey M, Schoofs T, et int., Klein F. "Antibody 10-1074 suppresses viremia in HIV-1-infected individuals" _Nat. Med._, 2017. [10.1038/nm.4268](https://doi.org/10.1038/nm.4268)
[^10.1038/s41586-018-0531-2]: Mendoza P, Gruell H, et int., Nussenzweig MC. "Combination therapy with anti-HIV-1 antibodies maintains viral suppression" _Nature_, 2018. [10.1038/s41586-018-0531-2](https://doi.org/10.1038/s41586-018-0531-2)
[^10.1007/s11684-017-0596-6]: Jin Y, Lei C, et int., Ying T. "Human monoclonal antibodies as candidate therapeutics against emerging viruses" _Front Med_, 2017. [10.1007/s11684-017-0596-6](https://doi.org/10.1007/s11684-017-0596-6)
[^10.3201/eid2607.200841]: Okba NMA, Müller MA, et int., Haagmans BL. "Severe Acute Respiratory Syndrome Coronavirus 2-Specific Antibody Responses in Coronavirus Disease 2019 Patients" _Emerging Infect. Dis._, 2020. [10.3201/eid2607.200841](https://doi.org/10.3201/eid2607.200841)
[^10.1101/2020.05.03.20084442]: Suthar MS, Zimmerman M, et int., Wrammert J. "Rapid generation of neutralizing antibody responses in COVID-19 patients" _medRxiv_, 2020. [10.1101/2020.05.03.20084442](https://doi.org/10.1101/2020.05.03.20084442)
[^10.1101/2020.07.10.20150557]: Lee WT, Girardin RC, et int., McDonough KA. "Neutralizing Antibody Responses in COVID-19 Convalescent Sera" _medRxiv_, 2020. [10.1101/2020.07.10.20150557](https://doi.org/10.1101/2020.07.10.20150557)
[^10.1002/rmv.706]: Coughlin MM, Prabhakar BS. "Neutralizing human monoclonal antibodies to severe acute respiratory syndrome coronavirus: target, mechanism of action, and therapeutic potential" _Rev. Med. Virol._, 2012. [10.1002/rmv.706](https://doi.org/10.1002/rmv.706)
[^10.3390/v10120680]: Han H, Liu J, et int., Yu X. "Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection" _Viruses_, 2018. [10.3390/v10120680](https://doi.org/10.3390/v10120680)
[^10.1080/22221751.2019.1624482]: Xu J, Jia W, et int., Zhang L. "Antibodies and vaccines against Middle East respiratory syndrome coronavirus" _Emerg Microbes Infect_, 2019. [10.1080/22221751.2019.1624482](https://doi.org/10.1080/22221751.2019.1624482)
[^10.1038/s41586-020-2381-y]: Shi R, Shan C, et int., Yan J. "A human neutralizing antibody targets the receptor binding site of SARS-CoV-2" _Nature_, 2020. [10.1038/s41586-020-2381-y](https://doi.org/10.1038/s41586-020-2381-y)
[^10.1126/science.abd0827]: Hansen J, Baum A, et int., Kyratsous CA. "Studies in humanized mice and convalescent humans yield a SARS-CoV-2 antibody cocktail" _Science_, 2020. [10.1126/science.abd0827](https://doi.org/10.1126/science.abd0827)
[^10.1126/science.abd0831]: Baum A, Fulton BO, et int., Kyratsous CA. "Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies" _Science_, 2020. [10.1126/science.abd0831](https://doi.org/10.1126/science.abd0831)
[^10.1126/science.abb7269]: Yuan M, Wu NC, et int., Wilson IA. "A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV" _Science_, 2020. [10.1126/science.abb7269](https://doi.org/10.1126/science.abb7269)
[^10.1016/j.chom.2020.06.010]: Huo J, Zhao Y, et int., Stuart DI. "Neutralization of SARS-CoV-2 by Destruction of the Prefusion Spike" _Cell Host Microbe_, 2020. [10.1016/j.chom.2020.06.010](https://doi.org/10.1016/j.chom.2020.06.010)
[^10.1080/22221751.2020.1729069]: Tian X, Li C, et int., Ying T. "Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody" _Emerg Microbes Infect_, 2020. [10.1080/22221751.2020.1729069](https://doi.org/10.1080/22221751.2020.1729069)
[^10.1038/s41586-020-2349-y]: Pinto D, Park Y, et int., Corti D. "Cross-neutralization of SARS-CoV-2 by a human monoclonal SARS-CoV antibody" _Nature_, 2020. [10.1038/s41586-020-2349-y](https://doi.org/10.1038/s41586-020-2349-y)
[^10.1126/science.abc7424]: Wec AZ, Wrapp D, et int., Walker LM. "Broad neutralization of SARS-related viruses by human monoclonal antibodies" _Science_, 2020. [10.1126/science.abc7424](https://doi.org/10.1126/science.abc7424)
[^10.1038/s41467-020-16256-y]: Wang C, Li W, et int., Bosch B. "A human monoclonal antibody blocking SARS-CoV-2 infection" _Nat Commun_, 2020. [10.1038/s41467-020-16256-y](https://doi.org/10.1038/s41467-020-16256-y)
[^10.1101/2020.05.13.093088]: Li W, Drelich A, et int., Dimitrov DS. "Potent neutralization of SARS-CoV-2 in vitro and in an animal model by a human monoclonal antibody" _bioRxiv_, 2020. [10.1101/2020.05.13.093088](https://doi.org/10.1101/2020.05.13.093088)
[^10.1101/2020.05.19.104281]: Yuan AQ. "Isolation of and Characterization of Neutralizing Antibodies to Covid-19 from a Large Human Naïve scFv Phage Display Library" _bioRxiv_, 2020. [10.1101/2020.05.19.104281](https://doi.org/10.1101/2020.05.19.104281)
[^10.1038/s41591-020-0998-x]: Zost SJ, Gilchuk P, et int., Crowe JE. "Rapid isolation and profiling of a diverse panel of human monoclonal antibodies targeting the SARS-CoV-2 spike protein" _Nat. Med._, 2020. [10.1038/s41591-020-0998-x](https://doi.org/10.1038/s41591-020-0998-x)
[^10.1126/science.abc5902]: Brouwer PJM, Caniels TG, et int., van Gils MJ. "Potent neutralizing antibodies from COVID-19 patients define multiple targets of vulnerability" _Science_, 2020. [10.1126/science.abc5902](https://doi.org/10.1126/science.abc5902)
[^10.1126/science.abc6952]: Chi X, Yan R, et int., Chen W. "A neutralizing human antibody binds to the N-terminal domain of the Spike protein of SARS-CoV-2" _Science_, 2020. [10.1126/science.abc6952](https://doi.org/10.1126/science.abc6952)
[^10.1038/s41586-020-2380-z]: Ju B, Zhang Q, et int., Zhang L. "Human neutralizing antibodies elicited by SARS-CoV-2 infection" _Nature_, 2020. [10.1038/s41586-020-2380-z](https://doi.org/10.1038/s41586-020-2380-z)
[^10.1101/2020.06.17.153486]: Liu L, Wang P, et int., Ho DD. "Potent Neutralizing Monoclonal Antibodies Directed to Multiple Epitopes on the SARS-CoV-2 Spike" _bioRxiv_, 2020. [10.1101/2020.06.17.153486](https://doi.org/10.1101/2020.06.17.153486)
[^10.1038/s41586-020-2456-9]: Robbiani DF, Gaebler C, et int., Nussenzweig MC. "Convergent antibody responses to SARS-CoV-2 in convalescent individuals" _Nature_, 2020. [10.1038/s41586-020-2456-9](https://doi.org/10.1038/s41586-020-2456-9)
[^10.1126/science.abc7520]: Rogers TF, Zhao F, et int., Burton DR. "Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model" _Science_, 2020. [10.1126/science.abc7520](https://doi.org/10.1126/science.abc7520)
[^10.1126/science.abc2241]: Wu Y, Wang F, et int., Liu L. "A noncompeting pair of human neutralizing antibodies block COVID-19 virus binding to its receptor ACE2" _Science_, 2020. [10.1126/science.abc2241](https://doi.org/10.1126/science.abc2241)
[^10.1016/j.cell.2020.05.025]: Cao Y, Su B, et int., Xie XS. "Potent Neutralizing Antibodies against SARS-CoV-2 Identified by High-Throughput Single-Cell Sequencing of Convalescent Patients’ B Cells" _Cell_, 2020. [10.1016/j.cell.2020.05.025](https://doi.org/10.1016/j.cell.2020.05.025)
[^10.1101/2020.05.22.111005]: Zost SJ, Gilchuk P, et int., Crowe JE. "Potently neutralizing human antibodies that block SARS-CoV-2 receptor binding and protect animals" _bioRxiv_, 2020. [10.1101/2020.05.22.111005](https://doi.org/10.1101/2020.05.22.111005)
[^10.1056/NEJMoa035026]: Lalezari JP, Henry K, et int., Salgo M. "Enfuvirtide, an HIV-1 Fusion Inhibitor, for Drug-Resistant HIV Infection in North and South America" _New England Journal of Medicine_, 2003. [10.1056/NEJMoa035026](https://doi.org/10.1056/NEJMoa035026)
[^10.1056/NEJMoa035211]: Lazzarin A, Clotet B, et int., Salgo M. "Efficacy of Enfuvirtide in Patients Infected with Drug-Resistant HIV-1 in Europe and Australia" _New England Journal of Medicine_, 2003. [10.1056/NEJMoa035211](https://doi.org/10.1056/NEJMoa035211)
[^10.1038/nrd1331]: Matthews T, Salgo M, et int., Bolognesi D. "Enfuvirtide: the first therapy to inhibit the entry of HIV-1 into host CD4 lymphocytes" _Nature Reviews Drug Discovery_, 2004. [10.1038/nrd1331](https://doi.org/10.1038/nrd1331)
[^10.1016/j.bbrc.2004.05.046]: Yuan K, Yi L, et int., Deng H. "Suppression of SARS-CoV entry by peptides corresponding to heptad regions on spike glycoprotein" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.05.046](https://doi.org/10.1016/j.bbrc.2004.05.046)
[^10.1038/ncomms4067]: Lu L, Liu Q, et int., Jiang S. "Structure-based discovery of Middle East respiratory syndrome coronavirus fusion inhibitor" _Nat Commun_, 2014. [10.1038/ncomms4067](https://doi.org/10.1038/ncomms4067)
[^10.1093/infdis/jiv325]: Channappanavar R, Lu L, et int., Jiang S. "Protective Effect of Intranasal Regimens Containing Peptidic Middle East Respiratory Syndrome Coronavirus Fusion Inhibitor Against MERS-CoV Infection" _J. Infect. Dis._, 2015. [10.1093/infdis/jiv325](https://doi.org/10.1093/infdis/jiv325)
[^10.1021/acs.jmedchem.8b00890]: Wang C, Zhao L, et int., Liu K. "De Novo Design of α-Helical Lipopeptides Targeting Viral Fusion Proteins: A Promising Strategy for Relatively Broad-Spectrum Antiviral Drug Discovery" _J. Med. Chem._, 2018. [10.1021/acs.jmedchem.8b00890](https://doi.org/10.1021/acs.jmedchem.8b00890)
[^10.1126/sciadv.aav4580]: Xia S, Yan L, et int., Lu L. "A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike" _Sci Adv_, 2019. [10.1126/sciadv.aav4580](https://doi.org/10.1126/sciadv.aav4580)
[^10.1128/JVI.00635-20]: Zhu Y, Yu D, et int., He Y. "Design of potent membrane fusion inhibitors against SARS-CoV-2, an emerging coronavirus with high fusogenic activity" _J. Virol._, 2020. [10.1128/JVI.00635-20](https://doi.org/10.1128/JVI.00635-20)
[^10.1016/j.antiviral.2020.104792]: Tang T, Bidon M, et int., Daniel S. "Coronavirus membrane fusion mechanism offers as a potential target for antiviral development" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104792](https://doi.org/10.1016/j.antiviral.2020.104792)
[^10.1038/s41423-020-0374-2]: Xia S, Zhu Y, et int., Lu L. "Fusion mechanism of 2019-nCoV and fusion inhibitors targeting HR1 domain in spike protein" _Cellular & Molecular Immunology_, 2020. [10.1038/s41423-020-0374-2](https://doi.org/10.1038/s41423-020-0374-2)
[^10.1128/JVI.01697-07]: Ujike M, Nishikawa H, et int., Taguchi F. "Heptad Repeat-Derived Peptides Block Protease-Mediated Direct Entry from the Cell Surface of Severe Acute Respiratory Syndrome Coronavirus but Not Entry via the Endosomal Pathway" _Journal of Virology_, 2008. [10.1128/JVI.01697-07](https://doi.org/10.1128/JVI.01697-07)
[^10.1038/s41422-020-0305-x]: Xia S, Liu M, et int., Lu L. "Inhibition of SARS-CoV-2 (previously 2019-nCoV) infection by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion" _Cell Res._, 2020. [10.1038/s41422-020-0305-x](https://doi.org/10.1038/s41422-020-0305-x)
[^10.1128/JVI.00288-17]: Chong H, Xue J, et int., He Y. "A Lipopeptide HIV-1/2 Fusion Inhibitor with Highly Potent In Vitro, Ex Vivo, and In Vivo Antiviral Activity" _J. Virol._, 2017. [10.1128/JVI.00288-17](https://doi.org/10.1128/JVI.00288-17)
[^10.1038/nature03712]: Imai Y, Kuba K, et int., Penninger JM. "Angiotensin-converting enzyme 2 protects from severe acute lung failure" _Nature_, 2005. [10.1038/nature03712](https://doi.org/10.1038/nature03712)
[^10.1038/nm1267]: Kuba K, Imai Y, et int., Penninger JM. "A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury" _Nat. Med._, 2005. [10.1038/nm1267](https://doi.org/10.1038/nm1267)
[^10.1186/s13054-017-1823-x]: Khan A, Benthin C, et int., Lazaar AL. "A pilot clinical trial of recombinant human angiotensin-converting enzyme 2 in acute respiratory distress syndrome" _Crit Care_, 2017. [10.1186/s13054-017-1823-x](https://doi.org/10.1186/s13054-017-1823-x)
[^10.1016/j.cell.2020.04.004]: Monteil V, Kwon H, et int., Penninger JM. "Inhibition of SARS-CoV-2 Infections in Engineered Human Tissues Using Clinical-Grade Soluble Human ACE2" _Cell_, 2020. [10.1016/j.cell.2020.04.004](https://doi.org/10.1016/j.cell.2020.04.004)
[^10.1016/j.bbrc.2004.05.114]: Hofmann H, Geier M, et int., Pöhlmann S. "Susceptibility to SARS coronavirus S protein-driven infection correlates with expression of angiotensin converting enzyme 2 and infection can be blocked by soluble receptor" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.05.114](https://doi.org/10.1016/j.bbrc.2004.05.114)
[^10.1016/j.cell.2020.03.045]: Wang Q, Zhang Y, et int., Qi J. "Structural and Functional Basis of SARS-CoV-2 Entry by Using Human ACE2" _Cell_, 2020. [10.1016/j.cell.2020.03.045](https://doi.org/10.1016/j.cell.2020.03.045)
[^10.1038/s41467-020-16048-4]: Lei C, Qian K, et int., Hu S. "Neutralization of SARS-CoV-2 spike pseudotyped virus by recombinant ACE2-Ig" _Nature Communications_, 2020. [10.1038/s41467-020-16048-4](https://doi.org/10.1038/s41467-020-16048-4)
[^10.1101/2020.03.19.999318]: Zhang G, Pomplun S, et int., Pentelute BL. "Investigation of ACE2 N-terminal fragments binding to SARS-CoV-2 Spike RBD" _bioRxiv_, 2020. [10.1101/2020.03.19.999318](https://doi.org/10.1101/2020.03.19.999318)
[^10.1016/j.virol.2006.01.029]: Han DP, Penn-Nicholson A, Cho MW. "Identification of critical determinants on ACE2 for SARS-CoV entry and development of a potent entry inhibitor" _Virology_, 2006. [10.1016/j.virol.2006.01.029](https://doi.org/10.1016/j.virol.2006.01.029)
[^10.1101/2020.04.28.066761]: Mycroft-West C, Su D, et int., Skidmore MA. "Heparin inhibits cellular invasion by SARS-CoV-2: structural dependence of the interaction of the surface protein (spike) S1 receptor binding domain with heparin." _bioRxiv_, 2020. [10.1101/2020.04.28.066761](https://doi.org/10.1101/2020.04.28.066761)
[^10.1101/2020.06.08.140236]: Tandon R, Sharp JS, et int., Linhardt RJ. "Effective Inhibition of SARS-CoV-2 Entry by Heparin and Enoxaparin Derivatives" _bioRxiv_, 2020. [10.1101/2020.06.08.140236](https://doi.org/10.1101/2020.06.08.140236)
[^10.1101/2020.06.24.169334]: Guimond SE, Mycroft-West CJ, et int., Turnbull JE. "Pixatimod (PG545), a clinical-stage heparan sulfate mimetic, is a potent inhibitor of the SARS-CoV-2 virus" _bioRxiv_, 2020. [10.1101/2020.06.24.169334](https://doi.org/10.1101/2020.06.24.169334)
[^PMID15918330]: Zheng B, Guan Y, et int., Huang J. "Synthetic peptides outside the spike protein heptad repeat regions as potent inhibitors of SARS-associated coronavirus" _Antivir. Ther. (Lond.)_, 2005. [PMID15918330](https://pubmed.ncbi.nlm.nih.gov/15918330)
[^10.1128/JVI.02322-09]: O'Keefe BR, Giomarelli B, et int., McCray PB. "Broad-spectrum in vitro activity and in vivo efficacy of the antiviral protein griffithsin against emerging viruses of the family Coronaviridae" _J. Virol._, 2010. [10.1128/JVI.02322-09](https://doi.org/10.1128/JVI.02322-09)
[^10.1016/j.antiviral.2016.07.011]: Millet JK, Séron K, et int., Belouzard S. "Middle East respiratory syndrome coronavirus infection is inhibited by griffithsin" _Antiviral Res._, 2016. [10.1016/j.antiviral.2016.07.011](https://doi.org/10.1016/j.antiviral.2016.07.011)
[^10.1016/j.antiviral.2014.04.006]: Blaising J, Polyak SJ, Pécheur E. "Arbidol as a broad-spectrum antiviral: an update" _Antiviral Res._, 2014. [10.1016/j.antiviral.2014.04.006](https://doi.org/10.1016/j.antiviral.2014.04.006)
[^10.1128/JVI.02077-15]: Pécheur E, Borisevich V, et int., Polyak SJ. "The Synthetic Antiviral Drug Arbidol Inhibits Globally Prevalent Pathogenic Viruses" _J. Virol._, 2016. [10.1128/JVI.02077-15](https://doi.org/10.1128/JVI.02077-15)
[^10.1016/j.antiviral.2008.10.009]: Leneva IA, Russell RJ, et int., Hay AJ. "Characteristics of arbidol-resistant mutants of influenza virus: implications for the mechanism of anti-influenza action of arbidol" _Antiviral Res._, 2009. [10.1016/j.antiviral.2008.10.009](https://doi.org/10.1016/j.antiviral.2008.10.009)
[^10.1038/s41421-020-0169-8]: Wang X, Cao R, et int., Wang M. "The anti-influenza virus drug, arbidol is an efficient inhibitor of SARS-CoV-2 in vitro" _Cell Discovery_, 2020. [10.1038/s41421-020-0169-8](https://doi.org/10.1038/s41421-020-0169-8)
[^10.1073/pnas.1617020114]: Kadam RU, Wilson IA. "Structural basis of influenza virus fusion inhibition by the antiviral drug Arbidol" _Proc. Natl. Acad. Sci. U.S.A._, 2017. [10.1073/pnas.1617020114](https://doi.org/10.1073/pnas.1617020114)
[^10.1101/2020.03.19.20038984]: Li Y, Xie Z, et int., Li L. "An exploratory randomized controlled study on the efficacy and safety of lopinavir/ritonavir or arbidol treating adult patients hospitalized with mild/moderate COVID-19 (ELACOI)" _medRxiv_, 2020. [10.1101/2020.03.19.20038984](https://doi.org/10.1101/2020.03.19.20038984)
[^10.1016/j.jinf.2020.03.002]: Deng L, Li C, et int., Xia J. "Arbidol combined with LPV/r versus LPV/r alone against Corona Virus Disease 2019:a retrospective cohort study" _J. Infect._, 2020. [10.1016/j.jinf.2020.03.002](https://doi.org/10.1016/j.jinf.2020.03.002)
[^10.1016/j.jinf.2020.03.060]: Zhu Z, Lu Z, et int., Xue Y. "Arbidol Monotherapy is Superior to Lopinavir/ritonavir in Treating COVID-19" _J. Infect._, 2020. [10.1016/j.jinf.2020.03.060](https://doi.org/10.1016/j.jinf.2020.03.060)
[^PMID32574262]: Zhou Q, Chen V, et int., Fish EN. "Interferon-α2b Treatment for COVID-19" _Front. Immunol._, 2020. [PMID32574262](https://pubmed.ncbi.nlm.nih.gov/32574262)
[^10.1038/srep22008]: Zhao H, Zhou J, et int., Zheng B. "A novel peptide with potent and broad-spectrum antiviral activities against multiple respiratory viruses" _Sci Rep_, 2016. [10.1038/srep22008](https://doi.org/10.1038/srep22008)
[^10.1016/j.chembiol.2004.07.013]: Kao RY, Tsui WHW, et int., Yuen K. "Identification of novel small-molecule inhibitors of severe acute respiratory syndrome-associated coronavirus by chemical genetics" _Chem. Biol._, 2004. [10.1016/j.chembiol.2004.07.013](https://doi.org/10.1016/j.chembiol.2004.07.013)
[^10.1101/2020.03.29.014183]: Milewska A, Chi Y, et int., Pyrc K. "HTCC as a highly effective polymeric inhibitor of SARS-CoV-2 and MERS-CoV" _bioRxiv_, 2020. [10.1101/2020.03.29.014183](https://doi.org/10.1101/2020.03.29.014183)
[^10.1371/journal.pone.0156552]: Milewska A, Kaminski K, et int., Pyrc K. "HTCC: Broad Range Inhibitor of Coronavirus Entry" _PLoS ONE_, 2016. [10.1371/journal.pone.0156552](https://doi.org/10.1371/journal.pone.0156552)
[^10.1016/j.antiviral.2011.12.012]: Struck A, Axmann M, et int., Meyer B. "A hexapeptide of the receptor-binding domain of SARS corona virus spike protein blocks viral entry into host cells via the human receptor ACE2" _Antiviral Res._, 2012. [10.1016/j.antiviral.2011.12.012](https://doi.org/10.1016/j.antiviral.2011.12.012)
[^10.1016/j.antiviral.2006.04.014]: Ho T, Wu S, et int., Hsiang C. "Emodin blocks the SARS coronavirus spike protein and angiotensin-converting enzyme 2 interaction" _Antiviral Res._, 2007. [10.1016/j.antiviral.2006.04.014](https://doi.org/10.1016/j.antiviral.2006.04.014)
[^10.1016/j.antiviral.2011.02.008]: Schwarz S, Wang K, et int., Schwarz W. "Emodin inhibits current through SARS-associated coronavirus 3a protein" _Antiviral Res._, 2011. [10.1016/j.antiviral.2011.02.008](https://doi.org/10.1016/j.antiviral.2011.02.008)
[^10.1093/infdis/jiu396]: Mair-Jenkins J, Saavedra-Campos M, et int., Convalescent Plasma Study Group. "The effectiveness of convalescent plasma and hyperimmune immunoglobulin for the treatment of severe acute respiratory infections of viral etiology: a systematic review and exploratory meta-analysis" _J. Infect. Dis._, 2015. [10.1093/infdis/jiu396](https://doi.org/10.1093/infdis/jiu396)
[^10.1016/j.tmrv.2020.04.001]: Sullivan HC, Roback JD. "Convalescent Plasma: Therapeutic Hope or Hopeless Strategy in the SARS-CoV-2 Pandemic" _Transfus Med Rev_, 2020. [10.1016/j.tmrv.2020.04.001](https://doi.org/10.1016/j.tmrv.2020.04.001)
[^10.1172/JCI138745]: Bloch EM, Shoham S, et int., Tobian AA. "Deployment of convalescent plasma for the prevention and treatment of COVID-19" _J. Clin. Invest._, 2020. [10.1172/JCI138745](https://doi.org/10.1172/JCI138745)
[^10.1172/JCI138003]: Casadevall A, Pirofski L. "The convalescent sera option for containing COVID-19" _J Clin Invest_, 2020. [10.1172/JCI138003](https://doi.org/10.1172/JCI138003)
[^10.1172/JCI140200]: Joyner MJ, Wright RS, et int., Casadevall A. "Early safety indicators of COVID-19 convalescent plasma in 5,000 patients" _J. Clin. Invest._, 2020. [10.1172/JCI140200](https://doi.org/10.1172/JCI140200)
[^10.1001/jama.2020.10044]: Li L, Zhang W, et int., Liu Z. "Effect of Convalescent Plasma Therapy on Time to Clinical Improvement in Patients With Severe and Life-threatening COVID-19: A Randomized Clinical Trial" _JAMA_, 2020. [10.1001/jama.2020.10044](https://doi.org/10.1001/jama.2020.10044)
[^10.1101/2020.05.20.20102236]: Liu STH, Lin H, et int., Bouvier NM. "Convalescent plasma treatment of severe COVID-19: A matched control study" _medRxiv_, 2020. [10.1101/2020.05.20.20102236](https://doi.org/10.1101/2020.05.20.20102236)
[^10.1101/2020.07.01.20139857]: Gharbharan A, Jordans CCE, et int., Rijnders B. "Convalescent Plasma for COVID-19. A randomized clinical trial" _medRxiv_, 2020. [10.1101/2020.07.01.20139857](https://doi.org/10.1101/2020.07.01.20139857)
[^10.1016/j.antiviral.2020.104791]: Sallard E, Lescure F, et int., Peiffer-Smadja N. "Type 1 interferons as a potential treatment against COVID-19" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104791](https://doi.org/10.1016/j.antiviral.2020.104791)
[^10.1038/nri3787]: McNab F, Mayer-Barber K, et int., O'Garra A. "Type I interferons in infectious disease" _Nat. Rev. Immunol._, 2015. [10.1038/nri3787](https://doi.org/10.1038/nri3787)
[^10.1016/j.smim.2019.101300]: Wang BX, Fish EN. "Global virus outbreaks: Interferons as 1st responders" _Semin. Immunol._, 2019. [10.1016/j.smim.2019.101300](https://doi.org/10.1016/j.smim.2019.101300)
[^10.1038/nrmicro.2016.81]: de Wit E, van Doremalen N, et int., Munster VJ. "SARS and MERS: recent insights into emerging coronaviruses" _Nat. Rev. Microbiol._, 2016. [10.1038/nrmicro.2016.81](https://doi.org/10.1038/nrmicro.2016.81)
[^10.1371/journal.ppat.1008737]: Ribero MS, Jouvenet N, et int., Nisole S. "Interplay between SARS-CoV-2 and the type I interferon response" _PLOS Pathogens_, 2020. [10.1371/journal.ppat.1008737](https://doi.org/10.1371/journal.ppat.1008737)
[^10.1016/j.coviro.2012.04.004]: Totura AL, Baric RS. "SARS coronavirus pathogenesis: host innate immune responses and viral antagonism of interferon" _Curr Opin Virol_, 2012. [10.1016/j.coviro.2012.04.004](https://doi.org/10.1016/j.coviro.2012.04.004)
[^10.1128/JVI.00985-20]: Vanderheiden A, Ralfs P, et int., Suthar MS. "Type I and Type III IFN Restrict SARS-CoV-2 Infection of Human Airway Epithelial Cultures" _Journal of Virology_, 2020. [10.1128/JVI.00985-20](https://doi.org/10.1128/JVI.00985-20)
[^10.1126/science.abc6027]: Hadjadj J, Yatim N, et int., Terrier B. "Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients" _Science_, 2020. [10.1126/science.abc6027](https://doi.org/10.1126/science.abc6027)
[^10.1016/j.cell.2020.04.026]: Blanco-Melo D, Nilsson-Payant BE, et int., tenOever BR. "Imbalanced Host Response to SARS-CoV-2 Drives Development of COVID-19" _Cell_, 2020. [10.1016/j.cell.2020.04.026](https://doi.org/10.1016/j.cell.2020.04.026)
[^10.1101/2020.03.07.982264]: Lokugamage KG, Schindewolf C, Menachery VD. "SARS-CoV-2 sensitive to type I interferon pretreatment" _bioRxiv_, 2020. [10.1101/2020.03.07.982264](https://doi.org/10.1101/2020.03.07.982264)
[^10.1074/jbc.AC120.013788]: Felgenhauer U, Schoen A, et int., Weber F. "Inhibition of SARS-CoV-2 by type I and type III interferons" _J. Biol. Chem._, 2020. [10.1074/jbc.AC120.013788](https://doi.org/10.1074/jbc.AC120.013788)
[^10.1016/S0140-6736(20)31042-4]: Hung IF, Lung K, et int., Yuen K. "Triple combination of interferon beta-1b, lopinavir–ritonavir, and ribavirin in the treatment of patients admitted to hospital with COVID-19: an open-label, randomised, phase 2 trial" _The Lancet_, 2020. [10.1016/S0140-6736(20)31042-4](https://doi.org/10.1016/S0140-6736(20)31042-4)
[^10.1128/AAC.01061-20]: Davoudi-Monfared E, Rahmani H, et int., Yekaninejad MS. "Efficacy and safety of interferon β-1a in treatment of severe COVID-19: A randomized clinical trial" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.01061-20](https://doi.org/10.1128/AAC.01061-20)
[^10.1101/2020.04.24.20077735]: Zheng F, Zhou Y, et int., Gong G. "A Novel Protein Drug, Novaferon, as the Potential Antiviral Drug for COVID-19" _medRxiv_, 2020. [10.1101/2020.04.24.20077735](https://doi.org/10.1101/2020.04.24.20077735)
[^10.3389/fimmu.2020.01061]: Zhou Q, Chen V, et int., Fish EN. "Interferon-α2b Treatment for COVID-19" _Front. Immunol._, 2020. [10.3389/fimmu.2020.01061](https://doi.org/10.3389/fimmu.2020.01061)
[^10.1016/j.chom.2020.07.005]: Wang N, Zhan Y, et int., Hong P. "Retrospective Multicenter Cohort Study Shows Early Interferon Therapy Is Associated with Favorable Clinical Responses in COVID-19 Patients" _Cell Host & Microbe_, 2020. [10.1016/j.chom.2020.07.005](https://doi.org/10.1016/j.chom.2020.07.005)
[^10.1172/JCI126363]: Channappanavar R, Fehr AR, et int., Perlman S. "IFN-I response timing relative to virus replication determines MERS coronavirus infection outcomes" _J. Clin. Invest._, 2019. [10.1172/JCI126363](https://doi.org/10.1172/JCI126363)
[^10.1016/j.cell.2020.04.035]: Ziegler CGK, Allon SJ, et int., HCA Lung Biological Network. "SARS-CoV-2 Receptor ACE2 Is an Interferon-Stimulated Gene in Human Airway Epithelial Cells and Is Detected in Specific Cell Subsets across Tissues" _Cell_, 2020. [10.1016/j.cell.2020.04.035](https://doi.org/10.1016/j.cell.2020.04.035)
[^10.15252/emmm.201606413]: Davidson S, McCabe TM, et int., Wack A. "IFNλ is a potent anti-influenza therapeutic without the inflammatory side effects of IFNα treatment" _EMBO Mol Med_, 2016. [10.15252/emmm.201606413](https://doi.org/10.15252/emmm.201606413)
[^10.1093/cid/ciaa453]: O'Brien TR, Thomas DL, et int., Hartmann R. "Weak Induction of Interferon Expression by SARS-CoV-2 Supports Clinical Trials of Interferon Lambda to Treat Early COVID-19" _Clin. Infect. Dis._, 2020. [10.1093/cid/ciaa453](https://doi.org/10.1093/cid/ciaa453)
[^10.1016/j.celrep.2020.107863]: Stanifer ML, Kee C, et int., Boulant S. "Critical Role of Type III Interferon in Controlling SARS-CoV-2 Infection in Human Intestinal Epithelial Cells" _Cell Rep_, 2020. [10.1016/j.celrep.2020.107863](https://doi.org/10.1016/j.celrep.2020.107863)
[^10.1016/j.chom.2020.05.008]: Park A, Iwasaki A. "Type I and Type III Interferons – Induction, Signaling, Evasion, and Application to Combat COVID-19" _Cell Host & Microbe_, 2020. [10.1016/j.chom.2020.05.008](https://doi.org/10.1016/j.chom.2020.05.008)
[^10.1016/j.cell.2020.02.052]: Hoffmann M, Kleine-Weber H, et int., Pöhlmann S. "SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor" _Cell_, 2020. [10.1016/j.cell.2020.02.052](https://doi.org/10.1016/j.cell.2020.02.052)
[^10.1101/2020.04.15.042085]: Bestle D, Heindl MR, et int., Bottcher-Friebertshauser E. "TMPRSS2 and furin are both essential for proteolytic activation and spread of SARS-CoV-2 in human airway epithelial cells and provide promising drug targets" _bioRxiv_, 2020. [10.1101/2020.04.15.042085](https://doi.org/10.1101/2020.04.15.042085)
[^10.1073/pnas.0505577102]: Simmons G, Gosalia DN, et int., Bates P. "Inhibitors of cathepsin L prevent severe acute respiratory syndrome coronavirus entry" _Proc. Natl. Acad. Sci. U.S.A._, 2005. [10.1073/pnas.0505577102](https://doi.org/10.1073/pnas.0505577102)
[^10.1128/JVI.02232-10]: Glowacka I, Bertram S, et int., Pöhlmann S. "Evidence that TMPRSS2 activates the severe acute respiratory syndrome coronavirus spike protein for membrane fusion and reduces viral control by the humoral immune response" _J. Virol._, 2011. [10.1128/JVI.02232-10](https://doi.org/10.1128/JVI.02232-10)
[^10.1128/JVI.02062-10]: Shulla A, Heald-Sargent T, et int., Gallagher T. "A transmembrane serine protease is linked to the severe acute respiratory syndrome coronavirus receptor and activates virus entry" _J. Virol._, 2011. [10.1128/JVI.02062-10](https://doi.org/10.1128/JVI.02062-10)
[^10.1128/JVI.00094-12]: Kawase M, Shirato K, et int., Matsuyama S. "Simultaneous treatment of human bronchial epithelial cells with serine and cysteine protease inhibitors prevents severe acute respiratory syndrome coronavirus entry" _J. Virol._, 2012. [10.1128/JVI.00094-12](https://doi.org/10.1128/JVI.00094-12)
[^10.1016/j.virol.2017.11.012]: Shirato K, Kawase M, Matsuyama S. "Wild-type human coronaviruses prefer cell-surface TMPRSS2 to endosomal cathepsins for cell entry" _Virology_, 2018. [10.1016/j.virol.2017.11.012](https://doi.org/10.1016/j.virol.2017.11.012)
[^10.3390/v12060629]: Yamamoto M, Kiso M, et int., Inoue J. "The Anticoagulant Nafamostat Potently Inhibits SARS-CoV-2 S Protein-Mediated Fusion in a Cell Fusion Assay System and Viral Infection In Vitro in a Cell-Type-Dependent Manner" _Viruses_, 2020. [10.3390/v12060629](https://doi.org/10.3390/v12060629)
[^10.1038/s41467-020-15562-9]: Ou X, Liu Y, et int., Qian Z. "Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV" _Nature Communications_, 2020. [10.1038/s41467-020-15562-9](https://doi.org/10.1038/s41467-020-15562-9)
[^10.1128/AAC.00754-20]: Hoffmann M, Schroeder S, et int., Pöhlmann S. "Nafamostat mesylate blocks activation of SARS-CoV-2: New treatment option for COVID-19" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.00754-20](https://doi.org/10.1128/AAC.00754-20)
[^10.1016/j.antiviral.2015.01.011]: Zhou Y, Vedantham P, et int., Simmons G. "Protease inhibitors targeting coronavirus and filovirus entry" _Antiviral Res._, 2015. [10.1016/j.antiviral.2015.01.011](https://doi.org/10.1016/j.antiviral.2015.01.011)
[^10.1128/JVI.01001-12]: Schneider M, Ackermann K, et int., Gilch S. "Severe acute respiratory syndrome coronavirus replication is severely impaired by MG132 due to proteasome-independent inhibition of M-calpain" _J. Virol._, 2012. [10.1128/JVI.01001-12](https://doi.org/10.1128/JVI.01001-12)
[^10.1073/pnas.1407087111]: Millet JK, Whittaker GR. "Host cell entry of Middle East respiratory syndrome coronavirus after two-step, furin-mediated activation of the spike protein" _Proc. Natl. Acad. Sci. U.S.A._, 2014. [10.1073/pnas.1407087111](https://doi.org/10.1073/pnas.1407087111)
[^10.1101/2020.04.21.053058]: Kang Y, Chou Y, et int., Kirchhausen T. "Inhibition of PIKfyve kinase prevents infection by EBOV and SARS-CoV-2" _bioRxiv_, 2020. [10.1101/2020.04.21.053058](https://doi.org/10.1101/2020.04.21.053058)
[^10.1038/s41586-020-2577-1]: Riva L, Yuan S, et int., Chanda SK. "Discovery of SARS-CoV-2 antiviral drugs through large-scale compound repurposing" _Nature_, 2020. [10.1038/s41586-020-2577-1](https://doi.org/10.1038/s41586-020-2577-1)
[^10.1016/j.cell.2020.06.034]: Bouhaddou M, Memon D, et int., Krogan NJ. "The Global Phosphorylation Landscape of SARS-CoV-2 Infection" _Cell_, 2020. [10.1016/j.cell.2020.06.034](https://doi.org/10.1016/j.cell.2020.06.034)
[^10.1016/s1473-3099(03)00806-5]: Savarino A, Boelaert JR, et int., Cauda R. "Effects of chloroquine on viral infections: an old drug against today's diseases?" _Lancet Infect Dis_, 2003. [10.1016/s1473-3099(03)00806-5](https://doi.org/10.1016/s1473-3099(03)00806-5)
[^10.1371/journal.ppat.1004502]: Burkard C, Verheije MH, et int., de Haan CAM. "Coronavirus cell entry occurs through the endo-/lysosomal pathway in a proteolysis-dependent manner" _PLoS Pathog._, 2014. [10.1371/journal.ppat.1004502](https://doi.org/10.1371/journal.ppat.1004502)
[^10.1186/1743-422X-2-69]: Vincent MJ, Bergeron E, et int., Nichol ST. "Chloroquine is a potent inhibitor of SARS coronavirus infection and spread" _Virol. J._, 2005. [10.1186/1743-422X-2-69](https://doi.org/10.1186/1743-422X-2-69)
[^10.1128/AAC.03036-14]: Dyall J, Coleman CM, et int., Frieman MB. "Repurposing of clinically developed drugs for treatment of Middle East respiratory syndrome coronavirus infection" _Antimicrob. Agents Chemother._, 2014. [10.1128/AAC.03036-14](https://doi.org/10.1128/AAC.03036-14)
[^10.3390/v10060283]: Shin JS, Jung E, et int., Go YY. "Saracatinib Inhibits Middle East Respiratory Syndrome-Coronavirus Replication In Vitro" _Viruses_, 2018. [10.3390/v10060283](https://doi.org/10.3390/v10060283)
[^10.1016/j.bbrc.2004.08.085]: Keyaerts E, Vijgen L, et int., Van Ranst M. "In vitro inhibition of severe acute respiratory syndrome coronavirus by chloroquine" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.08.085](https://doi.org/10.1016/j.bbrc.2004.08.085)
[^10.1038/s41421-020-0156-0]: Liu J, Cao R, et int., Wang M. "Hydroxychloroquine, a less toxic derivative of chloroquine, is effective in inhibiting SARS-CoV-2 infection in vitro" _Cell Discov_, 2020. [10.1038/s41421-020-0156-0](https://doi.org/10.1038/s41421-020-0156-0)
[^10.1093/cid/ciaa237]: Yao X, Ye F, et int., Liu D. "In Vitro Antiviral Activity and Projection of Optimized Dosing Design of Hydroxychloroquine for the Treatment of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)" _Clin. Infect. Dis._, 2020. [10.1093/cid/ciaa237](https://doi.org/10.1093/cid/ciaa237)
[^10.1101/2020.03.29.014407]: Clementi N, Criscuolo E, et int., Mancini N. "Combined prophylactic and therapeutic use maximizes hydroxychloroquine anti-SARS-CoV-2 effects in vitro" _bioRxiv_, 2020. [10.1101/2020.03.29.014407](https://doi.org/10.1101/2020.03.29.014407)
[^10.1101/2020.07.15.20151852]: Horby P, Mafham M, et int., Landray MJ. "Effect of Hydroxychloroquine in Hospitalized Patients with COVID-19: Preliminary results from a multi-centre, randomized, controlled trial." _medRxiv_, 2020. [10.1101/2020.07.15.20151852](https://doi.org/10.1101/2020.07.15.20151852)
[^10.7326/M20-4207]: Skipper CP, Pastick KA, et int., Boulware DR. "Hydroxychloroquine in Nonhospitalized Adults With Early COVID-19: A Randomized Trial" _Ann. Intern. Med._, 2020. [10.7326/M20-4207](https://doi.org/10.7326/M20-4207)
[^10.1056/NEJMoa2019014]: Cavalcanti AB, Zampieri FG, et int., Berwanger O. "Hydroxychloroquine with or without Azithromycin in Mild-to-Moderate Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2019014](https://doi.org/10.1056/NEJMoa2019014)
[^10.1101/2020.04.10.20060558]: Tang W, Cao Z, et int., Xie Q. "Hydroxychloroquine in patients with COVID-19: an open-label, randomized, controlled trial" _medRxiv_, 2020. [10.1101/2020.04.10.20060558](https://doi.org/10.1101/2020.04.10.20060558)
[^10.1093/cid/ciaa1009]: Mitjà O, Corbacho-Monné M, et int., Vall-Mayans M. "Hydroxychloroquine for Early Treatment of Adults with Mild Covid-19: A Randomized-Controlled Trial" _Clin Infect Dis_, 2020. [10.1093/cid/ciaa1009](https://doi.org/10.1093/cid/ciaa1009)
[^10.1101/2020.07.20.20157651]: Mitja O, Ubals M, et int., Clotet B. "A Cluster-Randomized Trial of Hydroxychloroquine as Prevention of Covid-19 Transmission and Disease" _medRxiv_, 2020. [10.1101/2020.07.20.20157651](https://doi.org/10.1101/2020.07.20.20157651)
[^10.1056/NEJMoa2016638]: Boulware DR, Pullen MF, et int., Hullsiek KH. "A Randomized Trial of Hydroxychloroquine as Postexposure Prophylaxis for Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2016638](https://doi.org/10.1056/NEJMoa2016638)
[^10.1111/j.1365-2125.1989.tb03439.x]: Tett SE, Cutler DJ, et int., Brown KF. "Bioavailability of hydroxychloroquine tablets in healthy volunteers" _Br J Clin Pharmacol_, 1989. [10.1111/j.1365-2125.1989.tb03439.x](https://doi.org/10.1111/j.1365-2125.1989.tb03439.x)
[^10.2165/00003088-199631040-00003]: Ducharme J, Farinotti R. "Clinical pharmacokinetics and metabolism of chloroquine. Focus on recent advancements" _Clin Pharmacokinet_, 1996. [10.2165/00003088-199631040-00003](https://doi.org/10.2165/00003088-199631040-00003)
[^10.1128/AAC.00339-08]: Lim H, Im J, et int., Park J. "Pharmacokinetics of hydroxychloroquine and its clinical implications in chemoprophylaxis against malaria caused by Plasmodium vivax" _Antimicrob. Agents Chemother._, 2009. [10.1128/AAC.00339-08](https://doi.org/10.1128/AAC.00339-08)
[^10.1002/cpt.1856]: Garcia-Cremades M, Solans BP, et int., Savic RM. "Optimizing hydroxychloroquine dosing for patients with COVID-19: An integrative modeling approach for effective drug repurposing" _Clin. Pharmacol. Ther._, 2020. [10.1002/cpt.1856](https://doi.org/10.1002/cpt.1856)
[^10.1101/2020.07.22.216150]: Ou T, Mou H, et int., Farzan M. "Hydroxychloroquine-mediated inhibition of SARS-CoV-2 entry is attenuated by TMPRSS2" _bioRxiv_, 2020. [10.1101/2020.07.22.216150](https://doi.org/10.1101/2020.07.22.216150)
[^10.1021/acsinfecdis.0c00052]: Xu J, Shi P, et int., Zhou J. "Broad Spectrum Antiviral Agent Niclosamide and Its Therapeutic Potential" _ACS Infect Dis_, 2020. [10.1021/acsinfecdis.0c00052](https://doi.org/10.1021/acsinfecdis.0c00052)
[^10.1016/j.cellsig.2017.04.001]: Chen W, Mook RA, et int., Wang J. "Niclosamide: Beyond an antihelminthic drug" _Cell. Signal._, 2018. [10.1016/j.cellsig.2017.04.001](https://doi.org/10.1016/j.cellsig.2017.04.001)
[^10.1021/jm070295s]: Wen C, Kuo Y, et int., Yang N. "Specific plant terpenoids and lignoids possess potent antiviral activities against severe acute respiratory syndrome coronavirus" _J. Med. Chem._, 2007. [10.1021/jm070295s](https://doi.org/10.1021/jm070295s)
[^10.1101/2020.02.25.965582]: Ko M, Chang SY, et int., Windisch MP. "Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19" _bioRxiv_, 2020. [10.1101/2020.02.25.965582](https://doi.org/10.1101/2020.02.25.965582)
[^10.1101/2020.04.15.997254]: Gassen NC, Papies J, et int., Muller MA. "Analysis of SARS-CoV-2-controlled autophagy reveals spermidine, MK-2206, and niclosamide as putative antiviral therapeutics" _bioRxiv_, 2020. [10.1101/2020.04.15.997254](https://doi.org/10.1101/2020.04.15.997254)
[^10.1038/nrd.2015.37]: Zumla A, Chan JFW, et int., Yuen K. "Coronaviruses - drug discovery and therapeutic options" _Nat Rev Drug Discov_, 2016. [10.1038/nrd.2015.37](https://doi.org/10.1038/nrd.2015.37)
[^10.7150/ijbs.45498]: Yang N, Shen H. "Targeting the Endocytic Pathway and Autophagy Process as a Novel Therapeutic Strategy in COVID-19" _Int. J. Biol. Sci._, 2020. [10.7150/ijbs.45498](https://doi.org/10.7150/ijbs.45498)
[^10.1371/journal.pone.0194868]: Cong Y, Hart BJ, et int., Holbrook MR. "MERS-CoV pathogenesis and antiviral efficacy of licensed drugs in human monocyte-derived antigen-presenting cells" _PLoS ONE_, 2018. [10.1371/journal.pone.0194868](https://doi.org/10.1371/journal.pone.0194868)
[^10.1101/2020.03.25.008482]: Weston S, Haupt R, et int., Frieman MB. "FDA approved drugs with broad anti-coronaviral activity inhibit SARS-CoV-2 in vitro" _bioRxiv_, 2020. [10.1101/2020.03.25.008482](https://doi.org/10.1101/2020.03.25.008482)
[^10.1371/journal.ppat.1004166]: Lundin A, Dijkman R, et int., Trybala E. "Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the middle East respiratory syndrome virus" _PLoS Pathog._, 2014. [10.1371/journal.ppat.1004166](https://doi.org/10.1371/journal.ppat.1004166)
[^10.1016/j.virusres.2018.01.002]: Rappe JCF, de Wilde A, et int., Thiel V. "Antiviral activity of K22 against members of the order Nidovirales" _Virus Res._, 2018. [10.1016/j.virusres.2018.01.002](https://doi.org/10.1016/j.virusres.2018.01.002)
[^10.1128/mBio.00165-13]: Josset L, Menachery VD, et int., Katze MG. "Cell host response to infection with novel human coronavirus EMC predicts potential antivirals and important differences with SARS coronavirus" _mBio_, 2013. [10.1128/mBio.00165-13](https://doi.org/10.1128/mBio.00165-13)
[^10.1128/mBio.00611-12]: Kindler E, Jónsdóttir HR, et int., Thiel V. "Efficient replication of the novel human betacoronavirus EMC on primary human epithelium highlights its zoonotic potential" _mBio_, 2013. [10.1128/mBio.00611-12](https://doi.org/10.1128/mBio.00611-12)
[^10.1128/AAC.03659-14]: Kindrachuk J, Ork B, et int., Jahrling PB. "Antiviral potential of ERK/MAPK and PI3K/AKT/mTOR signaling modulation for Middle East respiratory syndrome coronavirus infection as identified by temporal kinome analysis" _Antimicrob. Agents Chemother._, 2015. [10.1128/AAC.03659-14](https://doi.org/10.1128/AAC.03659-14)
[^10.1038/s41586-020-2286-9]: Gordon DE, Jang GM, et int., Krogan NJ. "A SARS-CoV-2 protein interaction map reveals targets for drug repurposing" _Nature_, 2020. [10.1038/s41586-020-2286-9](https://doi.org/10.1038/s41586-020-2286-9)
[^10.1128/JVI.01429-16]: Coleman CM, Sisk JM, et int., Frieman MB. "Abelson Kinase Inhibitors Are Potent Inhibitors of Severe Acute Respiratory Syndrome Coronavirus and Middle East Respiratory Syndrome Coronavirus Fusion" _J. Virol._, 2016. [10.1128/JVI.01429-16](https://doi.org/10.1128/JVI.01429-16)
[^10.1099/jgv.0.001047]: Sisk JM, Frieman MB, Machamer CE. "Coronavirus S protein-induced fusion is blocked prior to hemifusion by Abl kinase inhibitors" _J. Gen. Virol._, 2018. [10.1099/jgv.0.001047](https://doi.org/10.1099/jgv.0.001047)
[^10.1016/j.jiph.2016.04.001]: Rossignol J. "Nitazoxanide, a new drug candidate for the treatment of Middle East respiratory syndrome coronavirus" _J Infect Public Health_, 2016. [10.1016/j.jiph.2016.04.001](https://doi.org/10.1016/j.jiph.2016.04.001)
[^10.1101/2020.03.11.987016]: Matsuyama S, Kawase M, et int., Fukushi S. "The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15" _bioRxiv_, 2020. [10.1101/2020.03.11.987016](https://doi.org/10.1101/2020.03.11.987016)
[^10.1016/j.jiac.2020.04.007]: Iwabuchi K, Yoshie K, et int., Morishima T. "Therapeutic potential of ciclesonide inahalation for COVID-19 pneumonia: Report of three cases" _J Infect Chemother_, 2020. [10.1016/j.jiac.2020.04.007](https://doi.org/10.1016/j.jiac.2020.04.007)
[^10.3390/ph13030051]: Bleasel MD, Peterson GM. "Emetine, Ipecac, Ipecac Alkaloids and Analogues as Potential Antiviral Agents for Coronaviruses" _Pharmaceuticals (Basel)_, 2020. [10.3390/ph13030051](https://doi.org/10.3390/ph13030051)
[^10.4049/jimmunol.178.8.5200]: Gowen BB, Wong M, et int., Sidwell RW. "TLR3 is essential for the induction of protective immunity against Punta Toro Virus infection by the double-stranded RNA (dsRNA), poly(I:C12U), but not Poly(I:C): differential recognition of synthetic dsRNA molecules" _J. Immunol._, 2007. [10.4049/jimmunol.178.8.5200](https://doi.org/10.4049/jimmunol.178.8.5200)
[^10.1016/j.virol.2009.09.023]: Day CW, Baric R, et int., Barnard DL. "A new mouse-adapted strain of SARS-CoV as a lethal model for evaluating antiviral agents in vitro and in vivo" _Virology_, 2009. [10.1016/j.virol.2009.09.023](https://doi.org/10.1016/j.virol.2009.09.023)
[^10.1016/j.antiviral.2016.12.007]: Kumaki Y, Salazar AM, et int., Barnard DL. "Prophylactic and therapeutic intranasal administration with an immunomodulator, Hiltonol® (Poly IC:LC), in a lethal SARS-CoV-infected BALB/c mouse model" _Antiviral Res._, 2017. [10.1016/j.antiviral.2016.12.007](https://doi.org/10.1016/j.antiviral.2016.12.007)
[^10.1128/AAC.00440-20]: Ekins S, Madrid PB. "Tilorone: A Broad-Spectrum Antiviral For Emerging Viruses" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.00440-20](https://doi.org/10.1128/AAC.00440-20)
[^10.1016/j.virol.2018.06.011]: de Wilde AH, Pham U, et int., Snijder EJ. "Cyclophilins and cyclophilin inhibitors in nidovirus replication" _Virology_, 2018. [10.1016/j.virol.2018.06.011](https://doi.org/10.1016/j.virol.2018.06.011)
[^10.1099/vir.0.034983-0]: de Wilde AH, Zevenhoven-Dobbe JC, et int., van Hemert MJ. "Cyclosporin A inhibits the replication of diverse coronaviruses" _J. Gen. Virol._, 2011. [10.1099/vir.0.034983-0](https://doi.org/10.1099/vir.0.034983-0)
[^10.1099/vir.0.052910-0]: de Wilde AH, Raj VS, et int., van den Hoogen BG. "MERS-coronavirus replication induces severe in vitro cytopathology and is strongly inhibited by cyclosporin A or interferon-α treatment" _J. Gen. Virol._, 2013. [10.1099/vir.0.052910-0](https://doi.org/10.1099/vir.0.052910-0)
[^10.1016/j.virusres.2016.11.011]: de Wilde AH, Falzarano D, et int., Snijder EJ. "Alisporivir inhibits MERS- and SARS-coronavirus replication in cell culture, but not SARS-coronavirus infection in a mouse model" _Virus Res._, 2017. [10.1016/j.virusres.2016.11.011](https://doi.org/10.1016/j.virusres.2016.11.011)
[^10.1016/j.virusres.2014.02.010]: Carbajo-Lozoya J, Ma-Lauer Y, et int., von Brunn A. "Human coronavirus NL63 replication is cyclophilin A-dependent and inhibited by non-immunosuppressive cyclosporine A-derivatives including Alisporivir" _Virus Res._, 2014. [10.1016/j.virusres.2014.02.010](https://doi.org/10.1016/j.virusres.2014.02.010)
[^10.1016/j.antiviral.2018.05.007]: Li HS, Kuok DIT, et int., Nicholls JM. "Effect of interferon alpha and cyclosporine treatment separately and in combination on Middle East Respiratory Syndrome Coronavirus (MERS-CoV) replication in a human in-vitro and ex-vivo culture model" _Antiviral Res._, 2018. [10.1016/j.antiviral.2018.05.007](https://doi.org/10.1016/j.antiviral.2018.05.007)
[^10.1016/j.antiviral.2020.104787]: Caly L, Druce JD, et int., Wagstaff KM. "The FDA-approved Drug Ivermectin inhibits the replication of SARS-CoV-2 in vitro" _Antiviral Research_, 2020. [10.1016/j.antiviral.2020.104787](https://doi.org/10.1016/j.antiviral.2020.104787)
[^10.4269/ajtmh.20-0271]: Chaccour C, Hammann F, et int., Rabinovich NR. "Ivermectin and Novel Coronavirus Disease (COVID-19): Keeping Rigor in Times of Urgency" _Am. J. Trop. Med. Hyg._, 2020. [10.4269/ajtmh.20-0271](https://doi.org/10.4269/ajtmh.20-0271)
[^10.1101/2020.04.21.20073262]: Schmith VD, Zhou J, Lohmer LR. "The Approved Dose of Ivermectin Alone is not the Ideal Dose for the Treatment of COVID-19" _medRxiv_, 2020. [10.1101/2020.04.21.20073262](https://doi.org/10.1101/2020.04.21.20073262)
[^10.1101/2020.06.06.20124461]: Rajter JC, Sherman M, et int., Rajter J. "ICON (Ivermectin in COvid Nineteen) study: Use of Ivermectin is Associated with Lower Mortality in Hospitalized Patients with COVID19" _medRxiv_, 2020. [10.1101/2020.06.06.20124461](https://doi.org/10.1101/2020.06.06.20124461)


<!--stackedit_data:
eyJoaXN0b3J5IjpbLTg1MTcwNDU2OSwtNDE0Nzc2MTE4LDE0MD
cwMDY4Nyw2ODQ3ODE0ODcsNzkxMDA3NDYzLDEzNzE3MTQyOTYs
LTE0NTIzNTgxNDQsMjA5ODc3NTcwNSwxNDE5OTYxNzg5LC0xNz
U4Njc1MTU5LC0xNjQ3MTk4NTA0LC0xOTI2MTIwMjczLC0xMTky
MDE0ODY3LC0xNjQxNzk5NzUwLDEzNzMxNzk5MDEsODUzMTEwMT
E4LDEwMDkwMzE4NjYsLTE2MjU4NjQ4LC0xODI0NzE5NzkyLDE0
OTcwNTAyMF19
-->