
## Mission Statement
The Coronavirus Antiviral Research Database is designed to expedite the development of SARS-CoV-2 antiviral therapy. It will benefit global coronavirus drug development efforts by (1) promoting uniform reporting of experimental results to facilitate comparisons between different candidate antiviral compounds; (2) identifying gaps in coronavirus antiviral drug development research; (3) helping scientists, clinical investigators, public health officials, and funding agencies prioritize the most promising compounds and repurposed drugs for further development; (4) providing an objective, evidenced-based, source of information for the public; and (5) creating a hub for the exchange of ideas among coronavirus researchers whose feedback is sought and welcomed. By comprehensively reviewing all published laboratory, animal model, and clinical data on potential coronavirus therapies, the Database makes it unlikely that promising treatment approaches will be overlooked. In addition, by making it possible to compare the underlying data associated with competing treatment strategies, stakeholders will be better positioned to prioritize the most promising anti-coronavirus compounds for further development.

## Rationale
One underlying hypothesis of this project is that there already exists in the public domain ample data to rationally develop new drugs and design clinical trials. More than 15 years of research on coronaviruses including SARS-CoV and MERS-CoV have identified many compounds capable of inhibiting specific steps in the coronavirus replication cycle. Since the beginning of 2020, there has also been an outpouring of research on SARS-CoV-2 replication and multiple structures have been published of its spike protein, polymerase, replication complex, and protease enzymes. Many attempts are also being made to identify drugs or compounds already used in humans that can be repurposed to treat SARS-CoV-2 infections.

A second underlying hypothesis of this project is that drug development can be accelerated by creating a resource that juxtaposes data associated with treatments making it possible for  laboratory researchers, clinical investigators, public health officials, and funding agencies to prioritize the most promising anti-coronavirus compounds for further development. The current paradigm of drug development involves competing entities engaged in compound screening, drug optimization, preclinical studies, pharmacokinetics, and clinical trials. The process is vertically but not horizontally integrated with different laboratories and companies pursuing competing agendas. The process is opaque to most stakeholders, in large part because no one has yet created what we are working on – a publicly available integrated representation and synthesis of the diverse forms of antiviral drug development data.

The critical need for horizontal integration is evident from the fact that the vast majority of ongoing and planned clinical trials do not reflect the best available scientific research: two-thirds of trials include hydroxychloroquine or chloroquine and nearly one-fourth include non-coronavirus antiviral agents such as those targeting HIV-1 or influenza. Neither of these two approaches are supported by the available evidence.

## Current Status and Plans
The database website currently contains three tracks: (1) a database of published in vitro, animal model, and clinical data on small molecules, peptides, recombinant proteins, monoclonal antibodies, and interferons that have been studied inhibitory activity against one or more coronavirus species; (2) a linked registry of ongoing and planned clinical trials of potential anti-coronavirus compounds annotated according to study design and patient population; and (3) a comprehensively referenced and regularly updated evidence-based narrative summary of the state of coronavirus therapeutics. The narrative synthesizes data from the first two tracks, describe research findings relevant to coronavirus replication and drug development that do not lend themselves to inclusion in a database, and contain objective assessments of the most promising research directions.

Additional planned work includes (1) the collection of human pharmacokinetic data on those compounds with promising results in clinical studies; (2) detailed descriptions of SARS-CoV-2 monoclonal antibodies including information on their binding sites, epitopes, escape mutations, and Fc portions; and (3) regularly updated curated alignments of coronavirus spike (S), RNA-dependent RNA polymerase (RdRp), protease enzymes, and potentially additional targets of therapy. Separate alignments will be categorize amino acid variation at four phylogenetic levels: SARS-CoV-2, sarbecovirus, betacoronaviruses, and coronaviruses. Tracking SARS-CoV-2 S sequence variability will be particularly important to inform the development of monoclonal antibodies as novel mutations are now being identified in this protein [^10.1101/2020.04.29.069054]. Alignments of the key enzymatic targets of therapy are important for identifying conserved sites important for drug design and for understanding which compounds with activity against one coronavirus are likely to be active against other species.

## Work Completed To Date
As of April 28, 2020, we have extracted data linking antiviral compounds to coronaviruses from 225 publications including 180 peer-reviewed publications and 45 non-peer-reviewed pre-prints. The database contains results from 1121 cell culture experiments, 350 biochemical experiments, 308 entry assay experiments, 147 animal model experiments, and 36 clinical studies. More than 50% of the results are from studies of SARS-CoV (41%) or SARS-CoV-2 (15%); while the remainder are from studies of MERS-CoV (24%), endemic human coronaviruses (11%), and mammalian/bat coronaviruses (9%). There are currently 192 ongoing or planned clinical trials in which one or more of the treatment arms contains a compound intended to inhibit SARS-CoV-2 replication.

The schema of the database can be found at [covdb.stanford.edu/database-schema/](/database-schema/). The database has been populated with the data described in the preceding paragraph and these data have been linked to eight additional core look-up tables containing references, compounds, drug targets, virus species, virus strains, cell lines, virus measurements, and animal models. The look-up tables, which contain definitions and continually updated explanatory text, ensure referential integrity and are designed to be educational. Their explanations are continually updated.

Each compound is categorized according to its most likely target including both virus and host targets. The compounds with viral targets include entry inhibitors (monoclonal antibodies, fusion inhibitors, and additional small and large molecule inhibitors of receptor binding), nucleoside analog RdRp (polymerase) inhibitors; 3CL protease inhibitors; PL protease inhibitors; and helicase inhibitors. The compounds with host targets include interferons and inhibitors of host proteases that cleave coronavirus spike (S) proteins. A large number of additional compounds, particularly those purported to target host pathways, act by less well defined or completely uncertain mechanisms; these include inhibitors of endosomal trafficking, compounds that modulate signaling phosphorylation pathways, cyclophilin inhibitors, and compounds that may induce interferon production. Although coronaviruses contains four structural proteins (spike, envelope, membrane, and nucleocapsid), 16 non-structural proteins (nsp1-nsp16), and several accessory proteins [^10.1007/978-1-4939-2438-7_1][^10.1038/s41586-020-2286-9] most antiviral research has focused on the targets reviewed here.

## Polymerase Inhibitors
RNA-dependent RNA polymerases (RdRps) catalyze phosphodiester bond formation between nucleoside triphosphates in an RNA-templated manner. RdRps are highly conserved in their structural and functional features, even among diverse RNA viruses belonging to different families[^10.3390/v11080745]. Coronavirus RNA polymerization, however, is more complex than that of other viruses. First, coronaviruses contain a 3’-to-5’ exoribonuclease (ExoN; nsp14) which is required for replication fidelity[^10.1073/pnas.0508200103]. ExoN is responsible for the intrinsic resistance of coronavirus species such as SARS-CoV to ribavirin and several other nucleoside analogs [^10.1371/journal.ppat.1003565][^10.1073/pnas.1718806115]. Second, coronavirus RdRp (nsp12) requires two accessory proteins (nsp7 and nsp8). The 3D structure of the nsp12-nsp7-nsp8 complex of SARS-CoV [^10.1038/s41467-019-10280-3] and SARS-CoV-2 [^10.1126/science.abb7498][^10.1126/science.abc1560] have been solved by cryo-EM.

Polymerase inhibitors are the most commonly used antiviral agents and have formed the backbone of most HIV, hepatitis B, hepatitis C, and herpes virus treatments. Nucleoside analog inhibitors are also often broad-spectrum possessing activity against multiple related virus species. Most antiviral polymerase inhibitors act as nucleoside analog chain terminators – usually because they lack a 3’-hydroxyl group. Several nucleoside analogs, however, are not chain terminators. Rather, they become incorporated into viral genomes and inhibit replication by introducing mutations during subsequent rounds of replication.

### Remdesivir
Remdesivir (GS-5734) is the monophosphate prodrug of the parent 1’-cyano-substituted adenine C-nucleoside analogue GS-441524[^10.1021/acs.jmedchem.6b01594]. The presence of the phosphate group allows for more efficient metabolism of the prodrug to the active nucleoside triphosphate form by bypassing the rate-limiting first phosphorylation step[^10.1038/nature17180]. Remdesivir is a nonobligate chain terminator in that it contains a 3’-OH group. However, it causes delayed chain termination due to a steric clash after the addition of three trailing nucleosides[^10.1074/jbc.AC120.013056][^10.1074/jbc.RA120.013679][^10.1016/j.antiviral.2020.104793]. The mechanism by which it eludes coronavirus exonucleases is not known.

Remdesivir has broad spectrum activity against multiple RNA viruses and inhibits SARS-CoV, MERS-CoV, and SARS-CoV-2 with most reported half-maximal effective concentrations (EC<sub>50</sub>s) below 1 μM[^10.1126/scitranslmed.aal3653][^10.1128/mBio.00221-18][^10.1038/s41467-019-13940-6][^10.1038/s41422-020-0282-0][^10.1101/2020.04.03.024257][^10.1016/j.antiviral.2020.104786][^10.1101/2020.03.20.999730][^10.1101/2020.04.03.023846][^10.1101/2020.04.03.20052548][^10.1101/2020.04.27.064279]. It inhibits coronaviruses in multiple cell types including primary human airway epithelial cells[^10.1126/scitranslmed.aal3653][^10.1101/2020.04.27.064279]. Animal model experiments have demonstrated that remdesivir can reduce lung virus levels and lung damage in mice infected with MERS-CoV and SARS-CoV[^10.1126/scitranslmed.aal3653][^10.1016/j.antiviral.2020.104793], rhesus macaques infected with MERS-CoV[^10.1126/scitranslmed.aal3653][^10.1073/pnas.1922083117], and SARS-CoV-2[^10.1101/2020.04.15.043166].

Remdesivir resistance in a related coronavirus – murine hepatitis virus (MHV) – has been selected during prolonged _in vitro_ passage in the presence of remdesivir[^10.1128/mBio.00221-18]. The remdesivir-resistant mutations are in the RdRp fingers domain. Introducing the homologous substitutions into SARS-CoV (F480L + V557L) resulted in a virus with reduced replication and about 6-fold reduced remdesivir susceptibility [^10.1128/mBio.00221-18].

Remdesivir is administered intravenously with a loading dose of 200 mg followed by 100 mg daily for 5 to 10 days in adult patients[^10.1002/phar.2398][^10.1016/j.onehlt.2020.100128]. Following intravenous administration, the achievable maximum remdesivir plasma concentration (Cmax) is predicted to be at least twice as high as its most commonly reported EC<sub>50</sub>s[^10.1101/2020.04.16.20068379].

In 53 patients with severe SARS-CoV-2 pneumonia receiving remdesivir as part of two expanded access trials, 34 were receiving mechanical ventilation (including 3 on extracorporeal membrane oxygenation \[ECMO\]). Overall, 6 of the 34 ventilated patients and 1 of the 19 non-ventilated patients died[^10.1056/NEJMoa2007016]. This death rate appeared lower than that of historical controls with similar levels of disease severity.

A phase 3 placebo-controlled trial begun in China in patients with moderate-to-severe disease was discontinued prematurely after recruiting 237 rather than the planned 453 patients. Patients were randomized 2:1 to receive remdesivir or placebo. Although not statistically significant, patients receiving remdesivir had a faster time to clinical improvement than those receiving placebo (median 18 days \[IQR 12–28\] vs 23 days \[15–28\]; HR 1.52 \[0.95–2.43\]. The 28-day mortality was similar in both groups (14% for remdesivir and 13% for placebo). There was no difference in the rates of decline of nasopharyngeal viral RNA levels between the two groups. Adverse events leading to treatment discontinuation were 12% of the remdesivir group and 5% in the placebo group[^10.1016/S0140-6736(20)31022-9].

Preliminary results from the NIH Adaptive COVID-19 Treatment Trial (ACTT) were announced on April 29th, 2020. The trial randomly assigned 1,063 hospitalized patients to remdesivir or placebo. There was a statistically significant 31% shortening in the median time to study-defined recovery in the remdesivir group (11 vs. 15 days). Mortality in the remdesivir group was lower than in the placebo group  (8% vs.11%) but this finding did not reach statistical significance. Based on this study, the FDA issued an Emergency Use Authorization for the use of remdesivir in hospitalized patients with COVID-19 who have severe disease defined as having an SpO2 ≤ 94% on room air or requiring supplemental oxygen, mechanical ventilation, or extracorporeal membrane oxygenation (ECMO).

___Conclusion: Remdesivir is the first anti-SARS-CoV-2 compound that appears to be clinically beneficial. The optimal use of this drug will become clearer following the publication of the ACTT trial and two additional Gilead studies. The risks and benefits of using this drug in high-risk persons with mild-to-moderate disease will be important to study. Remdesivir will likely become the main comparator drug in future trials of moderate-to-severe SARS-CoV-2 infections.___

### β-D-N4-hydroxycytidine-5'-isopropyl ester (aka NHC and EIDD-2801)
β-D-N4-hydroxycytidine is a cytidine analogue that exerts its activity primarily through viral mutagenesis in that it incorporates into new RNA strands but leads to large numbers of replication errors during subsequent rounds of replication [^10.1016/j.coviro.2019.04.002]. β-D-N4-hydroxycytidine-5'-isopropyl ester is an orally available NHC prodrug. The mechanism by which β-D-N4-hydroxycytidine eludes the proof-reading function of the coronavirus exonucleases is not known.

β-D-N4-hydroxycytidine has broad-spectrum antiviral activity against multiple RNA viruses including MERS-CoV, SARS-CoV, and SARS-CoV-2 with most EC<sub>50</sub>s below 1 μM[^10.1177/095632020401500102][^10.1177/095632020601700505][^10.1128/JVI.01348-19][^10.1126/scitranslmed.abb5883]. It also has activity in primary human airway epithelial cells and reduced virus levels, disease, and lung damage in mouse models of SARS-CoV and MERS-CoV[^10.1126/scitranslmed.abb5883]. Deep sequencing studies have confirmed that viral mutagenesis is the main mechanism by which β-D-N4-hydroxycytidine inhibits coronaviruses [^10.1177/095632020401500102][^10.1177/095632020601700505][^10.1128/JVI.01348-19][^10.1126/scitranslmed.abb5883].

An Investigational New Drug Application for β-D-N4-hydroxycytidine-5'-isopropyl ester has been approved, enabling its manufacturer, Ridgeback Biotherapeutics LP, to initiate human clinical testing. Although viral mutagenesis has rarely been shown to be an effective mechanism of viral inhibition, the large size of coronavirus genomes may make coronaviruses particularly vulnerable to viral mutagenesis. There have also been past concerns that non-chain-terminating nucleosides may be associated with toxicity to host polymerases. Nonetheless, _in vitro_ studies have not identified effects on human RNA polymerases[^10.1128/AAC.01719-19][^10.1126/scitranslmed.aax5866]. Moreover, the risk of toxicity for short courses of therapy are likely to be low.

___Conclusion: The preclinical in vitro and animal model data associated with EIDD-2801 are comparable to those of remdesivir. However, EIDD-2801’s safety in humans has not yet been established. Conducting safety and efficacy trials will likely require a minimum of three months. Should these trials be successful, this drug may be a useful oral option to be used in high-risk outpatients. Moreover, used in combination with remdesivir it might further reduce morbidity and mortality in patients with severe disease.___

### Other Nucleoside Analogs
Ribavirin is a nucleoside analog with modest activity against several viruses. Although multiple mechanisms for its activity have been proposed, the most likely is an increase in viral mutagenesis. In cell culture studies, it has demonstrated little, if any, activity against coronaviruses including SARS-CoV, MERS-CoV, and SARS-CoV-2[^10.1016/s0140-6736(03)13615-x][^10.1177/095632020401500102][^10.1016/j.jinf.2013.09.029][^10.1099/vir.0.061911-0][^10.1038/srep01686][^10.1016/j.antiviral.2020.104786][^10.1016/S0140-6736(20)31022-9]. Structural and biochemical studies have shown that although ribavirin 5’-monophosphate can be incorporated during RNA synthesis, it is readily excised by nsp14, the coronavirus exonuclease. Multiple observational clinical studies of patients with SARS-CoV, and to a lesser extent in patients with MERS-CoV have shown no evidence for clinical benefit while demonstrating a high risk of toxicity[^10.1371/journal.pmed.0030343]. Ribavirin is rarely used or studied for treating SARS-CoV-2.

Favipiravir (T-705) is a purine analog prodrug that is ribosylated and phosphorylated intracellularly to form the active metabolite favipiravir ibofuranosyl-5′-triphosphate (aka T-705-RTP). It has broad spectrum activity against multiple viral RNA polymerases and appears to act by causing viral mutagenesis[^10.1016/j.antiviral.2013.09.015][^10.1016/j.antiviral.2018.03.003]. Favipiravir, approved in Japan for treating pandemic influenza infections, appears relatively safe and achieves high plasma concentrations following oral administration[^10.1002/cpt.1844][^10.1101/2020.04.16.20068379]. However, it has little, if any, inhibitory activity against SARS-CoV-2 in vitro, with EC<sub>50</sub>s ranging from 60 to >100 μM[^10.1038/s41422-020-0282-0][^10.1016/j.antiviral.2020.104786][^10.1101/2020.03.20.999730].

In a recent open-label randomized trial of 240 patients, of whom just 40% had PCR-confirmed COVID-19 pneumonia, favipiravir was associated with an improved 7 day clinical recovery rate compared to Arbidol in a subgroup of patients without critical disease (71% vs 56%; p=0.02)[^10.1101/2020.03.17.20037432]. However, the fact that diagnosis was confirmed in just 40% of patients raises the question as to what proportion of patients may have had influenza infections. As of April 20, 2020, there are 7 ongoing or planned clinical trials of favipiravir in about 700 SARS-CoV-2 patients; just one is placebo-controlled.

Galidesivir (BCX4430) is an adenosine analog that inhibits multiple viruses including filoviruses (Ebola virus and Marburg virus) _in vitro_ and in rodents and cynomolgus macaques[^10.1038/nature13027]. However, it has little, if any _in vitro_ activity against MERS-CoV and SARS-CoV with EC<sub>50</sub> levels of about 60 μM for MERS-CoV and SARS-CoV[^10.1038/nature13027] and of >100 μM for SARS-CoV-2[^10.1016/j.antiviral.2020.104786]. At least seven additional nucleoside analogs have been studied _in vitro_ against SARS-CoV, MERs-CoV, or SARS-CoV-2, including four that are either approved (gemcitabine, mizoribine, or mycophenolate) or being studied in humans (merimepodib) as immunosuppressants or anticancer agents[^10.1099/vir.0.061911-0][^10.1016/j.antiviral.2005.01.003][^10.1128/JVI.00023-19][^10.1101/2020.04.07.028589][^10.1016/j.jinf.2013.09.029].

___Conclusions: There appears to be little benefit for the use or further study of the nucleoside analogs in this section either because they lack meaningful SARS-CoV-2 inhibitory activity or because they are immunosuppressive.___

## Protease Inhibitors
Coronaviruses contain two protease enzymes: 3 chymotrypsin-like cysteine protease (3CLpro or Main \[M\]-pro; non-structural protein 5 \[nsp5\]) and papain-like serine protease (PLpro; nsp3)[^10.1002/cmdc.202000223]. 3CLpro cleaves polyprotein 1a/b at 11 sites. It is conserved among several families of RNA viruses and has similar cleavage site specificity to the picornavirus family of 3C proteases. 3CLpro contains 306 amino acids and functions as a homodimer. PLpro is part of a large multidomain transmembrane protein. It cleaves polyprotein 1a/b at the nsp1/2, nsp2/3, and nsp3/4 boundaries and also cleaves host proteins important for innate immunity such as ubiquitin interferon-stimulated gene product 15 (ISG-15). The 3D structure of SARS-CoV 3CLpro[^10.1126/science.1085658] and PLpro[^10.1021/jm401712t]; and of SARS-CoV-2 3CLpro[^10.1038/s41586-020-2223-y][^10.1126/science.abb3405] and [PLpro](https://www.wwpdb.org/pdb?id=pdb_00006w9c) are available for structure-based drug design.

There are more candidate 3CLpro than PLpro inhibitors because of this enzyme’s similarities to those of other virus species. The recent SARS-CoV-2 3CLpro structures have also identified tightly binding compounds with nanomolar-range half-maximal inhibitory concentrations in biochemical experiments (IC<sub>50</sub>s) and half-maximal effective concentrations in cell culture (EC<sub>50</sub>s). Most of the protease inhibitors (PIs) that are currently being studied clinically are curiously those that were approved for treating HIV-1 and display little if any SARS-CoV-2 inhibitory activity.

### HIV-1 Protease Inhibitors
None of the HIV-1 protease inhibitors (PIs) are expected to inhibit coronaviruses as the HIV-1 protease is an aspartate protease with a different specificity than the coronavirus 3CLpro and PLpro enzymes. Nonetheless, one particular HIV-1 PI combination – lopinavir boosted by ritonavir, which is an inhibitor of lopinavir metabolism (lopinavir/r) – has been used frequently to treat SARS-CoV, MERS-CoV, and SARS-CoV-2 infection.  Each of the other HIV-1 associated PIs has also been studied for potential anti-coronavirus activity.

Lopinavir/r was first used to treat a coronavirus infection when it was administered to a cohort of 75 hospitalized patients with SARS-CoV in Hong Kong in 2003[^PMID14660806][^10.1136/thorax.2003.012658]. These 75 hospitalized patients included 44 newly diagnosed SARS-CoV patients and 31 who had more severe disease. Among the 44 newly diagnosed patients, 2% died and none required mechanical ventilation; the 31 patients with more severe disease, 13% died and 10% required intubation. The mortality rate and need for mechanical ventilation among the 44 newly diagnosed patients were lower than those observed in historical controls (i.e., the very first group of SARS patients treated at their hospital). The authors noted in their conclusion that the favorable response to lopinavir/r in the subset of the 44 newly diagnosed patients compared to historical controls may have resulted in part from the experience at treating patients with SARS[^PMID14660806][^10.1136/thorax.2003.012658].

Lopinavir displays weak inhibitory activity against SARS-CoV, MERS-CoV, and SARS-CoV-2 with EC<sub>50</sub>s of between 5 and >25 μM or higher[^10.1136/thorax.2003.012658][^10.1016/j.jcv.2004.03.003][^10.1016/j.bbrc.2004.04.083][^10.1177/095632020601700505][^10.1016/j.jinf.2013.09.029][^10.1128/AAC.03011-14][^10.1038/s41467-019-13940-6][^10.1101/2020.03.20.999730][^10.1016/j.antiviral.2020.104786][^10.1101/2020.04.06.026476][^10.1101/2020.04.29.067983]. In addition, the selectivity index of lopinavir in cell culture is low suggesting that whatever inhibition is observed in cell culture may result from its toxic effects on the cells used to culture coronavirus rather than on the virus itself. In two biochemical assays with SARS-CoV 3CLpro, lopinavir displayed an IC<sub>50</sub>s of 50 μM[^10.1073/pnas.0403596101] and >20 μM (Ma et al. 2020)[^10.1101/2020.04.20.051581]. In one study of marmosets infected with MERS-CoV, two animals receiving lopinavir/r had reduced mortality, virus levels, and lung pathology compared with three untreated control animals[^10.1093/infdis/jiv392]. However, no benefit was observed in mice infected with MERS-CoV treated with lopinavir/r + IFN-β[^10.1038/s41467-019-13940-6].

Most importantly, an open-label randomized controlled trial of lopinavir/r vs. standard-of-care in 199 hospitalized adults with SARS-CoV-2 infection and SatO<sub>2</sub> levels 94% or PaO<sub>2</sub> /FiO<sub>2</sub>  <300, demonstrated no significant difference in days to virological clearance, clinical improvement, need for mechanical ventilation, or mortality[^10.1056/NEJMoa2001282]. The primary outcome of time to clinical improvement defined by a 2-point improvement on a 7-category ordinal scale or hospital discharge was 16 days in both groups of patients.

Among the remaining HIV-1 PIs, only nelfinavir and atazanavir appeared to have any coronavirus inhibitory activity. Nelfinavir had EC<sub>50</sub>s of 0.05 and 3.5 μM for SARS-CoV[^10.1177/095632020601700505][^10.1016/j.bbrc.2004.04.083], 1.1 μM and no activity for SARS-CoV-2[^10.1101/2020.04.06.026476][^10.1101/2020.04.29.067983], and 3.6 μM[^10.1101/2020.02.25.965582] and no activity for MERS-CoV[^10.1016/j.jinf.2013.09.029]. Its IC<sub>50</sub> against SARS-CoV 3CLpro in a biochemical experiment was 46 μM[^10.1016/j.bbrc.2005.05.095]. Atazanavir had had EC<sub>50</sub>s against SARS-CoV-2 between 0.2 and 2 μM in one study[^10.1101/2020.04.04.020925] but as high as 9.4 and >50 μM in three other studies[^10.1101/2020.03.20.999730][^10.1101/2020.04.06.026476][^10.1101/2020.04.29.067983]. Darunavir has displayed poor activity against SARS-CoV-2[^10.1101/2020.04.03.20052548][^10.1101/2020.04.06.026476].

___Conclusion: As of May 7, 2020, there were 17 recruiting trials and 8 planned trials containing a lopinavir/r treatment arm. However, considering that lopinavir/r does not inhibit coronaviruses in vitro and displays little or no clinical or virological activity, it is uncertain why it is still being studied.___

### 3CLpro Investigational Inhibitors
Most coronavirus PI drug discovery efforts have targeted 3CLpro: it shares a conserved substrate recognition pocket with the 3C like protease (3Cpro) of human picornaviruses, which have already been successfully targeted with antiviral agents. In addition, 3CLpro can also be purified in large quantities and readily crystallized[^10.1016/j.virusres.2007.02.015].

There have been multiple drug screens and several structure-based designs targeting 3CLpro. Several compounds have inhibited SARS-CoV and SARS-CoV-2 in biochemical assays with IC<sub>50</sub>s ranging between 0.02 and 0.2 μM[^10.1021/jm050184y][^10.1101/2020.04.20.051581][^10.1016/j.bmc.2016.05.013][^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb3405]. One of these, GC376 forms covalent bonds with three active site cysteines and inhibits multiple coronaviruses[^10.1128/JVI.01348-12]. Its IC<sub>50</sub> against SARS-CoV-2 was 0.03 μM while its EC<sub>50</sub> in cell culture was 3.4 μM[^10.1101/2020.04.20.051581]. GC376 also inhibits MERS-CoV (IC<sub>50</sub> = 1.7 μM) and displays sub-micromolar inhibitory against several other coronaviruses including the alphacoronavirus FIPV, TGEV, and HCoV 229E and the betacoronavirus BCV. It is highly effective in treating cats with the rapidly fatal coronavirus disease FIPV[^10.1371/journal.ppat.1005531]. Crystal structures of MERS-CoV 3CLpro bound to GC376 has led to the identification of piperidene peptidomimetics with greater inhibitory activity against SARS-CoV and MERS-CoV[^10.1016/j.ejmech.2018.03.004].

Another series of peptidomimetics has been shown to inhibit BtCoV-HKU4 (related to MERS-CoV). Asymmetric aromatic disulfides have had low micromolar activity against SARS-CoV[^10.1016/j.ejmech.2017.05.045]. Several other backbones have been shown to have activity against SARS-CoV[^10.1021/jm050184y][^10.1016/j.bmcl.2007.08.031].

The compounds with the most 3CLpro inhibitory activity identified to date are a series of peptidomimetic α-ketoamide and aldehyde derivatives[^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb4489] . Several of these compounds has been co-crystallized with SARS-CoV-2 3CLpro[^10.1021/acs.jmedchem.9b01828][^10.1126/science.abb4489]. Preliminary pharmacokinetics evaluations have demonstrated metabolic stability in mouse and human microsomes and encouraging plasma and bronco-alveolar lavage fluid levels whether administered subcutaneously or by inhalation[^10.1126/science.abb3405][^10.1126/science.abb4489]. Another SARS-CoV-2 crystal structure identified of several inhibitory compounds; including an investigational compound ebselen, which has been studied in humans[^10.1038/s41586-020-2223-y].

Disulfiram, the licensed oral alcohol antagonist has been reported to inhibit SARS-CoV-2 3CL protease with an IC<sub>50</sub> of about 10 μM[^10.1038/s41586-020-2223-y] and to inhibit SARS-CoV and MERS-CoV PL protease with a slightly higher IC<sub>50</sub> [^10.1016/j.antiviral.2017.12.015]. However, disulfiram has not yet been studied in cell culture, and existing PK data suggests that standard doses may be insufficient to yield inhibitory drug levels.

___Conclusion: There are many promising lead compounds that could be developed into highly effective SARS-CoV-2 drugs. Several of these including GC376 and ebselen might be useful as initial scaffolds because they display very low μM activity and have been studied in cats and humans, respectively. Other peptidomimetic compounds, such as the -ketoamides have greater inhibitory in vitro activity but have not yet been studied in animals or humans. ___

### PLpro Investigational Inhibitors
The crystal structure of SARS-CoV[^10.1073/pnas.0510851103][^10.1021/jm401712t] and SARS-CoV-2 () have been solved. However, in contrast to 3CLpro, the PLpro enzyme is membrane associated, making it more challenging to study in vitro. It does not have homologues in other viruses, so there are fewer compounds that can serve as starting points for drug development. Nonetheless, it is an essential coronavirus enzyme; high-throughput testing approaches have begun and several lead compounds have been identified[^10.1016/j.antiviral.2014.12.015][^10.1002/cmdc.202000223].

## Entry Inhibitors
Coronavirus cell entry is mediated by its spike (S) glycoprotein. S is a membrane protein essential for the two main steps of the entry process: attachment to the host cell and fusion of the viral and cellular membranes. The functional S protein is a trimer of identical subunits. For most coronaviruses, S is processed by host proteases to generate two functional subunits, designated S1 and S2. The S1 subunit forms the apex of the S trimer including the receptor binding domains (RBDs)[^10.1016/bs.aivir.2019.08.002] and is responsible for attachment to cellular receptors. The S2 unit is responsible for fusion to the cell membrane.

The 3D structure of the SARS-CoV-2 S trimer has been determined by cryo-EM[^10.1126/science.abb2507]. The cellular receptor for SARS-CoV and SARS-CoV-2 is the human angiotensin-converting enzyme 2 (ACE2)[^10.1038/nature02145][^10.1038/s41586-020-2012-7]. An extended insertion within the structurally similar SARS-CoV and SARS-CoV-2 RBDs called the receptor-binding motif (RBM) contains most of the S amino acids responsible for ACE2 binding[^10.1074/jbc.C300520200][^10.1038/s41586-020-2180-5][^10.1007/s11427-020-1637-5]. S binding to ACE2 does not interfere with ACE2’s catalytic activity[^10.1126/science.1116480]. However, ACE2 internalization following infection leads to the loss of ACE2’s lung protective effects. S1-ACE2 affinity is higher for SARS-CoV-2 than SARS-CoV-1, possibly explaining its greater infectiousness[^10.1038/s41586-020-2180-5].

S2 contains two complementary heptad repeat (HR) regions, designated HR1 and HR2, which are alpha helices with repeated patterns of seven amino acids that have regions that can bind to one another. HR1 projects out to the host cell membrane whereas HR2 is about 170 amino acids upstream and closer to the virus membrane[^10.1016/j.virusres.2014.11.021]. HR1 and HR2 collapse onto one another to create a hairpin which brings the viral and cell membranes together promoting fusion. As S2 (like S1) functions as a trimer, the hairpin formed by the interaction of HR1 and HR2 is referred to as a six-helix bundle.

SARS-CoV and SARS-CoV-2 have two main cleavage sites. One is at the S1/S2 boundary and the other is at a site within S2 referred to as the S2’ site. These sites must be cleaved by host proteases in order for the S1 and S2 to undergo the necessary conformational changes to shift from receptor binding to cell fusion[^10.1016/bs.aivir.2019.08.002]. The specific cleavage sites, the responsible host enzymes, and the cellular locations at which cleavage occurs varies among different coronaviruses and likely also in different cell types; this variation is believed to play a role in cellular tropism and pathogenesis[^10.1016/j.virusres.2014.11.021][^10.1016/bs.aivir.2019.08.002]. Several candidate host proteases appear to be particularly important for processing S1 and S2, including TMPRSS2, a surface transmembrane serine protease, endosomal cathepsins, and furin. In contrast, to SARS-CoV, SARS-CoV-2 S harbors a furin cleavage site at the S1/S2 boundary, which appears to be cleaved during biosynthesis[^10.1016/j.cell.2020.02.058][^10.1016/j.antiviral.2020.104742]. It has been speculated that the presence of this cleavage site may have expanded the transmissibility and tissue tropism of SARS-CoV-2 considering the more widespread expression of furin-like proteases in host cells.

This section describes interventions that interfere with virus entry including convalescent plasma, monoclonal antibodies that block S1 receptor binding, fusion inhibitors, and several other receptor binding inhibitors. Compounds that inhibit the host protease enzymes required for cleaving the S protein are described in the section on host-acting compounds.

### Convalescent Plasma
There are few randomized controlled studies of convalescent plasma for the treatment of severe viral infections. A systematic review and meta-analysis of seven studies of convalescent plasma or immunoglobulin for SARS-CoV was considered inconclusive with respect to potential benefits and harm[^10.1371/journal.pmed.0030343]. A larger, more recent meta-analysis of 32 studies of severe SARS-CoV and influenza, reported that administration of convalescent plasma early after symptom onset showed a statistically significant 75% reduction in the pooled odds of mortality compared with placebo or no therapy[^10.1093/infdis/jiu396]. Moreover, the administration of polyclonal immune globulin preparations have been shown to prevent hepatitis B and rabies infections.

Two risk-benefit analyses and several experts have argued that the possible large benefit associated with convalescent plasma for SARS-CoV-2 outweighs the small risk of undetected bloodborne infection and allergic reaction[^10.1016/j.tmrv.2020.04.001][^10.1172/JCI138745][^10.1007/s11684-017-0596-6][^10.1172/JCI138003]. According to FDA guidance, the administration of SARS-CoV-2 convalescent plasma must be under an investigational new drug application. Donors must have had resolution of symptoms 28 days before their donation or 14 days before if they have negative nasopharyngeal or blood PCR tests. The recommend neutralizing antibody titers should be 1:160 (minimum >1:80). However, because neutralizing antibodies cannot readily be detected in most clinical settings, this requirement is waived provided blood samples can be stored for possible eventual testing. Additionally, the magnitude of SARS-CoV-2 receptor-binding domain antibodies appears to correlate strongly with viral neutralization[^10.1101/2020.05.03.20084442].

Joyner et. al. reported the results of a safety review of the first 5,000 patients receiving convalescent plasma in the U.S. between April 3 and May 11 under the FDA Expanded Access Program[^10.1101/2020.05.12.20099879]. 81% were classified as having severe or life-threatening diseases and 19% were judged to be at high risk of progressing to severe or life-threatening disease. 66% were in the ICU. 36 (0.72%) had a serious adverse event in the first four hours after transfusion including 25 events considered related to transfusion, including death in 4, transfusion-associated lung injury in 11, circulatory overload in 7, and allergic reaction in 3. Overall, the seven day mortality was 14.9%[^10.1101/2020.05.12.20099879].

There are now at least five additional published observational studies of convalescent plasma in 50 generally severely ill SARS-CoV-2 patients, including 31 who were receiving mechanical ventilation[^10.1073/pnas.2004168117][^10.1001/jama.2020.4783][^10.1016/j.chest.2020.03.039][^10.1101/2020.05.08.20095471][^10.1093/infdis/jiaa228]. All patients received between 200 to 600 ml plasma, and in three studies of 19 patients the donors were generally documented to have neutralizing antibody titers >1:640[^10.1073/pnas.2004168117][^10.1001/jama.2020.4783][^10.1016/j.chest.2020.03.039]. Each of the 19 patients in these initial reports survived and appeared to experience clinical improvement within three days. However, five of six mechanically ventilated patients in another study died[^10.1093/infdis/jiaa228]. In the fifth and largest study, of 25 patients in Houston, the overall response to therapy also appeared to be favorable in that only one patient died despite 12 being mechanically ventilated[^10.1101/2020.05.08.20095471]. 

As of May 7, 2020, there are 46 ongoing (n=18) or planned (n=28) clinical trials containing at least one arm in which recipients will receive convalescent plasma. There are also two planned studies of convalescent immunoglobulin and one of pooled immunoglobulin from healthy donors. One ongoing study is randomizing patients to convalescent plasma or to standard donor plasma and another planned study aims to randomize high-risk exposed persons to convalescent plasma or placebo. The remaining studies are not placebo controlled. Convalescent plasma and potentially other forms of immunoglobulin therapy will likely remain a treatment option particularly in patients with severe disease.

___Conclusions: Although the use of convalescent plasma is not supported by controlled trials, substantial observational data supports this intervention. The usefulness of convalescent plasma also suggests that neutralizing monoclonal antibodies should be useful at treating what at least clinically is primarily a respiratory tract infection. However, human plasma is a complex mixture of compounds. Therefore, the two trials in which convalescent plasma will be compared to standard donor plasma (one for treatment and the other for prevention) would be  informative. An alternative approach to assessing the effectiveness of convalescent plasma would be a retrospective study to determine if there is a correlation between neutralizing antibody titer and clinical response.___

### Monoclonal Antibodies
Neutralizing monoclonal antibodies (MAbs) have been to treat Ebola[^10.1126/science.aad5224][^10.1056/NEJMoa1910993] and prevent respiratory syncytial virus[^10.2165/00003495-199958020-00009]. Neutralizing MAbs have also be used reduce virus levels in HIV-1-infected persons but resistance emerges unless a combination of neutralizing MAbs is administered[^10.1038/nm.4268][^10.1038/s41586-018-0531-2].

Recovery during SARS-CoV and MERS-CoV is usually associated with the development of neutralizing antibodies[^10.1007/s11684-017-0596-6]. Most SARS-CoV neutralizing MAbs target the part of the receptor binding domain (RBD) that binds ACE2 while most MERS-CoV neutralizing MAbs target the part of the RBD that binds its cellular receptor[^10.1016/j.bbrc.2004.05.114][^10.1074/jbc.C300520200][^10.4049/jimmunol.174.8.4908]. Several potent humanized neutralizing MAbs targeting SARS-CoV[^10.1002/rmv.706] and MERS-CoV[^10.1126/scitranslmed.3008140][^10.1038/ncomms8712][^10.3390/v10120680] have shown protection _in vitro_ and in animal models. Crystallization and site-directed mutagenesis studies have defined the specific RBD epitopes recognized by individual MAbs and identified those amino acid residues critical for MAb binding[^10.1038/cr.2015.113].

The most frequently studied SARS-CoV MAbs include (i) 80R, identified from a nonimmune human phage display library[^10.1073/pnas.0307140101]; (ii) S3.1, identified in a memory B cell repertoire of a person recovering from SARS-CoV[^10.1038/nm1080]; (iii) m396, crystallized in complex with SARS-CoV RBD[^10.1073/pnas.0701000104]; (iv) 201 and 68, which were generated by immunizing transgenic mice with SARS-CoV S glycoprotein[^10.1086/427242]: 201 recognized RBD positions 490-510 and directly interfered with S-ACE2 binding; 68 recognized RBD positions 130-150 and likely interfered with the conformational changes required for virus-cell membrane fusion; (v) CR3014, which was protective in ferrets[^10.1016/S0140-6736(04)16506-9] and CR3022, which neutralized CR3014-escape variants[^10.1371/journal.pmed.0030237]; (vi) 4D4, which was found to bind upstream of the RBD between amino acids 12 and 261 and to inhibit a post-binding event[^10.1016/j.virol.2009.07.028]; and (vii) S230.15[^10.1073/pnas.0701000104].

The most frequently studied MERS-CoV MAbs include (i) Mersmab1[^10.1128/JVI.00433-14]; (ii) m336, which was shown to protect rabbits from MERS-CoV[^10.1128/JVI.00912-14][^10.1093/infdis/jiw080] and which has been engineered into a novel antibody-peptide bispecific fusion protein[^10.3390/antib8040053]; (iii) REGN3048 and REGN3051 which have been highly potent _in vitro_ and in a mouse infection model (Pascal et al. 2015)[^10.1073/pnas.1510830112]; (iv) 3B11, which was isolated from a non-immune human Ab-phage library and characterized through the generation of escape variants[^10.1073/pnas.1402074111]; and (v) 4C and D12, two mouse neutralizing MAbs that have been co-crystallized with the MERS-CoV RBD[^10.1080/14728222.2017.1271415].

Most SARS-CoV MAbs do not inhibit SARS-CoV-2[^10.1016/j.cell.2020.03.045]. For example, three well-characterized SARS-CoV RBD-directed MAbs (S230, m396, and 80R) do not bind SARS-CoV-2 RBD at a concentration of 1μM[^10.1126/science.abb2507]. However, two MAbs that appear to bind at more conserved S1 sites that do not compete with ACE2 have been shown to inhibit SARS-CoV-2 including CR3022 and S309[^10.1080/22221751.2020.1729069][^10.1126/science.abb7269][^10.1101/2020.05.05.079202][^10.1101/2020.04.07.023903]. S309 has been shown to recognize an epitope distinct from the S1 receptor binding motif. It is also synergistic with MAbs that bind to the S1 receptor binding motif[^10.1101/2020.04.07.023903].

Most individuals recovering from SARS-CoV-2 develop significant titers of neutralizing antibodies[^10.1101/2020.03.30.20047365][^10.1101/2020.05.03.20084442]. An increasing number of studies have reported the identification and characterization of MAbs cloned from B  cells of persons recovering from SARS-CoV-2 infection. Chen et al. reported that among 26 COVID-19 convalescent patients, most developed IgG S1-specific antibodies and three had antibodies that blocked S1-ACE2 binding[^10.1038/s41423-020-0426-7]. Two MAbs cloned from these patients (311mab–31B5, 311mab−32D4) were shown to potently inhibit S1-ACE2 binding and block pseudovirus entry[^10.1038/s41423-020-0426-7]. Brouwer et. al. also cloned several highly potent ACE2-competing MAbs from convalescent patients including one which blocked infection at a concentration of 7 ng/ml (COVA-ID-2-15)[^10.1101/2020.05.12.088716].  Under an extremely accelerated schedule, it likely that several MAbs will be available for phase I studies this summer[^10.1038/s41587-020-0512-5]. Zost et. al. isolated 386 SARS-CoV-2-reactive MAbs from two convalescent patients and grouped them into five classes: Class I ind to t

___Conclusions: The preliminary data suggesting that convalescent plasma is useful for treating SARS-CoV-2 and the ability of monoclonal antibodies to successfully treat Ebola suggests that monoclonal antibodies will be useful at both treating and preventing SARS-CoV-2 infections.  Under an extremely accelerated schedule, it appears likely that several MAbs will be available for phase I studies this summer.___

### Fusion Inhibitors
Peptides that mimic HR1 or HR2 can block virus-cell fusion by preventing the interaction between HR1 and HR2. The HIV-1 fusion inhibitor enfuvirtide is a 36 amino acid peptide that acts by this mechanism to inhibit HIV-1 replication following subcutaneous administration[^10.1056/NEJMoa035026][^10.1056/NEJMoa035211]. Several peptide mimics of coronavirus HR2 have been shown to be potently inhibit MERS-CoV, SARS-CoV, and SARS-CoV-2 _in vitro_[^10.1016/j.bbrc.2004.05.046][^10.1038/ncomms4067][^10.1093/infdis/jiv325][^10.1021/acs.jmedchem.8b00890][^10.1126/sciadv.aav4580][^10.1101/2020.03.26.009233]. The interest in coronavirus fusion inhibitors derives in part because S2 is more conserved than S1 among different coronavirus species. For example, the S1 proteins of SARS-CoV and SARS-CoV-2 differ by more than 10%. However, their HR1 domains differ by 7% and their HR2 domains are essentially identical[^10.1038/s41423-020-0374-2].

Prior to the SARS-CoV-2 pandemic, an HR2 mimicking peptide named EK1 was identified as a potential broad-spectrum coronavirus inhibitor as it inhibited both SARS-CoV and MERS-CoV at sub-micromolar levels and was found to safe and protective when administered intra-nasally to SARS-CoV-infected mice[^10.1126/sciadv.aav4580]. With the emergence of SARS-CoV-2, the crystal structure of the SARS-CoV-2 six helix bundle was solved, and several modifications were made to the EK1 peptide. In addition, a cholesterol group was added[^10.1038/s41422-020-0305-x] as lipidation had recently been shown empirically to improve the pharmacokinetics and inhibitory activity of viral fusion inhibitors[^10.1128/JVI.00288-17]. This new lipopeptide, named EK1C4 was found to inhibit SARS-CoV-2 fusion 150 times more strongly than EK1, with SARS-CoV-2 EC<sub>50</sub>s of 0.04 μM and 0.004 μM in cell culture for SARS-CoV-2 and MERS-CoV, and with even lower EC<sub>50</sub>s in cell-cell fusion assays.

___Conclusion: In theory, safety and PK studies of a fusion inhibitor such as EK1C4 could begin immediately to determine the ideal mode of administration (parenteral, intranasal, inhalational), the risk of hypersensitivity, and the plasma half-life. Whereas peptides can be administered parenterally or by inhalation, it is not known whether the same would apply to a lipopeptide. In contrast to monoclonal antibodies, the half-life of fusion inhibitors will be short and could therefore be used solely for treatment.___

### Soluble ACE2 and Other Molecular Decoys
Soluble recombinant human ACE2 (rhACE2) has two potential beneficial effects. First, prior to the SARS-CoV-2 pandemic, rhACE2 has been studied to protect lungs from injury during acute respiratory distress syndrome (ARDS)[^10.1038/nature03712][^10.1038/nm1267][^10.1186/s13054-017-1823-x][^10.1016/j.cell.2020.04.004]. Second, rhACE2 inhibits the binding of SARS-CoV and SARS-CoV-2 to ACE2-expressing cells[^10.1016/j.bbrc.2004.05.114][^10.1016/j.cell.2020.03.045] and has displayed does-dependent activity _in vitro_ against SARS-CoV-2 in cultured cells and engineered human blood vessel and kidney organoids[^10.1016/j.cell.2020.04.004]. APN1 (Apeiron Biologics) is a recombinant form of human ACE2, that demonstrated safety in 89 non-SARS-CoV-2 patients and volunteers. It is now being evaluated in a 200-person placebo-controlled study of hospitalized persons with SARS-CoV-2 (NCT04335136). Another more recently described similar approach involving the use of a fusion protein – in which an inactive ACE2 mutant is linked to a human Fc IgG – is highly active _in vitro_ [^10.1038/s41467-020-16048-4].

The complete ACE2 molecule may not be necessary to inhibit viral infection if a synthetic peptide can be constructed with a critical minimal subset of ACE2 amino acids capable of competing with cellular ACE2 for the SARS-CoV-2 receptor binding domain (RBD). Preliminary data from SARS-CoV suggest that multiple residues from different parts of the ACE2 molecule will be required including amino acids between positions 24–90 and 329–354[^10.1126/science.1116480]. Two ACE2-derived peptides between positions 22–57 demonstrated some _in vitro_ activity against SARS-CoV[^10.1016/j.virol.2006.01.029]. Another peptide comprising two discontinuous ACE2 segments containing residues 22–44 and 351–357 artificially linked together by glycine, exhibited more potent _in vitro_ activity with an EC<sub>50</sub> of about 0.1 μM[^10.1016/j.virol.2006.01.029].

___Conclusion: rhACE2 may prove to be useful in patients with the most severe cases of SARS-CoV-2 pneumonia.___

## Additional Possible Entry Inhibitors
Spike-mimicking 20-mer peptides corresponding to different S1 and/or S2 regions were reported in one study to synergistically interfere with SARS-CoV entry into cells presumably by competing with complete virus for ACE2 binding[^PMID15918330]. However, few other studies have described this approach. The inhibition of ACE2 enzymatic function or down-regulation of ACE2 levels is precarious as the absence of ACE2 is associated with the same pulmonary and cardiovascular disturbances observed during severe SARS-CoV and presumably SARS-CoV-2 infection.

Defensins are small antimicrobial peptides produced by white blood cells and epithelial cells, which have a role in innate immunity. In a study of several peptides derived from mouse β-defensin-4, one 30 amino acid peptide called P9 was found to inhibit influenza A, MERS-Co-V, and SARS-CoV both _in vitro_ and in a mouse model[^10.1038/srep22008].

Griffithsin, a 121 amino acid protein isolated from red algae, inhibits SARS-CoV and MERS-CoV _in vitro_ with EC<sub>50</sub>s in the range of 0.01 to 0.1 μM without inducing cell toxicity. It also inhibits SARS-CoV when delivered intranasally in a mouse model[^10.1128/JVI.02322-09][^10.1016/j.antiviral.2016.07.011]. Time course experiments suggest that griffithsin interferes with virus cell entry[^10.1016/j.antiviral.2016.07.011]. While ELISA assays showed that it binds directly to the S glycoprotein, it does not inhibit S binding to ACE2; this suggests that griffithsin may prevent the subsequent steps needed for viral entry[^10.1128/JVI.02322-09].

VE607 was reported in one study to inhibit SARS-CoV S protein-ACE2-mediated cell entry[^10.1016/j.chembiol.2004.07.013] but does not appear to have been studied further. HTCC polymers are cationacally modified chitosan derivatives that appear to inhibit cell entry of multiple coronaviruses including MERS-CoV and SARS-CoV[^10.1101/2020.03.29.014183][^10.1371/journal.pone.0156552]. SARS-RBD-Hexapeptide is weak inhibitor of SARS-CoV viral entry, but nonetheless suggests the possibility that small peptides may be able to inhibit the interaction of coronavirus S1 receptor binding domains and ACE2[^10.1016/j.antiviral.2011.12.012]. Emodin is a plant derivative that appears to weakly inhibit SARS-CoV S protein-ACE2-mediated cell entry[^10.1016/j.antiviral.2006.04.014], although it has also been proposed to inhibit SARS-CoV release from cells[^10.1016/j.antiviral.2011.02.008].

Arbidol is an oral drug used in Russia and China to treat influenza and other viral infections[^10.1016/j.antiviral.2014.04.006][^10.1128/JVI.02077-15]. It is reported to have _in vitro_ activity against multiple viruses and to inhibit viral entry by binding viral surface glycoproteins[^10.1016/j.antiviral.2008.10.009]. Its SARS-CoV-2 EC<sub>50</sub>s has been reported to be between 5 and 10 μM with a selectivity index of about 8[^10.1101/2020.04.03.023846][^10.1038/s41421-020-0169-8]. A crystal structure of arbidol in complex with influenza hemagglutinin showed that arbidol binds to a hydrophobic cavity in the hemagglutinin trimer stem[^10.1073/pnas.1617020114]. There have been five small studies of arbidol in patients with SARS-CoV-2, including two open-label controlled studies[^10.1101/2020.03.17.20037432][^10.1101/2020.03.19.20038984] and three observational studies[^10.1016/j.jinf.2020.03.002][^10.1016/j.jinf.2020.03.060][^10.1101/2020.04.06.20042580]. One controlled trial suggested it was inferior to favipiravir at reducing time to clinical recovery[^10.1101/2020.03.17.20037432]; one observational trial suggested it was inferior to nebulized interferon α2b at reducing viral shedding; and one observational trial suggested that it may have been more effective than lopinavir/r at reducing viral shedding[^10.1016/j.jinf.2020.03.060].

One group has reported that certain anticoagulants such as heparin and enoxaparin bind to S1 receptor binding domains and weakly interfere with virus cell entry _in vitro_[^10.1101/2020.04.28.066761].

___Conclusion: Most of the compounds in this section have not been well studied and therefore appear unlikely at the present time to lead to useful treatments.___

## Interferons
Interferons (IFNs) respond to the presence of cellular changes suggestive of a viral infection by inducing numerous genes (IFN-stimulated genes \[ISGs\]) involved in inflammation, signaling, and immunomodulation to protect against uncontrolled infection. They interfere with viral replication by slowing cell metabolism, interfering with the membrane formation required for virus replication, and inducing secretion of cytokines that promote adaptive immunity[^10.1016/j.antiviral.2020.104791]. The importance of IFN is underscored by the fact that most viruses, and in particular coronaviruses, create many protein products specifically designed to antagonize cellular IFN signaling pathways[^10.1016/j.coviro.2012.04.004][^10.1016/j.smim.2019.101300]. While one might therefore expect that exogenously administered IFN would be a mainstay in the treatment of acute viral infections, this is surprisingly not the case.

There are three IFN families[^10.1038/nri3787]. Type I IFN includes 13 related IFN-α subtypes and a single IFN-β. Type II IFN consists of a single gene product IFN-γ. Type III IFN consists of IFN-λ1, IFN-λ2, and IFN-λ3. Several IFN-α products have been approved for the treatment of hepatitis B, hepatitis C, human papillomavirus, and several malignancies. IFN-β, IFN-γ, and IFN-λ have been approved for the treatment of multiple sclerosis, congenital immunodeficiency chronic granulomatous disease, and hepatitis delta infection, respectively, Commercial IFN-α products vary in the extent to which they include different IFN-α subtypes. The biological and clinical significance of the differences between the different IFN-α products are not known[^10.1016/j.smim.2019.101300]. Pegylated interferons have a longer half-life because the attached polyethylene glycol causes interferon to be cleared more slowly.

As of April 27, 2020, the Coronavirus Antiviral Research Database contains about 160 cell culture[^10.1016/s0140-6736(03)13615-x][^10.1089/1079990041535610][^10.1086/382597][^PMID15651759][^10.1016/j.bbrc.2004.08.085][^10.1016/j.virol.2004.08.011][^10.3201/eid1002.030482][^10.1016/j.bbrc.2004.06.076][^10.1080/00365540410021144][^10.1016/j.bbrc.2004.11.128][^10.1177/095632020601700505][^10.1128/JVI.02322-09][^10.1099/vir.0.052910-0][^10.1038/nm.3362][^10.1016/j.jinf.2013.09.029][^10.1099/vir.0.061911-0][^10.1016/j.antiviral.2018.05.007][^10.1016/j.antiviral.2017.01.012][^10.1101/2020.03.07.982264][^10.1101/2020.04.02.022764][^10.1038/s41467-019-13940-6] and 27 animal model experiments[^10.1038/nm1001][^10.1177/095632020601700505][^10.1016/j.antiviral.2010.11.007][^10.1038/nm.3362][^10.1093/infdis/jiv392][^10.1093/infdis/jiv325][^10.1172/JCI126363] reporting the inhibitory activity of different IFN preparations against SARS-CoV (100 results), MERS-CoV (49 results), SARS-CoV-2 (4 results), or another coronavirus species (35 results). IFN-α, IFN-β, and IFN-γ products were used in about 50%, 30%, and 10% of experiments, respectively. IFN-λ and IFN-β in combination with IFN-γ were used in the remaining 10% of experiments.

Both IFN-α and IFN-β appear to inhibit coronavirus _in vitro_ and in animal models, although several authors have argued that IFN-β is more effective than IFN-α at inhibiting coronaviruses[^10.1016/j.antiviral.2020.104791]. One study suggests that SARS-CoV-2 is more susceptible to IFN-α than SARS-CoV-1, due to a deletion in ORF3b, a 154 amino acid protein known to antagonize one of the proteins involved in the cellular IFN response[^10.1101/2020.03.07.982264].

As of May 7, 2020, there have been eight observational studies of IFN-α or IFN-β in patients with MERS-CoV (4 studies), SARS-CoV (2 studies), or SARS-CoV-2 (3 studies). One of the three SARS-CoV-2 was a randomized control trial of 127 patients with mild-to-moderate disease who were randomized 2:1 to receive IFN-β + ribavirin + lopinavir/r vs. lopinavir/r alone[^10.1016/S0140-6736(20)31042-4]. The patients receiving triple therapy had a significantly shorter time to a negative nasopharyngeal PCR (7 vs. 12 days) and a more rapid clinical improvement by several metrics. As ribavirin has generally not been efficacious in treating coronavirus infections, IFN-β is the most likely compound responsible for the improved outcome. The remaining clinical IFN studies did not contain control groups making it difficult to draw conclusions regarding clinical efficacy.

There are also 10 ongoing studies of subcutaneous or intranasal IFN-α or IFN-β, although nearly all of these studies contain multiple treatment arms and lack control groups. There are three planned studies of IFN-λ, including one with a placebo arm.

 Theoretical and empirical data suggest that IFNs should be administered as early as possible in the course of infection[^10.1172/JCI126363] when innate immunity is likely to be most needed; this would also avoid exacerbating the unregulated exuberant damaging immune response that often occurs in the later stages of severe coronavirus infections. Although far fewer data are available for IFN-λs, they have garnered much interest because they act at epithelial barriers and thus are theoretically optimal for treating viral respiratory tract infections and because they are reported to suppress viral spread without causing inflammation[^10.15252/emmm.201606413][^10.1093/cid/ciaa453].

___Conclusion: Considering the multiple proteins produced by coronaviruses to down regulate-host innate immunity and the generally favorable response to IFN in vitro and in animal models, rigorously designed studies of the early use of IFNs following SARS-CoV-2 infection are urgently needed to determine whether there already exists a highly effective SARS-CoV-2 treatment.___

## Additional Host-Targeting Compounds
With the exception of interferons, all approved antivirals target specific steps in a virus’s replication cycle. However, the concept of developing drugs that target host pathways has gained increasing interest during the past several years and has reached a fever pitch with the SARS-CoV-2 pandemic. Host-acting drugs have a number of theoretical advantages for treating acute pandemic infections. First, potential host acting antiviral drugs may have already been approved or undergone human testing for other conditions, making it possible to repurpose them to treat SARS-CoV-2. Second, host-acting drugs theoretically be able to inhibit multiple viruses, since many viruses depend on similar host factors and pathways. Third, although drugs targeting host factors may have a higher risk of cellular toxicity compared to drugs that specifically target a virus protein, such toxicity may be acceptable for the relatively few days required to treat an acute viral infection.

During the past 17 years, hundreds of compounds have been identified that appear to inhibit coronaviruses _in vitro_ by inhibiting a cellular protein, influencing a signaling pathway, or modifying a cellular organelle. For some of these compounds, it has been possible to determine their mechanism of action. However, it is difficult to pinpoint the mechanism of action for many other compounds. This section reviews compounds that appear to act primarily by influencing the host rather than the virus and that are described in multiple studies, demonstrate _in vitro_ potency, and have favorable safety profiles. These compounds have been categorized into those that (i) inhibit host proteases required for S protein cleavage; (ii) appear to inhibit virus endosomal trafficking; or (3) act by additional or uncertain mechanisms.

### Host Protease Inhibitors
The proteolytic activation of coronavirus spike proteins by host cell proteases is critical for the virus to transition from receptor attachment to cell fusion. Many host proteases have been shown to proteolytically process the spike (S) protein including cell surface proteases such as transmembrane protease/serine proteases (TMPRSSs), endosomal cathepsins, furin, and trypsin[^10.1016/j.virusres.2014.11.021]. Coronaviruses usually require that S is cleaved at two sites – the S1/S2 juncture and at a second site within S2 referred to as S2’. The specific enzymes involved depend in part on the virus species, the amino acid sequence at the S1/S2 boundary and at the S2’ site, and the infected cells.

In contrast to SARS-CoV, SARS-CoV-2 contains a functional poly-basic furin cleavage site at the S1/S2 boundary[^10.1016/j.cell.2020.02.058]. This observation and accumulating data support the tentative conclusion that furin and TMPRSS2 are likely to mediate cleavage at the S1/S2 boundary and at S2’, respectively, and thus are likely to be particularly important host proteases for SARS-CoV-2[^10.1016/j.cell.2020.02.052][^10.1101/2020.04.15.042085]. However, the endosomal cysteine proteases cathepsin B and L are considered essential host proteases for SARS-CoV[^10.1073/pnas.0505577102][^10.1128/JVI.02232-10][^10.1128/JVI.02062-10][^10.1128/JVI.00094-12][^10.1016/j.virol.2017.11.012][^10.1016/j.cell.2020.02.052][^10.1101/2020.04.15.042085] and, together with TMPRSS2, may also be involved in SARS-CoV-2 virus entry in certain cell types[^10.1101/2020.04.15.042085][^10.1101/2020.04.22.054981][^10.1038/s41467-020-15562-9].

Two serine protease inhibitors available for human use in Japan, camostat and nafamostat, are believed to inhibit coronaviruses primarily through their inhibition of TMPRSS2. Both drugs have potently inhibited MERS-CoV, SARS-CoV, and SARS-CoV-2 in pseudovirus entry inhibitor assays and in cell culture[^10.1016/j.cell.2020.02.052][^10.1128/AAC.00754-20][^10.1101/2020.04.22.054981][^10.1101/2020.04.03.024257]. Camostat has also protected mice from fatal SARS-CoV infection[^10.1016/j.antiviral.2015.01.011]. Both camostat and nafamostat appear to inhibit SARS-CoV-2 at lower concentrations than SARS-CoV[^10.1016/j.cell.2020.02.052][^10.1128/AAC.00754-20][^10.1101/2020.04.22.054981][^10.1101/2020.04.03.024257]. In Calu-3 lung epithelial cells, camostat inhibited SARS-CoV and SARS-CoV-2 with EC<sub>50</sub>s generally below 1 μM, while the EC<sub>50</sub>s for nafamostat have been about 10 to 100-fold lower. However, the EC<sub>50</sub>s for both drugs against both viruses are considerably higher in Vero cells [^10.1101/2020.03.20.999730][^10.1101/2020.04.22.054981][^10.1016/S0140-6736(20)31022-9] suggesting that TMPRSS2 inhibition alone may not be sufficient to suppress SARS-CoV-2 in these cells[^10.1101/2020.04.15.042085]. Two studies have reported that aprotonin, a third serine protease inhibitor available in Russia as an aerosol preparation displayed comparable inhibitory activity to camostat[^10.1101/2020.04.03.024257][^10.1101/2020.04.15.042085].

Several small molecule cathepsin inhibitors have also been shown to inhibit coronaviruses _in vitro_. K11777, an investigational inhibitor of Trypanosoma cruzi, the agent of Chagas disease, was found to inhibit a variety of cysteine proteases including human cathepsins[^10.1016/j.antiviral.2015.01.011]. It also potently inhibited SARS-CoV in cell culture and virus entry assays at sub-micromolar concentrations[^10.1016/j.antiviral.2015.01.011]. Several K11777 derivatives including SMDC256159 and SMDC256160 were also shown to be potent SARS-CoV inhibitors. K11777 and each of its derivatives synergistically inhibited SARS-CoV when administered in combination with camostat. MG132, MDL28170, and other calpain inhibitors have been found to inhibit SARS-CoV with EC<sub>50</sub>s in the low μM range[^10.1128/JVI.01001-12][^10.1073/pnas.0505577102][^10.1177/095632020601700505].

Several synthetic furin inhibitors have also shown to inhibit coronaviruses _in vitro_ including MI-1851, which inhibited SARS-CoV-2 alone and synergistically in combination with TMPRSS2 inhibitors[^10.1101/2020.04.15.042085], while dec-RVKR-CMK, which weakly inhibited MERS-CoV in a pseudovirus entry assay[^10.1073/pnas.1407087111].

___Conclusion: Clinical studies of camostat and nafamostat are just beginning but they are few in number These studies should be a high priority. It remains possible, however, that it may also be necessary to target furin and possibly cathepsins to inhibit SARS-CoV-2 in patients.___

### Endosomal Trafficking Inhibitors
Chloroquine and its analogs have been reported to have multiple intracellular actions that may interfere with virus replication and/or the immune response to viral infections. Chloroquine analogs are weak bases, that, in their non-protonated form, penetrate and concentrate within acidic intracellular organelles such as endosomes and lysosomes. In these organelles, chloroquine analogs may interfere with the trafficking of viruses that require a pH-dependent step for entry[^10.1016/s1473-3099(03)00806-5][^10.1371/journal.ppat.1004502]. Chloroquine analogs have also been reported to impair ACE2 glycosylation, which could theoretically diminish receptor binding by SARS-CoV and SARS-CoV-2[^10.1186/1743-422X-2-69]. They are also anti-inflammatory, and have been used for autoimmune conditions.

Chloroquine and hydroxychloroquine have demonstrated modest inhibitory activity against MERS-CoV[^10.1128/AAC.03036-14][^10.3390/v10060283][^10.1128/AAC.03011-14], SARS-CoV[^10.1186/1743-422X-2-69][^10.1177/095632020601700505][^10.1016/j.bbrc.2004.08.085][^10.1128/AAC.03011-14], and SARS-CoV-2[^10.1038/s41422-020-0282-0][^10.1038/s41421-020-0156-0][^10.1093/cid/ciaa237][^10.1101/2020.03.20.999730][^10.1101/2020.04.04.020925][^10.1101/2020.03.29.014407][^10.1101/2020.04.03.023846] primarily in Vero cells with most EC<sub>50</sub>s ranging between 1 μM and 10 μM. Chloroquine has also been shown to reduce mortality in a mouse model of HCoV-OC43 infection[^10.1128/AAC.01509-08][^10.1128/JVI.00023-19] but to have minimal if any effects on virus levels in a mouse SARS-CoV model[^10.1177/095632020601700505]. However, despite its _in vitro_ inhibitory activity against influenza and other viruses including dengue, large clinical trials of chloroquine have not shown any clinical benefit suggesting that the modest viral inhibition observed _in vitro_ may be cell culture artifacts[^10.1371/journal.pntd.0000785][^10.1016/S1473-3099(11)70065-2].

The pharmacokinetics of chloroquine and hydroxychloroquine are also complicated. Based on plasma levels alone, they appear to barely achieve sufficient inhibitory levels in patients receiving standard dosing regimens[^10.1111/j.1365-2125.1989.tb03439.x][^10.2165/00003088-199631040-00003][^10.1128/AAC.00339-08][^10.1002/cpt.1856][^10.1101/2020.04.16.20068379]. Mefloquine is an antimalarial that also weakly inhibits SARS-CoV, MERS-CoV, and SARS-CoV-2[^10.1128/AAC.03036-14][^10.1101/2020.03.20.999730] but achieves higher plasma levels than chloroquine or hydroxychloroquine[^10.1101/2020.04.16.20068379].

In a large observational study of 1,376 consecutive hospitalized patients of whom 58% received hydroxychloroquine, the risk of the primary endpoint, death or intubation, was compared with those patients not receiving hydroxychloroquine.  Overall 346 (25.1%) patients met the composite primary endpoint. In an unadjusted analysis that did not take into account disease severity, the risk of the primary endpoint was higher among those receiving hydroxychloroquine (HR:2.37; 95% CI: 0.82-1.32), although in the primary multivariable analysis, there was no difference between the two groups (HR:1.04;95%: 0.82-1.32)[^10.1056/NEJMoa2012410].

There have been at least 10 additional smaller studies including four open label randomized controlled studies and six observational studies. One observational study of 80 patients with mild SARS-CoV-2 infections reported more rapid virus clearance compared to historical controls in patients receiving hydroxychloroquine plus azithromycin[^10.1016/j.tmaid.2020.101663].  Of the remaining nine small studies, one observational study of 48 critically ill patients reported that hydroxychloroquine reduced mortality possibly through its anti-inflammatory effects as evidenced by reduced IL6 levels[^10.1101/2020.04.27.20073379] and another study of persons with mild-to-moderate disease reported that viral clearance was more rapid among patients receiving chloroquine compared with historical controls[^10.1101/2020.04.26.20081059]. None of the seven remaining studies, including the four controlled trials, reported benefit from the use of hydroxychloroquine or chloroquine; several highlighted the risk of ventricular arrythmias[^10.1101/2020.04.10.20060699][^10.1016/j.medmal.2020.03.006][^10.1101/2020.03.22.20040758][^10.1001/jamanetworkopen.2020.8857][^10.1101/2020.04.10.20060558][^10.3785/j.issn.1008-9292.2020.03.03]. A detailed critical review of a number of these studies has been published[^10.1093/ofid/ofaa130].

In addition, to the lack of sufficiently powered controlled trials, the interpretation of the clinical studies published to date has been complicated by several factors. First, hydroxychloroquine (which has been used more commonly than chloroquine) is administered with azithromycin in some studies but not others. The rationale for using azithromycin has been stated to possibly augment hydroxychloroquine's antiviral activity and/or to prevent bacterial pneumonia. However, each drug is associated with a risk of electrocardiographic QT prolongation and therefore potential fatal arrhythmias. Second, in most observational studies, those patients with more severe disease have been more likely to receive hydroxychloroquine (with or without azithromycin). Third, several studies have come from one research group whose objectivity and research methods have been questioned. The patients in their studies have typically had much milder disease than those reported in most other studies, which included primarily hospitalized patients. Based on the totality of evidence published so far, there does not appear to be strong evidence for the clinical efficacy of hydroxychloroquine or chloroquine.

Niclosamide is an oral anti-helminthic drug that has been used in millions of persons worldwide. It appears to exert it anti-helminthic effects by inhibiting oxidative phosphorylation and stimulating mitochondrial adenosine triphosphatase activity[^10.1021/acsinfecdis.0c00052]. In high-throughput drug repurposing screens, niclosamide has been found to have several additional biological effects that appear to be associated either with its effect on endosomal acidification or with its influence on signaling pathways[^10.1016/j.cellsig.2017.04.001]. Five research groups reported that niclosamide inhibits SARS-CoV[^10.1021/jm070295s][^10.1073/pnas.0403596101], MERS-CoV[^10.1101/2020.02.25.965582], or SARS-CoV-2[^10.1101/2020.04.15.997254][^10.1101/2020.03.20.999730] in cell culture with EC<sub>50</sub>s consistently below 1 μM. One study suggested that SARS-CoV-2 facilitates AKT1/SKPT-dependent degradation of autophagy-initiating Beclin-1 and that reversal of this effect by niclosamide and the AKT inhibitor MK-2206 markedly inhibits SARS-CoV-2[^10.1101/2020.04.15.997254]. Although niclosamide dosing has generally been designed to achieve high intestinal levels, a recent pharmacokinetic review suggested that even at current doses, its maximal plasma concentration may be sufficient to inhibit SARS-CoV-2 replication[^10.1101/2020.04.16.20068379].

Chlorpromazine is an antipsychotic drug used in treatment of schizophrenia that has been reported to inhibit several viruses by interfering with viral entry into cells by inhibiting clathrin-mediated endocytosis[^10.1038/nrd.2015.37][^10.7150/ijbs.45498]. Four studies have reported EC<sub>50</sub>s in the range of 1 to 14 μM for SARS-CoV and MERS-CoV[^10.1128/AAC.03036-14][^10.1128/AAC.03011-14][^10.1371/journal.pone.0194868][^10.1101/2020.03.25.008482].

K22 is a small molecule found to inhibit the double membrane formation and reticular vesicular network required for the replication of coronaviruses and other positive strand RNA viruses[^10.1371/journal.ppat.1004166][^10.1016/j.virusres.2018.01.002]. Although it has weak _in vitro_ activity, it is of interest because it appears to act at a novel stage of infection; furthermore, K22-resistant viruses containing nsp6 mutations were selected after 10 passages of HCoV-229E suggesting that this compound may also have specific antiviral activity[^10.1371/journal.ppat.1004166].

___Conclusion: The early hopes that chloroquine / hydroxychloroquine might be efficacious for treating SARS-CoV-2 no longer appear to be justified. These drugs are also associated with a high risk of toxicity especially when administered in combination with azithromycin. Many of the clinical trials containing these drugs are likely to be redesigned now that remdesivir appears to be efficacious. Among the endosomal trafficking inhibitors, niclosamide now appears more promising than hydroxychloroquine, although questions remain about its bioavailability. A phase I/II trial of niclosamide to evaluate its pharmacokinetics and potential antiviral activity would help determine whether this drug could be a safe and inexpensive oral SARS-CoV-2 treatment.___

### Miscellaneous Additional Host Targets
__Signaling pathway inhibitors__: High-throughput experiments have been performed for some viral infections to identify host proteins essential for virus infection but not cell viability. These have not yet been reported for coronavirus infections. However, several transcriptomic and proteomic analyses of SARS-CoV and MERS-CoV infection have described perturbations in cellular signaling pathways raising the possibility that drugs that influence the kinase proteins involved in these pathways might influence coronavirus replication[^10.1128/mBio.00165-13][^10.1128/mBio.00611-12][^10.1128/AAC.03659-14]. One encyclopedic proteomic SARS-CoV-2 study described the many interactions between virus and host proteins but was not primarily focused on signaling pathways[^10.1038/s41586-020-2286-9].

The kinase inhibitors dasatinib and imatinib were found to inhibit SARS-CoV and MERS-CoV with EC<sub>50</sub>s of 2.1 and 5.4 μM for dasatinib and 9.8 and 17.7 μM for imatinib[^10.1128/AAC.03036-14]. Further mechanistic studies demonstrated that imatinib inhibits the early stages of infection with SARS-CoV, MERS-CoV and other coronaviruses possibly by affecting viral fusion with endosomal membranes; inhibition was demonstrated to occur specifically through ABL kinase inhibition[^10.1128/JVI.01429-16][^10.1099/jgv.0.001047]. At least one pharmacokinetic study suggested, however, that standard doses of imatinib would be associated with plasma levels considerably lower than the EC<sub>50</sub>s reported for SARS-CoV and MERS-CoV[^10.1101/2020.04.16.20068379]. As of April 24, imatinib is listed in five ongoing or planned clinical trials including two placebo-controlled trials. It appears, however, that in these studies imatinib is being studied primarily for its role in treating pulmonary fibrosis. Another kinase inhibitor, saracatinib, has also recently been shown to inhibit several coronaviruses in the low μM range[^10.3390/v10060283]

__Nitazoxanide__: Nitazoxanide is licensed in the U.S. for the treatment of cryptosporidium infections. It has been reported to inhibit a wide range of viruses in vitro. Its potential mechanisms of antiviral activity are not known. It has been proposed to interfere with host regulated pathways involved in viral replication including those involving interferon or mTORC1[^10.1016/j.jiph.2016.04.001]. In one study its EC<sub>50</sub> for SARS-CoV-2 was 2.1 μM[^10.1038/s41422-020-0282-0]. One pharmacokinetic analysis suggested that nitazoxanide plasma levels could reach concentrations above its one reported EC<sub>50</sub> level[^10.1101/2020.04.16.20068379]. As of April 24, it is listed as being studied in six clinical trials, including two placebo-controlled studies.

__Ciclesonide__: Ciclesonide is an inhaled corticosteroid discovered in a high-throughput drug screen to inhibit coronavirus replication in the low μM range[^10.1101/2020.03.11.987016][^10.1101/2020.03.20.999730][^10.1101/2020.02.25.965582]. Although its mechanism of action is not known, virus passage in the presence of ciclesonide led to the selection of a ciclesonide-resistant mutant, characterized by a point mutation in the NSP15 gene. A single mutation in this gene was responsible for ~100-fold reduced susceptibility. A small case series of three patients from Japan who received inhaled ciclesonide has been published[^10.1016/j.jiac.2020.04.007], As of April 21, there is one ongoing and one planned study of inhaled ciclesonide in South Korea.

__Emetine__: Emetine is an FDA-approved drug for treating amebiasis. It is produced from the ipecac root and has been used to induce vomiting. Emetine has been shown in four studies to inhibit multiple coronaviruses including SARS-CoV, MERS-CoV, and SARS-CoV-2 _in vitro_, with EC<sub>50</sub>s consistently below 1.0 μM[^10.1128/AAC.03036-14][^10.1128/JVI.00023-19][^10.1101/2020.02.25.965582][^10.1016/j.antiviral.2020.104786]. Its mechanism of action is not known for certain, although it did inhibit MERS-CoV in an entry inhibitor assay using a pseudotype virus expressing S protein[^10.1128/JVI.00023-19]. After oral administration, its levels in the lungs are much higher than in the plasma and well above its reported EC<sub>50</sub> levels[^10.3390/ph13030051]. Emetine is currently not being studied for the treatment of SARS-CoV-2

__Interferon inducers__: Several compounds appear to inhibit viruses by inducing interferon. Ampligen® is a mismatched double-stranded RNA polymer that stimulates the production of interferon. It has been reported to have _in vitro_ activity against several viruses[^10.4049/jimmunol.178.8.5200] including SARS-CoV [^10.1177/095632020601700505]. It has also been reported to protect mice against SARS-CoV challenge in three studies[^10.1177/095632020601700505][^10.1016/j.virol.2009.09.023][^10.1016/j.antiviral.2016.12.007]. Hiltonol® is synthetic double-stranded polyriboinosinic-polyribocytidylic acid reported to protect mice from lethal SARS-CoV challenge when administered intranasally prior to infection[^10.1016/j.antiviral.2016.12.007]. Tilorone is a small molecule reported to inhibit MERS-CoV and other viruses _in vitro_ by inducing interferon[^10.1128/JVI.00023-19][^10.1128/AAC.00440-20].

__Cyclophilin inhibitors__: Cyclosporin and FK-506 are two immunosuppressive drugs commonly used to prevent rejection in transplant recipients. The discovery of these two compounds led to the identification of the family of cyclophilin host proteins that bind to cyclosporin and FK-506. Cyclophilins are involved in a wide range of cellular processes including protein folding, protein trafficking, and cell signaling. They have also been shown to play a role in the replication of several viruses including HIV-1, hepatitis C, influenza, and several coronavirus species[^10.1016/j.virol.2018.06.011]. Alisporivir is a non-immunosuppressive cyclophilin inhibitor which has been studied primarily for the treatment of hepatitis C virus. However, these compounds have generally had just weak inhibitory effects of on coronavirus replication and alisporivir did not protect mice from SARS-CoV infection[^10.1099/vir.0.034983-0][^10.1099/vir.0.052910-0][^10.1016/j.virusres.2016.11.011][^10.1016/j.virusres.2014.02.010][^10.1016/j.antiviral.2018.05.007][^10.1101/2020.03.20.999730].

__Ivermectin__: Ivermectin is one of an increasing number of additional compounds found to inhibit SARS-CoV-2 without a defined mechanism of action. It has a reported EC<sub>50</sub> of about 2 μM[^10.1016/j.antiviral.2020.104787]. Although its plasma levels following oral administration appear too low to inhibit coronavirus replication, its levels in lung tissue appear likely to be higher [^10.4269/ajtmh.20-0271][^10.1101/2020.04.16.20068379]. Ivermectin is included in four clinical trials – each containing multiple treatment arms.

<!-- start of references details -->

[^10.1101/2020.04.29.069054]: Korber B, Fischer W, et int., Group SCG. "Spike mutation pipeline reveals the emergence of a more transmissible form of SARS-CoV-2" _bioRxiv_, 2020. [10.1101/2020.04.29.069054](https://doi.org/10.1101/2020.04.29.069054)
[^10.1007/978-1-4939-2438-7_1]: Fehr AR, Perlman S. "Coronaviruses: an overview of their replication and pathogenesis" _Methods Mol. Biol._, 2015. [10.1007/978-1-4939-2438-7_1](https://doi.org/10.1007/978-1-4939-2438-7_1)
[^10.1038/s41586-020-2286-9]: Gordon DE, Jang GM, et int., Krogan NJ. "A SARS-CoV-2 protein interaction map reveals targets for drug repurposing" _Nature_, 2020. [10.1038/s41586-020-2286-9](https://doi.org/10.1038/s41586-020-2286-9)
[^10.3390/v11080745]: Peersen OB. "A Comprehensive Superposition of Viral Polymerase Structures" _Viruses_, 2019. [10.3390/v11080745](https://doi.org/10.3390/v11080745)
[^10.1073/pnas.0508200103]: Minskaia E, Hertzig T, et int., Ziebuhr J. "Discovery of an RNA virus 3'->5' exoribonuclease that is critically involved in coronavirus RNA synthesis" _Proc. Natl. Acad. Sci. U.S.A._, 2006. [10.1073/pnas.0508200103](https://doi.org/10.1073/pnas.0508200103)
[^10.1371/journal.ppat.1003565]: Smith EC, Blanc H, et int., Denison MR. "Coronaviruses lacking exoribonuclease activity are susceptible to lethal mutagenesis: evidence for proofreading and potential therapeutics" _PLoS Pathog._, 2013. [10.1371/journal.ppat.1003565](https://doi.org/10.1371/journal.ppat.1003565)
[^10.1073/pnas.1718806115]: Ferron F, Subissi L, et int., Imbert I. "Structural and molecular basis of mismatch correction and ribavirin excision from coronavirus RNA" _Proc. Natl. Acad. Sci. U.S.A._, 2018. [10.1073/pnas.1718806115](https://doi.org/10.1073/pnas.1718806115)
[^10.1038/s41467-019-10280-3]: Kirchdoerfer RN, Ward AB. "Structure of the SARS-CoV nsp12 polymerase bound to nsp7 and nsp8 co-factors" _Nat Commun_, 2019. [10.1038/s41467-019-10280-3](https://doi.org/10.1038/s41467-019-10280-3)
[^10.1126/science.abb7498]: Gao Y, Yan L, et int., Rao Z. "Structure of the RNA-dependent RNA polymerase from COVID-19 virus" _Science_, 2020. [10.1126/science.abb7498](https://doi.org/10.1126/science.abb7498)
[^10.1126/science.abc1560]: Yin W, Mao C, et int., Xu HE. "Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir" _Science_, 2020. [10.1126/science.abc1560](https://doi.org/10.1126/science.abc1560)
[^10.1021/acs.jmedchem.6b01594]: Siegel D, Hui HC, et int., Mackman RL. "Discovery and Synthesis of a Phosphoramidate Prodrug of a Pyrrolo[2,1-f][triazin-4-amino] Adenine C-Nucleoside (GS-5734) for the Treatment of Ebola and Emerging Viruses" _J. Med. Chem._, 2017. [10.1021/acs.jmedchem.6b01594](https://doi.org/10.1021/acs.jmedchem.6b01594)
[^10.1038/nature17180]: Warren TK, Jordan R, et int., Bavari S. "Therapeutic efficacy of the small molecule GS-5734 against Ebola virus in rhesus monkeys" _Nature_, 2016. [10.1038/nature17180](https://doi.org/10.1038/nature17180)
[^10.1074/jbc.AC120.013056]: Gordon CJ, Tchesnokov EP, et int., Gotte M. "The antiviral compound remdesivir potently inhibits RNA-dependent RNA polymerase from Middle East respiratory syndrome coronavirus" _J. Biol. Chem._, 2020. [10.1074/jbc.AC120.013056](https://doi.org/10.1074/jbc.AC120.013056)
[^10.1074/jbc.RA120.013679]: Gordon CJ, Tchesnokov EP, et int., Gotte M. "Remdesivir is a direct-acting antiviral that inhibits RNA-dependent RNA polymerase from severe acute respiratory syndrome coronavirus 2 with high potency" _J. Biol. Chem._, 2020. [10.1074/jbc.RA120.013679](https://doi.org/10.1074/jbc.RA120.013679)
[^10.1016/j.antiviral.2020.104793]: Shannon A, Tuyet Le NT, et int., Canard B. "Remdesivir and SARS-CoV-2: structural requirements at both nsp12 RdRp and nsp14 Exonuclease active-sites" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104793](https://doi.org/10.1016/j.antiviral.2020.104793)
[^10.1126/scitranslmed.aal3653]: Sheahan TP, Sims AC, et int., Baric RS. "Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses" _Sci Transl Med_, 2017. [10.1126/scitranslmed.aal3653](https://doi.org/10.1126/scitranslmed.aal3653)
[^10.1128/mBio.00221-18]: Agostini ML, Andres EL, et int., Denison MR. "Coronavirus Susceptibility to the Antiviral Remdesivir (GS-5734) Is Mediated by the Viral Polymerase and the Proofreading Exoribonuclease" _mBio_, 2018. [10.1128/mBio.00221-18](https://doi.org/10.1128/mBio.00221-18)
[^10.1038/s41467-019-13940-6]: Sheahan TP, Sims AC, et int., Baric RS. "Comparative therapeutic efficacy of remdesivir and combination lopinavir, ritonavir, and interferon beta against MERS-CoV" _Nat Commun_, 2020. [10.1038/s41467-019-13940-6](https://doi.org/10.1038/s41467-019-13940-6)
[^10.1038/s41422-020-0282-0]: Wang M, Cao R, et int., Xiao G. "Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro" _Cell Res._, 2020. [10.1038/s41422-020-0282-0](https://doi.org/10.1038/s41422-020-0282-0)
[^10.1101/2020.04.03.024257]: Bojkova D, McGreig JE, et int., Cinatl JN. "SARS-CoV-2 and SARS-CoV differ in their cell tropism and drug sensitivity profiles" _bioRxiv_, 2020. [10.1101/2020.04.03.024257](https://doi.org/10.1101/2020.04.03.024257)
[^10.1016/j.antiviral.2020.104786]: Choy K, Yin-Lam Wong A, et int., Yen H. "Remdesivir, lopinavir, emetine, and homoharringtonine inhibit SARS-CoV-2 replication in vitro" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104786](https://doi.org/10.1016/j.antiviral.2020.104786)
[^10.1101/2020.03.20.999730]: Jeon S, Ko M, et int., Kim S. "Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs" _bioRxiv_, 2020. [10.1101/2020.03.20.999730](https://doi.org/10.1101/2020.03.20.999730)
[^10.1101/2020.04.03.023846]: Touret F, Gilles M, et int., Coutard B. "In vitro screening of a FDA approved chemical library reveals potential inhibitors of SARS-CoV-2 replication" _bioRxiv_, 2020. [10.1101/2020.04.03.023846](https://doi.org/10.1101/2020.04.03.023846)
[^10.1101/2020.04.03.20052548]: Meyer SD, Bojkova D, et int., Ciesek S. "Lack of Antiviral Activity of Darunavir against SARS-CoV-2" _medRxiv_, 2020. [10.1101/2020.04.03.20052548](https://doi.org/10.1101/2020.04.03.20052548)
[^10.1101/2020.04.27.064279]: Pruijssers AJ, George AS, et int., Sheahan TP. "Remdesivir potently inhibits SARS-CoV-2 in human lung cells and chimeric SARS-CoV expressing the SARS-CoV-2 RNA polymerase in mice." _bioRxiv_, 2020. [10.1101/2020.04.27.064279](https://doi.org/10.1101/2020.04.27.064279)
[^10.1073/pnas.1922083117]: de Wit E, Feldmann F, et int., Feldmann H. "Prophylactic and therapeutic remdesivir (GS-5734) treatment in the rhesus macaque model of MERS-CoV infection" _Proc. Natl. Acad. Sci. U.S.A._, 2020. [10.1073/pnas.1922083117](https://doi.org/10.1073/pnas.1922083117)
[^10.1101/2020.04.15.043166]: Williamson B, Feldmann F, et int., Wit ED. "Clinical benefit of remdesivir in rhesus macaques infected with SARS-CoV-2" _bioRxiv_, 2020. [10.1101/2020.04.15.043166](https://doi.org/10.1101/2020.04.15.043166)
[^10.1002/phar.2398]: Barlow A, Landolf KM, et int., Heavner MS. "Review of Emerging Pharmacotherapy for the Treatment of Coronavirus Disease 2019" _Pharmacotherapy_, 2020. [10.1002/phar.2398](https://doi.org/10.1002/phar.2398)
[^10.1016/j.onehlt.2020.100128]: Amirian ES, Levy JK. "Current knowledge about the antivirals remdesivir (GS-5734) and GS-441524 as therapeutic options for coronaviruses" _One Health_, 2020. [10.1016/j.onehlt.2020.100128](https://doi.org/10.1016/j.onehlt.2020.100128)
[^10.1101/2020.04.16.20068379]: Arshad U, Pertinez H, et int., Owen A. "Prioritisation of potential anti-SARS-CoV-2 drug repurposing opportunities based on ability to achieve adequate target site concentrations derived from their established human pharmacokinetics" _medRxiv_, 2020. [10.1101/2020.04.16.20068379](https://doi.org/10.1101/2020.04.16.20068379)
[^10.1056/NEJMoa2007016]: Grein J, Ohmagari N, et int., Flanigan T. "Compassionate Use of Remdesivir for Patients with Severe Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2007016](https://doi.org/10.1056/NEJMoa2007016)
[^10.1016/S0140-6736(20)31022-9]: Wang Y, Zhang D, et int., Wang C. "Remdesivir in adults with severe COVID-19: a randomised, double-blind, placebo-controlled, multicentre trial" _The Lancet_, 2020. [10.1016/S0140-6736(20)31022-9](https://doi.org/10.1016/S0140-6736(20)31022-9)
[^10.1016/j.coviro.2019.04.002]: Pruijssers AJ, Denison MR. "Nucleoside analogues for the treatment of coronavirus infections" _Curr Opin Virol_, 2019. [10.1016/j.coviro.2019.04.002](https://doi.org/10.1016/j.coviro.2019.04.002)
[^10.1177/095632020401500102]: Barnard DL, Hubbard VD, et int., Sidwell RW. "Inhibition of severe acute respiratory syndrome-associated coronavirus (SARSCoV) by calpain inhibitors and beta-D-N4-hydroxycytidine" _Antivir. Chem. Chemother._, 2004. [10.1177/095632020401500102](https://doi.org/10.1177/095632020401500102)
[^10.1177/095632020601700505]: Barnard DL, Day CW, et int., Sidwell RW. "Evaluation of immunomodulators, interferons and known in vitro SARS-coV inhibitors for inhibition of SARS-coV replication in BALB/c mice" _Antivir. Chem. Chemother._, 2006. [10.1177/095632020601700505](https://doi.org/10.1177/095632020601700505)
[^10.1128/JVI.01348-19]: Agostini ML, Pruijssers AJ, et int., Denison MR. "Small-Molecule Antiviral β-d-N4-Hydroxycytidine Inhibits a Proofreading-Intact Coronavirus with a High Genetic Barrier to Resistance" _J. Virol._, 2019. [10.1128/JVI.01348-19](https://doi.org/10.1128/JVI.01348-19)
[^10.1126/scitranslmed.abb5883]: Sheahan TP, Sims AC, et int., Baric RS. "An orally bioavailable broad-spectrum antiviral inhibits SARS-CoV-2 in human airway epithelial cell cultures and multiple coronaviruses in mice" _Sci Transl Med_, 2020. [10.1126/scitranslmed.abb5883](https://doi.org/10.1126/scitranslmed.abb5883)
[^10.1128/AAC.01719-19]: Sticher ZM, Lu G, et int., Kolykhalov AA. "Analysis of the Potential for N4-Hydroxycytidine To Inhibit Mitochondrial Replication and Function" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.01719-19](https://doi.org/10.1128/AAC.01719-19)
[^10.1126/scitranslmed.aax5866]: Toots M, Yoon J, et int., Plemper RK. "Characterization of orally efficacious influenza drug with high resistance barrier in ferrets and human airway epithelia" _Sci Transl Med_, 2019. [10.1126/scitranslmed.aax5866](https://doi.org/10.1126/scitranslmed.aax5866)
[^10.1016/s0140-6736(03)13615-x]: Cinatl J, Morgenstern B, et int., Doerr HW. "Glycyrrhizin, an active component of liquorice roots, and replication of SARS-associated coronavirus" _Lancet_, 2003. [10.1016/s0140-6736(03)13615-x](https://doi.org/10.1016/s0140-6736(03)13615-x)
[^10.1016/j.jinf.2013.09.029]: Chan JFW, Chan K, et int., Yuen K. "Broad-spectrum antivirals for the emerging Middle East respiratory syndrome coronavirus" _J. Infect._, 2013. [10.1016/j.jinf.2013.09.029](https://doi.org/10.1016/j.jinf.2013.09.029)
[^10.1099/vir.0.061911-0]: Hart BJ, Dyall J, et int., Hensley L. "Interferon-β and mycophenolic acid are potent inhibitors of Middle East respiratory syndrome coronavirus in cell-based assays" _J. Gen. Virol._, 2014. [10.1099/vir.0.061911-0](https://doi.org/10.1099/vir.0.061911-0)
[^10.1038/srep01686]: Falzarano D, de Wit E, et int., Feldmann H. "Inhibition of novel β coronavirus replication by a combination of interferon-α2b and ribavirin" _Sci Rep_, 2013. [10.1038/srep01686](https://doi.org/10.1038/srep01686)
[^10.1371/journal.pmed.0030343]: Stockman LJ, Bellamy R, Garner P. "SARS: systematic review of treatment effects" _PLoS Med._, 2006. [10.1371/journal.pmed.0030343](https://doi.org/10.1371/journal.pmed.0030343)
[^10.1016/j.antiviral.2013.09.015]: Furuta Y, Gowen BB, et int., Barnard DL. "Favipiravir (T-705), a novel viral RNA polymerase inhibitor" _Antiviral Res._, 2013. [10.1016/j.antiviral.2013.09.015](https://doi.org/10.1016/j.antiviral.2013.09.015)
[^10.1016/j.antiviral.2018.03.003]: Delang L, Abdelnabi R, Neyts J. "Favipiravir as a potential countermeasure against neglected and emerging RNA viruses" _Antiviral Res._, 2018. [10.1016/j.antiviral.2018.03.003](https://doi.org/10.1016/j.antiviral.2018.03.003)
[^10.1002/cpt.1844]: Du Y, Chen X. "Favipiravir: pharmacokinetics and concerns about clinical trials for 2019-nCoV infection" _Clin. Pharmacol. Ther._, 2020. [10.1002/cpt.1844](https://doi.org/10.1002/cpt.1844)
[^10.1101/2020.03.17.20037432]: Chen C, Huang J, et int., Wang X. "Favipiravir versus Arbidol for COVID-19: A Randomized Clinical Trial" _MedRxiv_, 2020. [10.1101/2020.03.17.20037432](https://doi.org/10.1101/2020.03.17.20037432)
[^10.1038/nature13027]: Warren TK, Wells J, et int., Bavari S. "Protection against filovirus diseases by a novel broad-spectrum nucleoside analogue BCX4430" _Nature_, 2014. [10.1038/nature13027](https://doi.org/10.1038/nature13027)
[^10.1016/j.antiviral.2005.01.003]: Saijo M, Morikawa S, et int., Kurane I. "Inhibitory effect of mizoribine and ribavirin on the replication of severe acute respiratory syndrome (SARS)-associated coronavirus" _Antiviral Res._, 2005. [10.1016/j.antiviral.2005.01.003](https://doi.org/10.1016/j.antiviral.2005.01.003)
[^10.1128/JVI.00023-19]: Shen L, Niu J, et int., Tan W. "High-Throughput Screening and Identification of Potent Broad-Spectrum Inhibitors of Coronaviruses" _J. Virol._, 2019. [10.1128/JVI.00023-19](https://doi.org/10.1128/JVI.00023-19)
[^10.1101/2020.04.07.028589]: Bukreyeva N, Mantlo EK, et int., Zeldis J. "The IMPDH inhibitor merimepodib suppresses SARS-CoV-2 replication in vitro." _bioRxiv_, 2020. [10.1101/2020.04.07.028589](https://doi.org/10.1101/2020.04.07.028589)
[^10.1002/cmdc.202000223]: Ghosh AK, Brindisi M, et int., Mesecar AD. "Drug Development and Medicinal Chemistry Efforts Toward SARS-Coronavirus and Covid-19 Therapeutics" _ChemMedChem_, 2020. [10.1002/cmdc.202000223](https://doi.org/10.1002/cmdc.202000223)
[^10.1126/science.1085658]: Anand K, Ziebuhr J, et int., Hilgenfeld R. "Coronavirus main proteinase (3CLpro) structure: basis for design of anti-SARS drugs" _Science_, 2003. [10.1126/science.1085658](https://doi.org/10.1126/science.1085658)
[^10.1021/jm401712t]: Báez-Santos YM, Barraza SJ, et int., Mesecar AD. "X-ray structural and biological evaluation of a series of potent and highly selective inhibitors of human coronavirus papain-like proteases" _J. Med. Chem._, 2014. [10.1021/jm401712t](https://doi.org/10.1021/jm401712t)
[^10.1038/s41586-020-2223-y]: Jin Z, Du X, et int., Yang H. "Structure of Mpro from COVID-19 virus and discovery of its inhibitors" _Nature_, 2020. [10.1038/s41586-020-2223-y](https://doi.org/10.1038/s41586-020-2223-y)
[^10.1126/science.abb3405]: Zhang L, Lin D, et int., Hilgenfeld R. "Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors" _Science_, 2020. [10.1126/science.abb3405](https://doi.org/10.1126/science.abb3405)
[^PMID14660806]: Chan KS, Lai ST, et int., Yuen KY. "Treatment of severe acute respiratory syndrome with lopinavir/ritonavir: a multicentre retrospective matched cohort study" _Hong Kong Med J_, 2003. [PMID14660806](https://pubmed.ncbi.nlm.nih.gov/14660806)
[^10.1136/thorax.2003.012658]: Chu CM, Cheng VCC, et int., HKU/UCH SARS Study Group. "Role of lopinavir/ritonavir in the treatment of SARS: initial virological and clinical findings" _Thorax_, 2004. [10.1136/thorax.2003.012658](https://doi.org/10.1136/thorax.2003.012658)
[^10.1016/j.jcv.2004.03.003]: Chen F, Chan KH, et int., Yuen KY. "In vitro susceptibility of 10 clinical isolates of SARS coronavirus to selected antiviral compounds" _J. Clin. Virol._, 2004. [10.1016/j.jcv.2004.03.003](https://doi.org/10.1016/j.jcv.2004.03.003)
[^10.1016/j.bbrc.2004.04.083]: Yamamoto N, Yang R, et int., Yamamoto N. "HIV protease inhibitor nelfinavir inhibits replication of SARS-associated coronavirus" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.04.083](https://doi.org/10.1016/j.bbrc.2004.04.083)
[^10.1128/AAC.03011-14]: de Wilde AH, Jochmans D, et int., Snijder EJ. "Screening of an FDA-approved compound library identifies four small-molecule inhibitors of Middle East respiratory syndrome coronavirus replication in cell culture" _Antimicrob. Agents Chemother._, 2014. [10.1128/AAC.03011-14](https://doi.org/10.1128/AAC.03011-14)
[^10.1101/2020.04.06.026476]: Yamamoto N, Matsuyama S, et int., Yamamoto N. "Nelfinavir inhibits replication of severe acute respiratory syndrome coronavirus 2 in vitro." _bioRxiv_, 2020. [10.1101/2020.04.06.026476](https://doi.org/10.1101/2020.04.06.026476)
[^10.1101/2020.04.29.067983]: Liu S, Lien C, et int., Wang T. "Evaluation of 19 antiviral drugs against SARS-CoV-2 Infection" _bioRxiv_, 2020. [10.1101/2020.04.29.067983](https://doi.org/10.1101/2020.04.29.067983)
[^10.1073/pnas.0403596101]: Wu C, Jan J, et int., Wong C. "Small molecules targeting severe acute respiratory syndrome human coronavirus" _Proceedings of the National Academy of Sciences_, 2004. [10.1073/pnas.0403596101](https://doi.org/10.1073/pnas.0403596101)
[^10.1101/2020.04.20.051581]: Ma C, Hurst B, et int., Wang J. "Boceprevir, GC-376, and calpain inhibitors II, XII inhibit SARS-CoV-2 viral replication by targeting the viral main protease" _bioRxiv_, 2020. [10.1101/2020.04.20.051581](https://doi.org/10.1101/2020.04.20.051581)
[^10.1093/infdis/jiv392]: Chan JF, Yao Y, et int., Yuen K. "Treatment With Lopinavir/Ritonavir or Interferon-β1b Improves Outcome of MERS-CoV Infection in a Nonhuman Primate Model of Common Marmoset" _J. Infect. Dis._, 2015. [10.1093/infdis/jiv392](https://doi.org/10.1093/infdis/jiv392)
[^10.1056/NEJMoa2001282]: Cao B, Wang Y, et int., Wang C. "A Trial of Lopinavir–Ritonavir in Adults Hospitalized with Severe Covid-19" _NEJM_, 2020. [10.1056/NEJMoa2001282](https://doi.org/10.1056/NEJMoa2001282)
[^10.1101/2020.02.25.965582]: Ko M, Chang SY, et int., Windisch MP. "Screening of FDA-approved drugs using a MERS-CoV clinical isolate from South Korea identifies potential therapeutic options for COVID-19" _bioRxiv_, 2020. [10.1101/2020.02.25.965582](https://doi.org/10.1101/2020.02.25.965582)
[^10.1016/j.bbrc.2005.05.095]: Liu Y, Huang V, et int., Chow L. "Screening of drugs by FRET analysis identifies inhibitors of SARS-CoV 3CL protease" _Biochem. Biophys. Res. Commun._, 2005. [10.1016/j.bbrc.2005.05.095](https://doi.org/10.1016/j.bbrc.2005.05.095)
[^10.1101/2020.04.04.020925]: Fintelman-Rodrigues N, Sacramento CQ, et int., Souza TML. "Atazanavir inhibits SARS-CoV-2 replication and pro-inflammatory cytokine production" _bioRxiv_, 2020. [10.1101/2020.04.04.020925](https://doi.org/10.1101/2020.04.04.020925)
[^10.1016/j.virusres.2007.02.015]: Grum-Tokars V, Ratia K, et int., Mesecar AD. "Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery" _Virus Res._, 2008. [10.1016/j.virusres.2007.02.015](https://doi.org/10.1016/j.virusres.2007.02.015)
[^10.1021/jm050184y]: Shie J, Fang J, et int., Wong C. "Discovery of potent anilide inhibitors against the severe acute respiratory syndrome 3CL protease" _J. Med. Chem._, 2005. [10.1021/jm050184y](https://doi.org/10.1021/jm050184y)
[^10.1016/j.bmc.2016.05.013]: Kumar V, Tan K, et int., Liang P. "Identification, synthesis and evaluation of SARS-CoV and MERS-CoV 3C-like protease inhibitors" _Bioorg. Med. Chem._, 2016. [10.1016/j.bmc.2016.05.013](https://doi.org/10.1016/j.bmc.2016.05.013)
[^10.1021/acs.jmedchem.9b01828]: Zhang L, Lin D, et int., Hilgenfeld R. "α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment" _J. Med. Chem._, 2020. [10.1021/acs.jmedchem.9b01828](https://doi.org/10.1021/acs.jmedchem.9b01828)
[^10.1128/JVI.01348-12]: Kim Y, Lovell S, et int., Chang K. "Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses" _J. Virol._, 2012. [10.1128/JVI.01348-12](https://doi.org/10.1128/JVI.01348-12)
[^10.1371/journal.ppat.1005531]: Kim Y, Liu H, et int., Pedersen NC. "Reversal of the Progression of Fatal Coronavirus Infection in Cats by a Broad-Spectrum Coronavirus Protease Inhibitor" _PLoS Pathog._, 2016. [10.1371/journal.ppat.1005531](https://doi.org/10.1371/journal.ppat.1005531)
[^10.1016/j.ejmech.2018.03.004]: Galasiti Kankanamalage AC, Kim Y, et int., Groutas WC. "Structure-guided design of potent and permeable inhibitors of MERS coronavirus 3CL protease that utilize a piperidine moiety as a novel design element" _Eur J Med Chem_, 2018. [10.1016/j.ejmech.2018.03.004](https://doi.org/10.1016/j.ejmech.2018.03.004)
[^10.1016/j.ejmech.2017.05.045]: Wang L, Bao B, et int., Wang J. "Discovery of unsymmetrical aromatic disulfides as novel inhibitors of SARS-CoV main protease: Chemical synthesis, biological evaluation, molecular docking and 3D-QSAR study" _Eur J Med Chem_, 2017. [10.1016/j.ejmech.2017.05.045](https://doi.org/10.1016/j.ejmech.2017.05.045)
[^10.1016/j.bmcl.2007.08.031]: Ghosh AK, Xi K, et int., Mesecar AD. "Structure-based design, synthesis, and biological evaluation of peptidomimetic SARS-CoV 3CLpro inhibitors" _Bioorg. Med. Chem. Lett._, 2007. [10.1016/j.bmcl.2007.08.031](https://doi.org/10.1016/j.bmcl.2007.08.031)
[^10.1126/science.abb4489]: Dai W, Zhang B, et int., Liu H. "Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease" _Science_, 2020. [10.1126/science.abb4489](https://doi.org/10.1126/science.abb4489)
[^10.1016/j.antiviral.2017.12.015]: Lin M, Moses DC, et int., Chou C. "Disulfiram can inhibit MERS and SARS coronavirus papain-like proteases via different modes" _Antiviral Res._, 2018. [10.1016/j.antiviral.2017.12.015](https://doi.org/10.1016/j.antiviral.2017.12.015)
[^10.1073/pnas.0510851103]: Ratia K, Saikatendu KS, et int., Mesecar AD. "Severe acute respiratory syndrome coronavirus papain-like protease: structure of a viral deubiquitinating enzyme" _Proc. Natl. Acad. Sci. U.S.A._, 2006. [10.1073/pnas.0510851103](https://doi.org/10.1073/pnas.0510851103)
[^10.1016/j.antiviral.2014.12.015]: Báez-Santos YM, St John SE, Mesecar AD. "The SARS-coronavirus papain-like protease: structure, function and inhibition by designed antiviral compounds" _Antiviral Res._, 2015. [10.1016/j.antiviral.2014.12.015](https://doi.org/10.1016/j.antiviral.2014.12.015)
[^10.1016/bs.aivir.2019.08.002]: Tortorici MA, Veesler D. "Structural insights into coronavirus entry" _Adv. Virus Res._, 2019. [10.1016/bs.aivir.2019.08.002](https://doi.org/10.1016/bs.aivir.2019.08.002)
[^10.1126/science.abb2507]: Wrapp D, Wang N, et int., McLellan JS. "Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation" _Science_, 2020. [10.1126/science.abb2507](https://doi.org/10.1126/science.abb2507)
[^10.1038/nature02145]: Li W, Moore MJ, et int., Farzan M. "Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus" _Nature_, 2003. [10.1038/nature02145](https://doi.org/10.1038/nature02145)
[^10.1038/s41586-020-2012-7]: Zhou P, Yang X, et int., Shi Z. "A pneumonia outbreak associated with a new coronavirus of probable bat origin" _Nature_, 2020. [10.1038/s41586-020-2012-7](https://doi.org/10.1038/s41586-020-2012-7)
[^10.1074/jbc.C300520200]: Wong SK, Li W, et int., Farzan M. "A 193-amino acid fragment of the SARS coronavirus S protein efficiently binds angiotensin-converting enzyme 2" _J. Biol. Chem._, 2004. [10.1074/jbc.C300520200](https://doi.org/10.1074/jbc.C300520200)
[^10.1038/s41586-020-2180-5]: Lan J, Ge J, et int., Wang X. "Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor" _Nature_, 2020. [10.1038/s41586-020-2180-5](https://doi.org/10.1038/s41586-020-2180-5)
[^10.1007/s11427-020-1637-5]: Xu X, Chen P, et int., Hao P. "Evolution of the novel coronavirus from the ongoing Wuhan outbreak and modeling of its spike protein for risk of human transmission" _Sci. China Life Sci._, 2020. [10.1007/s11427-020-1637-5](https://doi.org/10.1007/s11427-020-1637-5)
[^10.1126/science.1116480]: Li F, Li W, et int., Harrison SC. "Structure of SARS coronavirus spike receptor-binding domain complexed with receptor" _Science_, 2005. [10.1126/science.1116480](https://doi.org/10.1126/science.1116480)
[^10.1016/j.virusres.2014.11.021]: Millet JK, Whittaker GR. "Host cell proteases: Critical determinants of coronavirus tropism and pathogenesis" _Virus Res._, 2015. [10.1016/j.virusres.2014.11.021](https://doi.org/10.1016/j.virusres.2014.11.021)
[^10.1016/j.cell.2020.02.058]: Walls AC, Park Y, et int., Veesler D. "Structure, Function, and Antigenicity of the SARS-CoV-2 Spike Glycoprotein" _Cell_, 2020. [10.1016/j.cell.2020.02.058](https://doi.org/10.1016/j.cell.2020.02.058)
[^10.1016/j.antiviral.2020.104742]: Coutard B, Valle C, et int., Decroly E. "The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade" _Antiviral Research_, 2020. [10.1016/j.antiviral.2020.104742](https://doi.org/10.1016/j.antiviral.2020.104742)
[^10.1093/infdis/jiu396]: Mair-Jenkins J, Saavedra-Campos M, et int., Convalescent Plasma Study Group. "The effectiveness of convalescent plasma and hyperimmune immunoglobulin for the treatment of severe acute respiratory infections of viral etiology: a systematic review and exploratory meta-analysis" _J. Infect. Dis._, 2015. [10.1093/infdis/jiu396](https://doi.org/10.1093/infdis/jiu396)
[^10.1073/pnas.2004168117]: Duan K, Liu B, et int., Yang X. "Effectiveness of convalescent plasma therapy in severe COVID-19 patients" _Proc. Natl. Acad. Sci. U.S.A._, 2020. [10.1073/pnas.2004168117](https://doi.org/10.1073/pnas.2004168117)
[^10.1001/jama.2020.4783]: Shen C, Wang Z, et int., Liu L. "Treatment of 5 Critically Ill Patients With COVID-19 With Convalescent Plasma" _JAMA_, 2020. [10.1001/jama.2020.4783](https://doi.org/10.1001/jama.2020.4783)
[^10.1016/j.chest.2020.03.039]: Zhang B, Liu S, et int., Zhang S. "Treatment with convalescent plasma for critically ill patients with SARS-CoV-2 infection" _Chest_, 2020. [10.1016/j.chest.2020.03.039](https://doi.org/10.1016/j.chest.2020.03.039)
[^10.1016/j.tmrv.2020.04.001]: Sullivan HC, Roback JD. "Convalescent Plasma: Therapeutic Hope or Hopeless Strategy in the SARS-CoV-2 Pandemic" _Transfus Med Rev_, 2020. [10.1016/j.tmrv.2020.04.001](https://doi.org/10.1016/j.tmrv.2020.04.001)
[^10.1172/JCI138745]: Bloch EM, Shoham S, et int., Tobian AA. "Deployment of convalescent plasma for the prevention and treatment of COVID-19" _J. Clin. Invest._, 2020. [10.1172/JCI138745](https://doi.org/10.1172/JCI138745)
[^10.1007/s11684-017-0596-6]: Jin Y, Lei C, et int., Ying T. "Human monoclonal antibodies as candidate therapeutics against emerging viruses" _Front Med_, 2017. [10.1007/s11684-017-0596-6](https://doi.org/10.1007/s11684-017-0596-6)
[^10.1172/JCI138003]: Casadevall A, Pirofski L. "The convalescent sera option for containing COVID-19" _J Clin Invest_, 2020. [10.1172/JCI138003](https://doi.org/10.1172/JCI138003)
[^10.1101/2020.05.03.20084442]: Suthar MS, Zimmerman M, et int., Wrammert J. "Rapid generation of neutralizing antibody responses in COVID-19 patients" _medRxiv_, 2020. [10.1101/2020.05.03.20084442](https://doi.org/10.1101/2020.05.03.20084442)
[^10.1126/science.aad5224]: Corti D, Misasi J, et int., Sullivan NJ. "Protective monotherapy against lethal Ebola virus infection by a potently neutralizing antibody" _Science_, 2016. [10.1126/science.aad5224](https://doi.org/10.1126/science.aad5224)
[^10.1056/NEJMoa1910993]: Mulangu S, Dodd LE, et int., PALM Consortium Study Team. "A Randomized, Controlled Trial of Ebola Virus Disease Therapeutics" _N. Engl. J. Med._, 2019. [10.1056/NEJMoa1910993](https://doi.org/10.1056/NEJMoa1910993)
[^10.2165/00003495-199958020-00009]: Scott LJ, Lamb HM. "Palivizumab" _Drugs_, 1999. [10.2165/00003495-199958020-00009](https://doi.org/10.2165/00003495-199958020-00009)
[^10.1038/nm.4268]: Caskey M, Schoofs T, et int., Klein F. "Antibody 10-1074 suppresses viremia in HIV-1-infected individuals" _Nat. Med._, 2017. [10.1038/nm.4268](https://doi.org/10.1038/nm.4268)
[^10.1038/s41586-018-0531-2]: Mendoza P, Gruell H, et int., Nussenzweig MC. "Combination therapy with anti-HIV-1 antibodies maintains viral suppression" _Nature_, 2018. [10.1038/s41586-018-0531-2](https://doi.org/10.1038/s41586-018-0531-2)
[^10.1016/j.bbrc.2004.05.114]: Hofmann H, Geier M, et int., Pöhlmann S. "Susceptibility to SARS coronavirus S protein-driven infection correlates with expression of angiotensin converting enzyme 2 and infection can be blocked by soluble receptor" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.05.114](https://doi.org/10.1016/j.bbrc.2004.05.114)
[^10.4049/jimmunol.174.8.4908]: He Y, Lu H, et int., Jiang S. "Receptor-binding domain of severe acute respiratory syndrome coronavirus spike protein contains multiple conformation-dependent epitopes that induce highly potent neutralizing antibodies" _J. Immunol._, 2005. [10.4049/jimmunol.174.8.4908](https://doi.org/10.4049/jimmunol.174.8.4908)
[^10.1002/rmv.706]: Coughlin MM, Prabhakar BS. "Neutralizing human monoclonal antibodies to severe acute respiratory syndrome coronavirus: target, mechanism of action, and therapeutic potential" _Rev. Med. Virol._, 2012. [10.1002/rmv.706](https://doi.org/10.1002/rmv.706)
[^10.1126/scitranslmed.3008140]: Jiang L, Wang N, et int., Zhang L. "Potent neutralization of MERS-CoV by human neutralizing monoclonal antibodies to the viral spike glycoprotein" _Sci Transl Med_, 2014. [10.1126/scitranslmed.3008140](https://doi.org/10.1126/scitranslmed.3008140)
[^10.1038/ncomms8712]: Wang L, Shi W, et int., Graham BS. "Evaluation of candidate vaccine approaches for MERS-CoV" _Nature Communications_, 2015. [10.1038/ncomms8712](https://doi.org/10.1038/ncomms8712)
[^10.3390/v10120680]: Han H, Liu J, et int., Yu X. "Neutralizing Monoclonal Antibodies as Promising Therapeutics against Middle East Respiratory Syndrome Coronavirus Infection" _Viruses_, 2018. [10.3390/v10120680](https://doi.org/10.3390/v10120680)
[^10.1038/cr.2015.113]: Li Y, Wan Y, et int., Yan J. "A humanized neutralizing antibody against MERS-CoV targeting the receptor-binding domain of the spike protein" _Cell Res._, 2015. [10.1038/cr.2015.113](https://doi.org/10.1038/cr.2015.113)
[^10.1073/pnas.0307140101]: Sui J, Li W, et int., Marasco WA. "Potent neutralization of severe acute respiratory syndrome (SARS) coronavirus by a human mAb to S1 protein that blocks receptor association" _Proc. Natl. Acad. Sci. U.S.A._, 2004. [10.1073/pnas.0307140101](https://doi.org/10.1073/pnas.0307140101)
[^10.1038/nm1080]: Traggiai E, Becker S, et int., Lanzavecchia A. "An efficient method to make human monoclonal antibodies from memory B cells: potent neutralization of SARS coronavirus" _Nat. Med._, 2004. [10.1038/nm1080](https://doi.org/10.1038/nm1080)
[^10.1073/pnas.0701000104]: Zhu Z, Chakraborti S, et int., Dimitrov DS. "Potent cross-reactive neutralization of SARS coronavirus isolates by human monoclonal antibodies" _Proc. Natl. Acad. Sci. U.S.A._, 2007. [10.1073/pnas.0701000104](https://doi.org/10.1073/pnas.0701000104)
[^10.1086/427242]: Greenough TC, Babcock GJ, et int., Ambrosino DM. "Development and characterization of a severe acute respiratory syndrome-associated coronavirus-neutralizing human monoclonal antibody that provides effective immunoprophylaxis in mice" _J. Infect. Dis._, 2005. [10.1086/427242](https://doi.org/10.1086/427242)
[^10.1016/S0140-6736(04)16506-9]: ter Meulen J, Bakker ABH, et int., Osterhaus ADME. "Human monoclonal antibody as prophylaxis for SARS coronavirus infection in ferrets" _Lancet_, 2004. [10.1016/S0140-6736(04)16506-9](https://doi.org/10.1016/S0140-6736(04)16506-9)
[^10.1371/journal.pmed.0030237]: ter Meulen J, van den Brink EN, et int., Goudsmit J. "Human monoclonal antibody combination against SARS coronavirus: synergy and coverage of escape mutants" _PLoS Med._, 2006. [10.1371/journal.pmed.0030237](https://doi.org/10.1371/journal.pmed.0030237)
[^10.1016/j.virol.2009.07.028]: Coughlin MM, Babcook J, Prabhakar BS. "Human monoclonal antibodies to SARS-coronavirus inhibit infection by different mechanisms" _Virology_, 2009. [10.1016/j.virol.2009.07.028](https://doi.org/10.1016/j.virol.2009.07.028)
[^10.1128/JVI.00433-14]: Du L, Zhao G, et int., Zhou Y. "A conformation-dependent neutralizing monoclonal antibody specifically targeting receptor-binding domain in Middle East respiratory syndrome coronavirus spike protein" _J. Virol._, 2014. [10.1128/JVI.00433-14](https://doi.org/10.1128/JVI.00433-14)
[^10.1128/JVI.00912-14]: Ying T, Du L, et int., Dimitrov DS. "Exceptionally potent neutralization of Middle East respiratory syndrome coronavirus by human monoclonal antibodies" _J. Virol._, 2014. [10.1128/JVI.00912-14](https://doi.org/10.1128/JVI.00912-14)
[^10.1093/infdis/jiw080]: Houser KV, Gretebeck L, et int., Subbarao K. "Prophylaxis With a Middle East Respiratory Syndrome Coronavirus (MERS-CoV)-Specific Human Monoclonal Antibody Protects Rabbits From MERS-CoV Infection" _J. Infect. Dis._, 2016. [10.1093/infdis/jiw080](https://doi.org/10.1093/infdis/jiw080)
[^10.3390/antib8040053]: Wang L, Xu J, et int., Ying T. "Engineering a Novel Antibody-Peptide Bispecific Fusion Protein Against MERS-CoV" _Antibodies (Basel)_, 2019. [10.3390/antib8040053](https://doi.org/10.3390/antib8040053)
[^10.1073/pnas.1510830112]: Pascal KE, Coleman CM, et int., Kyratsous CA. "Pre- and postexposure efficacy of fully human antibodies against Spike protein in a novel humanized mouse model of MERS-CoV infection" _Proc. Natl. Acad. Sci. U.S.A._, 2015. [10.1073/pnas.1510830112](https://doi.org/10.1073/pnas.1510830112)
[^10.1073/pnas.1402074111]: Tang X, Agnihothram SS, et int., Marasco WA. "Identification of human neutralizing antibodies against MERS-CoV and their role in virus adaptive evolution" _Proc. Natl. Acad. Sci. U.S.A._, 2014. [10.1073/pnas.1402074111](https://doi.org/10.1073/pnas.1402074111)
[^10.1080/14728222.2017.1271415]: Du L, Yang Y, et int., Jiang S. "MERS-CoV spike protein: a key target for antivirals" _Expert Opin. Ther. Targets_, 2017. [10.1080/14728222.2017.1271415](https://doi.org/10.1080/14728222.2017.1271415)
[^10.1016/j.cell.2020.03.045]: Wang Q, Zhang Y, et int., Qi J. "Structural and Functional Basis of SARS-CoV-2 Entry by Using Human ACE2" _Cell_, 2020. [10.1016/j.cell.2020.03.045](https://doi.org/10.1016/j.cell.2020.03.045)
[^10.1080/22221751.2020.1729069]: Tian X, Li C, et int., Ying T. "Potent binding of 2019 novel coronavirus spike protein by a SARS coronavirus-specific human monoclonal antibody" _Emerg Microbes Infect_, 2020. [10.1080/22221751.2020.1729069](https://doi.org/10.1080/22221751.2020.1729069)
[^10.1126/science.abb7269]: Yuan M, Wu NC, et int., Wilson IA. "A highly conserved cryptic epitope in the receptor-binding domains of SARS-CoV-2 and SARS-CoV" _Science_, 2020. [10.1126/science.abb7269](https://doi.org/10.1126/science.abb7269)
[^10.1101/2020.05.05.079202]: Huo J, Zhao Y, et int., Stuart DI. "Neutralization of SARS-CoV-2 by destruction of the prefusion Spike" _bioRxiv_, 2020. [10.1101/2020.05.05.079202](https://doi.org/10.1101/2020.05.05.079202)
[^10.1101/2020.04.07.023903]: Pinto D, Park Y, et int., Corti D. "Structural and functional analysis of a potent sarbecovirus neutralizing antibody" _bioRxiv_, 2020. [10.1101/2020.04.07.023903](https://doi.org/10.1101/2020.04.07.023903)
[^10.1101/2020.03.30.20047365]: Wu F, Wang A, et int., Huang J. "Neutralizing antibody responses to SARS-CoV-2 in a COVID-19 recovered patient cohort and their implications" _medRxiv_, 2020. [10.1101/2020.03.30.20047365](https://doi.org/10.1101/2020.03.30.20047365)
[^10.1016/j.it.2020.03.007]: Jiang S, Hillyer C, Du L. "Neutralizing Antibodies against SARS-CoV-2 and Other Human Coronaviruses" _Trends Immunol._, 2020. [10.1016/j.it.2020.03.007](https://doi.org/10.1016/j.it.2020.03.007)
[^10.1038/s41423-020-0426-7]: Chen X, Li R, et int., Ye L. "Human monoclonal antibodies block the binding of SARS-CoV-2 spike protein to angiotensin converting enzyme 2 receptor" _Cell. Mol. Immunol._, 2020. [10.1038/s41423-020-0426-7](https://doi.org/10.1038/s41423-020-0426-7)
[^10.1101/2020.05.12.088716]: Brouwer P, Caniels T, et int., Gils MV. "Potent neutralizing antibodies from COVID-19 patients define multiple targets of vulnerability" _bioRxiv_, 2020. [10.1101/2020.05.12.088716](https://doi.org/10.1101/2020.05.12.088716)
[^10.1038/s41587-020-0512-5]: Kelley B. "Developing therapeutic monoclonal antibodies at pandemic pace" _Nat. Biotechnol._, 2020. [10.1038/s41587-020-0512-5](https://doi.org/10.1038/s41587-020-0512-5)
[^10.1056/NEJMoa035026]: Lalezari JP, Henry K, et int., Salgo M. "Enfuvirtide, an HIV-1 Fusion Inhibitor, for Drug-Resistant HIV Infection in North and South America" _New England Journal of Medicine_, 2003. [10.1056/NEJMoa035026](https://doi.org/10.1056/NEJMoa035026)
[^10.1056/NEJMoa035211]: Lazzarin A, Clotet B, et int., Salgo M. "Efficacy of Enfuvirtide in Patients Infected with Drug-Resistant HIV-1 in Europe and Australia" _New England Journal of Medicine_, 2003. [10.1056/NEJMoa035211](https://doi.org/10.1056/NEJMoa035211)
[^10.1016/j.bbrc.2004.05.046]: Yuan K, Yi L, et int., Deng H. "Suppression of SARS-CoV entry by peptides corresponding to heptad regions on spike glycoprotein" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.05.046](https://doi.org/10.1016/j.bbrc.2004.05.046)
[^10.1038/ncomms4067]: Lu L, Liu Q, et int., Jiang S. "Structure-based discovery of Middle East respiratory syndrome coronavirus fusion inhibitor" _Nat Commun_, 2014. [10.1038/ncomms4067](https://doi.org/10.1038/ncomms4067)
[^10.1093/infdis/jiv325]: Channappanavar R, Lu L, et int., Jiang S. "Protective Effect of Intranasal Regimens Containing Peptidic Middle East Respiratory Syndrome Coronavirus Fusion Inhibitor Against MERS-CoV Infection" _J. Infect. Dis._, 2015. [10.1093/infdis/jiv325](https://doi.org/10.1093/infdis/jiv325)
[^10.1021/acs.jmedchem.8b00890]: Wang C, Zhao L, et int., Liu K. "De Novo Design of α-Helical Lipopeptides Targeting Viral Fusion Proteins: A Promising Strategy for Relatively Broad-Spectrum Antiviral Drug Discovery" _J. Med. Chem._, 2018. [10.1021/acs.jmedchem.8b00890](https://doi.org/10.1021/acs.jmedchem.8b00890)
[^10.1126/sciadv.aav4580]: Xia S, Yan L, et int., Lu L. "A pan-coronavirus fusion inhibitor targeting the HR1 domain of human coronavirus spike" _Sci Adv_, 2019. [10.1126/sciadv.aav4580](https://doi.org/10.1126/sciadv.aav4580)
[^10.1101/2020.03.26.009233]: Zhu Y, Yu D, et int., He Y. "Design of potent membrane fusion inhibitors against SARS-CoV-2, an emerging coronavirus with high fusogenic activity" _bioRxiv_, 2020. [10.1101/2020.03.26.009233](https://doi.org/10.1101/2020.03.26.009233)
[^10.1038/s41423-020-0374-2]: Xia S, Zhu Y, et int., Lu L. "Fusion mechanism of 2019-nCoV and fusion inhibitors targeting HR1 domain in spike protein" _Cellular & Molecular Immunology_, 2020. [10.1038/s41423-020-0374-2](https://doi.org/10.1038/s41423-020-0374-2)
[^10.1038/s41422-020-0305-x]: Xia S, Liu M, et int., Lu L. "Inhibition of SARS-CoV-2 (previously 2019-nCoV) infection by a highly potent pan-coronavirus fusion inhibitor targeting its spike protein that harbors a high capacity to mediate membrane fusion" _Cell Res._, 2020. [10.1038/s41422-020-0305-x](https://doi.org/10.1038/s41422-020-0305-x)
[^10.1128/JVI.00288-17]: Chong H, Xue J, et int., He Y. "A Lipopeptide HIV-1/2 Fusion Inhibitor with Highly Potent In Vitro, Ex Vivo, and In Vivo Antiviral Activity" _J. Virol._, 2017. [10.1128/JVI.00288-17](https://doi.org/10.1128/JVI.00288-17)
[^10.1038/nature03712]: Imai Y, Kuba K, et int., Penninger JM. "Angiotensin-converting enzyme 2 protects from severe acute lung failure" _Nature_, 2005. [10.1038/nature03712](https://doi.org/10.1038/nature03712)
[^10.1038/nm1267]: Kuba K, Imai Y, et int., Penninger JM. "A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury" _Nat. Med._, 2005. [10.1038/nm1267](https://doi.org/10.1038/nm1267)
[^10.1186/s13054-017-1823-x]: Khan A, Benthin C, et int., Lazaar AL. "A pilot clinical trial of recombinant human angiotensin-converting enzyme 2 in acute respiratory distress syndrome" _Crit Care_, 2017. [10.1186/s13054-017-1823-x](https://doi.org/10.1186/s13054-017-1823-x)
[^10.1016/j.cell.2020.04.004]: Monteil V, Kwon H, et int., Penninger JM. "Inhibition of SARS-CoV-2 Infections in Engineered Human Tissues Using Clinical-Grade Soluble Human ACE2" _Cell_, 2020. [10.1016/j.cell.2020.04.004](https://doi.org/10.1016/j.cell.2020.04.004)
[^10.1038/s41467-020-16048-4]: Lei C, Qian K, et int., Hu S. "Neutralization of SARS-CoV-2 spike pseudotyped virus by recombinant ACE2-Ig" _Nature Communications_, 2020. [10.1038/s41467-020-16048-4](https://doi.org/10.1038/s41467-020-16048-4)
[^10.1016/j.virol.2006.01.029]: Han DP, Penn-Nicholson A, Cho MW. "Identification of critical determinants on ACE2 for SARS-CoV entry and development of a potent entry inhibitor" _Virology_, 2006. [10.1016/j.virol.2006.01.029](https://doi.org/10.1016/j.virol.2006.01.029)
[^PMID15918330]: Zheng B, Guan Y, et int., Huang J. "Synthetic peptides outside the spike protein heptad repeat regions as potent inhibitors of SARS-associated coronavirus" _Antivir. Ther. (Lond.)_, 2005. [PMID15918330](https://pubmed.ncbi.nlm.nih.gov/15918330)
[^10.1038/srep22008]: Zhao H, Zhou J, et int., Zheng B. "A novel peptide with potent and broad-spectrum antiviral activities against multiple respiratory viruses" _Sci Rep_, 2016. [10.1038/srep22008](https://doi.org/10.1038/srep22008)
[^10.1128/JVI.02322-09]: O'Keefe BR, Giomarelli B, et int., McCray PB. "Broad-spectrum in vitro activity and in vivo efficacy of the antiviral protein griffithsin against emerging viruses of the family Coronaviridae" _J. Virol._, 2010. [10.1128/JVI.02322-09](https://doi.org/10.1128/JVI.02322-09)
[^10.1016/j.antiviral.2016.07.011]: Millet JK, Séron K, et int., Belouzard S. "Middle East respiratory syndrome coronavirus infection is inhibited by griffithsin" _Antiviral Res._, 2016. [10.1016/j.antiviral.2016.07.011](https://doi.org/10.1016/j.antiviral.2016.07.011)
[^10.1016/j.chembiol.2004.07.013]: Kao RY, Tsui WHW, et int., Yuen K. "Identification of novel small-molecule inhibitors of severe acute respiratory syndrome-associated coronavirus by chemical genetics" _Chem. Biol._, 2004. [10.1016/j.chembiol.2004.07.013](https://doi.org/10.1016/j.chembiol.2004.07.013)
[^10.1101/2020.03.29.014183]: Milewska A, Chi Y, et int., Pyrc K. "HTCC as a highly effective polymeric inhibitor of SARS-CoV-2 and MERS-CoV" _bioRxiv_, 2020. [10.1101/2020.03.29.014183](https://doi.org/10.1101/2020.03.29.014183)
[^10.1371/journal.pone.0156552]: Milewska A, Kaminski K, et int., Pyrc K. "HTCC: Broad Range Inhibitor of Coronavirus Entry" _PLoS ONE_, 2016. [10.1371/journal.pone.0156552](https://doi.org/10.1371/journal.pone.0156552)
[^10.1016/j.antiviral.2011.12.012]: Struck A, Axmann M, et int., Meyer B. "A hexapeptide of the receptor-binding domain of SARS corona virus spike protein blocks viral entry into host cells via the human receptor ACE2" _Antiviral Res._, 2012. [10.1016/j.antiviral.2011.12.012](https://doi.org/10.1016/j.antiviral.2011.12.012)
[^10.1016/j.antiviral.2006.04.014]: Ho T, Wu S, et int., Hsiang C. "Emodin blocks the SARS coronavirus spike protein and angiotensin-converting enzyme 2 interaction" _Antiviral Res._, 2007. [10.1016/j.antiviral.2006.04.014](https://doi.org/10.1016/j.antiviral.2006.04.014)
[^10.1016/j.antiviral.2011.02.008]: Schwarz S, Wang K, et int., Schwarz W. "Emodin inhibits current through SARS-associated coronavirus 3a protein" _Antiviral Res._, 2011. [10.1016/j.antiviral.2011.02.008](https://doi.org/10.1016/j.antiviral.2011.02.008)
[^10.1016/j.antiviral.2014.04.006]: Blaising J, Polyak SJ, Pécheur E. "Arbidol as a broad-spectrum antiviral: an update" _Antiviral Res._, 2014. [10.1016/j.antiviral.2014.04.006](https://doi.org/10.1016/j.antiviral.2014.04.006)
[^10.1128/JVI.02077-15]: Pécheur E, Borisevich V, et int., Polyak SJ. "The Synthetic Antiviral Drug Arbidol Inhibits Globally Prevalent Pathogenic Viruses" _J. Virol._, 2016. [10.1128/JVI.02077-15](https://doi.org/10.1128/JVI.02077-15)
[^10.1016/j.antiviral.2008.10.009]: Leneva IA, Russell RJ, et int., Hay AJ. "Characteristics of arbidol-resistant mutants of influenza virus: implications for the mechanism of anti-influenza action of arbidol" _Antiviral Res._, 2009. [10.1016/j.antiviral.2008.10.009](https://doi.org/10.1016/j.antiviral.2008.10.009)
[^10.1038/s41421-020-0169-8]: Wang X, Cao R, et int., Wang M. "The anti-influenza virus drug, arbidol is an efficient inhibitor of SARS-CoV-2 in vitro" _Cell Discovery_, 2020. [10.1038/s41421-020-0169-8](https://doi.org/10.1038/s41421-020-0169-8)
[^10.1073/pnas.1617020114]: Kadam RU, Wilson IA. "Structural basis of influenza virus fusion inhibition by the antiviral drug Arbidol" _Proc. Natl. Acad. Sci. U.S.A._, 2017. [10.1073/pnas.1617020114](https://doi.org/10.1073/pnas.1617020114)
[^10.1101/2020.03.19.20038984]: Li Y, Xie Z, et int., Li L. "An exploratory randomized controlled study on the efficacy and safety of lopinavir/ritonavir or arbidol treating adult patients hospitalized with mild/moderate COVID-19 (ELACOI)" _medRxiv_, 2020. [10.1101/2020.03.19.20038984](https://doi.org/10.1101/2020.03.19.20038984)
[^10.1016/j.jinf.2020.03.002]: Deng L, Li C, et int., Xia J. "Arbidol combined with LPV/r versus LPV/r alone against Corona Virus Disease 2019:a retrospective cohort study" _J. Infect._, 2020. [10.1016/j.jinf.2020.03.002](https://doi.org/10.1016/j.jinf.2020.03.002)
[^10.1016/j.jinf.2020.03.060]: Zhu Z, Lu Z, et int., Xue Y. "Arbidol Monotherapy is Superior to Lopinavir/ritonavir in Treating COVID-19" _J. Infect._, 2020. [10.1016/j.jinf.2020.03.060](https://doi.org/10.1016/j.jinf.2020.03.060)
[^10.1101/2020.04.06.20042580]: Zhou Q, Wei X, et int., Fish EN. "Interferon-a2b treatment for COVID-19" _medRxiv_, 2020. [10.1101/2020.04.06.20042580](https://doi.org/10.1101/2020.04.06.20042580)
[^10.1101/2020.04.28.066761]: Mycroft-West C, Su D, et int., Skidmore MA. "Heparin inhibits cellular invasion by SARS-CoV-2: structural dependence of the interaction of the surface protein (spike) S1 receptor binding domain with heparin." _bioRxiv_, 2020. [10.1101/2020.04.28.066761](https://doi.org/10.1101/2020.04.28.066761)
[^10.1016/j.antiviral.2020.104791]: Sallard E, Lescure F, et int., Peiffer-Smadja N. "Type 1 interferons as a potential treatment against COVID-19" _Antiviral Res._, 2020. [10.1016/j.antiviral.2020.104791](https://doi.org/10.1016/j.antiviral.2020.104791)
[^10.1016/j.coviro.2012.04.004]: Totura AL, Baric RS. "SARS coronavirus pathogenesis: host innate immune responses and viral antagonism of interferon" _Curr Opin Virol_, 2012. [10.1016/j.coviro.2012.04.004](https://doi.org/10.1016/j.coviro.2012.04.004)
[^10.1016/j.smim.2019.101300]: Wang BX, Fish EN. "Global virus outbreaks: Interferons as 1st responders" _Semin. Immunol._, 2019. [10.1016/j.smim.2019.101300](https://doi.org/10.1016/j.smim.2019.101300)
[^10.1038/nri3787]: McNab F, Mayer-Barber K, et int., O'Garra A. "Type I interferons in infectious disease" _Nat. Rev. Immunol._, 2015. [10.1038/nri3787](https://doi.org/10.1038/nri3787)
[^10.1089/1079990041535610]: Zheng B, He M, et int., Kung H. "Potent inhibition of SARS-associated coronavirus (SCOV) infection and replication by type I interferons (IFN-alpha/beta) but not by type II interferon (IFN-gamma)" _J. Interferon Cytokine Res._, 2004. [10.1089/1079990041535610](https://doi.org/10.1089/1079990041535610)
[^10.1086/382597]: Ströher U, DiCaro A, et int., Feldmann H. "Severe acute respiratory syndrome-related coronavirus is inhibited by interferon- alpha" _J. Infect. Dis._, 2004. [10.1086/382597](https://doi.org/10.1086/382597)
[^PMID15651759]: Scagnolari C, Vicenzi E, et int., Antonelli G. "Increased sensitivity of SARS-coronavirus to a combination of human type I and type II interferons" _Antivir. Ther. (Lond.)_, 2004. [PMID15651759](https://pubmed.ncbi.nlm.nih.gov/15651759)
[^10.1016/j.bbrc.2004.08.085]: Keyaerts E, Vijgen L, et int., Van Ranst M. "In vitro inhibition of severe acute respiratory syndrome coronavirus by chloroquine" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.08.085](https://doi.org/10.1016/j.bbrc.2004.08.085)
[^10.1016/j.virol.2004.08.011]: Sainz B, Mossel EC, et int., Garry RF. "Interferon-beta and interferon-gamma synergistically inhibit the replication of severe acute respiratory syndrome-associated coronavirus (SARS-CoV)" _Virology_, 2004. [10.1016/j.virol.2004.08.011](https://doi.org/10.1016/j.virol.2004.08.011)
[^10.3201/eid1002.030482]: Hensley LE, Fritz LE, et int., Geisbert TW. "Interferon-beta 1a and SARS coronavirus replication" _Emerging Infect. Dis._, 2004. [10.3201/eid1002.030482](https://doi.org/10.3201/eid1002.030482)
[^10.1016/j.bbrc.2004.06.076]: He R, Adonov A, et int., Li X. "Potent and selective inhibition of SARS coronavirus replication by aurintricarboxylic acid" _Biochem. Biophys. Res. Commun._, 2004. [10.1016/j.bbrc.2004.06.076](https://doi.org/10.1016/j.bbrc.2004.06.076)
[^10.1080/00365540410021144]: Dahl H, Linde A, Strannegård O. "In vitro inhibition of SARS virus replication by human interferons" _Scand. J. Infect. Dis._, 2004. [10.1080/00365540410021144](https://doi.org/10.1080/00365540410021144)
[^10.1016/j.bbrc.2004.11.128]: Morgenstern B, Michaelis M, et int., Cinatl J. "Ribavirin and interferon-beta synergistically inhibit SARS-associated coronavirus replication in animal and human cell lines" _Biochem. Biophys. Res. Commun._, 2005. [10.1016/j.bbrc.2004.11.128](https://doi.org/10.1016/j.bbrc.2004.11.128)
[^10.1099/vir.0.052910-0]: de Wilde AH, Raj VS, et int., van den Hoogen BG. "MERS-coronavirus replication induces severe in vitro cytopathology and is strongly inhibited by cyclosporin A or interferon-α treatment" _J. Gen. Virol._, 2013. [10.1099/vir.0.052910-0](https://doi.org/10.1099/vir.0.052910-0)
[^10.1038/nm.3362]: Falzarano D, de Wit E, et int., Feldmann H. "Treatment with interferon-α2b and ribavirin improves outcome in MERS-CoV-infected rhesus macaques" _Nat. Med._, 2013. [10.1038/nm.3362](https://doi.org/10.1038/nm.3362)
[^10.1016/j.antiviral.2018.05.007]: Li HS, Kuok DIT, et int., Nicholls JM. "Effect of interferon alpha and cyclosporine treatment separately and in combination on Middle East Respiratory Syndrome Coronavirus (MERS-CoV) replication in a human in-vitro and ex-vivo culture model" _Antiviral Res._, 2018. [10.1016/j.antiviral.2018.05.007](https://doi.org/10.1016/j.antiviral.2018.05.007)
[^10.1016/j.antiviral.2017.01.012]: Li L, Fu F, et int., Liu P. "IFN-lambda preferably inhibits PEDV infection of porcine intestinal epithelial cells compared with IFN-alpha" _Antiviral Res._, 2017. [10.1016/j.antiviral.2017.01.012](https://doi.org/10.1016/j.antiviral.2017.01.012)
[^10.1101/2020.03.07.982264]: Lokugamage KG, Schindewolf C, Menachery VD. "SARS-CoV-2 sensitive to type I interferon pretreatment" _bioRxiv_, 2020. [10.1101/2020.03.07.982264](https://doi.org/10.1101/2020.03.07.982264)
[^10.1101/2020.04.02.022764]: Mantlo EK, Bukreyeva N, et int., Huang C. "Potent Antiviral Activities of Type I Interferons to SARS-CoV-2 Infection" _bioRxiv_, 2020. [10.1101/2020.04.02.022764](https://doi.org/10.1101/2020.04.02.022764)
[^10.1038/nm1001]: Haagmans BL, Kuiken T, et int., Osterhaus ADME. "Pegylated interferon-alpha protects type 1 pneumocytes against SARS coronavirus infection in macaques" _Nat. Med._, 2004. [10.1038/nm1001](https://doi.org/10.1038/nm1001)
[^10.1016/j.antiviral.2010.11.007]: Kumaki Y, Ennis J, et int., Barnard DL. "Single-dose intranasal administration with mDEF201 (adenovirus vectored mouse interferon-alpha) confers protection from mortality in a lethal SARS-CoV BALB/c mouse model" _Antiviral Res._, 2011. [10.1016/j.antiviral.2010.11.007](https://doi.org/10.1016/j.antiviral.2010.11.007)
[^10.1172/JCI126363]: Channappanavar R, Fehr AR, et int., Perlman S. "IFN-I response timing relative to virus replication determines MERS coronavirus infection outcomes" _J. Clin. Invest._, 2019. [10.1172/JCI126363](https://doi.org/10.1172/JCI126363)
[^10.1016/S0140-6736(20)31042-4]: Hung IF, Lung K, et int., Yuen K. "Triple combination of interferon beta-1b, lopinavir–ritonavir, and ribavirin in the treatment of patients admitted to hospital with COVID-19: an open-label, randomised, phase 2 trial" _The Lancet_, 2020. [10.1016/S0140-6736(20)31042-4](https://doi.org/10.1016/S0140-6736(20)31042-4)
[^10.15252/emmm.201606413]: Davidson S, McCabe TM, et int., Wack A. "IFNλ is a potent anti-influenza therapeutic without the inflammatory side effects of IFNα treatment" _EMBO Mol Med_, 2016. [10.15252/emmm.201606413](https://doi.org/10.15252/emmm.201606413)
[^10.1093/cid/ciaa453]: O'Brien TR, Thomas DL, et int., Hartmann R. "Weak Induction of Interferon Expression by SARS-CoV-2 Supports Clinical Trials of Interferon Lambda to Treat Early COVID-19" _Clin. Infect. Dis._, 2020. [10.1093/cid/ciaa453](https://doi.org/10.1093/cid/ciaa453)
[^10.1016/j.cell.2020.02.052]: Hoffmann M, Kleine-Weber H, et int., Pöhlmann S. "SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor" _Cell_, 2020. [10.1016/j.cell.2020.02.052](https://doi.org/10.1016/j.cell.2020.02.052)
[^10.1101/2020.04.15.042085]: Bestle D, Heindl MR, et int., Bottcher-Friebertshauser E. "TMPRSS2 and furin are both essential for proteolytic activation and spread of SARS-CoV-2 in human airway epithelial cells and provide promising drug targets" _bioRxiv_, 2020. [10.1101/2020.04.15.042085](https://doi.org/10.1101/2020.04.15.042085)
[^10.1073/pnas.0505577102]: Simmons G, Gosalia DN, et int., Bates P. "Inhibitors of cathepsin L prevent severe acute respiratory syndrome coronavirus entry" _Proc. Natl. Acad. Sci. U.S.A._, 2005. [10.1073/pnas.0505577102](https://doi.org/10.1073/pnas.0505577102)
[^10.1128/JVI.02232-10]: Glowacka I, Bertram S, et int., Pöhlmann S. "Evidence that TMPRSS2 activates the severe acute respiratory syndrome coronavirus spike protein for membrane fusion and reduces viral control by the humoral immune response" _J. Virol._, 2011. [10.1128/JVI.02232-10](https://doi.org/10.1128/JVI.02232-10)
[^10.1128/JVI.02062-10]: Shulla A, Heald-Sargent T, et int., Gallagher T. "A transmembrane serine protease is linked to the severe acute respiratory syndrome coronavirus receptor and activates virus entry" _J. Virol._, 2011. [10.1128/JVI.02062-10](https://doi.org/10.1128/JVI.02062-10)
[^10.1128/JVI.00094-12]: Kawase M, Shirato K, et int., Matsuyama S. "Simultaneous treatment of human bronchial epithelial cells with serine and cysteine protease inhibitors prevents severe acute respiratory syndrome coronavirus entry" _J. Virol._, 2012. [10.1128/JVI.00094-12](https://doi.org/10.1128/JVI.00094-12)
[^10.1016/j.virol.2017.11.012]: Shirato K, Kawase M, Matsuyama S. "Wild-type human coronaviruses prefer cell-surface TMPRSS2 to endosomal cathepsins for cell entry" _Virology_, 2018. [10.1016/j.virol.2017.11.012](https://doi.org/10.1016/j.virol.2017.11.012)
[^10.1101/2020.04.22.054981]: Yamamoto M, Kiso M, et int., Inoue J. "The anticoagulant nafamostat potently inhibits SARS-CoV-2 infection in vitro: an existing drug with multiple possible therapeutic effects" _bioRxiv_, 2020. [10.1101/2020.04.22.054981](https://doi.org/10.1101/2020.04.22.054981)
[^10.1038/s41467-020-15562-9]: Ou X, Liu Y, et int., Qian Z. "Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV" _Nature Communications_, 2020. [10.1038/s41467-020-15562-9](https://doi.org/10.1038/s41467-020-15562-9)
[^10.1128/AAC.00754-20]: Hoffmann M, Schroeder S, et int., Pöhlmann S. "Nafamostat mesylate blocks activation of SARS-CoV-2: New treatment option for COVID-19" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.00754-20](https://doi.org/10.1128/AAC.00754-20)
[^10.1016/j.antiviral.2015.01.011]: Zhou Y, Vedantham P, et int., Simmons G. "Protease inhibitors targeting coronavirus and filovirus entry" _Antiviral Res._, 2015. [10.1016/j.antiviral.2015.01.011](https://doi.org/10.1016/j.antiviral.2015.01.011)
[^10.1128/JVI.01001-12]: Schneider M, Ackermann K, et int., Gilch S. "Severe acute respiratory syndrome coronavirus replication is severely impaired by MG132 due to proteasome-independent inhibition of M-calpain" _J. Virol._, 2012. [10.1128/JVI.01001-12](https://doi.org/10.1128/JVI.01001-12)
[^10.1073/pnas.1407087111]: Millet JK, Whittaker GR. "Host cell entry of Middle East respiratory syndrome coronavirus after two-step, furin-mediated activation of the spike protein" _Proc. Natl. Acad. Sci. U.S.A._, 2014. [10.1073/pnas.1407087111](https://doi.org/10.1073/pnas.1407087111)
[^10.1016/s1473-3099(03)00806-5]: Savarino A, Boelaert JR, et int., Cauda R. "Effects of chloroquine on viral infections: an old drug against today's diseases?" _Lancet Infect Dis_, 2003. [10.1016/s1473-3099(03)00806-5](https://doi.org/10.1016/s1473-3099(03)00806-5)
[^10.1371/journal.ppat.1004502]: Burkard C, Verheije MH, et int., de Haan CAM. "Coronavirus cell entry occurs through the endo-/lysosomal pathway in a proteolysis-dependent manner" _PLoS Pathog._, 2014. [10.1371/journal.ppat.1004502](https://doi.org/10.1371/journal.ppat.1004502)
[^10.1186/1743-422X-2-69]: Vincent MJ, Bergeron E, et int., Nichol ST. "Chloroquine is a potent inhibitor of SARS coronavirus infection and spread" _Virol. J._, 2005. [10.1186/1743-422X-2-69](https://doi.org/10.1186/1743-422X-2-69)
[^10.1128/AAC.03036-14]: Dyall J, Coleman CM, et int., Frieman MB. "Repurposing of clinically developed drugs for treatment of Middle East respiratory syndrome coronavirus infection" _Antimicrob. Agents Chemother._, 2014. [10.1128/AAC.03036-14](https://doi.org/10.1128/AAC.03036-14)
[^10.3390/v10060283]: Shin JS, Jung E, et int., Go YY. "Saracatinib Inhibits Middle East Respiratory Syndrome-Coronavirus Replication In Vitro" _Viruses_, 2018. [10.3390/v10060283](https://doi.org/10.3390/v10060283)
[^10.1038/s41421-020-0156-0]: Liu J, Cao R, et int., Wang M. "Hydroxychloroquine, a less toxic derivative of chloroquine, is effective in inhibiting SARS-CoV-2 infection in vitro" _Cell Discov_, 2020. [10.1038/s41421-020-0156-0](https://doi.org/10.1038/s41421-020-0156-0)
[^10.1093/cid/ciaa237]: Yao X, Ye F, et int., Liu D. "In Vitro Antiviral Activity and Projection of Optimized Dosing Design of Hydroxychloroquine for the Treatment of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2)" _Clin. Infect. Dis._, 2020. [10.1093/cid/ciaa237](https://doi.org/10.1093/cid/ciaa237)
[^10.1101/2020.03.29.014407]: Clementi N, Criscuolo E, et int., Mancini N. "Combined prophylactic and therapeutic use maximizes hydroxychloroquine anti-SARS-CoV-2 effects in vitro" _bioRxiv_, 2020. [10.1101/2020.03.29.014407](https://doi.org/10.1101/2020.03.29.014407)
[^10.1128/AAC.01509-08]: Keyaerts E, Li S, et int., Maes P. "Antiviral activity of chloroquine against human coronavirus OC43 infection in newborn mice" _Antimicrob. Agents Chemother._, 2009. [10.1128/AAC.01509-08](https://doi.org/10.1128/AAC.01509-08)
[^10.1371/journal.pntd.0000785]: Tricou V, Minh NN, et int., Simmons CP. "A randomized controlled trial of chloroquine for the treatment of dengue in Vietnamese adults" _PLoS Negl Trop Dis_, 2010. [10.1371/journal.pntd.0000785](https://doi.org/10.1371/journal.pntd.0000785)
[^10.1016/S1473-3099(11)70065-2]: Paton NI, Lee L, et int., Smith AW. "Chloroquine for influenza prevention: a randomised, double-blind, placebo controlled trial" _Lancet Infect Dis_, 2011. [10.1016/S1473-3099(11)70065-2](https://doi.org/10.1016/S1473-3099(11)70065-2)
[^10.1111/j.1365-2125.1989.tb03439.x]: Tett SE, Cutler DJ, et int., Brown KF. "Bioavailability of hydroxychloroquine tablets in healthy volunteers" _Br J Clin Pharmacol_, 1989. [10.1111/j.1365-2125.1989.tb03439.x](https://doi.org/10.1111/j.1365-2125.1989.tb03439.x)
[^10.2165/00003088-199631040-00003]: Ducharme J, Farinotti R. "Clinical pharmacokinetics and metabolism of chloroquine. Focus on recent advancements" _Clin Pharmacokinet_, 1996. [10.2165/00003088-199631040-00003](https://doi.org/10.2165/00003088-199631040-00003)
[^10.1128/AAC.00339-08]: Lim H, Im J, et int., Park J. "Pharmacokinetics of hydroxychloroquine and its clinical implications in chemoprophylaxis against malaria caused by Plasmodium vivax" _Antimicrob. Agents Chemother._, 2009. [10.1128/AAC.00339-08](https://doi.org/10.1128/AAC.00339-08)
[^10.1002/cpt.1856]: Garcia-Cremades M, Solans BP, et int., Savic RM. "Optimizing hydroxychloroquine dosing for patients with COVID-19: An integrative modeling approach for effective drug repurposing" _Clin. Pharmacol. Ther._, 2020. [10.1002/cpt.1856](https://doi.org/10.1002/cpt.1856)
[^10.1056/NEJMoa2012410]: Geleris J, Sun Y, et int., Schluger NW. "Observational Study of Hydroxychloroquine in Hospitalized Patients with Covid-19" _New England Journal of Medicine_, 2020. [10.1056/NEJMoa2012410](https://doi.org/10.1056/NEJMoa2012410)
[^10.1016/j.tmaid.2020.101663]: Gautret P, Lagier J, et int., Raoult D. "Clinical and microbiological effect of a combination of hydroxychloroquine and azithromycin in 80 COVID-19 patients with at least a six-day follow up: A pilot observational study" _Travel Med Infect Dis_, 2020. [10.1016/j.tmaid.2020.101663](https://doi.org/10.1016/j.tmaid.2020.101663)
[^10.1101/2020.04.27.20073379]: Yu B, Wang DW, Li C. "Hydroxychloroquine application is associated with a decreased mortality in critically ill patients with COVID-19" _medRxiv_, 2020. [10.1101/2020.04.27.20073379](https://doi.org/10.1101/2020.04.27.20073379)
[^10.1101/2020.04.26.20081059]: Huang M, Li M, et int., Shan H. "Preliminary evidence from a multicenter prospective observational study of the safety and efficacy of chloroquine for the treatment of COVID-19" _medRxiv_, 2020. [10.1101/2020.04.26.20081059](https://doi.org/10.1101/2020.04.26.20081059)
[^10.1101/2020.04.10.20060699]: Mahevas M, Tran V, et int., Costedoat N. "No evidence of clinical efficacy of hydroxychloroquine in patients hospitalized for COVID-19 infection with oxygen requirement: results of a study using routinely collected data to emulate a target trial" _medRxiv_, 2020. [10.1101/2020.04.10.20060699](https://doi.org/10.1101/2020.04.10.20060699)
[^10.1016/j.medmal.2020.03.006]: Molina JM, Delaugerre C, et int., de Castro N. "No Evidence of Rapid Antiviral Clearance or Clinical Benefit with the Combination of Hydroxychloroquine and Azithromycin in Patients with Severe COVID-19 Infection" _Médecine et Maladies Infectieuses_, 2020. [10.1016/j.medmal.2020.03.006](https://doi.org/10.1016/j.medmal.2020.03.006)
[^10.1101/2020.03.22.20040758]: Chen Z, Hu J, et int., Zhang Z. "Efficacy of hydroxychloroquine in patients with COVID-19: results of a randomized clinical trial" _medRxiv_, 2020. [10.1101/2020.03.22.20040758](https://doi.org/10.1101/2020.03.22.20040758)
[^10.1001/jamanetworkopen.2020.8857]: Borba MGS, Val FFA, et int., CloroCovid-19 Team. "Effect of High vs Low Doses of Chloroquine Diphosphate as Adjunctive Therapy for Patients Hospitalized With Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Infection: A Randomized Clinical Trial" _JAMA Netw Open_, 2020. [10.1001/jamanetworkopen.2020.8857](https://doi.org/10.1001/jamanetworkopen.2020.8857)
[^10.1101/2020.04.10.20060558]: Tang W, Cao Z, et int., Xie Q. "Hydroxychloroquine in patients with COVID-19: an open-label, randomized, controlled trial" _medRxiv_, 2020. [10.1101/2020.04.10.20060558](https://doi.org/10.1101/2020.04.10.20060558)
[^10.3785/j.issn.1008-9292.2020.03.03]: Chen J, Liu D. "硫酸羟氯喹治疗普通型2019冠状病毒病（COVID-19）患者初步研究" _J Zhejiang Univ_, 2020. [10.3785/j.issn.1008-9292.2020.03.03](https://doi.org/10.3785/j.issn.1008-9292.2020.03.03)
[^10.1093/ofid/ofaa130]: Pastick KA, Okafor EC, et int., Boulware DR. "Review: Hydroxychloroquine and Chloroquine for Treatment of SARS-CoV-2 (COVID-19)" _Open Forum Infect Dis_, 2020. [10.1093/ofid/ofaa130](https://doi.org/10.1093/ofid/ofaa130)
[^10.1021/acsinfecdis.0c00052]: Xu J, Shi P, et int., Zhou J. "Broad Spectrum Antiviral Agent Niclosamide and Its Therapeutic Potential" _ACS Infect Dis_, 2020. [10.1021/acsinfecdis.0c00052](https://doi.org/10.1021/acsinfecdis.0c00052)
[^10.1016/j.cellsig.2017.04.001]: Chen W, Mook RA, et int., Wang J. "Niclosamide: Beyond an antihelminthic drug" _Cell. Signal._, 2018. [10.1016/j.cellsig.2017.04.001](https://doi.org/10.1016/j.cellsig.2017.04.001)
[^10.1021/jm070295s]: Wen C, Kuo Y, et int., Yang N. "Specific plant terpenoids and lignoids possess potent antiviral activities against severe acute respiratory syndrome coronavirus" _J. Med. Chem._, 2007. [10.1021/jm070295s](https://doi.org/10.1021/jm070295s)
[^10.1101/2020.04.15.997254]: Gassen NC, Papies J, et int., Muller MA. "Analysis of SARS-CoV-2-controlled autophagy reveals spermidine, MK-2206, and niclosamide as putative antiviral therapeutics" _bioRxiv_, 2020. [10.1101/2020.04.15.997254](https://doi.org/10.1101/2020.04.15.997254)
[^10.1038/nrd.2015.37]: Zumla A, Chan JFW, et int., Yuen K. "Coronaviruses - drug discovery and therapeutic options" _Nat Rev Drug Discov_, 2016. [10.1038/nrd.2015.37](https://doi.org/10.1038/nrd.2015.37)
[^10.7150/ijbs.45498]: Yang N, Shen H. "Targeting the Endocytic Pathway and Autophagy Process as a Novel Therapeutic Strategy in COVID-19" _Int. J. Biol. Sci._, 2020. [10.7150/ijbs.45498](https://doi.org/10.7150/ijbs.45498)
[^10.1371/journal.pone.0194868]: Cong Y, Hart BJ, et int., Holbrook MR. "MERS-CoV pathogenesis and antiviral efficacy of licensed drugs in human monocyte-derived antigen-presenting cells" _PLoS ONE_, 2018. [10.1371/journal.pone.0194868](https://doi.org/10.1371/journal.pone.0194868)
[^10.1101/2020.03.25.008482]: Weston S, Haupt R, et int., Frieman MB. "FDA approved drugs with broad anti-coronaviral activity inhibit SARS-CoV-2 in vitro" _bioRxiv_, 2020. [10.1101/2020.03.25.008482](https://doi.org/10.1101/2020.03.25.008482)
[^10.1371/journal.ppat.1004166]: Lundin A, Dijkman R, et int., Trybala E. "Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the middle East respiratory syndrome virus" _PLoS Pathog._, 2014. [10.1371/journal.ppat.1004166](https://doi.org/10.1371/journal.ppat.1004166)
[^10.1016/j.virusres.2018.01.002]: Rappe JCF, de Wilde A, et int., Thiel V. "Antiviral activity of K22 against members of the order Nidovirales" _Virus Res._, 2018. [10.1016/j.virusres.2018.01.002](https://doi.org/10.1016/j.virusres.2018.01.002)
[^10.1128/mBio.00165-13]: Josset L, Menachery VD, et int., Katze MG. "Cell host response to infection with novel human coronavirus EMC predicts potential antivirals and important differences with SARS coronavirus" _mBio_, 2013. [10.1128/mBio.00165-13](https://doi.org/10.1128/mBio.00165-13)
[^10.1128/mBio.00611-12]: Kindler E, Jónsdóttir HR, et int., Thiel V. "Efficient replication of the novel human betacoronavirus EMC on primary human epithelium highlights its zoonotic potential" _mBio_, 2013. [10.1128/mBio.00611-12](https://doi.org/10.1128/mBio.00611-12)
[^10.1128/AAC.03659-14]: Kindrachuk J, Ork B, et int., Jahrling PB. "Antiviral potential of ERK/MAPK and PI3K/AKT/mTOR signaling modulation for Middle East respiratory syndrome coronavirus infection as identified by temporal kinome analysis" _Antimicrob. Agents Chemother._, 2015. [10.1128/AAC.03659-14](https://doi.org/10.1128/AAC.03659-14)
[^10.1128/JVI.01429-16]: Coleman CM, Sisk JM, et int., Frieman MB. "Abelson Kinase Inhibitors Are Potent Inhibitors of Severe Acute Respiratory Syndrome Coronavirus and Middle East Respiratory Syndrome Coronavirus Fusion" _J. Virol._, 2016. [10.1128/JVI.01429-16](https://doi.org/10.1128/JVI.01429-16)
[^10.1099/jgv.0.001047]: Sisk JM, Frieman MB, Machamer CE. "Coronavirus S protein-induced fusion is blocked prior to hemifusion by Abl kinase inhibitors" _J. Gen. Virol._, 2018. [10.1099/jgv.0.001047](https://doi.org/10.1099/jgv.0.001047)
[^10.1016/j.jiph.2016.04.001]: Rossignol J. "Nitazoxanide, a new drug candidate for the treatment of Middle East respiratory syndrome coronavirus" _J Infect Public Health_, 2016. [10.1016/j.jiph.2016.04.001](https://doi.org/10.1016/j.jiph.2016.04.001)
[^10.1101/2020.03.11.987016]: Matsuyama S, Kawase M, et int., Fukushi S. "The inhaled corticosteroid ciclesonide blocks coronavirus RNA replication by targeting viral NSP15" _bioRxiv_, 2020. [10.1101/2020.03.11.987016](https://doi.org/10.1101/2020.03.11.987016)
[^10.1016/j.jiac.2020.04.007]: Iwabuchi K, Yoshie K, et int., Morishima T. "Therapeutic potential of ciclesonide inahalation for COVID-19 pneumonia: Report of three cases" _J Infect Chemother_, 2020. [10.1016/j.jiac.2020.04.007](https://doi.org/10.1016/j.jiac.2020.04.007)
[^10.3390/ph13030051]: Bleasel MD, Peterson GM. "Emetine, Ipecac, Ipecac Alkaloids and Analogues as Potential Antiviral Agents for Coronaviruses" _Pharmaceuticals (Basel)_, 2020. [10.3390/ph13030051](https://doi.org/10.3390/ph13030051)
[^10.4049/jimmunol.178.8.5200]: Gowen BB, Wong M, et int., Sidwell RW. "TLR3 is essential for the induction of protective immunity against Punta Toro Virus infection by the double-stranded RNA (dsRNA), poly(I:C12U), but not Poly(I:C): differential recognition of synthetic dsRNA molecules" _J. Immunol._, 2007. [10.4049/jimmunol.178.8.5200](https://doi.org/10.4049/jimmunol.178.8.5200)
[^10.1016/j.virol.2009.09.023]: Day CW, Baric R, et int., Barnard DL. "A new mouse-adapted strain of SARS-CoV as a lethal model for evaluating antiviral agents in vitro and in vivo" _Virology_, 2009. [10.1016/j.virol.2009.09.023](https://doi.org/10.1016/j.virol.2009.09.023)
[^10.1016/j.antiviral.2016.12.007]: Kumaki Y, Salazar AM, et int., Barnard DL. "Prophylactic and therapeutic intranasal administration with an immunomodulator, Hiltonol® (Poly IC:LC), in a lethal SARS-CoV-infected BALB/c mouse model" _Antiviral Res._, 2017. [10.1016/j.antiviral.2016.12.007](https://doi.org/10.1016/j.antiviral.2016.12.007)
[^10.1128/AAC.00440-20]: Ekins S, Madrid PB. "Tilorone: A Broad-Spectrum Antiviral For Emerging Viruses" _Antimicrob. Agents Chemother._, 2020. [10.1128/AAC.00440-20](https://doi.org/10.1128/AAC.00440-20)
[^10.1016/j.virol.2018.06.011]: de Wilde AH, Pham U, et int., Snijder EJ. "Cyclophilins and cyclophilin inhibitors in nidovirus replication" _Virology_, 2018. [10.1016/j.virol.2018.06.011](https://doi.org/10.1016/j.virol.2018.06.011)
[^10.1099/vir.0.034983-0]: de Wilde AH, Zevenhoven-Dobbe JC, et int., van Hemert MJ. "Cyclosporin A inhibits the replication of diverse coronaviruses" _J. Gen. Virol._, 2011. [10.1099/vir.0.034983-0](https://doi.org/10.1099/vir.0.034983-0)
[^10.1016/j.virusres.2016.11.011]: de Wilde AH, Falzarano D, et int., Snijder EJ. "Alisporivir inhibits MERS- and SARS-coronavirus replication in cell culture, but not SARS-coronavirus infection in a mouse model" _Virus Res._, 2017. [10.1016/j.virusres.2016.11.011](https://doi.org/10.1016/j.virusres.2016.11.011)
[^10.1016/j.virusres.2014.02.010]: Carbajo-Lozoya J, Ma-Lauer Y, et int., von Brunn A. "Human coronavirus NL63 replication is cyclophilin A-dependent and inhibited by non-immunosuppressive cyclosporine A-derivatives including Alisporivir" _Virus Res._, 2014. [10.1016/j.virusres.2014.02.010](https://doi.org/10.1016/j.virusres.2014.02.010)
[^10.1016/j.antiviral.2020.104787]: Caly L, Druce JD, et int., Wagstaff KM. "The FDA-approved Drug Ivermectin inhibits the replication of SARS-CoV-2 in vitro" _Antiviral Research_, 2020. [10.1016/j.antiviral.2020.104787](https://doi.org/10.1016/j.antiviral.2020.104787)
[^10.4269/ajtmh.20-0271]: Chaccour C, Hammann F, et int., Rabinovich NR. "Ivermectin and Novel Coronavirus Disease (COVID-19): Keeping Rigor in Times of Urgency" _Am. J. Trop. Med. Hyg._, 2020. [10.4269/ajtmh.20-0271](https://doi.org/10.4269/ajtmh.20-0271)

<!--stackedit_data:
eyJoaXN0b3J5IjpbMTU3NDc4NDcxMiw4NzM1MDI0NTgsMTcxNz
UxMTU5LC03NDM0MjM3Myw0NjkxMDk3MjMsMTU1MDc5NDMwNiw3
NDg5ODYwMiwtMTIzNTY4MjA0MCwtMTYzNTczNTEsLTEzNDQzNj
Q4MDgsMTI3MjYyMDUsOTM0MzA2ODcwLC0xMTY5ODAyNDE3LC02
ODg4MDQ0Myw0MDk3NDQ1ODYsMjAyODY2MDI2NiwyOTIzNTcyMT
YsMTQ1OTI5OTE5NCw5NzQ0MDY5MTIsLTM5MzI2MDc5M119
-->