<div style="clear: both; height: 0;" />

## Variants of concern

<!-- B.1.1.7 -->
<section id="sec_b-1-1-7" class="figure-section wrap">

<div class="figure-caption">

### [B.1.1.7, aka 501Y.V1 variant, 20I/501Y.V1, and UK COVID variant](#sec_b-1-1-7)

Rambaut A, Loman N, et int., and Volz E. “Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations.” Virological.org, 2020. [^Rambaut20]

</div>

<div class="figure-image">

[genomemap]
b-1-1-7
[/genomemap]

</div>

</section>

<!-- B.1.1.7 E484K-->
<section id="sec_b-1-1-7-484k" class="figure-section wrap">

<div class="figure-caption">

### [B.1.1.7 with S:E484K](#sec_b-1-1-7-484k)

Latif AA, Gangavarapu K, et int., and the Center for Viral Systems Biology . [“B.1.1.7 Lineage with S:E484K Report.”](https://outbreak.info/situation-reports?country=United%20Kingdom&country=United%20States&division=California&pango=B.1.1.7&muts=S%3AE484K&selected=United%20States&selectedType=country) outbreak.info, 2021. [^B117E484]

</div>

<div class="figure-image">

[genomemap]
b-1-1-7-484k
[/genomemap]

</div>

</section>

<!-- B.1.351 -->
<section id="sec_b-1-351" class="figure-section wrap">

<div class="figure-caption">

### [B.1.351, aka 501.V2 variant, 20C/501Y.V2, and South African COVID-19 variant](#sec_b-1-351)

Tegally H, Wilkinson E, et int., and de Oliveira T. “Emergence and rapid spread of a new severe acute respiratory syndrome-related coronavirus 2 (SARS-CoV-2) lineage with multiple spike mutations in South Africa.” _medRxiv_, 2020. [doi.org/10.1101/2020.12.21.20248640](https://doi.org/10.1101/2020.12.21.20248640). [^Tegally20]

</div>

<div class="figure-image">

[genomemap]
b-1-351
[/genomemap]

</div>

</section>

<!-- P.1 -->
<section id="sec_p-1" class="figure-section wrap">

<div class="figure-caption">

### [P.1, aka B.1.1.28.1, 20J/501Y.V3 variant, and K417T/E484K/N501Y](#sec_p-1)

Naveca F, Nascimento V, et int., and Bello G. “Phylogenetic relationship of SARS-CoV-2 sequences from Amazonas with emerging Brazilian variants harboring mutations E484K and N501Y in the Spike protein.” Virological.org, 2021.[^Naveca21][^Faria21]

</div>

<div class="figure-image">

[genomemap]
p-1
[/genomemap]

</div>

</section>

## Variants of Interest

<!-- PHL-B.1.1.28 -->
<section id="sec_phl-b-1-1-28" class="figure-section wrap">

<div class="figure-caption">

### [PHL-B.1.1.28](#phl-b-1-1-28)

Tablizo FA, Kim KM, et int., and Saloma CP. “Genome sequencing and analysis of an emergent SARS-CoV-2 variant characterized by multiple spike protein mutations detected from the Central Visayas Region of the Philippines.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.03.21252812](https://doi.org/10.1101/2021.03.03.21252812
). [^Tablizo21]

</div>

<div class="figure-image">

[genomemap]
phl-b-1-1-28
[/genomemap]

</div>

</section>

<!-- B.1.141 -->
<section id="sec_n439k" class="figure-section wrap">

<div class="figure-caption">

### [B.1.141, aka N439K](#sec_n439k)

Thomson EC, Rosen LE, et int., and Snell G. “The circulating SARS-CoV-2 spike variant N439K maintains fitness while evading antibody-mediated immunity.” _bioRxiv_, 2020. [doi.org/10.1101/2020.11.04.355842](https://doi.org/10.1101/2020.11.04.355842). [^Thomson20]

</div>

<div class="figure-image">

[genomemap]
n439k
[/genomemap]

</div>

</section>

<!-- B.1.526 -->
<section id="sec_b-1-526" class="figure-section wrap">

<div class="figure-caption">

### [B.1.526](#sec_b-1-526)

West A, Barnes C, et int., and Bjorkman P. “SARS-CoV-2 lineage B.1.526 emerging in the New York region detected by software utility created to query the spike mutational landscape.” _bioRxiv_, 2020. [doi.org/10.1101/2021.02.14.431043](https://doi.org/10.1101/2021.02.14.431043). [^West21]

</div>

<div class="figure-image">

[genomemap]
b-1-526
[/genomemap]

</div>

</section>

<!-- b-1-429 -->
<section id="sec_b-1-429" class="figure-section wrap">

<div class="figure-caption">

### [B.1.429, aka CAL.20C](#sec_b-1-429)

Deng X, Garcia-Knight MA, et int., and Chiu CY. “Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation.” _medRxiv_, 2021. [doi.org/10.1101/2021.03.07.21252647](https://doi.org/10.1101/2021.03.07.21252647).[^Deng21]

</div>

<div class="figure-image">

[genomemap]
b-1-429
[/genomemap]

</div>

</section>

<!-- Cluster 5 -->
<section id="sec_cluster5" class="figure-section wrap">

<div class="figure-caption">

### [Cluster 5](#sec_cluster5)
Lassaunière R, Fonager J, et int., and Fomsgaard A. “SARS-CoV-2 spike mutations arising in Danish mink and their spread to humans.” Statens Serum Institut, 2020. [^Lassauniere20]

</div>

<div class="figure-image">

[genomemap]
cluster5
[/genomemap]

</div>

</section>

<!-- COH.20G/501Y -->
<section id="sec_tu-501y" class="figure-section wrap">

<div class="figure-caption">

### [COH.20G/501Y](#sec_tu-501y)
Tu H, Avenarius MR, et int., and Jones D. “Distinct Patterns of Emergence of SARS-CoV-2 Spike Variants including N501Y in Clinical Samples in Columbus Ohio.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.12.426407](https://doi.org/10.1101/2021.01.12.426407).[^Tu21]

</div>

<div class="figure-image">

[genomemap]
tu-501y
[/genomemap]

</div>

</section>

<!-- COH.20G/677H -->
<section id="sec_coh-20g-677h" class="figure-section wrap">

<div class="figure-caption">

### [COH.20G/677H](#sec_coh-20g-677h)
Tada T, Dcosta BM, et int., and Landau NR. “Neutralization of viruses with European, South African, and United States SARS-CoV-2 variant spike proteins by convalescent sera and BNT162b2 mRNA vaccine-elicited antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.05.430003](https://doi.org/10.1101/2021.02.05.430003).[^Tada21]

</div>

<div class="figure-image">

[genomemap]
coh-20g-677h
[/genomemap]

</div>

</section>

<!-- P.2 -->
<section id="sec_p-2" class="figure-section wrap">

<div class="figure-caption">

### [P.2, evolved from B.1.1.28](#sec_p-2)
Naveca F, Nascimento V, et int., and Bello G. “Phylogenetic relationship of SARS-CoV-2 sequences from Amazonas with emerging Brazilian variants harboring mutations E484K and N501Y in the Spike protein.” Virological.org, 2021.[^Naveca21][^Faria21]

</div>

<div class="figure-image">

[genomemap]
p-2
[/genomemap]

</div>

</section>

<!-- B.1.1.35 N9 -->
<section id="sec_b-1-1-33-n9" class="figure-section wrap">

<div class="figure-caption">

### [B.1.1.33 N9](#sec_p-1-1-33-n9)
Resende PC, Graf T, et int., and Siqueira MM. “A potential SARS-CoV-2 variant of interest (VOI) harboring mutation E484K in the Spike protein was identified within lineage B.1.1.33 circulating in Brazil.” _biorxiv_, 2021.[^Resende21]

</div>

<div class="figure-image">

[genomemap]
b-1-1-33-n9
[/genomemap]

</div>

</section>

## Prolonged Infections

<!-- Kemp S -->
<section id="sec_kemp21" class="figure-section wrap">

<div class="figure-caption">

### [Kemp S, Harvey W, et int., and Gupta RK.](#sec_kemp21)

Kemp S, Harvey W, et int., and Gupta RK. “SARS-CoV-2 evolution during treatment of chronic infection.” _Nature_, 2021. [doi.org/10.1038/s41586-021-03291-y](https://doi.org/10.1038/s41586-021-03291-y).[^Kemp21] Other detected spike mutations during prolonged infections: S:Y200H, S:T240I, S:P330S

</div>

<div class="figure-image">

[genomemap]
kemp21
[/genomemap]

</div>

</section>

<!-- McCarthy -->
<section id="sec_mccarthy20" class="figure-section wrap">

<div class="figure-caption">

### [McCarthy KR, Rennick LJ, et int., and Duprex WP.](#sec_mccarthy20)

McCarthy KR, Rennick LJ, et int., and Duprex WP. “Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape.” _bioRxiv_, 2020. [doi.org/10.1101/2020.11.19.389916](https://doi.org/10.1101/2020.11.19.389916).[^McCarthy20]

</div>

<div class="figure-image">

[genomemap]
mccarthy20
[/genomemap]

</div>

</section>

<!-- Choi B -->
<section id="sec_choi20" class="figure-section wrap">

<div class="figure-caption">

### [Choi B, Choudhary MC, et int., and Li JZ.](#sec_choi20)

Choi B, Choudhary MC, et int., and Li JZ. “Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host.” _New England Journal of Medicine_, 2020. [doi.org/10.1056/NEJMc2031364](https://doi.org/10.1056/NEJMc2031364). \[[PubMed33176080](https://www.ncbi.nlm.nih.gov/pubmed/33176080/)\] \[[PMC7673303](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673303/)\] [^Choi20]

</div>

<div class="figure-image">

[genomemap]
choi20
[/genomemap]

</div>

</section>

<!-- Avanzato VA -->
<section id="sec_avanzato20" class="figure-section wrap">

<div class="figure-caption">

### [Avanzato VA, Matson MJ, et int., and Munster VJ.](#sec_avanzato20)

Avanzato VA, Matson MJ, et int., and Munster VJ. “Case Study: Prolonged Infectious SARS-CoV-2 Shedding from an Asymptomatic Immunocompromised Individual with Cancer.” _Cell_, 2020. [doi.org/10.1016/j.cell.2020.10.049](https://doi.org/10.1016/j.cell.2020.10.049). \[[PubMed33248470](https://www.ncbi.nlm.nih.gov/pubmed/33248470/)\] \[[PMC7640888](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7640888/)\] [^Avanzato20]

</div>

<div class="figure-image">

[genomemap]
avanzato20
[/genomemap]

</div>

</section>

<!-- Bazykin G -->
<section id="sec_bazykin20" class="figure-section wrap">

<div class="figure-caption">

### [Bazykin G, Stanevich O, et int., and Lioznov D.](#sec_bazykin20)

Bazykin G, Stanevich O, et int., and Lioznov D. “Emergence of Y453F and Δ69-70HV mutations in a lymphoma patient with long-term COVID-19.” Virological.org, 2021. [^Bazykin21]

</div>

<div class="figure-image">

[genomemap]
bazykin20
[/genomemap]

</div>

</section>

<!-- Khatamzas E -->
<section id="sec_khatamzas20" class="figure-section wrap">

<div class="figure-caption">

### [Khatamzas E, Rehn A, et int., and Antwerpen M.](#sec_khatamzas20)

Khatamzas E, Rehn A, et int., and Antwerpen M. “Emergence of multiple SARS-CoV-2 mutations in an immunocompromised host.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.10.20248871](https://doi.org/10.1101/2021.01.10.20248871). [^Khatamzas21]

</div>


<div class="figure-image">

[genomemap]
khatamzas20
[/genomemap]

</div>

</section>
