[toc]
- [Variants with adaptive mutations](#variants.with.adaptive.mutations)
  - [Alpha variant, aka B.1.1.7, 501Y.V1, 20I/501Y.V1, and UK COVID variant](#sec_alpha)
  - [Alpha variant with S:E484K](#sec_alpha-484k)
  - [Beta variant, aka B.1.351, 501Y.V2, 20C/501Y.V2, and South African COVID-19 variant](#sec_beta)
  - [Gamma variant, aka P.1, B.1.1.28.1, 20J/501Y.V3, and K417T/E484K/N501Y](#sec_gamma)
  - [Delta variant, aka B.1.617.2](#sec_delta)
  - [Epsilon variant, aka B.1.427/B.1.429, and CAL.20C/S:452R](#sec_epsilon)
  - [Zeta variant, aka P.2, evolved from B.1.1.28](#sec_zeta)
  - [Eta variant, aka B.1.525](#sec_eta)
  - [Theta variant, aka P.3, and PHL-B.1.1.28](#sec_theta)
  - [Iota variant, aka B.1.526](#sec_iota)
  - [Kappa variant, aka B.1.617.1](#sec_kappa)
  - [Lambda variant, aka C.37, and B.1.1.1 sublineage in Peru and Chile](#sec_lambda)
  - [B.1.x](#sec_b-1-x)
  - [B.1.617](#sec_b-1-617)
  - [B.1.617.3](#sec_b-1-617-3)
  - [B.1.620](#sec_b-1-620)
  - [B.1.621](#sec_b-1-621)
  - [B.1.258, aka N439K](#sec_n439k)
  - [B.1.1.298, aka Cluster 5](#sec_cluster5)
  - [R.1](#sec_r-1)
  - [COH.20G/501Y, EPI_ISL_832378 and EPI_ISL_826521](#sec_coh-20g-501y)
  <!-- - [COH.20G/677H, EPI_ISL_826463](#sec_coh-20g-677h) -->
  - [N.9](#sec_n9)
  - [A.VOI.V2](#sec_a-voi-v2)
  - [A.27](#sec_a-27)
  - [AV.1](#sec_av-1)
- [Prolonged Infections](#prolonged.infections)
  - [Kemp S, Collier DA, et int., and Gupta RK.](#sec_kemp21)
  - [McCarthy KR, Rennick LJ, et int., and Duprex WP.](#sec_mccarthy20)
  - [Choi B, Choudhary MC, et int., and Li JZ.](#sec_choi20)
  - [Avanzato VA, Matson MJ, et int., and Munster VJ.](#sec_avanzato20)
  - [Bazykin G, Stanevich O, et int., and Lioznov D.](#sec_bazykin20)
  - [Khatamzas E, Rehn A, et int., and Antwerpen M.](#sec_khatamzas20)
  - [Chen L, Zody CM, et int., and Kreiswirth BN.](#sec_chen21b)
  - [Focosi D, Novazzi F, et int., and Maggi F.](#sec_focosi21b)
[/toc]

<div style="clear: both; height: 0;" />

## Variants with adaptive mutations

<!-- Alpha variant -->
<section id="sec_alpha" class="figure-section compact">

<div class="figure-caption">

### [Alpha variant, aka B.1.1.7, 501Y.V1, 20I/501Y.V1, and UK COVID variant](#sec_alpha)

Rambaut A, Loman N, et int., and Volz E. “Preliminary genomic characterisation of an emergent SARS-CoV-2 lineage in the UK defined by a novel set of spike mutations.” Virological.org, 2020. [^Rambaut20]

[Outbreak.info B.1.1.7 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.1.7) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-1-7
[/genomemap]

</div>

</section>

<!-- Alpha E484K-->
<section id="sec_alpha-484k" class="figure-section compact">

<div class="figure-caption">

### [Alpha variant with S:E484K](#sec_alpha-484k)

[B.1.1.7 Lineage with S:E484K Report](https://outbreak.info/situation-reports?pango=B.1.1.7&muts=S%3AE484K) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-1-7-484k
[/genomemap]

</div>

</section>

<!-- Beta variant -->
<section id="sec_beta" class="figure-section compact">

<div class="figure-caption">

### [Beta variant, aka B.1.351, 501Y.V2, 20C/501Y.V2, and South African COVID-19 variant](#sec_beta)

Tegally H, Wilkinson E, et int., and Oliveira T. “Detection of a SARS-CoV-2 variant of concern in South Africa.” _nature_, 2020. [doi.org/10.1038/s41586-021-03402-9](https://doi.org/10.1038/s41586-021-03402-9). [^Tegally21]

[Outbreak.info B.1.351 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.351) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-351
[/genomemap]

</div>

</section>

<!-- Gamma variant -->
<section id="sec_gamma" class="figure-section compact">

<div class="figure-caption">

### [Gamma variant, aka P.1, B.1.1.28.1, 20J/501Y.V3 variant, and K417T/E484K/N501Y](#sec_gamma)

Naveca F, Nascimento V, et int., and Bello G. “Phylogenetic relationship of SARS-CoV-2 sequences from Amazonas with emerging Brazilian variants harboring mutations E484K and N501Y in the Spike protein.” Virological.org, 2021.[^Naveca21][^Faria21]

[Outbreak.info P.1 Lineage Report](https://outbreak.info/situation-reports?pango=P.1) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
p-1
[/genomemap]

</div>

</section>

<!-- Delta variant -->
<section id="sec_delta" class="figure-section compact">

<div class="figure-caption">

### [Delta variant, aka B.1.617.2](#sec_delta)

[Outbreak.info B.1.617.2 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.617.2) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-617-2
[/genomemap]

</div>

</section>

<!-- Epsilon variant -->
<section id="sec_epsilon" class="figure-section compact">

<div class="figure-caption">

### [Epsilon, aka B.1.427/B.1.429, and CAL.20C/S:452R](#sec_epsilon)

Deng X, Garcia-Knight MA, et int., and Chiu CY. “Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation.” _medRxiv_, 2021. [doi.org/10.1101/2021.03.07.21252647](https://doi.org/10.1101/2021.03.07.21252647).[^Deng21]

[Outbreak.info B.1.427 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.427) [^Outbreak20]

[Outbreak.info B.1.429 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.429) [^Outbreak20]
</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-429
[/genomemap]

</div>

</section>

<!-- Zeta variant -->
<section id="sec_zeta" class="figure-section compact">

<div class="figure-caption">

### [Zeta variant, aka P.2, evolved from B.1.1.28](#sec_zeta)
Naveca F, Nascimento V, et int., and Bello G. “Phylogenetic relationship of SARS-CoV-2 sequences from Amazonas with emerging Brazilian variants harboring mutations E484K and N501Y in the Spike protein.” Virological.org, 2021.[^Naveca21][^Faria21]

[Outbreak.info P.2 Lineage Report](https://outbreak.info/situation-reports?pango=P.2) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
p-2
[/genomemap]

</div>

</section>


<!-- Eta variant -->
<section id="sec_eta" class="figure-section compact">

<div class="figure-caption">

### [Eta variant, aka B.1.525](#sec_eta)

[Outbreak.info B.1.525 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.525) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-525
[/genomemap]

</div>

</section>

<!-- Theta variant -->
<section id="sec_theta" class="figure-section compact">

<div class="figure-caption">

### [Theta variant, aka P.3, and PHL-B.1.1.28](#sec_theta)

Tablizo FA, Kim KM, et int., and Saloma CP. “Genome sequencing and analysis of an emergent SARS-CoV-2 variant characterized by multiple spike protein mutations detected from the Central Visayas Region of the Philippines.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.03.21252812](https://doi.org/10.1101/2021.03.03.21252812
). [^Tablizo21]

[Outbreak.info P.3 Lineage Report](https://outbreak.info/situation-reports?pango=P.3) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
phl-b-1-1-28
[/genomemap]

</div>

</section>

<!-- Iota variant -->
<section id="sec_iota" class="figure-section compact">

<div class="figure-caption">

### [Iota variant, aka B.1.526](#sec_iota)

West A, Barnes C, et int., and Bjorkman P. “SARS-CoV-2 lineage B.1.526 emerging in the New York region detected by software utility created to query the spike mutational landscape.” _bioRxiv_, 2020. [doi.org/10.1101/2021.02.14.431043](https://doi.org/10.1101/2021.02.14.431043). [^West21]

[Outbreak.info B.1.526 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.526) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-526
[/genomemap]

</div>

</section>

<!-- Kappa variant -->
<section id="sec_kappa" class="figure-section compact">

<div class="figure-caption">

### [Kappa variant, aka B.1.617.1](#sec_kappa)

[Outbreak.info B.1.617.1 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.617.1) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-617-1
[/genomemap]

</div>

</section>

<!-- Lambda variant -->
<section id="sec_lambda" class="figure-section compact">

<div class="figure-caption">

### [Lambda variant, aka C.37, and B.1.1.1 sublineage in Peru and Chile](#sec_lambda)
Romero PE, Davila-Barclay A, et int., and Tsukayama P. “Novel sublineage within B.1.1.1 currently expanding in Peru and Chile, with a convergent deletion in the ORF1a gene (Δ3675-3677) and a novel deletion in the Spike gene (Δ246-252, G75V, T76I, L452Q, F490S, T859N).” Virological.org, 2021. [^Romero21]

[Outbreak.info C.37 Lineage Report](https://outbreak.info/situation-reports?pango=C.37) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-1-1-sub
[/genomemap]

</div>

</section>

<!-- b-1-x -->
<section id="sec_b-1-x" class="figure-section compact">

<div class="figure-caption">

### [B.1.x](#sec_b-1-x)

Thornlow B, Hinrichs AS, et int., and Corbett-Detig R. “A new SARS-CoV-2 lineage that shares mutations with known Variants of Concern is rejected by automated sequence repository quality control.” _bioRxiv_, 2021. [doi.org/10.1101/2021.04.05.438352](https://doi.org/10.1101/2021.04.05.438352).[^Thornlow21]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-x
[/genomemap]

</div>

</section>


<!-- B.1.617 -->
<section id="sec_b-1-617" class="figure-section compact">

<div class="figure-caption">

### [B.1.617](#sec_b-1-617)

[Outbreak.info B.1.617 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.617) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-617
[/genomemap]

</div>

</section>

<!-- B.1.617.3 -->
<section id="sec_b-1-617-3" class="figure-section compact">

<div class="figure-caption">

### [B.1.617.3](#sec_b-1-617-3)

[Outbreak.info B.1.617.3 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.617.3) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-617-3
[/genomemap]

</div>

</section>


<!-- B.1.620 -->
<section id="sec_b-1-620" class="figure-section compact">

<div class="figure-caption">

### [B.1.620](#sec_b-1-620)

Dudas G, Hong SL, et int., and Baele G. “Travel-driven emergence and spread of SARS-CoV-2 lineage B.1.620 with multiple VOC-like mutations and deletions in Europe.” _medRxiv_, 2021. [doi.org/10.1101/2021.05.04.21256637](https://doi.org/10.1101/2021.05.04.21256637). [^Dudas21]

[Outbreak.info B.1.620 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.620) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-620
[/genomemap]

</div>

</section>


<!-- B.1.621 -->
<section id="sec_b-1-621" class="figure-section compact">

<div class="figure-caption">

### [B.1.621](#sec_b-1-621)

Laiton-Donato K, Franco-Munoz C, et int., and Mercado-Reyes M. “Characterization of the emerging B.1.621 variant of interest of SARS-CoV-2.” _medRxiv_, 2021. [doi.org/10.1101/2021.05.08.21256619](https://doi.org/10.1101/2021.05.08.21256619). [^Laiton-Donato21]

[Outbreak.info B.1.621 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.621) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
b-1-621
[/genomemap]

</div>

</section>

<!-- B.1.258 -->
<section id="sec_n439k" class="figure-section compact">

<div class="figure-caption">

### [B.1.258, aka N439K](#sec_n439k)

Thomson EC, Rosen LE, et int., and Snell G. “Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity.” _Cell_, 2021. [doi.org/10.1016/j.cell.2021.01.037](https://doi.org/10.1016/j.cell.2021.01.037). [^Thomson21]

[Outbreak.info B.1.258 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.258) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
n439k
[/genomemap]

</div>

</section>

<!-- Cluster 5 -->
<section id="sec_cluster5" class="figure-section compact">

<div class="figure-caption">

### [B.1.1.298, aka Cluster 5](#sec_cluster5)
Lassaunière R, Fonager J, et int., and Fomsgaard A. “SARS-CoV-2 spike mutations arising in Danish mink and their spread to humans.” Statens Serum Institut, 2020. [^Lassauniere20]

[Outbreak.info B.1.1.298 Lineage Report](https://outbreak.info/situation-reports?pango=B.1.1.298) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
cluster5
[/genomemap]

</div>

</section>

<!-- R1 -->
<section id="sec_r-1" class="figure-section compact">

<div class="figure-caption">

### [R.1](#sec_r-1)

Hirotsu Y and Omata M. "Detection of R.1 lineage severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) with spike protein W152L/E484K/G769V mutations in Japan." _PLOS PATHOGENS_, 2021. [doi.org/10.1371/journal.ppat.1009619](https://doi.org/10.1371/journal.ppat.1009619) [^Hirotsu21]


[Outbreak.info R.1 Lineage Report](https://outbreak.info/situation-reports?pango=R.1) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
r-1
[/genomemap]

</div>

</section>

<!-- COH.20G/501Y -->
<section id="sec_coh-20g-501y" class="figure-section compact">

<div class="figure-caption">

### [COH.20G/501Y, EPI_ISL_832378 and EPI_ISL_826521](#sec_coh-20g-501y)
Tu H, Avenarius MR, et int., and Jones D. “Distinct Patterns of Emergence of SARS-CoV-2 Spike Variants including N501Y in Clinical Samples in Columbus Ohio.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.12.426407](https://doi.org/10.1101/2021.01.12.426407).[^Tu21]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
coh-20g-501y
[/genomemap]

</div>

</section>

<!-- COH.20G/677H -->
<!-- <section id="sec_coh-20g-677h" class="figure-section compact">

<div class="figure-caption">

### [COH.20G/677H, EPI_ISL_826463](#sec_coh-20g-677h)

Tu H, Avenarius MR, et int., and Jones D. “Distinct Patterns of Emergence of SARS-CoV-2 Spike Variants including N501Y in Clinical Samples in Columbus Ohio.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.12.426407](https://doi.org/10.1101/2021.01.12.426407).[^Tu21]


</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
coh-20g-677h
[/genomemap]

</div>

</section> -->

<!-- N.9 -->
<section id="sec_n9" class="figure-section compact">

<div class="figure-caption">

### [N.9](#sec_n9)
Resende PC, Graf T, et int., and Siqueira MM. “A potential SARS-CoV-2 variant of interest (VOI) harboring mutation E484K in the Spike protein was identified within lineage B.1.1.33 circulating in Brazil.” _biorxiv_, 2021. [doi.org/10.1101/2021.03.12.434969](https://doi.org/10.1101/2021.03.12.434969) [^Resende21]

[Outbreak.info N.9 Lineage Report](https://outbreak.info/situation-reports?pango=N.9) [^Outbreak20]


</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
n9
[/genomemap]

</div>

</section>

<!-- A.VOI.V2 -->
<section id="sec_a-voi-v2" class="figure-section compact">

<div class="figure-caption">

### [A.VOI.V2](#sec_a-voi-v2)
Oliveira T, Lutucuta S, et int., and Lessells RJ. “A novel variant of interest of SARS-CoV-2 with multiple spike mutations is identified from travel surveillance in Africa.” _biorxiv_, 2021. [doi.org/10.1101/2021.03.30.21254323](https://doi.org/10.1101/2021.03.30.21254323)

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
a-voi-v2
[/genomemap]

</div>

</section>


<!-- A.27 -->
<section id="sec_a-27" class="figure-section compact">

<div class="figure-caption">

### [A.27](#sec_a-27)
Calvignac-Spencer S, Budt M, et int., and Holzer M. “Emergence of SARS-CoV-2 lineage A.27 in Germany, expressing viral spike proteins with several amino acid replacements of interest, including L18F, L452R, and N501Y in the absence of D614G.” Virological.org, 2021. [^Calvignac-Spencer21]

[Outbreak.info A.27 Lineage Report](https://outbreak.info/situation-reports?pango=A.27) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
a-27
[/genomemap]

</div>

</section>


<!-- AV.1 -->
<section id="sec_av-1" class="figure-section compact">

<div class="figure-caption">

### [AV.1](#sec_av-1)
Hsu S, Parker MD, et int., and de Silva TI. “Detection of Spike Mutations; D80G, T95I, G142D, △144, N439K, E484K, P681H, I1130V, and D1139H, in B.1.1.482 Lineage (AV.1) Samples from South Yorkshire, UK.” Virological.org, 2021. [^Hsu21]

[Outbreak.info AV.1 Lineage Report](https://outbreak.info/situation-reports?pango=AV.1) [^Outbreak20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
av-1
[/genomemap]

</div>

</section>


## Prolonged Infections

<!-- Kemp S -->
<section id="sec_kemp21" class="figure-section compact">

<div class="figure-caption">

### [Kemp S, Collier DA, et int., and Gupta RK.](#sec_kemp21)

Kemp S, Collier DA, et int., and Gupta RK. “SARS-CoV-2 evolution during treatment of chronic infection.” _Nature_, 2021. [doi.org/10.1038/s41586-021-03291-y](https://doi.org/10.1038/s41586-021-03291-y).[^Kemp21] Other detected spike mutations during prolonged infections: S:Y200H, S:T240I, S:P330S

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
kemp21
[/genomemap]

</div>

</section>

<!-- McCarthy -->
<section id="sec_mccarthy20" class="figure-section compact">

<div class="figure-caption">

### [McCarthy KR, Rennick LJ, et int., and Duprex WP.](#sec_mccarthy20)

McCarthy KR, Rennick LJ, et int., and Duprex WP. “Recurrent deletions in the SARS-CoV-2 spike glycoprotein drive antibody escape.” _bioRxiv_, 2020. [doi.org/10.1101/2020.11.19.389916](https://doi.org/10.1101/2020.11.19.389916).[^McCarthy20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
mccarthy20
[/genomemap]

</div>

</section>

<!-- Choi B -->
<section id="sec_choi20" class="figure-section compact">

<div class="figure-caption">

### [Choi B, Choudhary MC, et int., and Li JZ.](#sec_choi20)

Choi B, Choudhary MC, et int., and Li JZ. “Persistence and Evolution of SARS-CoV-2 in an Immunocompromised Host.” _New England Journal of Medicine_, 2020. [doi.org/10.1056/NEJMc2031364](https://doi.org/10.1056/NEJMc2031364). \[[PubMed33176080](https://www.ncbi.nlm.nih.gov/pubmed/33176080/)\] \[[PMC7673303](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7673303/)\] [^Choi20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
choi20
[/genomemap]

</div>

</section>

<!-- Avanzato VA -->
<section id="sec_avanzato20" class="figure-section compact">

<div class="figure-caption">

### [Avanzato VA, Matson MJ, et int., and Munster VJ.](#sec_avanzato20)

Avanzato VA, Matson MJ, et int., and Munster VJ. “Case Study: Prolonged Infectious SARS-CoV-2 Shedding from an Asymptomatic Immunocompromised Individual with Cancer.” _Cell_, 2020. [doi.org/10.1016/j.cell.2020.10.049](https://doi.org/10.1016/j.cell.2020.10.049). \[[PubMed33248470](https://www.ncbi.nlm.nih.gov/pubmed/33248470/)\] \[[PMC7640888](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7640888/)\] [^Avanzato20]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
avanzato20
[/genomemap]

</div>

</section>

<!-- Bazykin G -->
<section id="sec_bazykin20" class="figure-section compact">

<div class="figure-caption">

### [Bazykin G, Stanevich O, et int., and Lioznov D.](#sec_bazykin20)

Bazykin G, Stanevich O, et int., and Lioznov D. “Emergence of Y453F and Δ69-70HV mutations in a lymphoma patient with long-term COVID-19.” Virological.org, 2021. [^Bazykin21]

</div>

<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
bazykin20
[/genomemap]

</div>

</section>

<!-- Khatamzas E -->
<section id="sec_khatamzas20" class="figure-section compact">

<div class="figure-caption">

### [Khatamzas E, Rehn A, et int., and Antwerpen M.](#sec_khatamzas20)

Khatamzas E, Rehn A, et int., and Antwerpen M. “Emergence of multiple SARS-CoV-2 mutations in an immunocompromised host.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.10.20248871](https://doi.org/10.1101/2021.01.10.20248871). [^Khatamzas21]

</div>


<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
khatamzas20
[/genomemap]

</div>

</section>

<!-- Chen L -->
<section id="sec_chen21b" class="figure-section compact">

<div class="figure-caption">

### [Chen L, Zody CM, et int., and Kreiswirth BN.](#sec_chen21b)

Chen L, Zody CM, et int., and Kreiswirth BN. “Emergence of multiple SARS-CoV-2 antibody escape variants in an immunocompromised host undergoing convalescent plasma treatment.” _medRxiv_, 2021. [doi.org/10.1101/2021.04.08.21254791](https://doi.org/10.1101/2021.04.08.21254791). [^Chen21b]

</div>


<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
chen21b
[/genomemap]

</div>

</section>

<!-- Focosi21b -->
<section id="sec_focosi21b" class="figure-section compact">

<div class="figure-caption">

### [Focosi D, Novazzi F, et int., and Maggi F.](#sec_focosi21b)

Focosi D, Novazzi F, et int., and Maggi F. “Emergence of SARS-CoV-2 Spike Escape Mutation Q493r After Bamlanivimab/Etesevimab Treatment for COVID-19.” _Research Square_, 2021. [doi.org/10.21203/rs.3.rs-524959/v1
](https://doi.org/10.21203/rs.3.rs-524959/v1). [^Focosi21b]

</div>


<div class="figure-image" style="min-width: 61.8%;">

[genomemap]
focosi21b
[/genomemap]

</div>

</section>
