<section id="fig1" class="figure-section">

<div class="figure-image">

![SARS-CoV-2 Trimeric Spike (side view and top view)](spike-structures/fig1.png#!maxHeight=35rem)

</div>

<div class="figure-caption">

## [Figure 1](#fig1) | SARS-CoV-2 Trimeric Spike.

Side view and top view. Each color represents one chain in the trimer. (PDB: [6VXX])

</div>

[6VXX]: http://www.rcsb.org/structure/6VXX

</section>

<section id="fig2" class="figure-section">

<div class="figure-image">

![Trimeric spike structures illustrating RBD (receptor binding domains) up (open; ACE2 accessible) and down (closed) configurations](spike-structures/fig2.png#!maxHeight=20rem)

</div>

<div class="figure-caption">

## [Figure 2](#fig2) | Trimeric spike structures illustrating RBD (receptor binding domains) up (open; ACE2 accessible) and down (closed) configurations.

The structures on the left capture one of three chains having RBD in an up configuration (red) whereas the structures on the right capture show all three chains having RBD in the down configuration. In both sets of structures, the three RBDs are colored red, blue, and yellow whereas the individual spike protomers are in cyan, green, and magenta. (PDB: [6VXX] \[left two\], [6VYB] \[right two\])

</div>

[6VYB]: http://www.rcsb.org/structure/6VYB

</section>

<section id="fig3" class="figure-section">

<div class="figure-image">

![Spike RBD bound to ACE-2](spike-structures/fig3.png#!maxHeight=20rem)

</div>

<div class="figure-caption">

## [Figure 3](#fig3) | Spike RBD bound to ACE-2.

One of the three spike RBD bound to the human ACE2 protein. A structure of the RBD bound to ACE2 (PDB [6M0J]) superimposed onto the spike trimer. (PDB [6VSB])

</div>

[6M0J]: http://www.rcsb.org/structure/6M0J
[6VSB]: http://www.rcsb.org/structure/6VSB

</section>

<section id="fig4" class="figure-section">

<div class="figure-image">

![Spike S1 and S2 subunits](spike-structures/fig4.png#!maxHeight=20rem)

</div>

<div class="figure-caption">

## [Figure 4](#fig4) | Spike S1 and S2 subunits.

The spike protein is processed by host proteases to generate two functional subunits, designated S1 (shown in blue) and S2 (shown in pink). S1, which forms the apex of the trimer, is responsible for attachment to the ACE2 receptor. S2 is responsible for fusion to the cell membrane. (PDB: [6VSB])

</div>

[6VSB]: http://www.rcsb.org/structure/6VSB

</section>

<section id="fig5" class="figure-section">

<div class="figure-image">

![S1 domains](spike-structures/fig5.png#!maxHeight=20rem)

</div>

<div class="figure-caption">

## [Figure 5](#fig5) | S1 domains.

The four S1 domains â€“ N terminal domain (NTD; aka S1A; cyan), receptor binding domain (RBD; aka S1B; green), SD1 (aka S1C; red), SD2 (aka S1D; purple) -- of one S1 chain are shown within the context of the spike trimer. The RBD contains a subdomain at its apex referred to as the receptor binding motif (RBM; ACE2-competing residues). The RBM shown in yellow contains the residues that bind to ACE2. (PDB: [6VSB])

</div>

</section>

<section id="table1" class="figure-section wrap">

<div class="figure-image">

[table]
antibodyMeta
[/table]

</div>

<div class="figure-caption">

## [Table 1](#table1) | SARS-CoV-2 neutralizing monoclonal antibodies (mAbs) with structural information from Protein Data Bank(PDB)

</div>

</section>

<section id="fig6" class="figure-section wrap">

<div class="figure-image">

![RBD ACE2-competing mAbs with large overlap](spike-structures/fig6.png#!maxHeight=100rem)

</div>

<div class="figure-caption">

## [Figure 6](#fig6) | RBD ACE2-competing mAbs with large overlap (>50%).

ACE2-competing mAbs bind to a high proportion of ACE2-binding residues. B38, BD-236, BD-604, BD-629, CC12.1, CC12.3, COVA2-40, COVA2-39, and CV30 are encoded by the same variable heavy chain gene IGHV3-53. The bottom row shows the RBD rotated 180 degrees. The ACE2-binding residues are shown in green.

</div>

</section>

<section id="fig7" class="figure-section wrap">

<div class="figure-image">

![RBD ACE2-competing mAbs with small overlap](spike-structures/fig7.png#!maxHeight=100rem)

</div>

<div class="figure-caption">

## [Figure 7](#fig7) | RBD ACE2-competing mAbs with small overlap (<=50%).

ACE2-competing mAbs bind to a small proportion of ACE2-binding residues. These mAbs generally bind a lower proportion of ACE2-binding residues (compared with those in Figure 6). H11-H4 and H11-D4 are nanobodies that bind to the same region. Only C105 is encoded by IGHV3-53. C144 can bind to both ACE2-competing and non-ACE2-competing residues of adjacent RBDs. The bottom row shows the RBD rotated 180 degrees. The ACE2-binding residues are shown in green.

</div>

</section>

<section id="fig8" class="figure-section">

<div class="figure-image">

![RBD non-ACE2-competing mAbs](spike-structures/fig8.png#!maxHeight=35rem)

</div>

<div class="figure-caption">

## [Figure 8](#fig8) | RBD non-ACE2-competing mAbs.

RBD core is defined as the part of the RBD that does not contain the ACE2-competing residues. CR3022 and S309 are non-ACE2-competing mAbs that bind non-overlapping RBD core epitopes. EY6A, H014, and COVA1-16 are non-ACE2-competing mAbs that overlap with CR3022 (PDB: [6M0J]). The bottom row shows the RBD rotated 180 degrees. The ACE2-binding residues are shown in green.

</div>

[6M0J]: http://www.rcsb.org/structure/6M0J

</section>

<section id="fig9" class="figure-section">

<div class="figure-image">

![Regeneron ACE2-competing mAbs](spike-structures/fig9.png#!maxHeight=35rem)

</div>

<div class="figure-caption">

## [Figure 9](#fig9) | Regeneron ACE2-competing mAbs.

REGN19033 and REGN10987 are ACE2-competing mAbs that bind to non-overlapping epitopes.

</div>

</section>

<section id="fig10" class="figure-section">

<div class="figure-image">

![RBD variants and escape mutations](spike-structures/fig10.png#!maxHeight=35rem)

</div>

<div class="figure-caption">

## [Figure 10](#fig10) | Naturally occurring spike RBD variants and mAb escape mutations.

Loop representation of RBD positions 333 to 526 showing SARS-CoV-2 variants reported in >0.05% of reported virus isolates (blue) (A) and escape mutations (magenta) reported during in vitro virus passage (Baum et al DOI; Weisblum et al DOI) with mAbs C105 and REGN10987 and REGN10933 (B). ACE2-binding residues are in yellow.

</div>

</section>


<section id="fig11" class="figure-section">

<div class="figure-image">

![Phylogenetic tree, multiple sequence alignment, and sequence variability of the SARS-related coronavirus (SARSr-CoV) spike receptor binding domain (RBD)](spike-structures/fig11.png#!maxHeight=60rem)

</div>

<div class="figure-caption">

## [Figure 11](#fig11) | Phylogenetic tree, multiple sequence alignment, and sequence variability of the SARS-related coronavirus (SARSr-CoV) spike receptor binding domain (RBD).

The phylogenetic tree was created using the TN93 distance model. The average distance to the closest leaf algorithm was then used to remove sequences within a genetic distance of 1.0% from another sequence. Clustal Omega and JalView were used to create and view the multiple sequence alignment of the remaining sequence. The SARSr RBD consensus of the filtered sequence is shown at the top of each alignment and the SARS-CoV-2 sequence is highlighted. The sequence entropy at each position is displayed on a structural representation of the RBD (PDB: 6M0J) using a PyMol script.

</div>

</section>


<section id="fig12" class="figure-section">

<div class="figure-image">

![Phylogenetic tree, multiple sequence alignment, and sequence variability of the SARS-related coronavirus (SARSr-CoV) trimeric spike.](spike-structures/fig12.png#!maxHeight=100rem)

</div>

<div class="figure-caption">

## [Figure 12](#fig12) | Phylogenetic tree, multiple sequence alignment, and sequence variability of the SARS-related coronavirus (SARSr-CoV) trimeric spike.

The phylogenetic tree was created using the TN93 distance model. The average distance to the closest leaf algorithm was then used to remove sequences within a genetic distance of 1.0% from another sequence. Clustal Omega and JalView were used to create and view the multiple sequence alignment of the remaining sequence. The SARSr consensus of the filtered sequence is shown at the top of each alignment and the SARS-CoV-2 sequence is highlighted. The sequence entropy at each position is displayed on a structural representation of the trimeric spike (PDB: 6XR8) using a PyMol script.

</div>

</section>
