## Summaries

### Table 1: Virus Variants or Spike Mutations vs Monoclonal antibodies

Fold-reduced neutralizing susceptibility of viruses variants spike mutations to monoclonal antibodies in in phaese 3 clinical trials.


[table]
table_mab
[/table]


**Footnote:**

- B.1.1.7 contains the following spike mutations: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H.
- B.1.351 contains the following spike mutations: L18F, D80A, D215G, ∆242-244, R246I, K417N, E484K, N501Y, D614G, and A701V.
- P.1 contains the following spike mutations: L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
- B.1.526 contains the following spike mutations: L5F, T95I, D253G, S477N, E484K, D614G, A701V
- CAL.20C contains the following spike mutations: S13I, W152C, L452R, D614G
- The fold change is the median value of results, the subscript is the number of results. “-“ indicates absence of susceptibility data. “\*“ indicates conflict between two research groups, with one showing high-level of reduced susceptibility especially using IC80 rather than IC50.

MAb abbreviations

- **BAM**: Bamlanivimab/LY-CoV555
- **CAS**: Casirivimab/REGN10933
- **IMD**: Imdevimab/REGN10987
- **CAS/IMD**: Casirivimab+imdevimab/REGN-COV2
- **ETE**: Etesevimab/LY-CoV016/JS016/CB6
- **CIL**: Cilgavimab/COV2-2130/AZD1061
- **TIX**: Tixagevimab/COV2-2196/AZD8895
- **TIX-CIL**: Tixagevimab+Cilgavimab
- **BAM-ETE**: Bamlanivimab+Etesevimab
- **SOT**: Sotrovimab/Vir-7831/S309


<!-- ### Figure 1: Virus Variants or Spike Mutations vs Monoclonal antibodies

<section id="sec_mab-fold" class="figure-section wrap">

<div class="figure-image">

![Monoclonal antibodies](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/mab-fold-fig-1.png#!maxWidth=95%)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to monoclonal neutralizing antibodies (mAbs) associated with three variants of concern (B.1.1.7, B.1.351, and P.1) and six spike amino acid mutations E484K, N501Y, L452R, K417N, N439K, and Y453F. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. Each of the _in vitro_ susceptibility experiments were performed using pseudovirus assays. Some of the pseudovirus assays performed with single amino acid mutations contained D614G while others did not. D614G has been reported to slightly increase neutralization susceptibility. The Y-axis indicates the log-fold reduction in susceptibility. Data are displayed for mAbs in advanced clinical trials and several additional prototypical mAbs with high-resolution 3-D structures and defined mechanisms of mAb-RBD binding.

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility.
</p>


</div>

</section> -->



### Table 2: Virus Variants or Spike Mutations vs Convalescent Plasma and Plasma from Vaccinated Persons


[table]
table_plasma
[/table]


**Footnote:**

- Convalescent plasma in all studies is presumed likely to have been obtained from persons infected with spike variants lacking mutations associated with reduced neutralization susceptibility.
- B.1.1.7 contains the following spike mutations: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H.
- B.1.351 contains the following spike mutations: L18F, D80A, D215G, ∆242-244, R246I, K417N, E484K, N501Y, D614G, and A701V.
- P.1 contains the following spike mutations: L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
- B.1.526 contains the following spike mutations: L5F, T95I, D253G, S477N, E484K, D614G, A701V
- CAL.20C contains the following spike mutations: S13I, W152C, L452R, D614G


### Figure: Virus Variants or Spike Mutations vs Convalescent Plasma and Plasma from Vaccinated Persons

<section id="sec_plasma-fold" class="figure-section wrap">

<div class="figure-image">

![Convalescent and vaccine-recipient plasma](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/plasma-fold-fig-1.png#!maxWidth=95%)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to plasma from previously infected and vaccinated persons associated with three variants of concern (B.1.1.7, B.1.351, and P.1), variants of interest(CAL.20C) and two spike amino acid mutations E484K and N501Y. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. The Y-axis indicates the number of plasma samples tested.

"*": The study is using authentic virus assay.

Studies:

- **Becker21**: Becker M, Dulovic A, et int., and Schneiderhan-Marra N. “Immune response to SARS-CoV-2 variants of concern in vaccinated individuals.” _medRxiv_, 2021. [doi.org/10.1101/2021.03.08.21252958](https://doi.org/10.1101/2021.03.08.21252958).[^Becker21]
- **Brown21**: Brown JC, Goldhill DH, et int., and Barclay WS. “Increased transmission of SARS-CoV-2 lineage B.1.1.7 (VOC 2020212/01) is not accounted for by a replicative advantage in primary airway cells or antibody escape.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.24.432576](https://doi.org/10.1101/2021.02.24.432576).[^Brown21]
- **Cele21**: Cele S, Gazy I, et int., and Sigal A. “Escape of SARS-CoV-2 501Y.V2 from neutralization by convalescent plasma.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.26.21250224](https://doi.org/10.1101/2021.01.26.21250224).[^Cele21]
- **Chen21**: Chen RE, Zhang X, et int., and Diamond MS. “Resistance of SARS-CoV-2 variants to neutralization by monoclonal and serum-derived polyclonal antibodies.” _Nature Medicine_, 2021. [doi.org/10.1038/s41591-021-01294-w](https://doi.org/10.1038/s41591-021-01294-w). \[[PubMed33664494](https://www.ncbi.nlm.nih.gov/pubmed/33664494/)\][^Chen21]
- **Collier21**: Collier DA, De Marco A, et int., and Gupta RK. “SARS-CoV-2 B.1.1.7 sensitivity to mRNA vaccine-elicited, convalescent and monoclonal antibodies.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.19.21249840](https://doi.org/10.1101/2021.01.19.21249840). \[[PubMed33619509](https://www.ncbi.nlm.nih.gov/pubmed/33619509/)\] \[[PMC7899479](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7899479/)\][^Collier21]
- **Dejnirattisai21**: Dejnirattisai W, Zhou D, et int., and Screaton GR. “Antibody evasion by the Brazilian P.1 strain of SARS-CoV-2.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.12.435194](https://doi.org/10.1101/2021.03.12.435194).[^Dejnirattisai21]
- **Deng21**: Deng X, Garcia-Knight MA, et int., and Chiu CY. “Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation.” _medRxiv_, 2021. [doi.org/10.1101/2021.03.07.21252647](https://doi.org/10.1101/2021.03.07.21252647).[^Deng21],
- **Edara21**: Edara VV, Hudson WH, et int., and Suthar MS. “Neutralizing Antibodies Against SARS-CoV-2 Variants After Infection and Vaccination.” _JAMA_, 2021. [doi.org/10.1001/jama.2021.4388](https://doi.org/10.1001/jama.2021.4388). \[[PubMed33739374](https://www.ncbi.nlm.nih.gov/pubmed/33739374/)\][^Edara21]
- **Emary21**: Emary KRW, Golubchik T, et int., and Group OCVT. “Efficacy of ChAdOx1 nCoV-19&nbsp;(AZD1222)&nbsp;Vaccine Against SARS-CoV-2 VOC&nbsp;202012/01&nbsp;(B.1.1.7).” _SSRN Electronic Journal_, 2021. [doi.org/10.2139/ssrn.3779160](https://doi.org/10.2139/ssrn.3779160).[^Emary21]
- **Faulkner21**: Faulkner N, Ng KW, et int., and Kassiotis G. “Reduced antibody cross-reactivity following infection with B.1.1.7 than with parental SARS-CoV-2 strains.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.01.433314](https://doi.org/10.1101/2021.03.01.433314).[^Faulkner21]
- **Garcia-Beltran21**: Garcia-Beltran WF, Lam EC, et int., and Balazs AB. “Circulating SARS-CoV-2 variants escape neutralization by vaccine-induced humoral immunity.” _medRxiv_, 2021. [doi.org/10.1101/2021.02.14.21251704](https://doi.org/10.1101/2021.02.14.21251704). \[[PubMed33619506](https://www.ncbi.nlm.nih.gov/pubmed/33619506/)\] \[[PMC7899476](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7899476/)\][^Garcia-Beltran21]
- **Greaney21**: Greaney AJ, Loes AN, et int., and Bloom JD. “Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human serum antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2020.12.31.425021](https://doi.org/10.1101/2020.12.31.425021).[^Greaney21]
- **Hoffmann21**: Hoffmann M, Arora P, et int., and Pöhlmann S. “SARS-CoV-2 variants B.1.351 and B.1.1.248: Escape from therapeutic antibodies and antibodies induced by infection and vaccination.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.11.430787](https://doi.org/10.1101/2021.02.11.430787).[^Hoffmann21]
- **Hoffmann21b**: Hoffmann M, Zhang L, et int., and Pöhlmann S. “SARS-CoV-2 mutations acquired in mink reduce antibody-mediated neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.12.430998](https://doi.org/10.1101/2021.02.12.430998).[^Hoffmann21b]
- **Hu21**: Hu J, Peng P, et int., and Huang A. “Emerging SARS-CoV-2 variants reduce neutralization sensitivity to convalescent sera and monoclonal antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.22.427749](https://doi.org/10.1101/2021.01.22.427749).[^Hu21]
- **Jangra21**: Jangra S, Ye C, et int., and Schotsaert M. “The E484K mutation in the SARS-CoV-2 spike protein reduces but does not abolish neutralizing activity of human convalescent and post-vaccination sera.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.26.21250543](https://doi.org/10.1101/2021.01.26.21250543). \[[PubMed33532796](https://www.ncbi.nlm.nih.gov/pubmed/33532796/)\] \[[PMC7852247](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7852247/)\][^Jangra21]
- **Kemp21**: Kemp SA, Collier DA, et int., and Gupta RK. “SARS-CoV-2 evolution during treatment of chronic infection.” _Nature_, 2021. [doi.org/10.1038/s41586-021-03291-y](https://doi.org/10.1038/s41586-021-03291-y). \[[PubMed33545711](https://www.ncbi.nlm.nih.gov/pubmed/33545711/)\][^Kemp21]
- **Liu21**: Liu Z, VanBlargan LA, et int., and Whelan SP. “Identification of SARS-CoV-2 spike mutations that attenuate monoclonal and serum antibody neutralization.” _Cell Host & Microbe_, 2021. [doi.org/10.1016/j.chom.2021.01.014](https://doi.org/10.1016/j.chom.2021.01.014). \[[PubMed33535027](https://www.ncbi.nlm.nih.gov/pubmed/33535027/)\] \[[PMC7839837](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7839837/)\][^Liu21]
- **Liu21b**: Liu Y, Liu J, et int., and Shi P. “Neutralizing Activity of BNT162b2-Elicited Serum — Preliminary Report.” _New England Journal of Medicine_, 2021. [doi.org/10.1056/NEJMc2102017](https://doi.org/10.1056/NEJMc2102017). \[[PubMed33596352](https://www.ncbi.nlm.nih.gov/pubmed/33596352/)\][^Liu21b]
- **Madhi21**: Madhi SA, Baillie V, et int., and Izu A. “Safety and efficacy of the ChAdOx1 nCoV-19 (AZD1222) Covid-19 vaccine against the B.1.351 variant in South Africa.” _medRxiv_, 2021. [doi.org/10.1101/2021.02.10.21251247](https://doi.org/10.1101/2021.02.10.21251247).[^Madhi21]
- **Marot21**: Marot S, Malet I, et int., and Marcelin A. “Neutralization heterogeneity of United Kingdom and South-African SARS-CoV-2 variants in BNT162b2-vaccinated or convalescent COVID-19 healthcare workers.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.05.434089](https://doi.org/10.1101/2021.03.05.434089).[^Marot21]
- **Moyo-Gwete21**: Moyo-Gwete T, Madzivhandila M, et int., and Moore PL. “SARS-CoV-2 501Y.V2 (B.1.351) elicits cross-reactive neutralizing antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.06.434193](https://doi.org/10.1101/2021.03.06.434193).[^Moyo-Gwete21]
- **Muik21**: Muik A, Wallisch A, et int., and Şahin U. “Neutralization of SARS-CoV-2 lineage B.1.1.7 pseudovirus by BNT162b2 vaccine–elicited human sera.” _Science_, 2021. [doi.org/10.1126/science.abg6105](https://doi.org/10.1126/science.abg6105). \[[PubMed33514629](https://www.ncbi.nlm.nih.gov/pubmed/33514629/)\][^Muik21]
- **Planas21**: Planas D, Bruel T, et int., and Schwartz O. “Sensitivity of infectious SARS-CoV-2 B.1.1.7 and B.1.351 variants to neutralizing antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.12.430472](https://doi.org/10.1101/2021.02.12.430472).[^Planas21]
- **Rees-Spear21**: Rees-Spear C, Muir L, et int., and McCoy L. “The impact of Spike mutations on SARS-CoV-2 neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.15.426849](https://doi.org/10.1101/2021.01.15.426849).[^Rees-Spear21]
- **Shen21**: Shen X, Tang H, et int., and Montefiori DC. “SARS-CoV-2 variant B.1.1.7 is susceptible to neutralizing antibodies elicited by ancestral spike vaccines.” _Cell Host & Microbe_, 2021. [doi.org/10.1016/j.chom.2021.03.002](https://doi.org/10.1016/j.chom.2021.03.002). \[[PubMed33705729](https://www.ncbi.nlm.nih.gov/pubmed/33705729/)\] \[[PMC7934674](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7934674/)\][^Shen21]
- **Tada21**: Tada T, Dcosta BM, et int., and Landau NR. “Decreased neutralization of SARS-CoV-2 global variants by therapeutic anti-spike protein monoclonal antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.18.431897](https://doi.org/10.1101/2021.02.18.431897). \[[PubMed33619484](https://www.ncbi.nlm.nih.gov/pubmed/33619484/)\] \[[PMC7899448](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7899448/)\][^Tada21]
- **Tang21**: Tang J, Lee Y, et int., and Khurana S. “Reduced neutralization of SARS-CoV-2 variants by convalescent plasma and hyperimmune intravenous immunoglobulins for treatment of COVID-19.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.19.436183](https://doi.org/10.1101/2021.03.19.436183).[^Tang21]
- **Wang21**: Wang Z, Schmidt F, et int., and Nussenzweig MC. “mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants.” _Nature_, 2021. [doi.org/10.1038/s41586-021-03324-6](https://doi.org/10.1038/s41586-021-03324-6). \[[PubMed33567448](https://www.ncbi.nlm.nih.gov/pubmed/33567448/)\][^Wang21]
- **Wang21b**: Wang P, Nair MS, et int., and Ho DD. “Antibody Resistance of SARS-CoV-2 Variants B.1.351 and B.1.1.7.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.25.428137](https://doi.org/10.1101/2021.01.25.428137). \[[PubMed33532778](https://www.ncbi.nlm.nih.gov/pubmed/33532778/)\] \[[PMC7852271](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7852271/)\][^Wang21b]
- **Wang21c**: Wang C, van Haperen R, et int., and Bosch B. “Isolation of cross-reactive monoclonal antibodies against divergent human coronaviruses that delineate a conserved and vulnerable site on the spike protein.” _bioRxiv_, 2020. [doi.org/10.1101/2020.10.20.346916](https://doi.org/10.1101/2020.10.20.346916).[^Wang21c]
- **Wang21f**: Wang R, Zhang Q, et int., and Zhang L. “Spike mutations in SARS-CoV-2 variants confer resistance to antibody neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.09.434497](https://doi.org/10.1101/2021.03.09.434497).[^Wang21f]
- **Wibmer21**: Wibmer CK, Ayres F, et int., and Moore PL. “SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.18.427166](https://doi.org/10.1101/2021.01.18.427166). \[[PubMed33501446](https://www.ncbi.nlm.nih.gov/pubmed/33501446/)\] \[[PMC7836116](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7836116/)\][^Wibmer21]
- **Widera21**: Widera M, Wilhelm A, et int., and Ciesek S. “Bamlanivimab does not neutralize two SARS-CoV-2 variants carrying E484K in vitro.” _medRxiv_, 2021. [doi.org/10.1101/2021.02.24.21252372](https://doi.org/10.1101/2021.02.24.21252372).[^Widera21]
- **Wu21**: Wu K, Werner AP, et int., and Edwards DK. “Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine — Preliminary Report.” _New England Journal of Medicine_, 2021. [doi.org/10.1056/NEJMc2102179](https://doi.org/10.1056/NEJMc2102179). \[[PubMed33596346](https://www.ncbi.nlm.nih.gov/pubmed/33596346/)\][^Wu21]
- **Xie21b**: Xie X, Liu Y, et int., and Shi P. “Neutralization of SARS-CoV-2 spike 69/70 deletion, E484K and N501Y variants by BNT162b2 vaccine-elicited sera.” _Nature Medicine_, 2021. [doi.org/10.1038/s41591-021-01270-4](https://doi.org/10.1038/s41591-021-01270-4). \[[PubMed33558724](https://www.ncbi.nlm.nih.gov/pubmed/33558724/)\][^Xie21b]
- **Zhou21b**: Zhou D, Dejnirattisai W, et int., and Screaton GR. “Evidence of escape of SARS-CoV-2 variant B.1.351 from natural and vaccine-induced sera.” _Cell_, 2021. [doi.org/10.1016/j.cell.2021.02.037](https://doi.org/10.1016/j.cell.2021.02.037). \[[PubMed33730597](https://www.ncbi.nlm.nih.gov/pubmed/33730597/)\][^Zhou21b]

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility. The histogram labels indicate the studies’ authors and plasma characteristics. BNT: BNT162b2; Mod: mRNA-1273; NVX: NVX-CoV2373; AZD: AZD1222. Each of plasma units were obtained approximately one month following vaccination with the exception of one study.
</p>

</div>

</section>



## Raw data

- [Spike variants and mutation combinations / mAb neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-mab-neutralization)
- [Spike variants and mutation combinations / plasma neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-plasma-neutralization)
- [Spike mutation _in vitro_ selection](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-mutation-invitro-selection-cp-and-mab)



## Individual study details

- [Table S1: Neutralizing Sensitivity of Variants of Concern to Convalescent Plasma and Plasma from Vaccinated Persons](/page/susceptibility-studies#table.s1.neutralizing.sensitivity.of.variants.of.concern.to.convalescent.plasma.and.plasma.from.vaccinated.persons)
- [Table S2: Neutralizing Sensitivity of Variants of Concern to Monoclonal Antibodies (mAbs)](/page/susceptibility-studies#table.s2.neutralizing.sensitivity.of.variants.of.concern.to.monoclonal.antibodies.mabs.)
- [Table S3: Neutralizing Sensitivity of mutations to Convalescent Plasma and Plasma from Vaccinated Persons](/page/susceptibility-studies#table.s3.neutralizing.sensitivity.of.mutations.to.convalescent.plasma.and.plasma.from.vaccinated.persons)
- [Table S4: Neutralizing Sensitivity of Mutations to Monoclonal Antibodies (mAbs)](/page/susceptibility-studies#table.s4.neutralizing.sensitivity.of.mutations.to.monoclonal.antibodies.mabs.)


## Supplemantary data

### Statistics of Published _in vitro_ Neutralization Assays

Numbers of published _in vitro_ neutralization assays according to spike mutations, type of plasma, and monoclonal antibody


[table]
table_summary
[/table]
