[toc]

- [Summaries](#summaries)
  - [Table 1: Virus Variants and Spike Mutations vs Monoclonal Antibodies](#table.1.virus.variants.and.spike.mutations.vs.monoclonal.antibodies)
  - [Table 2: Virus Variants and Spike Mutations vs Convalescent Plasma](#table.2.virus.variants.and.spike.mutations.vs.convalescent.plasma)
  - [Table 3: Virus Variants and Spike Mutations vs Plasma from Vaccinated Persons](#table.3.virus.variants.and.spike.mutations.vs.plasma.from.vaccinated.persons)
  - [Table 4: Variants of Concern, Variants of Interest, and Other Variants for which Neutralization Data are Available in CoV-RDB](#table.4.variants.of.concern.variants.of.interest.and.other.variants.for.which.neutralization.data.are.available.in.cov-rdb)
  - [Table 5: Individual SARS-CoV-2 Spike Mutations for which Data are Available in CoV-RDB Receptor-binding domain (RBD)](#table.5.individual.sars-cov-2.spike.mutations.for.which.data.are.available.in.cov-rdb)
  - [Table 6. Monoclonal Antibodies (mAbs) with Emergency Use Authorization (EUA) or in Advanced Clinical Trials](#table.6.monoclonal.antibodies.mabs.with.emergency.use.authorization.eua.or.in.advanced.clinical.trials)
  - [Table 7. Numbers of Convalescent Plasma (CP) Neutralization Experiments in CoV-RDB According to the Infecting Virus, Time Since Infection, and SARS-CoV-2 Variant Tested](#table.7.numbers.of.convalescent.plasma.cp.neutralization.experiments.in.cov-rdb.according.to.the.infecting.virus.time.since.infection.and.sars-cov-2.variant.tested)
  - [Table 8. Numbers of Vaccinee Plasma (VP) Neutralization Experiments in CoV-RDB According to the Vaccine, History of Previous Infection, Time Since Infection, and SARS- CoV-2 Variant Tested](#table.8.numbers.of.vaccinee.plasma.vp.neutralization.experiments.in.cov-rdb.according.to.the.vaccine.history.of.previous.infection.time.since.infection.and.sars-.cov-2.variant.tested)
  <!-- - [Table 4: Vaccine Efficacy at Preventing Symptomatic SARS-CoV-2 Infection With Variants](#table.4.vaccine.efficacy.at.preventing.symptomatic.sars-cov-2.infection.with.variants) -->
  <!-- - [Figure: Virus Variants and Spike Mutations vs Convalescent Plasma and Plasma from Vaccinated Persons](/page/susceptibility-data/figure/#figure.virus.variants.and.spike.mutations.vs.convalescent.plasma.and.plasma.from.vaccinated.persons) -->
  - [Figure 1: Distribution of fold-reduction in susceptibilities (A) and of absolute neutralizing titers (B) of vaccinee plasma (VP) associated with eight vaccines to the four variants of concern (VOC): Alpha, Beta, Gamma, and Delta.](#figure.1.distribution.of.fold-reduction.in.susceptibilities.a.and.of.absolute.neutralizing.titers.b.of.vaccinee.plasma.vp.associated.with.eight.vaccines.to.the.four.variants.of.concern.voc.alpha.beta.gamma.and.delta.)
  - [Figure 2: Monoclonal antibodies (mAbs) with EUAs or in advanced clinical development: receptor binding domain (RBD) epitopes and immune escape positions.](#figure.2.monoclonal.antibodies.mabs.with.euas.or.in.advanced.clinical.development.receptor.binding.domain.rbd.epitopes.and.immune.escape.positions.)

- [References](#references)

[/toc]

## Summaries

### Table 1: Virus Variants and Spike Mutations vs Monoclonal Antibodies

#### Fold reduced neutralizing susceptibility to monoclonal antibodies under Emergency Use Authorization(EUA)

[table compact, lastCompact]
table_mab
[/table]

#### Fold reduced neutralizing susceptibility to monoclonal antibodies in other clinical trials

[table compact,lastCompact]
table_mab_non_eua
[/table]

- The fold change is the median value of samples, the subscript is the number of samples. “-“ indicates absence of susceptibility data. “\*“ indicates conflict between two research groups, with one showing high-level of reduced susceptibility especially using IC80 rather than IC50.
- Spike mutations:
  - **Alpha, B.1.1.7**: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H.
  - **Beta, B.1.351**: L18F, D80A, D215G, ∆242-244, R246I, K417N, E484K, N501Y, D614G, and A701V.
  - **Gamma, P.1**: L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
  - **Delta, B.1.617.2**: T19R, FR1570158∆, L452R, T478K, D614G, P681R, D950N.
  - **Iota, B.1.526**: L5F, T95I, D253G, E484K, D614G, A701V.
  - **Epsilon, B.1.427/9**: S13I, W152C, L452R, D614G.
  - **Kappa, B.1.617.1**: T95I, L452R, E484Q, D614G, P681R, Q1071H.
- Monoclonal antibody(mAb) abbreviations: **BAM**: Bamlanivimab/LY-CoV555, **CAS**: Casirivimab/REGN10933, **IMD**: Imdevimab/REGN10987, **CAS/IMD**: Casirivimab+imdevimab/REGN-COV2, **ETE**: Etesevimab/LY-CoV016/JS016/CB6, **CIL**: Cilgavimab/COV2-2130/AZD1061, **TIX**: Tixagevimab/COV2-2196/AZD8895, **TIX/CIL**: Tixagevimab+Cilgavimab, **BAM/ETE**: Bamlanivimab+Etesevimab, **SOT**: Sotrovimab/Vir-7831/S309, **REG**: Regdanvimab/CT-P59.
- The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility.


***


### Table 2: Virus Variants and Spike Mutations vs Convalescent Plasma

[table compact,lastCompact]
table_cp
[/table]

- Convalescent plasma in all studies is presumed likely to have been obtained from persons infected with spike variants lacking mutations associated with reduced neutralization susceptibility.
- Spike mutations:
  - **Alpha, B.1.1.7**: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H.
  - **Beta, B.1.351**: L18F, D80A, D215G, ∆242-244, R246I, K417N, E484K, N501Y, D614G, and A701V.
  - **Gamma, P.1**: L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
  - **Delta, B.1.617.2**: T19R, FR1570158∆, L452R, T478K, D614G, P681R, D950N.
  - **Iota, B.1.526**: L5F, T95I, D253G, E484K, D614G, A701V.
  - **Epsilon, B.1.427/9**: S13I, W152C, L452R, D614G.
  - **Kappa, B.1.617.1**: T95I, L452R, E484Q, D614G, P681R, Q1071H.

***


### Table 3: Virus Variants and Spike Mutations vs Plasma from Vaccinated Persons

[table compact,lastCompact]
table_vp
[/table]

- Spike mutations:
  - **Alpha, B.1.1.7**: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H.
  - **Beta, B.1.351**: L18F, D80A, D215G, ∆242-244, R246I, K417N, E484K, N501Y, D614G, and A701V.
  - **Gamma, P.1**: L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
  - **Delta, B.1.617.2**: T19R, FR1570158∆, L452R, T478K, D614G, P681R, D950N.
  - **Iota, B.1.526**: L5F, T95I, D253G, E484K, D614G, A701V.
  - **Epsilon, B.1.427/9**: S13I, W152C, L452R, D614G.
  - **Kappa, B.1.617.1**: T95I, L452R, E484Q, D614G, P681R, Q1071H.

***
<!-- ### Table 4: Vaccine Efficacy at Preventing Symptomatic SARS-CoV-2 Infection With Variants

[table compact,lastCompact]
table_ve
[/table]

- Includes persons receiving a full course of vaccination: 1 to 2 weeks following two vaccine doses for BNT162b, AZD1222, mRNA-1273, NVX-2373, and Coronavac; 4 weeks after one vaccine dose for AD26.CoV2.
- AZD1222 efficacy data includes some patients receiving the standard vaccine dose at both time points and others receiving a lower vaccine dose at the first time point.
- AD26.CoV2 efficacy data based on preventing moderate to severe disease; pre-variant efficacy based on U.S. data.
- Coronovac efficacy against data against pre-variant isolates may include some persons infected with the VOC gamma and the VOI zeta. The test-negative case control study of vaccine efficacy against gamma was performed in persons 70 years or older.
Abbreviations: Trial – post hoc analysis of clinical trial data; TNCC – test negative case control study
 -->

### Table 4: Variants of Concern, Variants of Interest, and Other Variants for which Neutralization Data are Available in CoV-RDB

#### Variants of concern

[table compact,lastCompact]
table_variants_of_concern
[/table]

#### Variants of interest

[table compact,lastCompact]
table_variants_of_interest
[/table]

#### Other variants containing >=1 RBD mutation

[table compact,lastCompact]
table_variants_other_rbd
[/table]

The most common ancestral variants lacking immune escape mutations used as controls include A
variants (n=3720), B variants (Wuhan consensus; n=19123), B.1 variants (Wuhan consensus + D614G; n=18299).

***

### Table 5: Individual SARS-CoV-2 Spike Mutations for which Data are Available in CoV-RDB

#### Receptor-binding domain (RBD)

[table compact,lastCompact]
table_mut_rbd
[/table]

#### N-terminal domain (NTD)

[table compact,lastCompact]
table_mut_ntd
[/table]

#### C-terminal domain of S1 (CTD)

[table compact,lastCompact]
table_mut_ctd
[/table]

#### S2

[table compact,lastCompact]
table_mut_s2
[/table]

The most commonly studied mutation combinations that are not VOCs or VOIs included K417N+E484K+N501Y (n=19 studies; 569 results), E484K+N501Y (n=5 studies; 189 results), ∆69/70+N501Y (n=6 studies; 135 results), ∆69/70+Y453F (n=2 studies; 74 results), L452R+E484Q+P681R (n=3 studies; 55 results) K417N+N501Y (n=2 studies; 75 results), K417N+E484K (n=2 studies; 75 results), and L452R+E484Q (n=4 studies; 55 results)

***


### Table 6. Monoclonal Antibodies (mAbs) with Emergency Use Authorization (EUA) or in Advanced Clinical Trials

[table compact,lastCompact]
table_mab_detail
[/table]

RBM (receptor binding motif) is the part of the receptor binding domain (RBD) that contains the spike ACE2 binding residues. RBM class 1 mAbs bind epitopes dominated by ACE2-binding residues and as a result bind solely when the RBD is in the up/open position. RBM class 2 mAbs have a smaller ACE2-binding footprint and can often bind the RBD in down/closed position. RBD core mAbs target a surface accessible part of the RBD that is separate from the RBM. 2The publications and results include those describing both in vitro selection and neutralization experiments.

***


### Table 7. Numbers of Convalescent Plasma (CP) Neutralization Experiments in CoV-RDB According to the Infecting Virus, Time Since Infection, and SARS-CoV-2 Variant Tested

#### Infecting virus

[table compact,lastCompact]
table_cp_infection
[/table]

#### Time since infection

[table compact,lastCompact]
table_cp_time
[/table]

#### Variant / mutations undergoing neutralization

[table compact,lastCompact]
table_cp_variant
[/table]

***


### Table 8. Numbers of Vaccinee Plasma (VP) Neutralization Experiments in CoV-RDB According to the Vaccine, History of Previous Infection, Time Since Infection, and SARS- CoV-2 Variant Tested

#### Vaccine

[table compact,lastCompact]
table_vp_vaccine
[/table]

#### Previously SARS-CoV-2 infected

[table compact,lastCompact]
table_vp_infection
[/table]

#### Time since vaccination

[table compact,lastCompact]
table_vp_time
[/table]

#### Variant / mutations undergoing neutralization

[table compact,lastCompact]
table_vp_variant
[/table]

#### Number of vaccinations

[table compact,lastCompact]
table_vp_dosage
[/table]


The numbers of references and results obtained from persons receiving uncommonly studied vaccines or vaccine combinations are not shown.

***

### Figure 1: Distribution of fold-reduction in susceptibilities (A) and of absolute neutralizing titers (B) of vaccinee plasma (VP) associated with eight vaccines to the four variants of concern (VOC): Alpha, Beta, Gamma, and Delta.

<div class="figure-image">

![fold titer 1 month](susceptibility-data/fold-titer.png#!maxHeight=35rem&marginLeft=0)


The X axes indicate the Greek letter associated with each VOC. The Y axes indicate the number of neutralizing assays. The numbers above the stacked bars indicate the number of studies reporting the experimental results. The figure includes VP obtained solely from previously uninfected persons receiving a complete vaccination schedule.

</div>

***

### Figure 2: Monoclonal antibodies (mAbs) with EUAs or in advanced clinical development: receptor binding domain (RBD) epitopes and immune escape positions.

<div class="figure-image">

![mab](susceptibility-data/mab.png#!maxHeight=50rem&marginLeft=0)


For each mAb, the top of the RBD and two side views are depicted using coordinates from PDB 6M0J. ACE2 binding residues are shown in red; the mAb epitope defined as those residues within 4.5 angstroms of the RBD is shown in dark blue; and ACE2 binding residues within the mAb epitope are shown in purple. Those positions containing mutations that were either selected by the mAb in vitro (“SEL”), reduced binding in a deep mutational scanning assay (“DMS”), and/or reduced in vitro neutralizing susceptibility by a median of 4-fold in CoV-RDB (drug resistance; “DR”) are also indicated. The mAb epitopes for BAM (bamlanivimab), ETE (etesevimab), CAS (casirivimab), IMD (imdevimab), SOT (sotrovirmab), CIL (cilgavimab) and TIX (tixagevimab) were determined from their PDB structures.

</div>


<section>

<div class="figure-caption">


## References

- **Abu-Raddad21**: [^Abu-Raddad21#inline]
- **Alenquer21**: [^Alenquer21#inline]
- **Andreano21**: [^Andreano21#inline]
- **Bates21**: [^Bates21#inline]
- **Baum20**: [^Baum20#inline]
- **Becker21**: [^Becker21#inline]
- **Bernal21**: [^Bernal21#inline]
- **Betton21**: [^Betton21#inline]
- **Brown21**: [^Brown21#inline]
- **Cathcart21**: [^Cathcart21#inline]
- **Cele21**: [^Cele21#inline]
- **Chen21**: [^Chen21#inline]
- **Collier21**: [^Collier21#inline]
- **Copin21**: [^Copin21#inline]
- **Dejnirattisai21**: [^Dejnirattisai21#inline]
- **Deng21**: [^Deng21#inline]
- **Dong21**: [^Dong21#inline]
- **Du20**: [^Du20#inline]
- **Edara21**: [^Edara21#inline]
- **Edara21b**: [^Edara21b#inline]
- **Edara21c**: [^Edara21c#inline]
- **Emary21**: [^Emary21#inline]
- **FDA21**: [^FDA21#inline]
- **Faulkner21**: [^Faulkner21#inline]
- **Ferreira21**: [^Ferreira21#inline]
- **Garcia-Beltran21**: [^Garcia-Beltran21#inline]
- **Goel21**: [^Goel21#inline]
- **Gottlieb21**: [^Gottlieb21#inline]
- **Greaney20**: [^Greaney20#inline]
- **Greaney21**: [^Greaney21#inline]
- **Greaney21c**: [^Greaney21c#inline]
- **Gu21**: [^Gu21#inline]
- **Haas21**: [^Haas21#inline]
- **Heath21**: [^Heath21#inline]
- **Hoffmann21**: [^Hoffmann21#inline]
- **Hoffmann21b**: [^Hoffmann21b#inline]
- **Hoffmann21c**: [^Hoffmann21c#inline]
- **Hu21**: [^Hu21#inline]
- **Huang21b**: [^Huang21b#inline]
- **Ikegame21**: [^Ikegame21#inline]
- **Jangra21**: [^Jangra21#inline]
- **Kemp21**: [^Kemp21#inline]
- **Khoury21**: [^Khoury21#inline]
- **Kuzmina21**: [^Kuzmina21#inline]
- **Lee21**: [^Lee21#inline]
- **Leier21**: [^Leier21#inline]
- **Li20h**: [^Li20h#inline]
- **Li21c**: [^Li21c#inline]
- **Li21d**: [^Li21d#inline]
- **Lien21**: [^Lien21#inline]
- **Liu21**: [^Liu21#inline]
- **Liu21b**: [^Liu21b#inline]
- **Madhi21**: [^Madhi21#inline]
- **Mallm21**: [^Mallm21#inline]
- **Marot21**: [^Marot21#inline]
- **McCallum21**: [^McCallum21#inline]
- **McCallum21b**: [^McCallum21b#inline]
- **Monin**: [^Monin21#inline]
- **Moyo-Gwete21**: [^Moyo-Gwete21#inline]
- **Muik21**: [^Muik21#inline]
- **Ranzani21**: [^Ranzani21#inline]
- **Planas21**: [^Planas21#inline]
- **Polack20**: [^Polack20#inline]
- **Rees-Spear21**: [^Rees-Spear21#inline]
- **Rondinone21**: [^Rondinone21#inline]
- **Rothenberger21**: [^Rothenberger21#inline]
- **Sadoff21**: [^Sadoff21#inline]
- **Sahin20**: [^Sahin20#inline]
- **Sapkal21**: [^Sapkal21#inline]
- **Sapkal21b**: [^Sapkal21b#inline]
- **Sheikh21**: [^Sheikh21#inline]
- **Shen21**: [^Shen21#inline]
- **Shen21b**: [^Shen21b#inline]
- **Shinde21**: [^Shinde21#inline]
- **Skelly21**: [^Skelly21#inline]
- **Stamatatos21**: [^Stamatatos21#inline]
- **Stankov21**: [^Stankov21#inline]
- **Starr21**: [^Starr21#inline]
- **Supasa21**: [^Supasa21#inline]
- **Tada21**: [^Tada21#inline]
- **Tada21c**: [^Tada21c#inline]
- **Tada21d**: [^Tada21d#inline]
- **Tada21e**: [^Tada21e#inline]
- **Tang21**: [^Tang21#inline]
- **Tortorici20**: [^Tortorici20#inline]
- **Trinite21**: [^Trinite21#inline]
- **Vanhove21**: [^Vanhove21#inline]
- **Virtanen21**: [^Virtanen21#inline]
- **Voysey20**: [^Voysey20#inline]
- **Wang21**: [^Wang21#inline]
- **Wang21b**: [^Wang21b#inline]
- **Wang21d**: [^Wang21d#inline]
- **Wang21e**: [^Wang21e#inline]
- **Wang21f**: [^Wang21f#inline]
- **Wang21g**: [^Wang21g#inline]
- **Weigang21**: [^Weigang21#inline]
- **Weisblum20**: [^Weisblum20#inline]
- **West21**: [^West21#inline]
- **Westendorf21**: [^Westendorf21#inline]
- **Wibmer21**: [^Wibmer21#inline]
- **Widera21**: [^Widera21#inline]
- **Wu21**: [^Wu21#inline]
- **Wu21b**: [^Wu21b#inline]
- **Xie21**: [^Xie21#inline]
- **Yadav21**: [^Yadav21#inline]
- **Yao21**: [^Yao21#inline]
- **Yuan21**: [^Yuan21#inline]
- **Zhou21b**: [^Zhou21b#inline]
- **Zhou21c**: [^Zhou21c#inline]
- **Zuckerman21**: [^Zuckerman21#inline]
- **deSouza21**: [^deSouza21#inline]

</div>

</section>
