<div style="clear: both; height: 0;" />

## Susceptibility results of convalescent and vaccine-recipient plasma

<section id="sec_plasma-fold" class="figure-section wrap">

<div class="figure-image">

![Convalescent and vaccine-recipient plasma](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/plasma-fold-fig-1.png)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to plasma from previously infected and vaccinated persons associated with three variants of concern (B.1.1.7, B.1.351, and P.1) and two spike amino acid mutations E484K and N501Y. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. The Y-axis indicates the number of plasma units tested.

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility. The histogram labels indicate the studies’ authors and plasma characteristics. BNT: BNT162b2; Mod: mRNA-1273; NVX: NVX-CoV2373. Each of plasma units were obtained approximately one month following vaccination with the exception of one study.
</p>

- **Cele21**: Cele S, Gazy I, et int., and Sigal A. “Escape of SARS-CoV-2 501Y.V2 variants from neutralization by convalescent plasma.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.26.21250224](https://doi.org/10.1101/2021.01.26.21250224).
- **Collier21**: Collier D, Marco AD, et int., and Gupta R. “SARS-CoV-2 B.1.1.7 escape from mRNA vaccine-elicited neutralizing antibodies.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.19.21249840](https://doi.org/10.1101/2021.01.19.21249840).
- **Deng21**: Deng X, Garcia-Knight MA, et int., and Chiu CY. “Transmission, infectivity, and antibody neutralization of an emerging SARS-CoV-2 variant in California carrying a L452R spike protein mutation.” _medRxiv_, 2021. [doi.org/10.1101/2021.03.07.21252647](https://doi.org/10.1101/2021.03.07.21252647).
- **Diamond21**: Chen RE, Zhang X, et int., and Diamond MS. “Resistance of SARS-CoV-2 variants to neutralization by monoclonal and serum-derived polyclonal antibodies.” _nature medicine_, 2021. [doi.org/10.1038/s41591-021-01294-w](https://doi.org/10.1038/s41591-021-01294-w). \[[PubMed33664494](https://pubmed.ncbi.nlm.nih.gov/33664494/)\]
- **Edara21**: Edara VV, Floyd K, et int., and Suthar MS. “Infection and mRNA-1273 vaccine antibodies neutralize SARS-CoV-2 UK variant.” _medRxiv_, 2021. [doi.org/10.1101/2021.02.02.21250799](https://doi.org/10.1101/2021.02.02.21250799).
- **Faulkner21**: Faulkner N, Ng KW, et int., and Kassiotis G. “Reduced antibody cross-reactivity following infection with B.1.1.7 than with parental SARS-CoV-2 strains.” _biorxiv_, 2021. [doi.org/10.1101/2021.03.01.433314](https://doi.org/10.1101/2021.03.01.433314).
- **Greaney21**: Greaney AJ, Loes AN, et int., and Bloom JD. “Comprehensive mapping of mutations to the SARS-CoV-2 receptor-binding domain that affect recognition by polyclonal human serum antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2020.12.31.425021](https://doi.org/10.1101/2020.12.31.425021).
- **Hoffmann21**: Hoffmann, M, Arora P, et int., and Muench Jan. “SARS-CoV-2 variants B.1.351 and B.1.1.248: Escape from therapeutic antibodies and antibodies induced by infection and vaccination.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.11.430787](https://doi.org/10.1101/2021.02.11.430787).
- **Hu21**: Hu J, Peng P, et int., and Huang A. “Emerging SARS-CoV-2 variants reduce neutralization sensitivity to convalescent sera and monoclonal antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.22.427749](https://doi.org/10.1101/2021.01.22.427749).
- **Jangra21**: Jangra S, Ye C, et int., and Schotsaert M. “The E484K mutation in the SARS-CoV-2 spike protein reduces but does not abolish neutralizing activity of human convalescent and post-vaccination sera.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.26.21250543](https://doi.org/10.1101/2021.01.26.21250543).
- **Liu21**: Liu Z, VanBlargan LA, et int., and Whelan SPJ. “Identification of SARS-CoV-2 spike mutations that attenuate monoclonal and serum antibody neutralization.” _Cell Host & Microbe_, 2021. [doi.org/10.1016/j.chom.2021.01.014](https://doi.org/10.1016/j.chom.2021.01.014).
- **Liu21b**: Liu Y, Liu J, et int., and Shi PY. “Neutralizing Activity of BNT162b2-Elicited Serum - Preliminary Report.” _NEJM_, 2021. [doi.org/10.1056/NEJMc2102017](https://doi.org/10.1056/NEJMc2102017). \[[PubMed33596352](https://pubmed.ncbi.nlm.nih.gov/33596352/)\]
- **Marot**: Marot S, Malet I, et int., and Marcelin AG. “Neutralization heterogeneity of United Kingdom and South-African SARS-CoV-2 variants in BNT162b2-vaccinated or convalescent COVID-19 healthcare workers.” _biorxiv_, 2021. [doi.org/10.1101/2021.03.05.434089](https://doi.org/10.1101/2021.03.05.434089).
- **Moyo-Gwete21**: Moyo-Gwete T, Madzivhandila M, et int., and Moore PL. “SARS-CoV-2 501Y.V2 (B.1.351) elicits cross-reactive neutralizing antibodies.” _biorxiv_, 2021. [doi.org/10.1101/2021.03.06.434193](https://doi.org/10.1101/2021.03.06.434193).
- **Muik21**: Muik A, Wallisch A, et int., and Sahin U. “Neutralization of SARS-CoV-2 lineage B.1.1.7 pseudovirus by BNT162b2 vaccine-elicited human sera.” _Science_, 2021. [doi.org/10.1126/science.abg6105](https://doi.org/10.1126/science.abg6105).
- **Planas21**: Planas D, Bruel T, et int., and Schwartz O. “Sensitivity of infectious SARS-CoV-2 B.1.1.7 and B.1.351 variants to neutralizing antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.12.430472](https://doi.org/10.1101/2021.02.12.430472).
- **Rondinone21**: Rondinone V, Pace L, et int., and Galante D. “VOC 202012/01 Variant Is Effectively Neutralized by Antibodies Produced by Patients Infected before Its Diffusion in Italy.” _Viruses_, 2021. [doi.org/10.3390/v13020276](https://doi.org/10.3390/v13020276).
- **RSC21**: Rees-Spear C, Muir L, et int., and McCoy L. “The impact of Spike mutations on SARS-CoV-2 neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.15.426849](https://doi.org/10.1101/2021.01.15.426849).
- **Shen21**: Shen X, Tang H, et int., and Montefiori DC. “SARS-CoV-2 variant B.1.1.7 is susceptible to neutralizing antibodies elicited by ancestral Spike vaccines.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.27.428516](https://doi.org/10.1101/2021.01.27.428516).
- **Stamatatos21**: Stamatatos L, Czartoski J, et int., and McGuire A. “Antibodies elicited by SARS-CoV-2 infection and boosted by vaccination neutralize an emerging variant and SARS-CoV-1.”, _medRxiv_, 2021. [doi.org/10.1101/2021.02.05.21251182](https://doi.org/10.1101/2021.02.05.21251182).
- **Tada21**: Tada T, Dcosta BM, et int., and Landau NR. “Neutralization of viruses with European, South African, and United States SARS-CoV-2 variant spike proteins by convalescent sera and BNT162b2 mRNA vaccine-elicited antibodies.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.05.430003](https://doi.org/10.1101/2021.02.05.430003).
- **Wang21**: Wang Z, Schmidt F, et int., and Nussenzweig MC. “mRNA vaccine-elicited antibodies to SARS-CoV-2 and circulating variants.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.15.426911](https://doi.org/10.1101/2021.01.15.426911). \[[PubMed33501451](https://pubmed.ncbi.nlm.nih.gov/33501451/)\] \[[PMC7836122](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7836122/)\]
- **Wang21b**: Wang P, Liu L, et int., and Ho DD. “Increased Resistance of SARS-CoV-2 Variants B.1.351 and B.1.1.7 to Antibody Neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.25.428137](https://doi.org/10.1101/2021.01.25.428137).
- **Wang21c**: Wang P, Wang M, et int., and Ho DD. “Increased Resistance of SARS-CoV-2 Variant P.1 to Antibody Neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.01.433466](https://doi.org/10.1101/2021.03.01.433466).
- **Wang21f**: Wang R, Zhang Q, et int., and Zhang L. “Spike mutations in SARS-CoV-2 variants confer resistance to antibody neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.09.434497](https://doi.org/10.1101/2021.03.09.434497).
- **Wibmer21**: Wibmer CK, Ayres F, et int., and Moore PL. “SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma.” _nature_, 2021. [doi.org/10.1038/s41591-021-01285-x](https://doi.org/10.1038/s41591-021-01285-x). \[[PubMed33654292](https://pubmed.ncbi.nlm.nih.gov/33654292/)\]
- **Widera21**: Widera M, Wilhelm A, et int., and Ciesek S. “Bamlanivimab does not neutralize two SARS-CoV-2 variants carrying E484K in vitro.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.24.21252372](https://doi.org/10.1101/2021.02.24.21252372).
- **Wu21**: Wu K, Werner AP, et int., and Edwards DK. “Serum Neutralizing Activity Elicited by mRNA-1273 Vaccine — Preliminary Report.” _NEJM_, 2021. [doi.org/10.1056/NEJMc2102179](https://doi.org/10.1056/NEJMc2102179). \[[PubMed33596346](https://pubmed.ncbi.nlm.nih.gov/33596346/)\]
- **Xie21**: Xie X, Zou J, et int., and Shi P. “Neutralization of N501Y mutant SARS-CoV-2 by BNT162b2 vaccine-elicited sera.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.07.425740](https://doi.org/10.1101/2021.01.07.425740). \[[PubMed33442691](https://pubmed.ncbi.nlm.nih.gov/33442691/)\] \[[PMC7805448](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7805448/)\]
- **Xie21b**: Xie X, Liu Y, et int., and Shi P. “Neutralization of SARS-CoV-2 spike 69/70 deletion, E484K and N501Y variants by BNT162b2 vaccine-elicited sera.” _Nature_, 2021. [doi.org/10.1038/s41591-021-01270-4](https://doi.org/10.1038/s41591-021-01270-4).


</div>

</section>


## Susceptibility results of monoclonal neutralizing antibodies

<section id="sec_mab-fold" class="figure-section wrap">

<div class="figure-image">

![Monoclonal antibodies](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/mab-fold-fig-1.png)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to monoclonal neutralizing antibodies (mAbs) associated with three variants of concern (B.1.1.7, B.1.351, and P.1) and six spike amino acid mutations E484K, N501Y, L452R, K417N, N439K, and Y453F. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. Each of the _in vitro_ susceptibility experiments were performed using pseudovirus assays. Some of the pseudovirus assays performed with single amino acid mutations contained D614G while others did not. D614G has been reported to slightly increase neutralization susceptibility. The Y-axis indicates the log-fold reduction in susceptibility. Data are displayed for mAbs in advanced clinical trials and several additional prototypical mAbs with high-resolution 3-D structures and defined mechanisms of mAb-RBD binding.

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility.
</p>

Studies:

- **Baum20**: Baum A, Fulton BO, et int., and Kyratsous CA. “Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies.” _Science_, 2020. [doi.org/10.1126/science.abd0831](https://doi.org/10.1126/science.abd0831). \[[PubMed32540904](https://pubmed.ncbi.nlm.nih.gov/32540904/)\] \[[PMC7299283](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7299283/)\]
- **Collier21**: Collier D, Marco AD, et int., and Gupta R. “SARS-CoV-2 B.1.1.7 escape from mRNA vaccine-elicited neutralizing antibodies.” _medRxiv_, 2021. [doi.org/10.1101/2021.01.19.21249840](https://doi.org/10.1101/2021.01.19.21249840).
- **Diamond21**: Chen RE, Zhang X, et int., and Diamond MS. “Resistance of SARS-CoV-2 variants to neutralization by monoclonal and serum-derived polyclonal antibodies.” _nature medicine_, 2021. [doi.org/10.1038/s41591-021-01294-w](https://doi.org/10.1038/s41591-021-01294-w). \[[PubMed33664494](https://pubmed.ncbi.nlm.nih.gov/33664494/)\]
- **Dong21**: Dong J, Zost SJ, et int., and Crowe JE. “Genetic and structural basis for recognition of SARS-CoV-2 spike protein by a two-antibody cocktail.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.27.428529](https://doi.org/10.1101/2021.01.27.428529).
- **Du20**: Du S, Cao Y, et int., and Qin C. “Structurally Resolved SARS-CoV-2 Antibody Shows High Efficacy in Severely Infected Hamsters and Provides a Potent Cocktail Pairing Strategy.” _Cell_, 2020. [doi.org/10.1016/j.cell.2020.09.035](https://doi.org/10.1016/j.cell.2020.09.035). \[[PubMed32970990](https://www.ncbi.nlm.nih.gov/pubmed/32970990/)\] \[[PMC7489885](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7489885/)\]
- **Gottlieb21**: Gottlieb RL, Nirula A, et int., and Skovronsky DM. “Effect of Bamlanivimab as Monotherapy or in Combination With Etesevimab on Viral Load in Patients With Mild to Moderate COVID-19.” _JAMA_, 2021. [doi.org/10.1001/jama.2021.0202](https://doi.org/10.1001/jama.2021.0202). \[[PubMed33475701](https://pubmed.ncbi.nlm.nih.gov/33475701/)\] \[[PMC7821080](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7821080/)\]
- **Gu21**: Gu C, Cao X, et int., and Deng SJ. “A human antibody with blocking activity to RBD proteins of multiple SARS-CoV-2 variants including B.1.351 showed potent prophylactic and therapeutic efficacy against SARS-CoV-2 in rhesus macaques.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.07.429299](https://doi.org/10.1101/2021.02.07.429299)
- **Hoffmann21**: Hoffmann, M, Arora P, et int., and Pöhlmann S. “SARS-CoV-2 variants B.1.351 and B.1.1.248: Escape from therapeutic antibodies and antibodies induced by infection and vaccination.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.11.430787](https://doi.org/10.1101/2021.02.11.430787).
- **Hoffmann21b**: Hoffmann, M, Zhang L, et int., and Pöhlmann S. “SARS-CoV-2 mutations acquired in mink reduce antibody-mediated neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.12.430998](https://doi.org/10.1101/2021.02.12.430998).
- **Li20h**: Li Q, Wu J, et int., and Wang Y. “The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity.” _Cell_, 2020. [doi.org/10.1016/j.cell.2020.07.012](https://doi.org/10.1016/j.cell.2020.07.012). \[[PubMed32730807](https://www.ncbi.nlm.nih.gov/pubmed/32730807/)\] \[[PMC7366990](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366990/)\]
- **RSC21**: Rees-Spear C, Muir L, et int., and McCoy L. “The impact of Spike mutations on SARS-CoV-2 neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.15.426849](https://doi.org/10.1101/2021.01.15.426849).
- **Rothenberger21**: Rothenberger S, Walser M, et int., and Stumpp MT. “Multi-specific DARPin® therapeutics demonstrate very high potency against mutated SARS-CoV-2 variants in vitro.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.03.429164](https://doi.org/10.1101/2021.02.03.429164).
- **Shen21**: Shen X, Tang H, et int., and Montefiori DC. “SARS-CoV-2 variant B.1.1.7 is susceptible to neutralizing antibodies elicited by ancestral Spike vaccines.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.27.428516](https://doi.org/10.1101/2021.01.27.428516).
- **Stamatatos21**: Stamatatos L, Czartoski J, et int., and McGuire A. “Antibodies elicited by SARS-CoV-2 infection and boosted by vaccination neutralize an emerging variant and SARS-CoV-1.”, _medRxiv_, 2021. [doi.org/10.1101/2021.02.05.21251182](https://doi.org/10.1101/2021.02.05.21251182).
- **Starr21**: Starr TN, Greaney AJ, et int., and Bloom JD. “Prospective mapping of viral mutations that escape antibodies used to treat COVID-19.” _Science_, 2021. [doi.org/10.1126/science.abf9302](https://doi.org/10.1126/science.abf9302). \[[PubMed33495308](https://www.ncbi.nlm.nih.gov/pubmed/33495308/)\]
- **Tortorici20**: Tortorici MA, Beltramello M, et int., and Veesler D. “Ultrapotent human antibodies protect against SARS-CoV-2 challenge via multiple mechanisms.” _Science_, 2020. [doi.org/10.1126/science.abe3354](https://doi.org/10.1126/science.abe3354). \[[PubMed32972994](https://www.ncbi.nlm.nih.gov/pubmed/32972994/)\]
- **Wang21b**: Wang P, Liu L, et int., and Ho DD. “Increased Resistance of SARS-CoV-2 Variants B.1.351 and B.1.1.7 to Antibody Neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.25.428137](https://doi.org/10.1101/2021.01.25.428137).
- **Wang21c**: Wang P, Wang M, et int., and Ho DD. “Increased Resistance of SARS-CoV-2 Variant P.1 to Antibody Neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.01.433466](https://doi.org/10.1101/2021.03.01.433466).
- **Wang21e**: Wang L, Zhou T, et int., and Misasi J. “Antibodies with potent and broad neutralizing activity against antigenically diverse and highly transmissible SARS-CoV-2 variants.” _biorxiv_, 2021. [doi.org/10.1101/2021.02.25.432969](https://doi.org/10.1101/2021.02.25.432969).
- **Wang21f**: Wang R, Zhang Q, et int., and Zhang L. “Spike mutations in SARS-CoV-2 variants confer resistance to antibody neutralization.” _bioRxiv_, 2021. [doi.org/10.1101/2021.03.09.434497](https://doi.org/10.1101/2021.03.09.434497).
- **Weisblum20**: Weisblum Y, Schmidt F, et int., and Bieniasz PD. “Escape from neutralizing antibodies by SARS-CoV-2 spike protein variants.” _eLife_, 2020. [doi.org/10.7554/elife.61312](https://doi.org/10.7554/elife.61312). \[[PubMed33112236](https://pubmed.ncbi.nlm.nih.gov/33112236/)\] \[[PMC7723407](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7723407/)\]
- **Wibmer21**: Wibmer CK, Ayres F, et int., and Moore PL. “SARS-CoV-2 501Y.V2 escapes neutralization by South African COVID-19 donor plasma.” _nature_, 2021. [doi.org/10.1038/s41591-021-01285-x](https://doi.org/10.1038/s41591-021-01285-x). \[[PubMed33654292](https://pubmed.ncbi.nlm.nih.gov/33654292/)\]
- **Widera21**: Widera M, Wilhelm A, et int., and Ciesek S. “Bamlanivimab does not neutralize two SARS-CoV-2 variants carrying E484K in vitro.” _bioRxiv_, 2021. [doi.org/10.1101/2021.02.24.21252372](https://doi.org/10.1101/2021.02.24.21252372).
- **Yao21**: Yao W, Wang Y, Zhong G. “Spike mutations decrease SARS-CoV-2 sensitivity to neutralizing antibodies but not ACE2-Ig in vitro.” _bioRxiv_, 2021. [doi.org/10.1101/2021.01.27.428353](https://doi.org/10.1101/2021.01.27.428353).

MAb abbreviations:

- **BAM**: Bamlanivimab/LY-CoV555
- **CAS**: Casirivimab/REGN10933
- **IMD**: Imdevimab/REGN10987
- **CAS/IMD**: Casirivimab+imdevimab/REGN-COV2
- **ETE**: Etesevimab/LY-CoV016/JS016/CB6
- **CIL**: Cilgavimab/COV2-2130/AZD1061
- **TIX**: Tixagevimab/COV2-2196/AZD8895
- **TIX-CIL**: Tixagevimab+Cilgavimab
- **BAM-ETE**: Bamlanivimab+Etesevimab
- **SOT**: Sotrovimab/Vir-7831/S309

</div>

</section>

## Primary data

- [Individual spike mutations / mAb neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#individual-spike-mutations-mab-neutralization)
- [Individual spike mutations / plasma neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#individual-spike-mutations-plasma-neutralization)
- [Spike variants and mutation combinations / mAb neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-mab-neutralization)
- [Spike variants and mutation combinations / plasma neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-plasma-neutralization)
- [Spike mutation _in vitro_ selection](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-mutation-invitro-selection-cp-and-mab)

## Tabular summaries

### Numbers of published in vitro neutralization assays according to spike mutations, type of plasma, and monoclonal antibody

[table]
TableS3
[/table]

### Fold-Reduced Neutralizing Susceptibility of Pseudotyped Viruses with Individual or Combinations of Spike Mutations to Monoclonal Antibodies According to General Mode of Binding

[table]
Table3
[/table]


**Footnote:**

- RBM-I: mAbs binding to the receptor binding motif - class I; RBM-II: mAbs binding to the RBM - class II; Core-I: mAbs binding to the RBD core. CAS (casirivimab), ETE (etesevimab), TIX (tixagevimab), BAM (bamlanivimab), CIL (cilgavimab), IMD (imdevimab), and SOT (Sotrovimab) are in phase III clinical trials. C144-LS and C135-LS are in phase I clinical trials. S2E12 is used with Sotrovimab in an antibody cocktail.
- B.1.1.7 contains the following spike mutations: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H. B.1.351 contains L18F, D80A, D215G, ∆242-244, K417N, E484K, N501Y, D614G, and A701V. P.1 contains L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
- “-“ indicates absence of susceptibility data. “\*“ indicates conflict between two research groups, with one showing high-level of reduced susceptibility especially using IC80 rather than IC50.

### Fold Reduction in Neutralizing Susceptibility of Convalescent Plasma and Plasma from Vaccinated Persons for Pseudotyped Viruses and Isogenic Clones Containing Individual Mutations or Combinations of Mutations

[table]
Table4
[/table]


**Footnote:** Convalescent plasma in all studies is presumed likely to have been obtained from persons infected with spike variants lacking mutations associated with reduced neutralization susceptibility. B.1.1.7 contains the following spike mutations: Δ69/70, Δ144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H. B.1.351 contains L18F, D80A, D215G, Δ242-244, K417N, E484K, N501Y, D614G, and A701V. P.1 contains L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.

### Neutralizing Sensitivity of Variants of Concern to Convalescent Plasma and Plasma from Vaccinated Persons

[table]
TableS4
[/table]

### Neutralizing Sensitivity of Variants of Concern to Monoclonal Antibodies (mAbs)

[table]
TableS5
[/table]

### Neutralizing Sensitivity of mutations to Convalescent Plasma and Plasma from Vaccinated Persons

[table]
TableS6
[/table]

### Neutralizing Sensitivity of Mutations to Monoclonal Antibodies (mAbs)

[table]
TableS7
[/table]
