<div style="clear: both; height: 0;" />


## Raw data

- [Spike variants and mutation combinations / mAb neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-mab-neutralization)
- [Spike variants and mutation combinations / plasma neutralization](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-variants-and-mutation-combinations-plasma-neutralization)
- [Spike mutation _in vitro_ selection](https://hivdb.github.io/covid-drdb-reports/resistance-dt.html#spike-mutation-invitro-selection-cp-and-mab)

## Summaries

### Table 1: Statistics of Published _in vitro_ Neutralization Assays

Numbers of published _in vitro_ neutralization assays according to spike mutations, type of plasma, and monoclonal antibody


[table]
table_summary
[/table]

### Table 2: Monoclonal Antibodies / Viruses Strains or Spike Mutations

Fold-Reduced Neutralizing Susceptibility of Viruses with Individual or Combinations of Spike Mutations to Monoclonal Antibodies According to General Mode of Binding


[table]
table_mab
[/table]


**Footnote:**

- RBM-I: mAbs binding to the receptor binding motif - class I; RBM-II: mAbs binding to the RBM - class II; Core-I: mAbs binding to the RBD core. CAS (casirivimab), ETE (etesevimab), TIX (tixagevimab), BAM (bamlanivimab), CIL (cilgavimab), IMD (imdevimab), and SOT (Sotrovimab) are in phase III clinical trials. C144-LS and C135-LS are in phase I clinical trials. S2E12 is used with Sotrovimab in an antibody cocktail.
- B.1.1.7 contains the following spike mutations: ∆69/70, ∆144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H. B.1.351 contains L18F, D80A, D215G, ∆242-244, K417N, E484K, N501Y, D614G, and A701V. P.1 contains L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.
- The fold change is the median value of results, the subscript is the number of results. “-“ indicates absence of susceptibility data. “\*“ indicates conflict between two research groups, with one showing high-level of reduced susceptibility especially using IC80 rather than IC50.

### Table 3: Convalescent Plasma and Plasma from Vaccinated Persons / Viruses Strains or Spike Mutations

Fold Reduction in Neutralizing Susceptibility of Convalescent Plasma and Plasma from Vaccinated Persons for Viruses and Isogenic Clones Containing Individual Mutations or Combinations of Mutations


[table]
table_plasma
[/table]


**Footnote:** Convalescent plasma in all studies is presumed likely to have been obtained from persons infected with spike variants lacking mutations associated with reduced neutralization susceptibility. B.1.1.7 contains the following spike mutations: Δ69/70, Δ144, N501Y, A570D, D614G, P681H, T716I, S982A, and D1118H. B.1.351 contains L18F, D80A, D215G, Δ242-244, K417N, E484K, N501Y, D614G, and A701V. P.1 contains L18F, T20N, P26S, D138Y, R190S, K417T, E484K, N501Y, D614G, H655Y, T1027I, and V1176F.


### Figure 1: Susceptibility results of convalescent and vaccine-recipient plasma

<section id="sec_plasma-fold" class="figure-section wrap">

<div class="figure-image">

![Convalescent and vaccine-recipient plasma](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/plasma-fold-fig-1.png#!maxWidth=95vw)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to plasma from previously infected and vaccinated persons associated with three variants of concern (B.1.1.7, B.1.351, and P.1), variants of interest(CAL.20C) and two spike amino acid mutations E484K and N501Y. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. The Y-axis indicates the number of plasma samples tested.

"*": The study is using authentic virus assay.

Studies:

Becker21[^Becker21],Brown21[^Brown21],Cele21[^Cele21],Chen21[^Chen21],Collier21[^Collier21],Dejnirattisai21[^Dejnirattisai21],Deng21[^Deng21],Edara21[^Edara21],Emary21[^Emary21],Faulkner21[^Faulkner21],Garcia-Beltran21	^[Garcia-Beltran21],Greaney21[^Greaney21],Hoffmann21[^Hoffmann21],Hoffmann21b[^Hoffmann21b],Hu21[^Hu21],Jangra21[^Jangra21],Kemp21[^Kemp21],Liu21[^Liu21],Liu21b[^Liu21b],Madhi21[^Madhi21],Marot21[^Marot21],Moyo-Gwete21[^Moyo-Gwete21],Muik21[^Muik21],Planas21[^Planas21],Rees-Spear21[^Rees-Spear21],Shen21[^Shen21],Tada21[^Tada21],Tang21[^Tang21],Wang21[^Wang21],Wang21b[^Wang21b],Wang21c[^Wang21c],Wang21f[^Wang21f],Wibmer21[^Wibmer21],Widera21[^Widera21],Wu21[^Wu21],Xie21b[^Xie21b],Zhou21b[^Zhou21b]

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility. The histogram labels indicate the studies’ authors and plasma characteristics. BNT: BNT162b2; Mod: mRNA-1273; NVX: NVX-CoV2373; AZD: AZD1222. Each of plasma units were obtained approximately one month following vaccination with the exception of one study.
</p>

</div>

</section>


### Figure 2: Susceptibility results of monoclonal neutralizing antibodies

<section id="sec_mab-fold" class="figure-section wrap">

<div class="figure-image">

![Monoclonal antibodies](https://hivdb.github.io/covid-drdb-reports/susceptibility-data_files/figure-html/mab-fold-fig-1.png#!maxWidth=95vw)

</div>

<div class="figure-caption">

Fold reduced _in vitro_ neutralization susceptibility to monoclonal neutralizing antibodies (mAbs) associated with three variants of concern (B.1.1.7, B.1.351, and P.1) and six spike amino acid mutations E484K, N501Y, L452R, K417N, N439K, and Y453F. B.1.1.7 contains N501Y plus two S1 N-terminal domain (NTD) deletions, three S1 C-terminal domain (CTD) mutations, and three S2-mutations. B.1.351 contains E484K and N501Y plus four S1 NTD mutations, two receptor binding domain (RBD) mutations, one S1 CTD mutation, and one S2 mutation. P.1 contains E484K and N501Y plus five S1 NTD mutations, one RBD mutation, two S1 CTD mutations, and two S2 mutations. Each of the _in vitro_ susceptibility experiments were performed using pseudovirus assays. Some of the pseudovirus assays performed with single amino acid mutations contained D614G while others did not. D614G has been reported to slightly increase neutralization susceptibility. The Y-axis indicates the log-fold reduction in susceptibility. Data are displayed for mAbs in advanced clinical trials and several additional prototypical mAbs with high-resolution 3-D structures and defined mechanisms of mAb-RBD binding.

<p>
The color scheme indicates the fold-reduction in neutralization: <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #ffffff; color: black;">absence of color</span> – &lt;3-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #7fcbee; color: black;">light blue</span> – 3 to 9.9-fold reduced susceptibility; <span style="padding: .1rem .2rem; margin: 0 .2rem; border: 1px solid black; background-color: #146aa8; color: white;">dark blue</span> – ≥10-fold reduced susceptibility.
</p>


</div>

</section>


## Individual study details

### Table S1: Neutralizing Sensitivity of Variants of Concern to Convalescent Plasma and Plasma from Vaccinated Persons

[table]
table_plasma_variant
[/table]

"*": The study is using authentic virus assay.
"†": Aggregated results, the study provides mean fold change, not individual result for each sample.


### Table S2: Neutralizing Sensitivity of Variants of Concern to Monoclonal Antibodies (mAbs)

[table]
table_mab_variant
[/table]


### Table S3: Neutralizing Sensitivity of mutations to Convalescent Plasma and Plasma from Vaccinated Persons

[table]
table_plasma_muts
[/table]

"*": The study is using authentic virus assay.
"†": Aggregated results, the study provides mean fold change, not individual result for each sample.

### Table S4: Neutralizing Sensitivity of Mutations to Monoclonal Antibodies (mAbs)

[table]
table_mab_muts
[/table]



## Supplemantary data

### MAb abbreviations

- **BAM**: Bamlanivimab/LY-CoV555
- **CAS**: Casirivimab/REGN10933
- **IMD**: Imdevimab/REGN10987
- **CAS/IMD**: Casirivimab+imdevimab/REGN-COV2
- **ETE**: Etesevimab/LY-CoV016/JS016/CB6
- **CIL**: Cilgavimab/COV2-2130/AZD1061
- **TIX**: Tixagevimab/COV2-2196/AZD8895
- **TIX-CIL**: Tixagevimab+Cilgavimab
- **BAM-ETE**: Bamlanivimab+Etesevimab
- **SOT**: Sotrovimab/Vir-7831/S309
